{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "from transformers import AutoTokenizer, AutoModelForTokenClassification\n",
    "from transformers import pipeline\n",
    "from tqdm.auto import tqdm\n",
    "from torch.utils.data import Dataset, DataLoader\n",
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "ners = [\"DISEASE\",\"EFFECT\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "l = []\n",
    "for ner in [\"DISEASE\"]:\n",
    "\ttest_sentences = pd.read_json(f\"../NER/{ner}/test.json\", lines=True)\n",
    "\ttest_output = pd.read_json(f\"../NER_output/{ner}/test_outputs.json\")\n",
    "\tmerged = pd.merge(test_sentences, test_output, left_index=True, right_index=True)\n",
    "\tmerged.loc[:,\"ner\"] = ner\n",
    "\tl.append(merged)\n",
    "\n",
    "df = pd.concat(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = df.reset_index().drop(columns=[\"index\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>tokens</th>\n",
       "      <th>ner_tags</th>\n",
       "      <th>id</th>\n",
       "      <th>predictions</th>\n",
       "      <th>label_ids</th>\n",
       "      <th>word_ids</th>\n",
       "      <th>ner</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>[The, current, study, did, show, that, alanine...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>0</td>\n",
       "      <td>[[-2.891744613647461, -3.85470700263977, 6.500...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 1...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>[The, results, are, presented, as, a, phylogen...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>1</td>\n",
       "      <td>[[-2.951909065246582, -3.7594962120056152, 6.2...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, 2, 2, 2, -100, 2, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 5, 6, 7, 7, 8, 9, 10, 11...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>[The, virulent, serotype, 2, ST7, strain, SC84...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, B, I, I, I, ...</td>\n",
       "      <td>2</td>\n",
       "      <td>[[-2.828230142593384, -3.822613000869751, 6.24...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, -100, 2, 2, -100, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10,...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>[A, human, stool-derived, Bilophila, wadsworth...</td>\n",
       "      <td>[O, O, O, O, O, O, O, B, I, O, O, O, O, O, O, O]</td>\n",
       "      <td>3</td>\n",
       "      <td>[[-2.458846092224121, -3.563410997390747, 6.05...</td>\n",
       "      <td>[-100, 2, 2, 2, -100, -100, 2, -100, 2, -100, ...</td>\n",
       "      <td>[None, 0, 1, 2, 2, 2, 3, 3, 4, 4, 4, 4, 5, 6, ...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>[The, L-H, group, showed, significantly, highe...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>4</td>\n",
       "      <td>[[-2.737249374389648, -3.846871137619018, 6.35...</td>\n",
       "      <td>[-100, 2, 2, -100, -100, 2, 2, 2, 2, 2, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 1, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>604</th>\n",
       "      <td>[S., citri, strain, BLH-13, was, isolated, in,...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>604</td>\n",
       "      <td>[[-2.632442951202392, -3.8172717094421382, 6.4...</td>\n",
       "      <td>[-100, 2, -100, 2, -100, 2, 2, -100, -100, -10...</td>\n",
       "      <td>[None, 0, 0, 1, 1, 2, 3, 3, 3, 3, 4, 5, 6, 7, ...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>605</th>\n",
       "      <td>[Nonetheless, ,, ADO, genes, with, high, cover...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>605</td>\n",
       "      <td>[[-2.719882726669311, -4.03209114074707, 6.377...</td>\n",
       "      <td>[-100, 2, 2, 2, -100, 2, 2, 2, 2, 2, 2, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>606</th>\n",
       "      <td>[In, particular, ,, the, combination, of, 5‐FU...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>606</td>\n",
       "      <td>[[-2.6313037872314453, -4.045229434967041, 5.8...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 5, 6, 6, 6, 7, 8, 9, 10,...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>607</th>\n",
       "      <td>[The, average, colony, growth, rate, (, 10.72,...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...</td>\n",
       "      <td>607</td>\n",
       "      <td>[[-2.942236423492431, -3.877107381820678, 6.71...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 5, 6, 6, 6, 7, 8, 9, 10,...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>608</th>\n",
       "      <td>[Among, rare, taxa, that, showed, sex-dependen...</td>\n",
       "      <td>[O, O, O, O, O, O, O, O, B, O, O, O, O, O, O, ...</td>\n",
       "      <td>608</td>\n",
       "      <td>[[-2.335780382156372, -3.918740034103393, 5.72...</td>\n",
       "      <td>[-100, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, 0, ...</td>\n",
       "      <td>[None, 0, 1, 2, 3, 4, 5, 5, 5, 6, 7, 8, 9, 10,...</td>\n",
       "      <td>DISEASE</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>609 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                tokens  \\\n",
       "0    [The, current, study, did, show, that, alanine...   \n",
       "1    [The, results, are, presented, as, a, phylogen...   \n",
       "2    [The, virulent, serotype, 2, ST7, strain, SC84...   \n",
       "3    [A, human, stool-derived, Bilophila, wadsworth...   \n",
       "4    [The, L-H, group, showed, significantly, highe...   \n",
       "..                                                 ...   \n",
       "604  [S., citri, strain, BLH-13, was, isolated, in,...   \n",
       "605  [Nonetheless, ,, ADO, genes, with, high, cover...   \n",
       "606  [In, particular, ,, the, combination, of, 5‐FU...   \n",
       "607  [The, average, colony, growth, rate, (, 10.72,...   \n",
       "608  [Among, rare, taxa, that, showed, sex-dependen...   \n",
       "\n",
       "                                              ner_tags   id  \\\n",
       "0    [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...    0   \n",
       "1    [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...    1   \n",
       "2    [O, O, O, O, O, O, O, O, O, O, O, B, I, I, I, ...    2   \n",
       "3     [O, O, O, O, O, O, O, B, I, O, O, O, O, O, O, O]    3   \n",
       "4    [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...    4   \n",
       "..                                                 ...  ...   \n",
       "604  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...  604   \n",
       "605  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...  605   \n",
       "606  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...  606   \n",
       "607  [O, O, O, O, O, O, O, O, O, O, O, O, O, O, O, ...  607   \n",
       "608  [O, O, O, O, O, O, O, O, B, O, O, O, O, O, O, ...  608   \n",
       "\n",
       "                                           predictions  \\\n",
       "0    [[-2.891744613647461, -3.85470700263977, 6.500...   \n",
       "1    [[-2.951909065246582, -3.7594962120056152, 6.2...   \n",
       "2    [[-2.828230142593384, -3.822613000869751, 6.24...   \n",
       "3    [[-2.458846092224121, -3.563410997390747, 6.05...   \n",
       "4    [[-2.737249374389648, -3.846871137619018, 6.35...   \n",
       "..                                                 ...   \n",
       "604  [[-2.632442951202392, -3.8172717094421382, 6.4...   \n",
       "605  [[-2.719882726669311, -4.03209114074707, 6.377...   \n",
       "606  [[-2.6313037872314453, -4.045229434967041, 5.8...   \n",
       "607  [[-2.942236423492431, -3.877107381820678, 6.71...   \n",
       "608  [[-2.335780382156372, -3.918740034103393, 5.72...   \n",
       "\n",
       "                                             label_ids  \\\n",
       "0    [-100, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, ...   \n",
       "1    [-100, 2, 2, 2, 2, 2, 2, 2, 2, -100, 2, 2, 2, ...   \n",
       "2    [-100, 2, 2, 2, 2, 2, -100, 2, 2, -100, 2, 2, ...   \n",
       "3    [-100, 2, 2, 2, -100, -100, 2, -100, 2, -100, ...   \n",
       "4    [-100, 2, 2, -100, -100, 2, 2, 2, 2, 2, 2, 2, ...   \n",
       "..                                                 ...   \n",
       "604  [-100, 2, -100, 2, -100, 2, 2, -100, -100, -10...   \n",
       "605  [-100, 2, 2, 2, -100, 2, 2, 2, 2, 2, 2, 2, 2, ...   \n",
       "606  [-100, 2, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, ...   \n",
       "607  [-100, 2, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, ...   \n",
       "608  [-100, 2, 2, 2, 2, 2, 2, -100, -100, 2, 2, 0, ...   \n",
       "\n",
       "                                              word_ids      ner  \n",
       "0    [None, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 1...  DISEASE  \n",
       "1    [None, 0, 1, 2, 3, 4, 5, 6, 7, 7, 8, 9, 10, 11...  DISEASE  \n",
       "2    [None, 0, 1, 2, 3, 4, 4, 5, 6, 6, 7, 8, 9, 10,...  DISEASE  \n",
       "3    [None, 0, 1, 2, 2, 2, 3, 3, 4, 4, 4, 4, 5, 6, ...  DISEASE  \n",
       "4    [None, 0, 1, 1, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,...  DISEASE  \n",
       "..                                                 ...      ...  \n",
       "604  [None, 0, 0, 1, 1, 2, 3, 3, 3, 3, 4, 5, 6, 7, ...  DISEASE  \n",
       "605  [None, 0, 1, 2, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11...  DISEASE  \n",
       "606  [None, 0, 1, 2, 3, 4, 5, 6, 6, 6, 7, 8, 9, 10,...  DISEASE  \n",
       "607  [None, 0, 1, 2, 3, 4, 5, 6, 6, 6, 7, 8, 9, 10,...  DISEASE  \n",
       "608  [None, 0, 1, 2, 3, 4, 5, 5, 5, 6, 7, 8, 9, 10,...  DISEASE  \n",
       "\n",
       "[609 rows x 7 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "label_ids = df[[\"label_ids\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('We', 'O', -100)\n",
      "('started', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('investigate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('factor-like', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('TpF1', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('major', 'O', 2)\n",
      "('antigen', 'O', -100)\n",
      "('expressed', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('T.', 'B', 2)\n",
      "('pallidum10', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('comparing', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('proliferative', 'O', 1)\n",
      "('rate', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('umbilical', 'O', 2)\n",
      "('vein', 'O', 2)\n",
      "('endothelial', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('HUVECs', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('exposed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('recombinant', 'O', 2)\n",
      "('TpF1', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('VEGF', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('free', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Gram-positive', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Gram-negative', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('contaminants', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('Suppl', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('L.', 'B', -100)\n",
      "('paracasei', 'I', 0)\n",
      "('FAM18149', 'I', -100)\n",
      "('was', 'O', 1)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('chemically', 'O', -100)\n",
      "('defined', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('CDM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('L.', 'O', 2)\n",
      "('helveticus', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Christensen', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Steele', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2003', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('omitting', 'O', 2)\n",
      "('either', 'O', -100)\n",
      "('cysteine', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('cysteine-deficient', 'O', 2)\n",
      "('CDM', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('or', 'O', -100)\n",
      "('methionine', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('methionine-deficient', 'O', 2)\n",
      "('CDM', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('25', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('30°C', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Colonies', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('BO2', 'B', 2)\n",
      "('strain', 'O', 0)\n",
      "('ranged', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('size', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('punctuate', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('1.5', 'O', 2)\n",
      "('mm', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('diameter', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('they', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('non-motile', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('mucoid', 'O', 2)\n",
      "('colonies', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('MacConkey', 'O', 2)\n",
      "('agar', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('positive', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('oxidase', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('catalase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('exhibited', 'O', -100)\n",
      "('nitrate', 'O', -100)\n",
      "('reduction', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('gas', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('rapid', 'O', 2)\n",
      "('urease', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('<', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('min', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Addition', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('models-a', 'O', 2)\n",
      "('mouse', 'O', 2)\n",
      "('acute', 'O', -100)\n",
      "('lung', 'O', -100)\n",
      "('infection', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CF', 'O', 2)\n",
      "('clinical', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('aeruginosa', 'I', 2)\n",
      "('LESB58-SED21', 'I', 2)\n",
      "('grown', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('SCFM2-captured', 'O', 1)\n",
      "('all', 'O', 1)\n",
      "('but', 'O', -100)\n",
      "('54', 'O', -100)\n",
      "('P.', 'O', -100)\n",
      "('aeruginosa', 'O', -100)\n",
      "('PAO1', 'O', -100)\n",
      "('genes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('SI', 'O', 2)\n",
      "('Appendix', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('S5', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Bartonella', 'B', -100)\n",
      "('henselae', 'I', 0)\n",
      "('Houston-1', 'I', -100)\n",
      "('effector', 'O', 1)\n",
      "('proteins', 'O', -100)\n",
      "('cloned', 'O', -100)\n",
      "('into', 'O', 1)\n",
      "('pENTR', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('gift', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Alexei', 'O', 2)\n",
      "('Savchenko', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('University', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('Toronto', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('ability', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('single', 'O', 2)\n",
      "('electron', 'O', 2)\n",
      "('reduced', 'O', 2)\n",
      "('E', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('bind', 'O', 2)\n",
      "('NO', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('investigated', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('K354M', 'O', 2)\n",
      "('mutant', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('Paracoccus', 'O', -100)\n",
      "('denitrificans', 'O', 2)\n",
      "('CcOX', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('38', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Bifidobacterium', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('correlated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('21', 'O', 2)\n",
      "('metabolites', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('5B', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('including', 'O', 2)\n",
      "('M149T215', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Bisphenol', 'O', 2)\n",
      "('b', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('M431T154', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('alpha-Tocopherol', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Vitamin', 'O', 2)\n",
      "('E', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('positive', 'O', -100)\n",
      "('correlations', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('WhiB3', 'O', -100)\n",
      "('subsequently', 'O', 2)\n",
      "('mediates', 'O', -100)\n",
      "('subversion', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('lysosomal', 'O', 2)\n",
      "('trafficking', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('macrophages', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('granuloma', 'O', 2)\n",
      "('formation', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vitro', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('^20', 'O', -100)\n",
      "('^', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('^21', 'O', 2)\n",
      "('We', 'O', 2)\n",
      "('then', 'O', 2)\n",
      "('used', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('aggregation', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('estimate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('formation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('KT1115', 'B', 2)\n",
      "('in', 'O', 2)\n",
      "('liquid', 'O', 2)\n",
      "('culture', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Here', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('quest', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('such', 'O', -100)\n",
      "('enzymes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('native', 'O', 2)\n",
      "('thermophilic', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('namely', 'O', 2)\n",
      "('Cohnella', 'O', 2)\n",
      "('sp', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Cultivars', 'O', -100)\n",
      "('“Cavolo', 'O', 2)\n",
      "('di', 'O', 2)\n",
      "('Napoli”', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('“Patriot”', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('“Costa', 'O', -100)\n",
      "('Rica', 'O', -100)\n",
      "('No.', 'O', 2)\n",
      "('4”', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('“Romana', 'O', -100)\n",
      "('Romabella', 'O', -100)\n",
      "('30', 'O', 2)\n",
      "('CN”', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('susceptible', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('highly', 'O', -100)\n",
      "('susceptible', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('while', 'O', 2)\n",
      "('“Banchu', 'O', -100)\n",
      "('Red', 'O', 2)\n",
      "('Fire”', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('partially', 'O', -100)\n",
      "('resistant', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('two', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('04750888', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Fus', 'O', 2)\n",
      "('1.01', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('assigned', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('newly', 'O', 2)\n",
      "('described', 'O', -100)\n",
      "('race', 'O', -100)\n",
      "('4', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('S1', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('F52', 'B', -100)\n",
      "(',', 'O', 0)\n",
      "('selected', 'O', -100)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('enhanced', 'O', 2)\n",
      "('biocontrol', 'O', 2)\n",
      "('capacity', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('sequenced', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('further', 'O', 2)\n",
      "('analyses', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('below', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Single', 'O', -100)\n",
      "('colonies', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('each', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('morphologically', 'O', 2)\n",
      "('distinct', 'O', 2)\n",
      "('soymilk', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('UCD_MED5', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('UCD_MED7', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 0)\n",
      "('aseptically', 'O', -100)\n",
      "('transferred', 'O', -100)\n",
      "('into', 'O', -100)\n",
      "('soymilk', 'O', -100)\n",
      "('tubes', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Overall', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('involved', 'O', 2)\n",
      "('65', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('pumilus', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('B.', 'O', 2)\n",
      "('safensis', 'O', -100)\n",
      "('genomes', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('including', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('FO-36b', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('MERTA', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('SAFR-032', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('ATCC7061', 'B', 2)\n",
      "('genomes', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Cultures', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Acanthamoeba', 'O', 2)\n",
      "('sp.', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('infected', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('TUM19329', 'B', 2)\n",
      "('^T', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('Philadelphia-1', 'B', 0)\n",
      "('at', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('multiplicity', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('infection', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('MOI', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('10', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('incubated', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('attachment', 'O', -100)\n",
      "('by', 'O', 2)\n",
      "('centrifugation', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('500', 'O', 2)\n",
      "('×', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('min', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Nitratiruptor', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('infected', 'O', -100)\n",
      "('multiple', 'O', -100)\n",
      "('times', 'O', -100)\n",
      "('by', 'O', 2)\n",
      "('diverse', 'O', 2)\n",
      "('phages', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('diversified', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('acquisition', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('prophage', 'O', -100)\n",
      "('genes', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('various', 'O', 2)\n",
      "('immune', 'O', 2)\n",
      "('mechanisms', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('phage', 'O', 2)\n",
      "('repressor', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('restriction-modification', 'O', 2)\n",
      "('system', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CRISPR-Cas', 'O', 2)\n",
      "('systems', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('explored', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genomic', 'O', 2)\n",
      "('variations', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('set', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Shewanella', 'O', 2)\n",
      "('baltica', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('stratified', 'O', -100)\n",
      "('basin', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('central', 'O', 2)\n",
      "('Baltic', 'O', 2)\n",
      "('Sea', 'O', 2)\n",
      "('during', 'O', 2)\n",
      "('1986', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('1987', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('1998', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('12-14', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Among', 'O', -100)\n",
      "('gram-negative', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('ortho-ring', 'O', -100)\n",
      "('cleavage', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('proposed', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('second', 'O', 2)\n",
      "('ring', 'O', 2)\n",
      "('cleavage', 'O', 2)\n",
      "('event', 'O', 2)\n",
      "('during', 'O', 2)\n",
      "('phenanthrene', 'O', 2)\n",
      "('biotransformation', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('Burkholderia', 'B', 2)\n",
      "('sp.', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('C3', 'I', 2)\n",
      "('whereby', 'O', 2)\n",
      "('1,2-dihydroxynaphthalene', 'O', 0)\n",
      "('was', 'O', 1)\n",
      "('converted', 'O', -100)\n",
      "('into', 'O', 1)\n",
      "('2-carboxycinnamic', 'O', 1)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Seo', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2007', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('isobutanol', 'O', 2)\n",
      "('biosynthesis', 'O', 2)\n",
      "('pathway', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('S.', 'O', 2)\n",
      "('blattae', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('p424IbPSO', 'B', 2)\n",
      "(')', 'O', -100)\n",
      "('uses', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('endogenous', 'O', -100)\n",
      "('NADPH-dependent', 'O', 2)\n",
      "('alcohol', 'O', 0)\n",
      "('dehydrogenase', 'O', -100)\n",
      "('YqhD', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('complete', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('pathway', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('sequences', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('sets', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('essential', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('namely', 'O', 2)\n",
      "('Pseudomonas', 'B', 2)\n",
      "('aeruginosa', 'I', 0)\n",
      "('strain', 'I', 1)\n",
      "('PA14', 'I', 1)\n",
      "('[', 'O', 1)\n",
      "('14', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Bacillus', 'O', 2)\n",
      "('subtilis', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Escherichia', 'B', 2)\n",
      "('coli', 'I', 2)\n",
      "('strain', 'I', 0)\n",
      "('MG1655', 'I', 1)\n",
      "('[', 'O', 1)\n",
      "('38', 'O', 1)\n",
      "(']', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Database', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Essential', 'O', 2)\n",
      "('Genes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('DEG', 'O', 2)\n",
      "('6.5', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('http', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('tubic.tju.edu.cn', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('deg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('39', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('40', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('62', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Salmonella', 'O', -100)\n",
      "('Typhimurium', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Salmonella', 'B', 2)\n",
      "('enterica', 'I', 0)\n",
      "('serovar', 'I', 1)\n",
      "('Typhimurium', 'I', 1)\n",
      "('strain', 'I', 1)\n",
      "('LT2', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('ATCC', 'B', -100)\n",
      "('700720', 'I', 2)\n",
      "(')', 'O', 0)\n",
      "('were', 'O', 1)\n",
      "('cultured', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Luria', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('M9', 'O', 2)\n",
      "('minimal', 'O', 2)\n",
      "('medium', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('previously', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('27', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('silico', 'O', 2)\n",
      "('resistome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('Marseille-Q3630', 'B', 2)\n",
      "('shows', 'O', 2)\n",
      "('erm', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('X', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('_4', 'O', -100)\n",
      "('gene', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('94.74', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('identity', 'O', 2)\n",
      "('percentage', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('notably', 'O', 2)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('erythromycin', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('high', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('CAZymes', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('predicted', 'O', 2)\n",
      "('secretomes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Hymenoscyphus', 'O', 2)\n",
      "('species', 'O', -100)\n",
      "('could', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('therefore', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('attributed', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('their', 'O', 2)\n",
      "('trophic', 'O', 2)\n",
      "('mode', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Stenlid', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('Rhizobium', 'O', 2)\n",
      "('etli', 'O', 2)\n",
      "('gshB', 'O', -100)\n",
      "('mutant', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('decrease', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('BNF', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('correlated', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('strong', 'O', -100)\n",
      "('reduction', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('nodule', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('deficiency', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('glutamine', 'O', 2)\n",
      "('transport', 'O', 2)\n",
      "('through', 'O', 2)\n",
      "('Aap', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Bra', 'O', 2)\n",
      "('transporter', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Taté', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2012', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('B13', 'B', -100)\n",
      "('cells', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('treated', 'O', 2)\n",
      "('initially', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "('days', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Dex', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "('days', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('EGF', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('EGF', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('then', 'O', 2)\n",
      "('withdrawn', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('fixed', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "('days', 'O', 2)\n",
      "('later', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('stained', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('amylase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CK7', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CK20', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Seoane', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('IBBTEC-UC', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Cantabria', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Spain', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('S.', 'O', -100)\n",
      "('pneumoniae', 'O', 2)\n",
      "('non-pathogenic', 'O', 2)\n",
      "('vaccine', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('49619-19F', 'B', 2)\n",
      "('was', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('commercially', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('ATCC', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Strain', 'O', -100)\n",
      "('AB26', 'B', 2)\n",
      "('contains', 'O', 0)\n",
      "('22', 'O', -100)\n",
      "('histidine', 'O', 2)\n",
      "('kinases', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('20', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('regulators', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('dedicates', 'O', 2)\n",
      "('~16', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('its', 'O', 2)\n",
      "('genome', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('transport', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Genomic', 'O', -100)\n",
      "('analysis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Enterococcus', 'B', 2)\n",
      "('durans', 'I', 0)\n",
      "('LAB18S', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('potential', 'O', -100)\n",
      "('probiotic', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('cheese', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('prepare', 'O', 2)\n",
      "('electrocompetent', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('overnight', 'O', -100)\n",
      "('cultures', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('UW551', 'B', 2)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('CPG', 'O', 0)\n",
      "('medium', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('iced', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('washed', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('times', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('ice-cold', 'O', -100)\n",
      "('sterile', 'O', 2)\n",
      "('water', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('resuspended', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('0.1X', 'O', -100)\n",
      "('volume', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('concentrating', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('10-fold', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('original', 'O', 2)\n",
      "('culture', 'O', 2)\n",
      "('density', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('This', 'O', -100)\n",
      "('surveillance', 'O', 2)\n",
      "('system', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('successful', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('detecting', 'O', 2)\n",
      "('changes', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('seroprevalence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('antibodies', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('nucleocapsid', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('spike', 'O', -100)\n",
      "('antigens', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('SARS-CoV-2', 'O', 2)\n",
      "('over', 'O', 2)\n",
      "('time', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('availability', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('gapped', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('methylotrophic', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Methylobacterium', 'B', -100)\n",
      "('extorquens', 'I', 2)\n",
      "('AM1', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('has', 'O', -100)\n",
      "('accelerated', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('definition', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('methylotrophy', 'O', 1)\n",
      "('modules', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('reasonably', 'O', 2)\n",
      "('complete', 'O', 2)\n",
      "('metabolic', 'O', 2)\n",
      "('reconstruction', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('available', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('Tn-Seq', 'O', 2)\n",
      "('pool', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('generated', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('aeruginosa', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('using', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('mariner', 'O', 2)\n",
      "('Tn', 'O', 2)\n",
      "('Himar1', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('randomly', 'O', 2)\n",
      "('inserts', 'O', 2)\n",
      "('into', 'O', -100)\n",
      "('TA', 'O', 2)\n",
      "('sites', 'O', 2)\n",
      "('within', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Lampe', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1998', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('At', 'O', -100)\n",
      "('60', 'O', 2)\n",
      "('±', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('drops', 'O', 2)\n",
      "('abruptly', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('decreases', 'O', -100)\n",
      "('by', 'O', 2)\n",
      "('58', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Staphylococcus', 'B', -100)\n",
      "('sciuri', 'I', 0)\n",
      "('strain', 'I', 1)\n",
      "('CCL', 'I', -100)\n",
      "('101', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('Micrococcus', 'B', 1)\n",
      "('luteus', 'I', 1)\n",
      "('strains', 'I', 2)\n",
      "('ATCC', 'I', 0)\n",
      "('15307', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('NCIMB', 'B', -100)\n",
      "('9278', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('NCTC', 'B', 1)\n",
      "('2665', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('Streptococcus', 'B', 2)\n",
      "('pyogenes', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('NCTC', 'I', 1)\n",
      "('12696', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('Staphylococcus', 'B', -100)\n",
      "('aureus', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('NCIMB9518', 'I', 2)\n",
      "('were', 'O', 0)\n",
      "('streaked', 'O', 1)\n",
      "('onto', 'O', 1)\n",
      "('nutrient', 'O', 1)\n",
      "('agar', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('NA', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Oxoid', 'O', 0)\n",
      "(',', 'O', 1)\n",
      "('Basingstoke', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('UK', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('grown', 'O', 2)\n",
      "('overnight', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('37', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('homolog', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('dps', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Bacillus', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('‘', 'O', 2)\n",
      "('mrgA’', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('101', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('highly', 'O', 2)\n",
      "('conserved', 'O', 2)\n",
      "('amongst', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('spore-producing', 'O', 2)\n",
      "('FO-36b', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('SAFR-032', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('well', 'O', 0)\n",
      "('as', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('non-resistant', 'O', -100)\n",
      "('spore-producing', 'O', 2)\n",
      "('ATCC7061', 'B', 0)\n",
      "('strain', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('iso-C', 'O', 2)\n",
      "('_11:0', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('specific', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('SA29-B', 'B', -100)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('YU21-B', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('KA5-B', 'B', 0)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('while', 'O', -100)\n",
      "('C', 'O', 2)\n",
      "('_18:0', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('iso-C', 'O', 0)\n",
      "('_18:0', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('specific', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('SA29-B', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('YU21-B', 'B', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('objective', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('evaluate', 'O', 2)\n",
      "('whether', 'O', 2)\n",
      "('species-specific', 'O', 2)\n",
      "('VOCs', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('PGPR', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('Paenibacillus', 'B', 2)\n",
      "('polymyxa', 'I', 2)\n",
      "('E681', 'I', -100)\n",
      "('can', 'O', 2)\n",
      "('promote', 'O', 0)\n",
      "('growth', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('induce', 'O', -100)\n",
      "('resistance', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Arabidopsis', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('A', 'O', -100)\n",
      "('single', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('gloeosporioides', 'O', 2)\n",
      "('s.l.', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('C.', 'O', -100)\n",
      "('thyoides', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', -100)\n",
      "('study', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('sister', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('Coll887', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('diseased', 'O', 2)\n",
      "('fruit', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('V.', 'O', 2)\n",
      "('oxycoccos', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('West', 'O', 0)\n",
      "('Virginia', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('For', 'O', -100)\n",
      "('example', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Enterobacter', 'O', 2)\n",
      "('cloacae', 'O', 2)\n",
      "('subsp.', 'O', 2)\n",
      "('dissolvens', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('previously', 'O', -100)\n",
      "('described', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('Erwinia', 'O', -100)\n",
      "('dissolvens', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('diseased', 'O', -100)\n",
      "('corn', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('8', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('endosphere', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('rice', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('without', 'O', 2)\n",
      "('causing', 'O', 2)\n",
      "('apparent', 'O', 2)\n",
      "('harm', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('host', 'O', -100)\n",
      "('plant', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('9', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('plasmid', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('RP4', 'O', 2)\n",
      "('conjugal', 'O', 2)\n",
      "('origin', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('transfer', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('origins', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('replication', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('pUC', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('Gram-positive', 'O', 2)\n",
      "('bacteria', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Enterococcus', 'B', 2)\n",
      "('pAMβ1', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('an', 'O', -100)\n",
      "('erythromycin', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('gene', 'O', 0)\n",
      "('from', 'O', 1)\n",
      "('Streptococcus', 'B', -100)\n",
      "('pneumoniae', 'I', -100)\n",
      "('Tn1545', 'I', 2)\n",
      "('that', 'O', 2)\n",
      "('functions', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('Gram-negative', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Gram-positive', 'O', 0)\n",
      "('bacteria', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('These', 'O', -100)\n",
      "('numbers', 'O', 2)\n",
      "('fall', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('range', 'O', 2)\n",
      "('seen', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('sequenced', 'O', 2)\n",
      "('HCH-degrading', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('13−26', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('much', 'O', -100)\n",
      "('greater', 'O', -100)\n",
      "('than', 'O', 2)\n",
      "('seen', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('non−HCH-degrading', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('used', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('our', 'O', -100)\n",
      "('comparative', 'O', -100)\n",
      "('analysis', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('Novosphingobium', 'B', 2)\n",
      "('aromaticivorans', 'I', 2)\n",
      "('DSM12444', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('0', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Sphingomonas', 'B', 2)\n",
      "('wittichii', 'I', 2)\n",
      "('RW-1', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Sphingobium', 'O', 1)\n",
      "('sp', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('^27', 'O', -100)\n",
      "('As', 'O', 2)\n",
      "('expected', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('KT1115', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('PAO1Δ', 'O', 0)\n",
      "('wspF', 'O', -100)\n",
      "('showed', 'O', -100)\n",
      "('significantly', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('c-di-GMP', 'O', -100)\n",
      "('levels', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('early', 'O', -100)\n",
      "('stage', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('fermentation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('GFP', 'O', -100)\n",
      "('intensity', 'O', -100)\n",
      "('decreased', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('56.1', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('64.3', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('late', 'O', 2)\n",
      "('stage', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('4', 'O', -100)\n",
      "('A', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('previously', 'O', 2)\n",
      "('stated', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Section', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('4188', 'B', 2)\n",
      "('elicits', 'O', 0)\n",
      "('galls', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('beet', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('gypsophila', 'O', 2)\n",
      "('whilst', 'O', 2)\n",
      "('824-1', 'B', 2)\n",
      "('incites', 'O', 2)\n",
      "('galls', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('gypsophila', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('causes', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('hypersensitive', 'O', -100)\n",
      "('response', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('HR', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('beet', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('genomic', 'O', 2)\n",
      "('G+C', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('R.', 'O', -100)\n",
      "('centenum', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('70.5', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('higher', 'O', -100)\n",
      "('than', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genome', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('purple', 'O', 2)\n",
      "('non-sulfur', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('Rhodospirillum', 'B', 2)\n",
      "('rubrum', 'I', 2)\n",
      "('ATCC', 'I', 2)\n",
      "('11170', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('65.4', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(';', 'O', -100)\n",
      "('GenBank', 'O', -100)\n",
      "('accession', 'O', 2)\n",
      "('CP000230', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('its', 'O', -100)\n",
      "('closest', 'O', -100)\n",
      "('sequenced', 'O', 1)\n",
      "('relative', 'O', -100)\n",
      "('.', 'O', 1)\n",
      "('The', 'O', -100)\n",
      "('human', 'O', 2)\n",
      "('SCLC', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('NCI-H69', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('NCI-H82', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('NCI-H146', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('NCI-H187', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('NCI-H209', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('NCI-H526', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('human', 'O', -100)\n",
      "('adenocarcinoma', 'O', 2)\n",
      "('cell', 'O', 0)\n",
      "('lines', 'O', -100)\n",
      "('NCI-H322', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('NCI-H441', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('A549', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('carcinoid', 'O', 2)\n",
      "('cell', 'O', 0)\n",
      "('line', 'O', -100)\n",
      "('NCI-H727', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('squamous', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('NCI-H226', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('NCI-H2170', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('NCI-H520', 'B', 2)\n",
      "('were', 'O', 0)\n",
      "('purchased', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('American', 'O', 2)\n",
      "('Type', 'O', 2)\n",
      "('Culture', 'O', 0)\n",
      "('Collection', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Manassas', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('VA', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Potential', 'O', -100)\n",
      "('targets', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('PHY906', 'O', 2)\n",
      "('include', 'O', 2)\n",
      "('DUSP5', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('DUSP6', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('DUSP7', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('DUSP9', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', -100)\n",
      "('are', 'O', -100)\n",
      "('reported', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('selectively', 'O', -100)\n",
      "('dephosphorylate', 'O', -100)\n",
      "('ERK1', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('226', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Acinetobacter', 'B', -100)\n",
      "('baylyi', 'I', 0)\n",
      "('ADP1', 'I', 1)\n",
      "('is', 'O', -100)\n",
      "('highly', 'O', -100)\n",
      "('competent', 'O', 1)\n",
      "('for', 'O', -100)\n",
      "('DNA', 'O', 2)\n",
      "('uptake', 'O', 2)\n",
      "('during', 'O', 2)\n",
      "('normal', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Regarding', 'O', -100)\n",
      "('chemical', 'O', 2)\n",
      "('methods', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('optimization', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('prodigiosin', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Chilczuk', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('designed', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('culture', 'O', -100)\n",
      "('system', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('commercial', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('Serratia', 'O', 2)\n",
      "('sp', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('TSA1^T', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('=JCM', 'B', -100)\n",
      "('18858^T=KCTC', 'I', -100)\n",
      "('62391^T', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('rice', 'O', 1)\n",
      "('paddy', 'O', -100)\n",
      "('soil', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Tokyo', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Japan', 'O', -100)\n",
      "('.', 'O', 0)\n",
      "('In', 'O', -100)\n",
      "('order', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('compare', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('new', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RT988', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('989', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('RT883', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('884', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('885', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('neighbor-joining', 'O', 2)\n",
      "('tree', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('constructed', 'O', 2)\n",
      "('using', 'O', 0)\n",
      "('concatenated', 'O', -100)\n",
      "('MLST', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('case', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('ppm', 'O', 2)\n",
      "('HM', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('MDA', 'O', 2)\n",
      "('ranged', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('5.29', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('15.14', 'O', -100)\n",
      "('nmol', 'O', -100)\n",
      "('g^-1', 'O', 2)\n",
      "('FW', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('whereas', 'O', -100)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 2)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', -100)\n",
      "('decreased', 'O', -100)\n",
      "('shoot', 'O', 2)\n",
      "('MDA', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('by', 'O', 0)\n",
      "('56', 'O', -100)\n",
      "('%', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 1)\n",
      "('lowered', 'O', 1)\n",
      "('MDA', 'O', -100)\n",
      "('content', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('63', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('ESCC', 'O', -100)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('KYSE30', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('KYSE180', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('generously', 'O', 2)\n",
      "('provided', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('Professor', 'O', 2)\n",
      "('Guan', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('SYSUCC', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('cultured', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Roswell', 'O', -100)\n",
      "('Park', 'O', 2)\n",
      "('Memorial', 'O', 2)\n",
      "('Institute', 'O', 2)\n",
      "('1640', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('RPMI1640', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Gibco', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('supplemented', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('fetal', 'O', 2)\n",
      "('bovine', 'O', 2)\n",
      "('serum', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('66S1MB^T', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('=LMG', 'B', -100)\n",
      "('31547^T=HAMBI', 'I', -100)\n",
      "('3720^T', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('root', 'O', 1)\n",
      "('nodule', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('soybean', 'O', -100)\n",
      "('plant', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('was', 'O', 0)\n",
      "('inoculated', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('root-zone', 'O', -100)\n",
      "('soil', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Amphicarpaea', 'O', 2)\n",
      "('bracteata', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('growing', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Quebec', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Canada', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('feature', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('largest', 'O', 2)\n",
      "('fold', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('479.183', 'O', 2)\n",
      "('m', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('z', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('matched', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('protonated', 'O', 2)\n",
      "('mass', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('N', 'O', 2)\n",
      "('-acetyl-β-', 'O', 2)\n",
      "('d', 'O', 2)\n",
      "('-glucosamine-1,6-anhydro-', 'O', 2)\n",
      "('N', 'O', 2)\n",
      "('-acetyl-β-', 'O', 2)\n",
      "('d', 'O', -100)\n",
      "('-muramate', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('C', 'O', -100)\n",
      "('_19', 'O', 2)\n",
      "('H', 'O', 2)\n",
      "('_29', 'O', -100)\n",
      "('N', 'O', -100)\n",
      "('_2', 'O', -100)\n",
      "('O', 'O', -100)\n",
      "('_12', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('predicted', 'O', 2)\n",
      "('SBW25', 'B', 2)\n",
      "('metabolite', 'O', -100)\n",
      "('involved', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('peptidoglycan', 'O', -100)\n",
      "('construction', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('recycling', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('30', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('low-pathogenic', 'O', 2)\n",
      "('PRRSV-2', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('LP-PRRSV', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('CH-1R', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('kindly', 'O', -100)\n",
      "('provided', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('Professor', 'O', 2)\n",
      "('Enmin', 'O', 2)\n",
      "('Zhou', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Northwest', 'O', 2)\n",
      "('A', 'O', 2)\n",
      "('&', 'O', 2)\n",
      "('F', 'O', -100)\n",
      "('University', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Then', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('donor', 'O', 2)\n",
      "('K', 'B', 2)\n",
      "('.', 'I', 0)\n",
      "('pneumoniae', 'I', 1)\n",
      "('SW1780', 'I', 1)\n",
      "('and', 'O', 1)\n",
      "('recipient', 'O', -100)\n",
      "('E', 'B', -100)\n",
      "('.', 'I', 2)\n",
      "('coli', 'I', 2)\n",
      "('J53', 'I', 0)\n",
      "('were', 'O', 1)\n",
      "('incubated', 'O', 1)\n",
      "('in', 'O', 1)\n",
      "('LB', 'O', -100)\n",
      "('medium', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('S.', 'B', -100)\n",
      "('lactosutens', 'I', 0)\n",
      "('DS20^T', 'I', -100)\n",
      "('[', 'O', 1)\n",
      "('26', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('S.', 'B', 1)\n",
      "('chinhatense', 'I', -100)\n",
      "('IP26^T', 'I', -100)\n",
      "('[', 'O', -100)\n",
      "('27', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('S.', 'B', 2)\n",
      "('ummariense', 'I', 0)\n",
      "('RL3^T', 'I', -100)\n",
      "('[', 'O', 1)\n",
      "('28', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('S.', 'B', 1)\n",
      "('quisquilarium', 'I', -100)\n",
      "('P25^T', 'I', -100)\n",
      "('[', 'O', -100)\n",
      "('29', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Sphingobium', 'O', 0)\n",
      "('sp', 'O', -100)\n",
      "('.', 'O', 1)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('several', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('suggest', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cytotoxic', 'O', 2)\n",
      "('actions', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('limited', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('PCa', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('α-antagonists', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('also', 'O', -100)\n",
      "('shown', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('induce', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('death', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('some', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('following', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('Bladder', 'O', 2)\n",
      "('HT1376', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Ovarian', 'O', 2)\n",
      "('SKOV-3', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Renal', 'O', -100)\n",
      "('Carcinoma', 'O', 2)\n",
      "('ACHN', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Caki-2', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('CB-inhibitors', 'O', 2)\n",
      "('alone', 'O', 2)\n",
      "('did', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('any', 'O', 2)\n",
      "('effect', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('F.', 'B', -100)\n",
      "('faecale', 'I', 0)\n",
      "('WV33', 'I', -100)\n",
      "('^T', 'O', 1)\n",
      "('was', 'O', -100)\n",
      "('aerobically', 'O', 1)\n",
      "('cultured', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Luria-Bertani', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('LB', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('10', 'O', -100)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('tryptone', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('5', 'O', -100)\n",
      "('g', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('L', 'O', -100)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('NaCl', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('rotary', 'O', 2)\n",
      "('shaker', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('250', 'O', 2)\n",
      "('rpm', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('^34', 'O', -100)\n",
      "('^', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('^35', 'O', 2)\n",
      "('^', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('^36', 'O', -100)\n",
      "('On', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('hand', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('speculated', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('RSCV', 'O', 2)\n",
      "('phenotype', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('KT1115', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('PAO1Δ', 'O', -100)\n",
      "('wspF', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('promote', 'O', 0)\n",
      "('stable', 'O', -100)\n",
      "('aggregate', 'O', -100)\n",
      "('formation', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('threshold', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('biofilm', 'O', -100)\n",
      "('dispersal', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Figures', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('S1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('led', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('complete', 'O', 2)\n",
      "('dispersal', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('required', 'O', 2)\n",
      "('more', 'O', 2)\n",
      "('RLs', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('P285', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('O277', 'B', -100)\n",
      "('were', 'O', 2)\n",
      "('Gram', 'O', 0)\n",
      "('negative', 'O', -100)\n",
      "('short', 'O', 2)\n",
      "('rod', 'O', 2)\n",
      "('arranged', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('pairs', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('single', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('non-motile', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AP2^T', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('=', 'O', -100)\n",
      "('CSUR', 'B', -100)\n",
      "('P195', 'I', -100)\n",
      "('=', 'O', 2)\n",
      "('DSM', 'B', 2)\n",
      "('26143', 'I', 0)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('fecal', 'O', 1)\n",
      "('flora', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('French', 'O', 2)\n",
      "('Caucasian', 'O', 2)\n",
      "('female', 'O', 2)\n",
      "('suffering', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('severe', 'O', 2)\n",
      "('restrictive', 'O', 2)\n",
      "('anorexia', 'O', 2)\n",
      "('nervosa', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('diverse', 'O', 2)\n",
      "('degree', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('multidrug', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MDR', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('namely', 'O', 2)\n",
      "('NK07', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('six', 'O', -100)\n",
      "('drugs', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('antimicrobial', 'O', 2)\n",
      "('classes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('5MDR', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('UDRT09', 'B', 2)\n",
      "('(', 'O', -100)\n",
      "('four', 'O', -100)\n",
      "('antimicrobial', 'O', 2)\n",
      "('classes', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('4MDR', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('NK01', 'B', 2)\n",
      "('(', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('antimicrobial', 'O', 2)\n",
      "('classes', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('3MDR', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('NK02', 'B', 0)\n",
      "('(', 'O', -100)\n",
      "('gentamicin', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CNRT12', 'B', 2)\n",
      "('(', 'O', -100)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('only', 'O', 0)\n",
      "('beta-lactam', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Sense', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('included', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('subsequent', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('investigation', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('According', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('phylogenetic', 'O', 2)\n",
      "('tree', 'O', 2)\n",
      "('constructed', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('partial', 'O', 2)\n",
      "('^blaCTX-M', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('^blaOXY', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('K1-DNA', 'O', -100)\n",
      "('sequences', 'O', -100)\n",
      "('located', 'O', -100)\n",
      "('between', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('two', 'O', -100)\n",
      "('universal', 'O', -100)\n",
      "('degenerated', 'O', 2)\n",
      "('primers', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('nine', 'O', 2)\n",
      "('K.', 'O', 2)\n",
      "('pneumoniae', 'O', 2)\n",
      "('clinical', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('formed', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('unique', 'O', 2)\n",
      "('cluster', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('^blaCTX-M-15', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('GenBank', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('AY044436', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('^blaCTX-M-28', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('GenBank', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('AJ549244', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('which', 'O', -100)\n",
      "('itself', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('closely', 'O', -100)\n",
      "('related', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('freundii', 'O', 2)\n",
      "('CTX-M-3', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('GenBank', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('Y10278', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('K.', 'O', -100)\n",
      "('oxytoca', 'O', 2)\n",
      "('^blaCTX-M-3', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('GenBank', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('AB185840', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('^blaCTX-M-1', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('GenBank', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('X92506', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('cluster', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('GT-16', 'B', -100)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('stomach', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('gastric', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('22', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('following', 'O', 2)\n",
      "('years', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('emergingly', 'O', 2)\n",
      "('discovered', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('23', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('24', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('growth', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('N110', 'B', 2)\n",
      "('reached', 'O', 0)\n",
      "('its', 'O', -100)\n",
      "('stational', 'O', 2)\n",
      "('phase', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "('h', 'O', -100)\n",
      "('after', 'O', 2)\n",
      "('culture', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('highest', 'O', 2)\n",
      "('yield', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('crude', 'O', 2)\n",
      "('biosurfactants', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('developed', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('h', 'O', -100)\n",
      "('after', 'O', -100)\n",
      "('culture', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('5E', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('HLT2-23', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('=CGMCC', 'B', -100)\n",
      "('1.11085', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "('=', 'O', -100)\n",
      "('NBRC', 'B', 2)\n",
      "('114046', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('ice', 'O', 2)\n",
      "('sample', 'O', -100)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('supraglacial', 'O', -100)\n",
      "('zone', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('Hailuogou', 'O', -100)\n",
      "('glacier', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('Sichuan', 'O', 2)\n",
      "('province', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Long-read', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', -100)\n",
      "('data', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('pomegranate', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('blight', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('Xanthomonas', 'B', -100)\n",
      "('citri', 'I', -100)\n",
      "('pv.', 'I', -100)\n",
      "('punicae', 'I', 2)\n",
      "('119', 'I', 2)\n",
      "('So', 'O', -100)\n",
      "('far', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('besides', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('OGT', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('only', 'O', -100)\n",
      "('structures', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('members', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('GT41', 'O', 2)\n",
      "('family', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', -100)\n",
      "('determined', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('OGT', 'O', 2)\n",
      "('orthologues', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Drosophila', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('DmOGT', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('46', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('plant', 'O', -100)\n",
      "('pathogen', 'O', 2)\n",
      "('Xanthomonas', 'O', 2)\n",
      "('campestris', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('XcOGT', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('47', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('soil', 'O', -100)\n",
      "('thermophile', 'O', -100)\n",
      "('Thermobaculum', 'O', 2)\n",
      "('terrenum', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('TtOGT', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('48', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Actinobacillus', 'O', 2)\n",
      "('pleuropneumoniae', 'O', 2)\n",
      "('HMW1C', 'O', -100)\n",
      "('protein', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('ApHMW1C', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('49', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('These', 'O', -100)\n",
      "('findings', 'O', 2)\n",
      "('indicate', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('sucrose-derived', 'O', 2)\n",
      "('EPS', 'O', 2)\n",
      "('not', 'O', -100)\n",
      "('only', 'O', -100)\n",
      "('promotes', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('maturation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('impairs', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C3b-mediated', 'O', 2)\n",
      "('opsonophagocytosis', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('PMN', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('most', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('including', 'O', -100)\n",
      "('blood', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('SK678', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('SK1056', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Concentration', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('2.0', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('NaCl', 'O', -100)\n",
      "('inhibited', 'O', -100)\n",
      "('growth', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('AN20', 'B', 2)\n",
      "('^T', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('1.5', 'O', 0)\n",
      "('%', 'O', -100)\n",
      "('NaCl', 'O', 2)\n",
      "('inhibited', 'O', -100)\n",
      "('growth', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('AN421', 'B', -100)\n",
      "('^T', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('AN502', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('Erwinia', 'O', -100)\n",
      "('amylovora', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('pathogen', 'O', -100)\n",
      "('causing', 'O', -100)\n",
      "('fire', 'O', 2)\n",
      "('blight', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('pear', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('apple', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('cotoneaster', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('Rosaceae', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Arcobacter', 'B', -100)\n",
      "('butzleri', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('RM4018', 'I', 1)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('motile', 'O', -100)\n",
      "('bacterium', 'O', 1)\n",
      "('that', 'O', 1)\n",
      "('synthesizes', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('polar', 'O', 2)\n",
      "('flagellum', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('contrast', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Khare', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "('observed', 'O', 2)\n",
      "('microscopic', 'O', 2)\n",
      "('granulomas', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('tonsils', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('10', 'O', -100)\n",
      "('Ravenel-infected', 'B', 2)\n",
      "('cattle', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('An', 'O', -100)\n",
      "('individual', 'O', 2)\n",
      "('colony', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('SBW25', 'B', 2)\n",
      "('on', 'O', 0)\n",
      "('an', 'O', -100)\n",
      "('M9', 'O', -100)\n",
      "('agar', 'O', 2)\n",
      "('plate', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('inoculated', 'O', -100)\n",
      "('into', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('M9', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('incubated', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('30°C', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('total', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('9', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('namely', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Botrytis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Cladosporium', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Coniochaeta', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Hygrocybe', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Lachnum', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Laetisaria', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Leucosporidiella', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Ophiocordyceps', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Trichoderma', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('unique', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('external', 'O', -100)\n",
      "('mycelial', 'O', -100)\n",
      "('cortices', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('analyze', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('pattern', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('V.', 'O', 2)\n",
      "('anguillarum', 'O', 2)\n",
      "('during', 'O', -100)\n",
      "('iron', 'O', 2)\n",
      "('starvation', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('cold-', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('warm-water', 'O', 2)\n",
      "('temperature', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('RNA-Seq', 'O', -100)\n",
      "('assay', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('performed', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('cells', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('V.', 'B', 2)\n",
      "('anguillarum', 'I', 2)\n",
      "('RV22', 'I', -100)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('CM9', 'O', 2)\n",
      "('minimal', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('with', 'O', 0)\n",
      "('50', 'O', -100)\n",
      "('μM', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('iron', 'O', -100)\n",
      "('chelator', 'O', 1)\n",
      "('2,2′-dipyridyl', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('low-iron', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('25°C', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Subsequent', 'O', -100)\n",
      "('studies', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('SR-A', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('recognises', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('Gram-positive', 'O', 2)\n",
      "('cell-wall', 'O', 2)\n",
      "('component', 'O', -100)\n",
      "('lipoteichoic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('LTA', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('9', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Desulfovibrio', 'O', -100)\n",
      "('desulfuricans', 'O', 2)\n",
      "('subsp.', 'O', -100)\n",
      "('desulfuricans', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('ATCC27774', 'B', 2)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('purchased', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('ATCC', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('ATCC', 'O', 0)\n",
      "('Medium', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('1249', 'O', -100)\n",
      "('Modified', 'O', 2)\n",
      "('Baar’s', 'O', 2)\n",
      "('Medium', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('MBM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('sulfate', 'O', 2)\n",
      "('reducers', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('made', 'O', 2)\n",
      "('according', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('ATCC', 'O', 2)\n",
      "('protocol', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Three', 'O', -100)\n",
      "('hundred', 'O', 2)\n",
      "('seven', 'O', 2)\n",
      "('ORFs', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('FO-36b', 'B', 2)\n",
      "('are', 'O', 0)\n",
      "('not', 'O', -100)\n",
      "('shared', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('SAFR-032', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('Recently', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('Gram-positive', 'O', 2)\n",
      "('coccus', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('HKU75', 'B', 2)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('purulent', 'O', 2)\n",
      "('discharge', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('wound', 'O', 2)\n",
      "('induced', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('guinea', 'O', 2)\n",
      "('pig', 'O', 2)\n",
      "('bite', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('example', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('SK115', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('SK678', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('SK1056', 'B', 0)\n",
      "('formed', 'O', -100)\n",
      "('silky', 'O', -100)\n",
      "('flocculates', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BHI-sucrose', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('CDM-sucrose', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('did', 'O', -100)\n",
      "('not', 'O', 2)\n",
      "('resemble', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('clump', 'O', -100)\n",
      "('aggregates', 'O', -100)\n",
      "('mediated', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sucrose-derived', 'O', 2)\n",
      "('glucan', 'O', -100)\n",
      "('typical', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('cariogenic', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('mutans', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Alves', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('2016', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('Harth-Chu', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2019', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2C', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('To', 'O', -100)\n",
      "('provide', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('standard', 'O', 2)\n",
      "('RNA', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('use', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('2-color', 'O', 2)\n",
      "('microarrays', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('total', 'O', 2)\n",
      "('RNA', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('LNCaP', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('DU145', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('PC3', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('CWR22', 'B', 2)\n",
      "('cell', 'O', 0)\n",
      "('lines', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('American', 'O', 2)\n",
      "('Type', 'O', 0)\n",
      "('Culture', 'O', -100)\n",
      "('Collection', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Manassas', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('VA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('growing', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('log', 'O', 2)\n",
      "('phase', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('dye-free', 'O', -100)\n",
      "('RPMI-1640', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('fetal', 'O', 2)\n",
      "('bovine', 'O', 2)\n",
      "('serum', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('FBS', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('Life', 'O', -100)\n",
      "('Technologies', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Rockville', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('MD', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Erysipelotrichaceae_UCG-004', 'O', -100)\n",
      "('had', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('negative', 'O', -100)\n",
      "('correlation', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('LDL-C', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('ALT', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('UA', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('IL-1β', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('IL-8', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CK-MB', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('LDH', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('P.', 'B', -100)\n",
      "('aeruginosa', 'I', 0)\n",
      "('MPAO1', 'I', -100)\n",
      "('was', 'O', 1)\n",
      "('grown', 'O', 1)\n",
      "('planktonically', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('SCFM1', 'O', 2)\n",
      "('shaking', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('250', 'O', 2)\n",
      "('rpm', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('37', 'O', 2)\n",
      "('°C', 'O', -100)\n",
      "('overnight', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Thus', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('issue', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('arctica', 'I', 0)\n",
      "('A', 'I', -100)\n",
      "('37-1-2', 'I', 1)\n",
      "('^T', 'O', -100)\n",
      "('belonging', 'O', 1)\n",
      "('to', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('species', 'O', -100)\n",
      "('P.', 'O', -100)\n",
      "('distincta', 'O', -100)\n",
      "('required', 'O', 2)\n",
      "('different', 'O', -100)\n",
      "('methods', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('applied', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('order', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('delineate', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('reclassify', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('anthocyanin', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('proanthocyanidins', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('selected', 'O', -100)\n",
      "('transgenic', 'O', -100)\n",
      "('lines', 'O', -100)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('anthocyanin', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('OE9', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('OE10', 'B', 2)\n",
      "('lines', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('higher', 'O', 0)\n",
      "('reaching', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('it', 'O', 0)\n",
      "('maximum', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('0.8', 'O', 2)\n",
      "('mg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('wild-type', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('remaining', 'O', 2)\n",
      "('transgenic', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('About', 'O', -100)\n",
      "('2000', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Y.', 'B', 2)\n",
      "('pestis', 'I', 0)\n",
      "('91001', 'I', -100)\n",
      "('strain', 'O', 1)\n",
      "('grown', 'O', -100)\n",
      "('at', 'O', 1)\n",
      "('26', 'O', -100)\n",
      "('°C', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('TMH', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('multiple', 'O', 2)\n",
      "('strategies', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', -100)\n",
      "('SDS-PAGE', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2D', 'O', 2)\n",
      "('liquid', 'O', 2)\n",
      "('chromatography', 'O', 2)\n",
      "('peptide', 'O', 2)\n",
      "('separation', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('intact', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('separation', 'O', 2)\n",
      "('combined', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('LTQ-FT', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('27', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('test', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('hypothesis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('effects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('OsiSh-2', 'B', 2)\n",
      "('spores', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('E+M+', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('dead', 'O', 2)\n",
      "('OsiSh-2', 'B', 2)\n",
      "('spores', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('DE+M+', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cell-free', 'O', 0)\n",
      "('culture', 'O', -100)\n",
      "('filtrate', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('CF+M+', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('rice', 'O', -100)\n",
      "('after', 'O', -100)\n",
      "('M.', 'O', -100)\n",
      "('oryzae', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('examined', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('initially', 'O', 2)\n",
      "('screened', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('HCV', 'O', 2)\n",
      "('replication', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('immortalized', 'O', 2)\n",
      "('Met', 'O', 2)\n",
      "('Mouse', 'O', 2)\n",
      "('Hepatocytes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MMH', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('cells', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('however', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('because', 'O', 2)\n",
      "('HCV', 'O', 2)\n",
      "('replication', 'O', 2)\n",
      "('efficiency', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('Huh-7', 'B', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 0)\n",
      "('can', 'O', -100)\n",
      "('vary', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('tested', 'O', 2)\n",
      "('HCV', 'O', 2)\n",
      "('replication', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('independently', 'O', 2)\n",
      "('derived', 'O', 2)\n",
      "('MMH', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('MMHD3', 'B', 2)\n",
      "('[', 'O', 2)\n",
      "('18', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('MMH1-1', 'B', 2)\n",
      "('[', 'O', 0)\n",
      "('19', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('parasite', 'O', 2)\n",
      "('T.', 'O', 2)\n",
      "('spiralis', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('ISS534', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('acquired', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('domestic', 'O', 2)\n",
      "('pig', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('central', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('25', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('maintained', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('laboratory', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('serial', 'O', 2)\n",
      "('passaging', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BALB', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('c', 'O', 2)\n",
      "('mice', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Subsequently', 'O', -100)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('discovered', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('occurring', 'O', 2)\n",
      "('sympatrically', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('B.', 'O', -100)\n",
      "('burgdorferi', 'O', -100)\n",
      "('sensu', 'O', 2)\n",
      "('lato', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('several', 'O', 2)\n",
      "('Ixodes', 'O', 2)\n",
      "('species', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('also', 'O', -100)\n",
      "('transmit', 'O', 2)\n",
      "('Lyme', 'O', 2)\n",
      "('disease', 'O', 2)\n",
      "('spirochetes', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('result', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('CPCC', 'B', 2)\n",
      "('101082', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('CPCC', 'B', -100)\n",
      "('101083', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('sandy', 'O', -100)\n",
      "('soil', 'O', 1)\n",
      "('samples', 'O', -100)\n",
      "('collected', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('Badain', 'O', -100)\n",
      "('Jaran', 'O', 2)\n",
      "('Desert', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('indole-3-acetic', 'O', -100)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('IAA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('-producing', 'O', 2)\n",
      "('activity', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Briefly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('The', 'O', 2)\n",
      "('CM-5', 'O', 2)\n",
      "('chip', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('activated', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('1:1', 'O', 2)\n",
      "('mixture', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('0.4', 'O', 2)\n",
      "('M', 'O', -100)\n",
      "('1-ethyl-3-', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('3-dimethylaminopropyl', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('carbodiimide', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('0.1', 'O', 2)\n",
      "('M', 'O', 2)\n",
      "('N-hydroxysuccinimide', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('7', 'O', -100)\n",
      "('min', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('As', 'O', -100)\n",
      "('previously', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 0)\n",
      "('5.1', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('7.1-7.3', 'O', 1)\n",
      "('strains', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('biopsy', 'O', -100)\n",
      "('tissues', 'O', -100)\n",
      "('obtained', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('OLP', 'O', -100)\n",
      "('patient', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('It', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('worth', 'O', 2)\n",
      "('noting', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('deletion', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('nidulans', 'O', 2)\n",
      "('atfA', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Fusarium', 'O', 2)\n",
      "('verticillioides', 'O', -100)\n",
      "('FvatfA', 'O', 2)\n",
      "('also', 'O', -100)\n",
      "('resulted', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('drastically', 'O', 2)\n",
      "('decreased', 'O', -100)\n",
      "('mycotoxin', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('namely', 'O', 2)\n",
      "('sterigmatocystin', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('fumonisin', 'O', 2)\n",
      "('productions', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('fungi', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Szabó', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2020', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('Kocsis', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2022', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Two', 'O', -100)\n",
      "('independent', 'O', 2)\n",
      "('clones', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Genbank', 'O', 2)\n",
      "('accessions', 'O', 2)\n",
      "('AF435838', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('AF435839', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('recovered', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('former', 'O', -100)\n",
      "('being', 'O', 2)\n",
      "('structurally', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('EST', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('BDGP', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('clot', 'O', 2)\n",
      "('#', 'O', 2)\n",
      "('7147', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('mRNA', 'O', 2)\n",
      "('level', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('α-SMA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('vimentin', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('fibronectin', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('upregulated', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('MDCK', 'B', 2)\n",
      "('cells', 'O', 2)\n",
      "('exposed', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('TGF-β1', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Our', 'O', -100)\n",
      "('study', 'O', 2)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('M.', 'O', 2)\n",
      "('catarrhalis', 'O', 2)\n",
      "('OMVs', 'O', -100)\n",
      "('induce', 'O', 2)\n",
      "('efficient', 'O', -100)\n",
      "('cell', 'O', -100)\n",
      "('death', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('apoptosis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('A549', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('complete', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('agarivorans', 'I', 0)\n",
      "('Hao', 'I', -100)\n",
      "('2018', 'I', 1)\n",
      "('strain', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('microbial', 'O', 1)\n",
      "('membrane', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('abalone', 'O', 2)\n",
      "('seedlings', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('via', 'O', 2)\n",
      "('whole-genome', 'O', 2)\n",
      "('sequencing', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('cysteine', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('methionine', 'O', 2)\n",
      "('metabolism', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('paracasei', 'I', 0)\n",
      "('FAM18149', 'I', -100)\n",
      "('grown', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('chemically', 'O', 1)\n",
      "('defined', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('CDM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('cysteine', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sole', 'O', 2)\n",
      "('sulfur', 'O', 2)\n",
      "('source', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('CDM', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('methionine', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sole', 'O', 2)\n",
      "('sulfur', 'O', 2)\n",
      "('source', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Here', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('report', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('characterization', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('Marseille-P9525', 'B', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('human', 'O', -100)\n",
      "('gut', 'O', -100)\n",
      "('microbiota', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('representative', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('new', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Indeed', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('families', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('divergent', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('G.', 'O', 2)\n",
      "('intestinalis', 'O', 2)\n",
      "('WB', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('GS', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Genera', 'O', -100)\n",
      "('Blautia', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CF231', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Prevotella', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Butyrivibrio', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Mogibacterium', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Moryella', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('undefined', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('families', 'O', -100)\n",
      "('Ruminococacceae', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('BS11', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Christensenellaceae', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Mogibacteriaceae', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Clostridiaceae', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('S24-7', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Desulfovibrionaceae', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Lachnospiraceae', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('order', 'O', -100)\n",
      "('Clostridiales', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('positively', 'O', 2)\n",
      "('correlated', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('isovaleric', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('isobutyric', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('total', 'O', -100)\n",
      "('BCVFA', 'O', -100)\n",
      "('proportions', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('whereas', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('Coprococcus', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Prevotella', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('undefined', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('family', 'O', -100)\n",
      "('RF16', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('negatively', 'O', 2)\n",
      "('correlated', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('them', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Fan', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('IOP', 'O', -100)\n",
      "('inhibits', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('progression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('gastric', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('stimulating', 'O', 2)\n",
      "('lymphocyte', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fan', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2012', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', -100)\n",
      "('Lee', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "('illustrated', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('IOP', 'O', 2)\n",
      "('inhibits', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('invasion', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('migration', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('B16-F10', 'O', 2)\n",
      "('melanoma', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Lee', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2014', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Three', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('acidophile', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('ferriphilum', 'I', 2)\n",
      "('DSM', 'I', 0)\n",
      "('14647^T', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('S.', 'B', -100)\n",
      "('thermosulfidooxidans', 'I', -100)\n",
      "('DSM', 'I', -100)\n",
      "('9293^T', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', 1)\n",
      "('A.', 'B', -100)\n",
      "('caldus', 'I', -100)\n",
      "('DSM', 'I', -100)\n",
      "('8584^T', 'I', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('FE', 'O', 2)\n",
      "('models', 'O', 2)\n",
      "('differed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('medial-lateral', 'O', 2)\n",
      "('positioning', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('bone', 'O', 2)\n",
      "('graft', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ranging', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('flush', 'O', 2)\n",
      "('position', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('4.5', 'O', 2)\n",
      "('mm', 'O', 2)\n",
      "('lateral', 'O', 2)\n",
      "('position', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('respect', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('flush', 'O', 2)\n",
      "('position', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('BG-AF3-A', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('=DSM', 'B', -100)\n",
      "('110574', 'I', -100)\n",
      "('^T', 'O', -100)\n",
      "('=LMG', 'B', -100)\n",
      "('31633', 'I', 2)\n",
      "('^T', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('which', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('jejunum', 'O', -100)\n",
      "('of', 'O', 0)\n",
      "('yellow-necked', 'O', -100)\n",
      "('mouse', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Apodemus', 'O', -100)\n",
      "('flavicollis', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('caught', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Vilnius', 'O', 2)\n",
      "('area', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Lithuania', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('For', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('purification', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('they', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('transformed', 'O', 2)\n",
      "('into', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 0)\n",
      "('Rosetta', 'I', -100)\n",
      "('strain', 'O', 1)\n",
      "('containing', 'O', 1)\n",
      "('pREP4', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Qiagen', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('report', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('effect', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('light', 'O', 2)\n",
      "('intensity', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('spectral', 'O', 2)\n",
      "('composition', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('ROS', 'O', 2)\n",
      "('generation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('measured', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('real-time', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('laser-photoacoustic', 'O', -100)\n",
      "('gas', 'O', 2)\n",
      "('detector', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('three', 'O', -100)\n",
      "('different', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('NIR', 'O', 2)\n",
      "('utilizing', 'O', 2)\n",
      "('cyanobacteria', 'O', 2)\n",
      "('belonging', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genus', 'O', 2)\n",
      "('Acaryochloris', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('new', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('named', 'O', -100)\n",
      "('Acaryochloris', 'B', -100)\n",
      "('CRS', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('phototrophic', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('growing', 'O', 2)\n",
      "('on', 'O', 0)\n",
      "('dead', 'O', -100)\n",
      "('coral', 'O', -100)\n",
      "('branches', 'O', -100)\n",
      "('collected', 'O', 1)\n",
      "('on', 'O', 2)\n",
      "('Heron', 'O', 2)\n",
      "('Island', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Australia', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Compared', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('American', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Haitian', 'O', 2)\n",
      "('HCoV-229E', 'O', 2)\n",
      "('genome', 'O', -100)\n",
      "('has', 'O', 2)\n",
      "('unique', 'O', -100)\n",
      "('nucleotide', 'O', -100)\n",
      "('polymorphisms', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('result', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('changes', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('deduced', 'O', 2)\n",
      "('amino', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('replicase', 'O', 2)\n",
      "('polyprotein', 'O', 2)\n",
      "('1ab', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('spike', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('accessory', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('membrane', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('envelope', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('nucleocapsid', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('complete', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('determined', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('combination', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('short-', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('long-read', 'O', -100)\n",
      "('sequencing', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('three', 'O', -100)\n",
      "('Lactiplantibacillus', 'O', 2)\n",
      "('plantarum', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('M8', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('M17', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('M19', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('Iranian', 'O', 2)\n",
      "('motal', 'O', 0)\n",
      "('cheese', 'O', -100)\n",
      "('are', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Correlation', 'O', -100)\n",
      "('analyses', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('relative', 'O', 2)\n",
      "('abundance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('markedly', 'O', 2)\n",
      "('altered', 'O', 2)\n",
      "('gut', 'O', 2)\n",
      "('microbiota', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('hyperlipidemia-related', 'O', 2)\n",
      "('parameters', 'O', 2)\n",
      "('-', 'O', -100)\n",
      "('TG', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('TC', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('LDL-C', 'O', 2)\n",
      "('-', 'O', 2)\n",
      "('suggested', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('abundances', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('Bifidobacterium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Ruminococcaceae_UCG-014', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('negatively', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('p', 'O', 2)\n",
      "('<', 'O', 2)\n",
      "('0.05', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('correlated', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('levels', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('TG', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('TC', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('LDL-C', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('serum', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figures', 'O', 2)\n",
      "('6H-M', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('abundance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Lactobacillus', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('positively', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('p', 'O', -100)\n",
      "('<', 'O', 2)\n",
      "('0.05', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('correlated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('levels', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('TG', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('TC', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('LDL-C', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('serum', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figures', 'O', 2)\n",
      "('6N-P', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('data', 'O', 2)\n",
      "('article', 'O', 2)\n",
      "('presents', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('high-quality', 'O', 2)\n",
      "('long-read', 'O', 2)\n",
      "('genome', 'O', -100)\n",
      "('sequence', 'O', -100)\n",
      "('data', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('pomegranate', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('blight', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('Xanthomonas', 'B', 2)\n",
      "('citri', 'I', 2)\n",
      "('pv.', 'I', -100)\n",
      "('punicae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('119', 'I', 2)\n",
      "('[', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('MT-4', 'B', 2)\n",
      "('cells', 'O', 0)\n",
      "('transduced', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('CPSF6ΔNCs', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('GFP', 'O', 2)\n",
      "('dual-expressing', 'O', -100)\n",
      "('lentiviral', 'O', -100)\n",
      "('vector', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('assessed', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('HIV', 'O', 2)\n",
      "('susceptibility', 'O', 2)\n",
      "('using', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('VSV', 'O', 2)\n",
      "('pseudotyped', 'O', 2)\n",
      "('CFP-expressing', 'O', 2)\n",
      "('lentiviral', 'O', 2)\n",
      "('vector', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('contrast', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('low-identity', 'O', 2)\n",
      "('subgroup', 'O', 2)\n",
      "('contains', 'O', -100)\n",
      "('S.', 'O', -100)\n",
      "('pneumoniae', 'O', 2)\n",
      "('WchF', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('known', 'O', 2)\n",
      "('d-Glc-β-1,4-l-rhamnosyltransferase', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('84', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('2', 'O', -100)\n",
      "('×', 'O', 2)\n",
      "('10^6', 'O', 2)\n",
      "('MCF7', 'O', 2)\n",
      "('cells', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('seeded', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('25', 'O', 2)\n",
      "('cm^2', 'O', 2)\n",
      "('flasks', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Analysis', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('fatty', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('profiles', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('iso-C', 'O', 2)\n",
      "('15:0', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('anteiso-C', 'O', -100)\n",
      "('15:0', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('C', 'O', -100)\n",
      "('16:0', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('C', 'O', -100)\n",
      "('18:1', 'O', -100)\n",
      "('ω9c', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('iso-C', 'O', -100)\n",
      "('17:0', 'O', -100)\n",
      "('3OH', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('major', 'O', -100)\n",
      "('fatty', 'O', -100)\n",
      "('acids', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('three', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('additionally', 'O', -100)\n",
      "('C', 'O', -100)\n",
      "('16:0', 'O', 2)\n",
      "('3OH', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('AN421', 'B', -100)\n",
      "('T', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('AN502', 'B', -100)\n",
      "('.', 'O', -100)\n",
      "('BUN14', 'B', -100)\n",
      "(',', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('hydrocarbon-polluted', 'O', 2)\n",
      "('sediments', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('Bizerte', 'O', -100)\n",
      "('coast', 'O', 2)\n",
      "('refinery', 'O', 2)\n",
      "('harbour', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('north', 'O', -100)\n",
      "('Tunisia', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('exhibits', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('capacity', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('hydrocarbon', 'O', 2)\n",
      "('degradation', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('ATCC', 'B', 2)\n",
      "('BAA-835', 'I', 0)\n",
      "('^T', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('=', 'O', -100)\n",
      "('KCTC', 'B', -100)\n",
      "('15667', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 0)\n",
      "('healthy', 'O', -100)\n",
      "('adult', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('Netherlands', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('included', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('fluorescence', 'O', 2)\n",
      "('intensity', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('reuteri', 'I', 0)\n",
      "('AMBV339', 'I', -100)\n",
      "('was', 'O', 1)\n",
      "('close', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('riboflavin', 'O', -100)\n",
      "('control', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('25', 'O', 2)\n",
      "('μg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('reuteri', 'I', 2)\n",
      "('AMBV336', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('reuteri', 'I', 2)\n",
      "('RC-14', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('L.', 'B', 1)\n",
      "('rhamnosus', 'I', -100)\n",
      "('GG', 'I', -100)\n",
      "('culture', 'O', 1)\n",
      "('fluorescence', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('MRS', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('much', 'O', 0)\n",
      "('less', 'O', -100)\n",
      "('pronounced', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Complete', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Bifidobacterium', 'B', 2)\n",
      "('adolescentis', 'I', 0)\n",
      "('P2P3', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('human', 'O', -100)\n",
      "('gut', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('possessing', 'O', 2)\n",
      "('strong', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('starch-degrading', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('L.', 'B', -100)\n",
      "('monocytogenes', 'I', 0)\n",
      "('F4244', 'I', -100)\n",
      "('(', 'I', 1)\n",
      "('serovar', 'I', 1)\n",
      "('4b', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('clinical', 'O', 1)\n",
      "('epidemic', 'O', 1)\n",
      "('strain', 'O', 1)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('cultured', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('tryptone', 'O', 2)\n",
      "('soy', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('0.6', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('TSB‐YE', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('or', 'O', -100)\n",
      "('brain', 'O', 2)\n",
      "('heart', 'O', 2)\n",
      "('infusion', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BHI', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('broth', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('37°C', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('18', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('may', 'O', 2)\n",
      "('raise', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('serious', 'O', 2)\n",
      "('public', 'O', 2)\n",
      "('health', 'O', 2)\n",
      "('concern', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('because', 'O', 2)\n",
      "('HAdV-55', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('potential', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('spread', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cause', 'O', 2)\n",
      "('severe', 'O', 2)\n",
      "('epidemics', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('become', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('major', 'O', 2)\n",
      "('etiological', 'O', 2)\n",
      "('agent', 'O', 2)\n",
      "('causing', 'O', 2)\n",
      "('pneumonia', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Chinese', 'O', 2)\n",
      "('population', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('dominant', 'O', 2)\n",
      "('band', 'O', 2)\n",
      "('appearing', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('DGGE', 'O', 2)\n",
      "('analyses', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('subcloned', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('DGGE', 'O', -100)\n",
      "('amplicon', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('pGEM-T', 'O', -100)\n",
      "('easy', 'O', 2)\n",
      "('vector', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Promega', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Leiden', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('The', 'O', 2)\n",
      "('Netherlands', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('transformed', 'O', -100)\n",
      "('into', 'O', -100)\n",
      "('Stratagene', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', -100)\n",
      "('XL-1', 'I', 2)\n",
      "('Blue', 'I', 2)\n",
      "('competent', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Agilent', 'O', 2)\n",
      "('Technologies', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Amstelveen', 'O', -100)\n",
      "(',', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('Netherlands', 'O', 1)\n",
      "(')', 'O', 1)\n",
      "('according', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('manufacturers’', 'O', 2)\n",
      "('specifications', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Though', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('pathogenic', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('potato', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('D.', 'B', 2)\n",
      "('dadantii', 'I', 0)\n",
      "('3937', 'I', -100)\n",
      "('is', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('well-characterized', 'O', -100)\n",
      "('model', 'O', 1)\n",
      "('while', 'O', -100)\n",
      "('D.', 'O', 2)\n",
      "('dianthicola', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('ME23', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('67-19', 'B', 2)\n",
      "('are', 'O', 2)\n",
      "('recent', 'O', 2)\n",
      "('isolates', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Six', 'O', -100)\n",
      "('different', 'O', 2)\n",
      "('copies', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('starvation', 'O', 2)\n",
      "('lipoprotein', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Slp', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('no', 'O', 2)\n",
      "('proven', 'O', 2)\n",
      "('function', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('co-expressed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('acid', 'O', -100)\n",
      "('resistance', 'O', -100)\n",
      "('island', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Mates', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('2007', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('identified', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('M.', 'B', -100)\n",
      "('infernorum', 'I', 2)\n",
      "('V4', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('orthologs', 'O', 2)\n",
      "('also', 'O', 0)\n",
      "('identified', 'O', -100)\n",
      "('in', 'O', 1)\n",
      "('both', 'O', -100)\n",
      "('other', 'O', -100)\n",
      "('complete', 'O', 1)\n",
      "('genomes', 'O', -100)\n",
      "('M.', 'O', 2)\n",
      "('fumariolicum', 'O', 2)\n",
      "('SoIV', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('M.', 'O', -100)\n",
      "('kamchatkense', 'O', 2)\n",
      "('Kam1', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('adenoviral-mediated', 'O', 2)\n",
      "('ectopic', 'O', 2)\n",
      "('expression', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('mPdx1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('mNgn3', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('mMafA', 'O', -100)\n",
      "('induced', 'O', -100)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('endocrine', 'O', -100)\n",
      "('genes', 'O', -100)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('Ins2', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('IAPP', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('NeuroD1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Pax4', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('both', 'O', -100)\n",
      "('AR42J', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('B13', 'O', -100)\n",
      "('cells', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('although', 'O', -100)\n",
      "('much', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('levels', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('ranging', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('1.5-fold', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('30-fold', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('B13', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1E-1H', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('draft', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Raphidiopsis', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Cylindrospermopsis', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('raciborskii', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('GIHE', 'B', -100)\n",
      "('2018', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('filamentous', 'O', 2)\n",
      "('nitrogen-fixing', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('potentially', 'O', -100)\n",
      "('harmful', 'O', 2)\n",
      "('cyanobacterium', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('is', 'O', 1)\n",
      "('reported', 'O', 2)\n",
      "('here', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Gene', 'O', -100)\n",
      "('family', 'O', 2)\n",
      "('clustering', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('Orthovenn2', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('default', 'O', -100)\n",
      "('parameters', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Xu', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "('2019', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('sequences', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('Enterobacter', 'B', 2)\n",
      "('mori', 'I', 2)\n",
      "('CX01', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('Enterobacter', 'B', 2)\n",
      "('mori', 'I', 0)\n",
      "('LMG', 'I', 1)\n",
      "('25706', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('Enterobacter', 'O', -100)\n",
      "('cloacae', 'O', 2)\n",
      "('(', 'O', 0)\n",
      "('NC_014121.1', 'O', 1)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('Enterobacter', 'O', -100)\n",
      "('cancerogenus', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('NZ_CP025225.1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Enterobacter', 'O', -100)\n",
      "('hormaechei', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('NZ_CP017179.1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Additionally', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('basis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('findings', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('AbcA', 'O', 2)\n",
      "('transporter', 'O', 2)\n",
      "('can', 'O', -100)\n",
      "('be', 'O', 2)\n",
      "('associated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('BC12', 'B', 2)\n",
      "('resistance', 'O', 2)\n",
      "('towards', 'O', 0)\n",
      "('polymyxin', 'O', -100)\n",
      "('B', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('39', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('quinolone', 'O', 2)\n",
      "('pipemidic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('sequenced', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('assembled', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('whole-genome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P', 'B', -100)\n",
      "('.', 'I', -100)\n",
      "('agglomerans', 'I', 2)\n",
      "('KM1', 'I', 0)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', 1)\n",
      "('kimchi', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('South', 'O', 1)\n",
      "('Korea', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('CSIRO-Mk3.6.0', 'O', -100)\n",
      "('has', 'O', 2)\n",
      "('prescribed', 'O', -100)\n",
      "('vegetation', 'O', -100)\n",
      "('properties', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('cell-wall', 'O', 2)\n",
      "('peptidoglycan', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('BG-AF3-A', 'B', -100)\n",
      "('^T', 'O', 2)\n",
      "('contains', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('amino', 'O', -100)\n",
      "('acids', 'O', -100)\n",
      "('alanine', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Ala', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('glutamic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Glu', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('lysine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Lys', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('aspartic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Asp', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('molar', 'O', 2)\n",
      "('ratio', 'O', 2)\n",
      "('1.3', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ala', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':0.8', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Asp', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':1.0', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Glu', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(':1.0', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Lys', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('suggesting', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('cell-wall', 'O', 2)\n",
      "('peptidoglycan', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('A4α', 'O', 2)\n",
      "('l', 'O', -100)\n",
      "('-Lys', 'O', -100)\n",
      "('-d', 'O', -100)\n",
      "('-Asp', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('38', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('vietnamiensis', 'I', 0)\n",
      "('G4', 'I', -100)\n",
      "('bacterium', 'O', 1)\n",
      "('was', 'O', -100)\n",
      "('exposed', 'O', 1)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('PAH', 'O', 2)\n",
      "('benzo', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('pyrene', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('differentially', 'O', 2)\n",
      "('expressed', 'O', 2)\n",
      "('when', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('contact', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('compound', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('evaluated', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Thus', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('if', 'O', 2)\n",
      "('BB0017', 'O', 2)\n",
      "('does', 'O', 2)\n",
      "('bind', 'O', -100)\n",
      "('ligands', 'O', -100)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('copper', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('c-di-AMP', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('previously', 'O', -100)\n",
      "('shown', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('PII', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('PII-like', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('does', 'O', 2)\n",
      "('so', 'O', 2)\n",
      "('via', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('unique', 'O', 2)\n",
      "('mechanism.', 'O', -100)\n",
      "('c-di-AMP', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('produced', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('burgdorferi', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('although', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('function', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('second', 'O', -100)\n",
      "('messenger', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('this', 'O', -100)\n",
      "('organism', 'O', 2)\n",
      "('remains', 'O', 2)\n",
      "('unclear', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('79,80', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('assembled', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('Wolbachia', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('wMelCS', 'B', 2)\n",
      "('^112', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('wMelPlus', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 0)\n",
      "('sequencing', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('material', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('ovaries', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('homogenized', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whole-female', 'O', 2)\n",
      "('infected', 'O', 2)\n",
      "('flies', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Drosophila', 'O', 2)\n",
      "('melanogaster', 'O', -100)\n",
      "('Bi90', 'O', 2)\n",
      "('^wMelCS112', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Bi90', 'O', -100)\n",
      "('^wMelPlus', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('see', 'O', 2)\n",
      "('Section', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('details', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Besides', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AB211', 'B', 2)\n",
      "('showed', 'O', 0)\n",
      "('positive', 'O', -100)\n",
      "('motility', 'O', -100)\n",
      "('through', 'O', 2)\n",
      "('diffused', 'O', 2)\n",
      "('cloudy', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('away', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('line', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('inoculation', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('semisolid', 'O', 2)\n",
      "('M9', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Supplementary', 'O', 2)\n",
      "('Figure', 'O', -100)\n",
      "('S7D', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('THP-1', 'B', -100)\n",
      "('cells', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('pre-treated', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('before', 'O', 2)\n",
      "('infection', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('PGE2', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('μM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('vehicle', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ethanol', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('then', 'O', 2)\n",
      "('infected', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('mentioned', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('either', 'O', 2)\n",
      "('S', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('suggests', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('luminescens', 'I', 0)\n",
      "('HIM3', 'I', -100)\n",
      "('has', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('larger', 'O', -100)\n",
      "('arsenal', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('insecticidal', 'O', 2)\n",
      "('toxins', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('greater', 'O', 2)\n",
      "('specificity', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('D.', 'O', 2)\n",
      "('magnifactella', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('CPCC', 'B', 2)\n",
      "('101083', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('=NBRC', 'B', -100)\n",
      "('113514', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "('=', 'O', -100)\n",
      "('KCTC', 'B', 2)\n",
      "('62853', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('sandy', 'O', 2)\n",
      "('dune', 'O', -100)\n",
      "('sample', 'O', 2)\n",
      "('with', 'O', 0)\n",
      "('moss-dominated', 'O', -100)\n",
      "('crusts', 'O', 1)\n",
      "('collected', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Badain', 'O', 2)\n",
      "('Jaran', 'O', -100)\n",
      "('desert', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Unlike', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('UPR', 'O', 2)\n",
      "('mutants', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ire1Δ', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('hxl1Δ', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('deletion', 'O', 2)\n",
      "('mutants', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('CNAG_00871', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CNAG_07560', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CNAG_07940', 'O', 2)\n",
      "('did', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('show', 'O', -100)\n",
      "('any', 'O', -100)\n",
      "('increased', 'O', 2)\n",
      "('sensitivity', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('tunicamycin', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('DTT', 'O', -100)\n",
      "('compared', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('wild-type', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('WT', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('indicating', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('bZIP', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('involved', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('ER', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('responses', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Interestingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('cells', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('expressing', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('functional', 'O', 2)\n",
      "('LinA1', 'O', 2)\n",
      "('S-glutathione', 'O', 2)\n",
      "('transferase', 'O', 2)\n",
      "('fusion', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('preferentially', 'O', -100)\n",
      "('transformed', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('+', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('enantiomer', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('α-HCH', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Suar', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Basic', 'O', -100)\n",
      "('morphological', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('biochemical', 'O', 2)\n",
      "('tests', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('carried', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('MMB-1', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('including', 'O', -100)\n",
      "('Gram-staining', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('indole', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('catalase', 'O', -100)\n",
      "('production', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('according', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('standard', 'O', 2)\n",
      "('practice', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('8R', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('-8-Bromo-10-epi-β-snyderol', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('78', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('sesquiterpene', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Laurencia', 'O', -100)\n",
      "('obtuse', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Rhodomelaceae', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('red', 'O', -100)\n",
      "('alga', 'O', -100)\n",
      "('displayed', 'O', -100)\n",
      "('activity', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('falciparum', 'I', 2)\n",
      "('D6', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('W2', 'B', -100)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('D6', 'B', 2)\n",
      "('=', 'O', -100)\n",
      "('0.012', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('W2', 'B', 2)\n",
      "('=', 'O', -100)\n",
      "('0.017', 'O', 2)\n",
      "('μM', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('SV-A', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('KS04105', 'O', -100)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('LLC-PK', 'B', 2)\n",
      "('cells', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('purified', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('caesium', 'O', 2)\n",
      "('chloride', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('CsCl', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('gradient', 'O', 2)\n",
      "('centrifugation', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('previously', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Kim', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('2016', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('sum', 'O', 2)\n",
      "('up', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('multiple-HM', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('severely', 'O', -100)\n",
      "('affected', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('physiology', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('wheat', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('inoculation', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('HM-resistant', 'O', 2)\n",
      "('PGP', 'O', 2)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 2)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', -100)\n",
      "('and', 'O', 2)\n",
      "('Enterobacter', 'B', 0)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', 1)\n",
      "('JWM6', 'I', -100)\n",
      "('either', 'O', -100)\n",
      "('alone', 'O', -100)\n",
      "('or', 'O', -100)\n",
      "('in', 'O', 1)\n",
      "('consortium', 'O', 1)\n",
      "('reduced', 'O', -100)\n",
      "('Cd', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('Pb', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('Ni', 'O', 1)\n",
      "('uptake', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('toxicity', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('enhanced', 'O', 2)\n",
      "('wheat', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Enteritidis', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Central', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('Eastern', 'O', 2)\n",
      "('African', 'O', 2)\n",
      "('clade', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CP255', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('D7795', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('two', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('Global', 'O', 2)\n",
      "('Epidemic', 'O', 2)\n",
      "('clade', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('P125109', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('A1636', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('mMVs', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('DQ12-45-1b', 'B', -100)\n",
      "('appear', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('differ', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('those', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('other', 'O', -100)\n",
      "('Gram-positive', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('various', 'O', 2)\n",
      "('aspects', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Draft', 'O', -100)\n",
      "('genomic', 'O', 2)\n",
      "('alignment', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sample', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('melitensis', 'I', 0)\n",
      "('16M', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('Assembly', 'O', -100)\n",
      "('ID', 'O', 1)\n",
      "(':', 'O', -100)\n",
      "('GCA_000007125.1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('suis', 'I', 2)\n",
      "('1330', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('Assembly', 'O', -100)\n",
      "('ID', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('GCA_000007505.1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('performed', 'O', 0)\n",
      "('using', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('MUMmer', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Kurtz', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2004', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('LASTZ', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Chiaromonte', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2002', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('tools', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('date', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('six', 'O', 2)\n",
      "('soil-born', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('B.', 'O', -100)\n",
      "('petrii', 'O', 2)\n",
      "('whose', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('has', 'O', -100)\n",
      "('been', 'O', 2)\n",
      "('sequenced', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Type', 'O', 2)\n",
      "('Strain', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('petrii', 'I', 2)\n",
      "('DSM', 'I', 2)\n",
      "('12804', 'I', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('complete', 'O', 0)\n",
      "('genome', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('reported', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('river', 'O', 1)\n",
      "('sediment', 'O', -100)\n",
      "('enriched', 'O', -100)\n",
      "('dechlorinating', 'O', 2)\n",
      "('bioreactor', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('27', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('identify', 'O', 2)\n",
      "('TC2-1-specific', 'B', 2)\n",
      "('genes', 'O', 0)\n",
      "('related', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('host', 'O', -100)\n",
      "('specialization', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('phylogenetic', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('TC2-1', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('X.', 'O', 2)\n",
      "('perforans', 'O', 2)\n",
      "('strains', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('NCBI', 'O', -100)\n",
      "('RefSeq', 'O', 2)\n",
      "('database', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('120', 'O', -100)\n",
      "('conserved', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('single-copy', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('selected', 'O', 2)\n",
      "('five', 'O', -100)\n",
      "('representative', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('further', 'O', 2)\n",
      "('comparative', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('tomato', 'O', 2)\n",
      "('leaves', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('NI1', 'B', 2)\n",
      "('from', 'O', 2)\n",
      "('Northern', 'O', 2)\n",
      "('guinea', 'O', 2)\n",
      "('savannah', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('Xp11-2', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('GEV2392', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CFBP', 'B', 2)\n",
      "('7293', 'I', 2)\n",
      "('from', 'O', 2)\n",
      "('different', 'O', 0)\n",
      "('regions', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('US', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('one', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AL66', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('pepper', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('transcriptomic', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('UW551', 'B', 2)\n",
      "('grown', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('CPG', 'O', -100)\n",
      "('medium', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('2.33-fold', 'O', 2)\n",
      "('decrease', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('ripS1', 'O', 2)\n",
      "('expression', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('20°C', 'O', -100)\n",
      "('compared', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('28°C', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Meng', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2015', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('generated', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('kinds', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('stable', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('HEK293', 'B', 2)\n",
      "('cells', 'O', 0)\n",
      "('transfected', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('β-arrestin', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ARRB2', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('different', 'O', 2)\n",
      "('GPCRs', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('connected', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('pair', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('ELuc', 'O', 2)\n",
      "('fragments', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('vagans', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('our', 'O', -100)\n",
      "('research-SRS89-was', 'B', 2)\n",
      "('able', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('synthesize', 'O', 0)\n",
      "('several', 'O', -100)\n",
      "('types', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('growth', 'O', -100)\n",
      "('regulators', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('great', 'O', 2)\n",
      "('importance', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('GA', 'O', 2)\n",
      "('_3', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('abundant', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Methylobacillus', 'B', -100)\n",
      "('flagellatus', 'I', 0)\n",
      "('KT', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('Methylomicrobium', 'B', -100)\n",
      "('alcaliphilum', 'I', -100)\n",
      "('20Z', 'I', 1)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 0)\n",
      "('can', 'O', -100)\n",
      "('utilize', 'O', -100)\n",
      "('methane', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('have', 'O', -100)\n",
      "('sucrose', 'O', -100)\n",
      "('utilizing', 'O', -100)\n",
      "('clusters', 'O', 1)\n",
      "('similar', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('Synechococcus', 'O', 2)\n",
      "('sp.', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('mfas', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('locus_tag', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('CP000284.1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('ID', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('ABE50875.1', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('maas', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('gene', 'O', 2)\n",
      "('locus_tag', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('FO082060.1', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('ID', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('CCE22312.1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Methylobacillus', 'B', -100)\n",
      "('flagellatus', 'I', 2)\n",
      "('KT', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Methylomicrobium', 'B', 2)\n",
      "('alcaliphilum', 'I', -100)\n",
      "('20Z', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('successfully', 'O', 2)\n",
      "('expressed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('E.', 'O', -100)\n",
      "('coli', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('typical', 'O', -100)\n",
      "('AS', 'O', -100)\n",
      "('activity', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('But', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2015', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('Jeong', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2014', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('successfully', 'O', 2)\n",
      "('engineered', 'O', 2)\n",
      "('AM1', 'B', 2)\n",
      "('to', 'O', 0)\n",
      "('produce', 'O', -100)\n",
      "('ITA', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('With', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('aim', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('evaluate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('influence', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('attachment', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('effect', 'O', 2)\n",
      "('over', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('solubilization', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('polymetallic', 'O', 2)\n",
      "('mineral', 'O', 2)\n",
      "('concentrate', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('behavior', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('microbial', 'O', 2)\n",
      "('consortium', 'O', 2)\n",
      "('formed', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('Acidithiobacillus', 'B', 2)\n",
      "('thiooxidans', 'I', 2)\n",
      "('DSM', 'I', 0)\n",
      "('14887T', 'I', -100)\n",
      "('and', 'O', -100)\n",
      "('Leptospirillum', 'B', 1)\n",
      "('ferrooxidans', 'I', -100)\n",
      "('DSM', 'I', -100)\n",
      "('2705T', 'I', 1)\n",
      "('previously', 'O', 1)\n",
      "('induced', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('D-galactose', 'O', 2)\n",
      "('for', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('early', 'O', -100)\n",
      "('formation', 'O', -100)\n",
      "('of', 'O', 1)\n",
      "('EPS', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('studied', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('One', 'O', -100)\n",
      "('crystal', 'O', 2)\n",
      "('shows', 'O', 2)\n",
      "('essentially', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('features', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('above', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('chain', 'O', 2)\n",
      "('A', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('adenine', 'O', 2)\n",
      "('instead', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('AMP', 'O', 2)\n",
      "('moiety', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('FAD', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('well', 'O', 2)\n",
      "('defined', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('species', 'O', 2)\n",
      "('Pseudoalteromonas', 'O', 2)\n",
      "('paragorgicola', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('single', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('KMM', 'B', -100)\n",
      "('3548', 'I', -100)\n",
      "('^T', 'O', -100)\n",
      "('has', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('recently', 'O', 2)\n",
      "('reclassified', 'O', 0)\n",
      "('as', 'O', -100)\n",
      "('P.', 'B', 1)\n",
      "('distincta', 'I', -100)\n",
      "('KMM', 'I', 2)\n",
      "('3548', 'I', -100)\n",
      "('[', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Moreover', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('neopeptin', 'O', 2)\n",
      "('A-C', 'O', 2)\n",
      "('are', 'O', -100)\n",
      "('identified', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('S.', 'B', 2)\n",
      "('hygroscopicus', 'I', -100)\n",
      "('NRRL', 'I', -100)\n",
      "('B-1477', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('S.', 'B', 2)\n",
      "('rimosus', 'I', 0)\n",
      "('NRRL', 'I', -100)\n",
      "('WC387', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('Streptomyces', 'O', -100)\n",
      "('sp', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Circular', 'O', -100)\n",
      "('dichroism', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CD', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('spectroscopy', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('atomic', 'O', 2)\n",
      "('force', 'O', 2)\n",
      "('microscopy', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AFM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('indicated', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('structure', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('preserved', 'O', 2)\n",
      "('even', 'O', 2)\n",
      "('if', 'O', 2)\n",
      "('B800', 'O', 2)\n",
      "('BChl', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('converted', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('3-acetyl', 'O', 2)\n",
      "('Chl', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('result', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('following', 'O', 2)\n",
      "('seven', 'O', 2)\n",
      "('drugs', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('finally', 'O', 2)\n",
      "('selected', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('hit', 'O', 2)\n",
      "('compounds', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('TA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('norepinephrine', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('diflunisal', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('althiazide', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('hyoscyamine', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('chlorophyllide', 'O', 2)\n",
      "('Cu', 'O', -100)\n",
      "('complex', 'O', -100)\n",
      "('Na', 'O', 2)\n",
      "('salt', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('pirenperone', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Table', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('With', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('enzyme-encoding', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('discussed', 'O', -100)\n",
      "('above', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('metabolic', 'O', 2)\n",
      "('networks', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('typhimurium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('K.', 'O', 2)\n",
      "('pneumoniae', 'O', -100)\n",
      "('can', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('reconstructed', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('organisms', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('previous', 'O', 2)\n",
      "('work', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('L.', 'O', 2)\n",
      "('kunkeei', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('named', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('MP2', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('gut', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('Chilean', 'O', 2)\n",
      "('honey', 'O', 2)\n",
      "('bees', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('María', 'O', 2)\n",
      "('Pinto', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Melipilla', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Olmos', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('2014', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('test', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('hypothesis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('investigated', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('effect', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('diverse', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('artichoke', 'O', 2)\n",
      "('bracts', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('fermentation', 'O', 2)\n",
      "('quality', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('berseem', 'O', 2)\n",
      "('silage', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('ensiling', 'O', 2)\n",
      "('times', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Nevertheless', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('set', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('investigate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('pathogenicity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('Hymenoscyphus', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('studied', 'O', -100)\n",
      "('here', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Instead', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('forming', 'O', 2)\n",
      "('fibrils', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('insoluble', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('high-molecular-weight', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('A30P', 'O', -100)\n",
      "('forms', 'O', -100)\n",
      "('relatively', 'O', -100)\n",
      "('soluble', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('partially', 'O', 2)\n",
      "('aggregated', 'O', -100)\n",
      "('species', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Conway', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('On', 'O', -100)\n",
      "('day', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('p.i.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('titer', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('R.', 'O', 2)\n",
      "('parkeri-specific', 'O', 2)\n",
      "('IgG', 'O', 2)\n",
      "('antibody', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('serum', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('mice', 'O', -100)\n",
      "('i.v.', 'O', -100)\n",
      "('infected', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('R.', 'B', 2)\n",
      "('parkeri', 'I', 2)\n",
      "('3A2', 'I', 2)\n",
      "('was', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('different', 'O', -100)\n",
      "('compared', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('mice', 'O', 2)\n",
      "('infected', 'O', 2)\n",
      "('with', 'O', 0)\n",
      "('R.', 'O', -100)\n",
      "('parkeri', 'O', 1)\n",
      "('WT', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Figure', 'O', -100)\n",
      "('4J', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('titer', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('approximately', 'O', 2)\n",
      "('1:1000', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('differential', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('MDA-MB-435S', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('clones', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('decreased', 'O', -100)\n",
      "('expression', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('integrin', 'O', 2)\n",
      "('αV', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('key', 'O', 2)\n",
      "('components', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('integrin', 'O', 2)\n",
      "('αVβ5', 'O', 2)\n",
      "('adhesion', 'O', -100)\n",
      "('complexes', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('talins', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('α-actinins', 'O', -100)\n",
      "('1', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('filamins', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('plectin', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('vinculin', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('Enterococcus', 'B', 2)\n",
      "('faecium', 'I', 0)\n",
      "('strain', 'I', 1)\n",
      "('19434', 'I', 1)\n",
      "('(', 'O', 1)\n",
      "('dosed', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('5.7', 'O', 2)\n",
      "('×', 'O', 2)\n",
      "('10^6', 'O', 2)\n",
      "('CFU', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Staphylococcus', 'B', 2)\n",
      "('aureus', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JE2', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('dosed', 'O', 2)\n",
      "('at', 'O', 0)\n",
      "('1.3', 'O', 1)\n",
      "('×', 'O', 1)\n",
      "('10^7', 'O', 1)\n",
      "('CFU', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Klebsiella', 'B', 2)\n",
      "('pneumoniae', 'I', 2)\n",
      "('strain', 'I', -100)\n",
      "('KP4640', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('dosed', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('1.1', 'O', -100)\n",
      "('×', 'O', 2)\n",
      "('10^7', 'O', 2)\n",
      "('CFU', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "(',', 'O', 1)\n",
      "('Acinetobacter', 'B', 1)\n",
      "('baumannii', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('AB5075', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('dosed', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('1.5', 'O', 2)\n",
      "('×', 'O', -100)\n",
      "('10^7', 'O', -100)\n",
      "('CFU', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Pseudomonas', 'B', -100)\n",
      "('aeruginosa', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('PAO1', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('dosed', 'O', 1)\n",
      "('at', 'O', 1)\n",
      "('1.9', 'O', 1)\n",
      "('×', 'O', -100)\n",
      "('10^5', 'O', -100)\n",
      "('CFU', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Enterobacter', 'B', 2)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('23391', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('dosed', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('1.2', 'O', 2)\n",
      "('×', 'O', 2)\n",
      "('10^7', 'O', 2)\n",
      "('CFU', 'O', 0)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', 1)\n",
      "('an', 'O', -100)\n",
      "('extra-intestinal', 'O', 2)\n",
      "('pathogenic', 'O', 2)\n",
      "('Escherichia', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('ExPEC', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('15051', 'B', -100)\n",
      "('(', 'O', -100)\n",
      "('dosed', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('2.3', 'O', 2)\n",
      "('×', 'O', 0)\n",
      "('10^7', 'O', 1)\n",
      "('CFU', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('The', 'O', -100)\n",
      "('P-S', 'O', 2)\n",
      "('bonded', 'O', 2)\n",
      "('model', 'O', -100)\n",
      "('OP', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('echothiophate', 'O', 2)\n",
      "('iodide', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('enzymes', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('G117H', 'O', 2)\n",
      "('mutant', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('BChE', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('hydrolyze', 'O', 2)\n",
      "('OPs', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('new', 'O', 2)\n",
      "('multiple', 'O', 2)\n",
      "('mutant', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('Brevundimonas', 'O', 2)\n",
      "('diminuta', 'O', -100)\n",
      "('PTE', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('GG1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('specially', 'O', 2)\n",
      "('designed', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('rapid', 'O', 2)\n",
      "('hydrolytic', 'O', 2)\n",
      "('detoxification', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('V-agents', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('30', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('ZH07', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('peanut', 'O', 2)\n",
      "('rhizosphere', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('utilizing', 'O', 2)\n",
      "('multiple', 'O', 2)\n",
      "('autotoxins', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('sources', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Dietzia', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('skin', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('confluent', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('reticulated', 'O', 2)\n",
      "('papillomatosis', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('40', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('aimed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('compare', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vivo', 'O', 2)\n",
      "('virulence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('MBO', 'B', 2)\n",
      "('Ravenel', 'I', 0)\n",
      "('to', 'O', -100)\n",
      "('MBO', 'B', 1)\n",
      "('95-1315', 'I', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('original', 'O', 0)\n",
      "('bovine', 'O', -100)\n",
      "('host', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('discover', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('pathways', 'O', 2)\n",
      "('impacted', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('mutations', 'O', 2)\n",
      "('across', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('MBO', 'B', 2)\n",
      "('Ravenel', 'I', 2)\n",
      "('genome', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('It', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('Thauera', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('able', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('degrade', 'O', 2)\n",
      "('toluene', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('aerobic', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('anaerobic', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('oxygen', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('toluene-dioxygenase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('tod', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('enzyme', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('initial', 'O', 2)\n",
      "('activation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('aromatic', 'O', 2)\n",
      "('ring', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Shinoda', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('K.', 'B', 2)\n",
      "('koreensis', 'I', 0)\n",
      "('DSM', 'I', -100)\n",
      "('16069', 'I', 1)\n",
      "('was', 'O', -100)\n",
      "('found', 'O', 1)\n",
      "('to', 'O', 1)\n",
      "('exhibit', 'O', -100)\n",
      "('putative', 'O', 2)\n",
      "('nitrate', 'O', 2)\n",
      "('reduction', 'O', 2)\n",
      "('capability', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('line', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vivo', 'O', 2)\n",
      "('observations', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('demonstrated', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('purified', 'O', 2)\n",
      "('LPS', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('pseudomallei', 'I', 0)\n",
      "('1026b', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('also', 'O', -100)\n",
      "('used', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('vivo', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('activates', 'O', 2)\n",
      "('HEK293', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('via', 'O', 2)\n",
      "('TLR2', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('via', 'O', 2)\n",
      "('TLR4', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Mycobacterium', 'B', -100)\n",
      "('bovis', 'I', 0)\n",
      "('BCG', 'I', 1)\n",
      "('infection', 'O', 1)\n",
      "('induces', 'O', 2)\n",
      "('macrophage', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('GM-CSF', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('may', 'O', -100)\n",
      "('contribute', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('mycobacterial', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('favoring', 'O', 2)\n",
      "('macrophage', 'O', 2)\n",
      "('M1', 'O', 2)\n",
      "('polarization', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Benmerzoug', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2018', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('expression', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('putative', 'O', 2)\n",
      "('laccase', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('FVEG_04196', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('FVEG_13405', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('found', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('significantly', 'O', -100)\n",
      "('upregulated', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('F.', 'O', 2)\n",
      "('verticillioides', 'O', 2)\n",
      "('ΔFvAtg4', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ΔFvAtg8', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('ΔFvAtg4-8', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('5', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('c', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('identified', 'O', 2)\n",
      "('1,838', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('high', 'O', -100)\n",
      "('confidence', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('matched', 'O', 2)\n",
      "('cyanobacterium', 'B', 2)\n",
      "('ESFC-1', 'I', 2)\n",
      "('in', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('least', 'O', 0)\n",
      "('2', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('5', 'O', -100)\n",
      "('replicate', 'O', -100)\n",
      "('samples', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('After', 'O', -100)\n",
      "('20', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('fixed', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('paraformaldehyde', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('washed', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('PBS', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('permeabilised', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('0.2', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('Triton', 'O', -100)\n",
      "('X-100', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('PBS', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('30', 'O', 2)\n",
      "('min', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Earlier', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('A11', 'B', 2)\n",
      "('has', 'O', 0)\n",
      "('been', 'O', -100)\n",
      "('reported', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('plant-growth', 'O', 2)\n",
      "('promoting', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('PGP', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('multi-metal-resistant', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MMR', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('capable', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('producing', 'O', -100)\n",
      "('rhamnolipids', 'O', -100)\n",
      "('while', 'O', 2)\n",
      "('growing', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('glycerol', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('minimal', 'O', 2)\n",
      "('salt', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('MSM', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('16', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Among', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('63', 'O', 2)\n",
      "('protective', 'O', 2)\n",
      "('nrMAGs', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('dominant', 'O', -100)\n",
      "('species', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('Blautia_A', 'O', 2)\n",
      "('obeum', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Bariatricus', 'O', -100)\n",
      "('comes', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('9', 'O', -100)\n",
      "('nrMAGs', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Faecalibacterium', 'O', -100)\n",
      "('prausnitzii_D', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "('nrMAGs', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Blautia_A', 'O', 2)\n",
      "('wexlerae', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('6', 'O', -100)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Faecalibacterium', 'O', 2)\n",
      "('sp900539945', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('4', 'O', 2)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Dorea', 'O', -100)\n",
      "('longicatena_B', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('3', 'O', 2)\n",
      "('nrMAGs', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Blautia_A', 'O', -100)\n",
      "('sp003480185', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Blautia_A', 'O', -100)\n",
      "('sp003471165', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('3', 'O', -100)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Dorea', 'O', 2)\n",
      "('formicigenerans', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('2', 'O', 2)\n",
      "('nrMAGs', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Fusicatenibacter', 'O', -100)\n",
      "('saccharivorans', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('2', 'O', -100)\n",
      "('nrMAGs', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('GCA-900066135', 'O', 2)\n",
      "('sp900543575', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('2', 'O', -100)\n",
      "('nrMAGs', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('isolate', 'O', 2)\n",
      "('A17', 'B', 2)\n",
      "('had', 'O', 0)\n",
      "('been', 'O', -100)\n",
      "('recovered', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "('year', 'O', 2)\n",
      "('old', 'O', 2)\n",
      "('male', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Ethiopia', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('PBek', 'B', 2)\n",
      "('had', 'O', 2)\n",
      "('been', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('German', 'O', 2)\n",
      "('National', 'O', 2)\n",
      "('Reference', 'O', 2)\n",
      "('Centre', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('Borrelia', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('traveler', 'O', 2)\n",
      "('returning', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Ethiopia', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('suspected', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('belong', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('recurrentis', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('A.', 'B', 2)\n",
      "('acetoxydans', 'I', 0)\n",
      "(',', 'I', -100)\n",
      "('strain', 'I', 1)\n",
      "('INE', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('=DSM', 'B', -100)\n",
      "('29876^T', 'I', 1)\n",
      "('=', 'O', 1)\n",
      "('JCM', 'B', 1)\n",
      "('30553^T', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('Tinto', 'O', -100)\n",
      "('River', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('Spain', 'O', -100)\n",
      "('.', 'O', 1)\n",
      "('Bacillus', 'B', -100)\n",
      "('aryabhattai', 'I', 0)\n",
      "('AB211', 'I', 1)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('Gram-positive', 'O', -100)\n",
      "('rod', 'O', -100)\n",
      "('shaped', 'O', -100)\n",
      "('spore', 'O', 1)\n",
      "('forming', 'O', -100)\n",
      "('firmicute', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Assuming', 'O', -100)\n",
      "('damage', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('damage', 'O', 2)\n",
      "('indexes', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('created', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('quantify', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('MFL', 'O', 2)\n",
      "('signals', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('extracting', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('magnetic', 'O', 2)\n",
      "('characteristics', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('MFL', 'O', 2)\n",
      "('signals', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('little', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('known', 'O', 2)\n",
      "('about', 'O', 2)\n",
      "('factors', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('underlying', 'O', 2)\n",
      "('children’s', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('parents’', 'O', -100)\n",
      "('self-efficacy', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('WTS', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('PET', 'O', 2)\n",
      "('hydrolase', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('purified', 'O', 2)\n",
      "('SM14est', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('assessed', 'O', -100)\n",
      "('using', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('suspension-based', 'O', 2)\n",
      "('assay', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('subsequent', 'O', 2)\n",
      "('analysis', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('reaction', 'O', 2)\n",
      "('products', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('UV-spectrophotometry', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('RP-HPLC', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('L.', 'B', -100)\n",
      "('reuteri', 'I', 0)\n",
      "('E81', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('LR5A', 'B', -100)\n",
      "('were', 'O', -100)\n",
      "('both', 'O', 1)\n",
      "('found', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('contain', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('putative', 'O', 2)\n",
      "('enterolysin-A', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('rhamnosus', 'I', 2)\n",
      "('LR2', 'I', 2)\n",
      "('had', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('carnocin_CP52', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('L.', 'B', 2)\n",
      "('plantarum', 'I', 0)\n",
      "('PFC-311', 'I', -100)\n",
      "('had', 'O', 1)\n",
      "('plantaricin-A', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('-E', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('-F', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('-J', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('-K', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('-N', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('To', 'O', -100)\n",
      "('confirm', 'O', 2)\n",
      "('accurate', 'O', 2)\n",
      "('phylogenetic', 'O', 2)\n",
      "('placement', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('14', 'O', 2)\n",
      "('hiHh', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('‘H.', 'O', -100)\n",
      "('intermedius’', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Haemophilus', 'O', 2)\n",
      "('spp.', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('average', 'O', -100)\n",
      "('nucleotide', 'O', 2)\n",
      "('identity', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ANI', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('H.', 'B', 2)\n",
      "('haemolyticus', 'I', 2)\n",
      "('NCTC', 'I', 2)\n",
      "('10839', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('non-typeable', 'O', 2)\n",
      "('H.', 'B', 2)\n",
      "('influenzae', 'I', 2)\n",
      "('86-028', 'I', 2)\n",
      "('NP', 'I', 0)\n",
      "('and', 'O', -100)\n",
      "('H.', 'B', 1)\n",
      "('parainfluenzae', 'I', -100)\n",
      "('T3T1', 'I', 1)\n",
      "('genomes', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('calculated', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('fastANI', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('51', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('with', 'O', 0)\n",
      "('default', 'O', -100)\n",
      "('parameters', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('Parasitological', 'O', -100)\n",
      "('screening', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('also', 'O', 2)\n",
      "('extended', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('residents', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('houses', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('radius', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('1-km', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('site', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('While', 'O', -100)\n",
      "('CDT2', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('moderately', 'O', -100)\n",
      "('expressed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('HEK293', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Du145', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('H1229', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('INTS7', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('strongly', 'O', 2)\n",
      "('detected', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('cell', 'O', -100)\n",
      "('lines', 'O', 2)\n",
      "('examined', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('MCF7', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('6A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('biofilm', 'O', 2)\n",
      "('exopolysaccharides', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('significantly', 'O', -100)\n",
      "('decreased', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('KT1115', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('PAO1Δ', 'B', 2)\n",
      "('wspF', 'I', 2)\n",
      "('after', 'O', 0)\n",
      "('PA4781', 'O', -100)\n",
      "('overexpressed', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Figures', 'O', 0)\n",
      "('4', 'O', -100)\n",
      "('B-4D', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Out', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('101', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('twenty', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('belonging', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('independent', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('71-1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('45-1', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('45-3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('44-1', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('4-1', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('shown', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('Southern', 'O', -100)\n",
      "('blot', 'O', -100)\n",
      "('analysis', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('contain', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('1.5', 'O', -100)\n",
      "('kb', 'O', 2)\n",
      "('hybridizing', 'O', 2)\n",
      "('band', 'O', 2)\n",
      "('corresponding', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('intact', 'O', 2)\n",
      "('bar', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Data', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('survival', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "('days', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 0)\n",
      "('O157', 'I', -100)\n",
      "(':', 'I', 1)\n",
      "('H7', 'I', 1)\n",
      "('populations', 'O', 1)\n",
      "('decreased', 'O', 1)\n",
      "('by', 'O', 2)\n",
      "('ca', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('P', 'B', -100)\n",
      "('.', 'I', 0)\n",
      "('agglomerans', 'I', 1)\n",
      "('KM1', 'I', 1)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('short-term', 'O', -100)\n",
      "('fermented', 'O', 2)\n",
      "('homemade', 'O', 2)\n",
      "('kimchi', 'O', 2)\n",
      "('pH', 'O', 2)\n",
      "('4.5', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Typical', 'O', -100)\n",
      "('resistance', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('related', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('specific', 'O', 2)\n",
      "('immune', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('toward', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('external', 'O', 2)\n",
      "('adverse', 'O', 2)\n",
      "('stimuli', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('MD15G1391000', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('show', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('similar', 'O', -100)\n",
      "('CN', 'O', -100)\n",
      "('trend', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('5C', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('toxin', 'O', 2)\n",
      "('genotyping', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('JXJA17', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('Cp', 'B', -100)\n",
      "('FORC_003', 'I', -100)\n",
      "('were', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('A', 'O', 0)\n",
      "(',', 'O', 1)\n",
      "('notwithstanding', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('fact', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('Cp', 'B', 2)\n",
      "('FORC_003', 'I', 2)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('aquarium', 'O', -100)\n",
      "('water', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('South', 'O', 2)\n",
      "('Korea', 'O', 0)\n",
      "('.', 'O', 1)\n",
      "('Moreover', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Q8', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('shown', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('degrade', 'O', 2)\n",
      "('CMC', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('xylan', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "('hours', 'O', 2)\n",
      "('incubation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('thus', 'O', 2)\n",
      "('demonstrating', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('more', 'O', 2)\n",
      "('polymer-degrading', 'O', 2)\n",
      "('activities', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('data', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('shown', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Moreover', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('heterologous', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('AtFAD3', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('AtFAD8', 'O', -100)\n",
      "('genes', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('tobbacco', 'O', 2)\n",
      "('showed', 'O', -100)\n",
      "('enhanced', 'O', -100)\n",
      "('osmotic', 'O', -100)\n",
      "('tolerance', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('Gram-positive', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('used', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('Bacillus', 'O', 2)\n",
      "('subtilus', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('NCIM', 'B', 2)\n",
      "('2718', 'I', 2)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Staphylococcus', 'O', 0)\n",
      "('aureus', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('ATCC', 'B', -100)\n",
      "('25923', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Here', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('report', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('complete', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Mucilaginibacter', 'B', 2)\n",
      "('strain', 'I', 0)\n",
      "('21P', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('an', 'O', 1)\n",
      "('estuarine', 'O', 1)\n",
      "('soil', 'O', -100)\n",
      "('contaminated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('iron', 'O', 2)\n",
      "('mine', 'O', 2)\n",
      "('tailings', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('Samarco', 'O', 2)\n",
      "('disaster', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('occurred', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('2015', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Minas', 'O', 2)\n",
      "('Gerais', 'O', -100)\n",
      "('State', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Brazil', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Finally', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('clear', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('clinical', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('can', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('advantageous', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('studying', 'O', 2)\n",
      "('specific', 'O', 2)\n",
      "('functions', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('mucoidy', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('this', 'O', -100)\n",
      "('again', 'O', -100)\n",
      "('does', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('seem', 'O', 2)\n",
      "('critical', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('many', 'O', 2)\n",
      "('aspects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('aeruginosa', 'O', 2)\n",
      "('physiology', 'O', 2)\n",
      "('since', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('LESB58-SED21', 'B', 2)\n",
      "('strain', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('SCFM2', 'O', 2)\n",
      "('did', 'O', 0)\n",
      "('only', 'O', -100)\n",
      "('somewhat', 'O', -100)\n",
      "('better', 'O', -100)\n",
      "('than', 'O', -100)\n",
      "('PAO1', 'B', -100)\n",
      "('in', 'O', 2)\n",
      "('SCFM2', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('missing', 'O', -100)\n",
      "('579', 'O', 2)\n",
      "('versus', 'O', 2)\n",
      "('681', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('shared', 'O', 2)\n",
      "('by', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('two', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('P', 'O', 2)\n",
      "('>', 'O', 2)\n",
      "('0.05', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('t', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('therefore', 'O', 2)\n",
      "('set', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('identify', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('malate', 'O', 2)\n",
      "('transporter', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('malate-producing', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('E.', 'O', -100)\n",
      "('coli', 'O', -100)\n",
      "('Δmdh', 'O', 2)\n",
      "('Δmqo', 'O', 2)\n",
      "('ΔackA-pta', 'O', -100)\n",
      "('ΔmaeA', 'O', 2)\n",
      "('ΔmaeB', 'O', 2)\n",
      "('pACT3w-ppc_K620S', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Sy280', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('deleting', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('permease', 'O', -100)\n",
      "('DctA', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('required', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('uptake', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('dicarboxylic', 'O', 2)\n",
      "('acids', 'O', -100)\n",
      "('aerobic', 'O', -100)\n",
      "('conditions', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('43', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('44', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('permeases', 'O', -100)\n",
      "('DcuA', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('DcuB', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('are', 'O', -100)\n",
      "('responsible', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('uptake', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('dicarboxylic', 'O', 2)\n",
      "('acids', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('anaerobic', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('45', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Salmonella', 'B', -100)\n",
      "('enterica', 'I', 0)\n",
      "('serotype', 'I', 1)\n",
      "('Braenderup', 'I', 1)\n",
      "('H9812', 'I', 1)\n",
      "('was', 'O', -100)\n",
      "('digested', 'O', -100)\n",
      "('with', 'O', 1)\n",
      "('XbaI', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('molecular', 'O', 2)\n",
      "('weight', 'O', 2)\n",
      "('standard', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('All', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BG11', 'O', 2)\n",
      "('media', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('25', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('room', 'O', 2)\n",
      "('temperature', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('Calothrix', 'B', 2)\n",
      "('parasitica', 'I', 2)\n",
      "('NIES-267', 'I', 0)\n",
      "('(', 'O', -100)\n",
      "('f', 'O', -100)\n",
      "('/', 'O', 1)\n",
      "('2', 'O', -100)\n",
      "('marine', 'O', 1)\n",
      "('medium', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('26', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Hydrococcus', 'B', 2)\n",
      "('rivularis', 'I', 2)\n",
      "('NIES-593', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('CB', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('27', 'O', 0)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Acaryochloris', 'O', 1)\n",
      "('marina', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('K', 'O', 1)\n",
      "('+', 'O', -100)\n",
      "('ESM', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('28', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('blood', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SK678', 'B', 2)\n",
      "('produced', 'O', 0)\n",
      "('reduced', 'O', -100)\n",
      "('levels', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('H', 'O', 2)\n",
      "('_2', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "('_2', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('consistently', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('reduced', 'O', -100)\n",
      "('amounts', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('eDNA', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('defects', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('initiation', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('WGS', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('Kp36', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Kp36-117R', 'O', 2)\n",
      "('unraveled', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('evolutionary', 'O', 2)\n",
      "('mechanism', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('phage', 'O', -100)\n",
      "('117', 'O', -100)\n",
      "('resistance', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('K.', 'B', 2)\n",
      "('pneumoniae', 'I', 2)\n",
      "('Kp36', 'I', 2)\n",
      "('under', 'O', 2)\n",
      "('pressure', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('persistent', 'O', 2)\n",
      "('phage', 'O', 2)\n",
      "('predation', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('Marseille-P9525', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('CSUR', 'O', -100)\n",
      "('P9525', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('after', 'O', -100)\n",
      "('3', 'O', 2)\n",
      "('days', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('pH', 'O', 2)\n",
      "('7.5.', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('anaerobic', 'O', 2)\n",
      "('atmosphere', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('fresh', 'O', 2)\n",
      "('stool', 'O', 2)\n",
      "('sample', 'O', 2)\n",
      "('collected', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('28-year-old', 'O', -100)\n",
      "('Burkinabe', 'O', -100)\n",
      "('woman', 'O', -100)\n",
      "('suspected', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('having', 'O', 2)\n",
      "('tuberculosis', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Thus', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('suspected', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('pathogenicity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('V.', 'B', 2)\n",
      "('fujianensis', 'I', 0)\n",
      "('FJ201301', 'I', -100)\n",
      "('^T', 'O', 1)\n",
      "('was', 'O', -100)\n",
      "('due', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('multi-interaction', 'O', -100)\n",
      "('with', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('variety', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('virulence', 'O', -100)\n",
      "('factors', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('For', 'O', -100)\n",
      "('ciprofloxacin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('MICs', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sensitive', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('E7946', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('resistant', 'O', -100)\n",
      "('clinical', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('EN160', 'B', 2)\n",
      "('remained', 'O', 2)\n",
      "('unchanged', 'O', 2)\n",
      "('when', 'O', 0)\n",
      "('catalase', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('added', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('media', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('aerobic', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Briefly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('could', 'O', 2)\n",
      "('induce', 'O', 2)\n",
      "('tissue', 'O', 2)\n",
      "('colonization', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cause', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('mortality', 'O', 2)\n",
      "('within', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('onset', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('immersion', 'O', 2)\n",
      "('challenge', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('assigned', 'O', 2)\n",
      "('highly', 'O', 2)\n",
      "('virulent', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('HV', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('case', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('carp', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('trout', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('04017018', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('JIP', 'B', -100)\n",
      "('P11', 'I', -100)\n",
      "('/', 'I', 2)\n",
      "('91', 'I', 0)\n",
      "('[', 'O', -100)\n",
      "('10', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('11', 'O', 1)\n",
      "(']', 'O', 1)\n",
      "('.', 'O', 2)\n",
      "('Previously', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('removing', 'O', 2)\n",
      "('anthrose', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('spore', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('reduced', 'O', 2)\n",
      "('germination', 'O', 2)\n",
      "('efficiency', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('increased', 'O', 2)\n",
      "('sporulation', 'O', 2)\n",
      "('rates', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('heterologous', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('anthracis', 'I', 2)\n",
      "('Sterne', 'I', 0)\n",
      "('model', 'O', -100)\n",
      "('[', 'O', 1)\n",
      "('18', 'O', -100)\n",
      "(']', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Kassa', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('GLA', 'O', -100)\n",
      "('1103', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Aurora', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Messay', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('NC', 'O', 2)\n",
      "('58', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Babylon', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('selected', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('future', 'O', 2)\n",
      "('breeding', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('genetics', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('physiological', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('Al', 'O', 2)\n",
      "('tolerance', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('faba', 'O', 2)\n",
      "('bean', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Although', 'O', -100)\n",
      "('35', 'O', 2)\n",
      "('candidate', 'O', 2)\n",
      "('T3Es', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('present', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('tested', 'O', 2)\n",
      "('R.', 'O', 2)\n",
      "('pseudosolanacearum', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('no', 'O', 2)\n",
      "('specific', 'O', -100)\n",
      "('effector', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('found', 'O', -100)\n",
      "('among', 'O', -100)\n",
      "('phylotype', 'O', 2)\n",
      "('III', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('LMG', 'B', 2)\n",
      "('9673', 'I', 2)\n",
      "('^T', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CMR15', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('RUN2279', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('RUN2474', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('UW386', 'B', 1)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('cell-wall', 'O', 2)\n",
      "('peptidoglycan', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('WF-MT5-A', 'B', -100)\n",
      "('^T', 'O', 2)\n",
      "('contains', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('amino', 'O', -100)\n",
      "('acids', 'O', -100)\n",
      "('alanine', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Ala', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('glutamic', 'O', -100)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Glu', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('lysine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Lys', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('aspartic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Asp', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('serine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ser', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('threonine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Thr', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('molar', 'O', 2)\n",
      "('ratio', 'O', 2)\n",
      "('1.1', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ala', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':0.8', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Asp', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':1.0', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Glu', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(':0.7', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Lys', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':0.1', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Thr', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(':0.1', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ser', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('suggesting', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('cell-wall', 'O', 2)\n",
      "('peptidoglycan', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('A4α', 'O', 2)\n",
      "('l', 'O', -100)\n",
      "('-Lys', 'O', -100)\n",
      "('-d', 'O', -100)\n",
      "('-Asp', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('38', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Lactobacillus', 'B', -100)\n",
      "('paracasei', 'I', 0)\n",
      "('FAM18149', 'I', 1)\n",
      "('that', 'O', -100)\n",
      "('had', 'O', -100)\n",
      "('been', 'O', 1)\n",
      "('grown', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('25', 'O', -100)\n",
      "('h', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('30°C', 'O', 2)\n",
      "('either', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('cysteine-deficient', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('methionine-deficient', 'O', 2)\n",
      "('CDM', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('collected', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('centrifugation', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('After', 'O', -100)\n",
      "('incubating', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('25', 'O', 2)\n",
      "('µM', 'O', 2)\n",
      "('PABA', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('NO', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('min', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('co-staining', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('eNOS', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('S-glutathionylated', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('antibodies', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('HL60', 'O', 2)\n",
      "('cells', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('detected', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('panel', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Here', 'O', -100)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 1)\n",
      "('alone', 'O', 1)\n",
      "('increased', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('chlorophyll', 'O', 0)\n",
      "('b', 'O', 1)\n",
      "('content', 'O', -100)\n",
      "('by', 'O', 1)\n",
      "('9.8', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('10', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('11', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('2.28', 'O', 2)\n",
      "('mg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('FW', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('treated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('consortium', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('comparison', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('untreated', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('MAR', 'O', 2)\n",
      "('index', 'O', 2)\n",
      "('results', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('more', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('half', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('K.', 'O', 2)\n",
      "('pneumoniae', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('had', 'O', -100)\n",
      "('resistance', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('least', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('antibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MAR', 'O', 2)\n",
      "('index', 'O', 2)\n",
      "('≥', 'O', 2)\n",
      "('0.26', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('one', 'O', -100)\n",
      "('isolate', 'O', -100)\n",
      "('showed', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('17', 'O', 2)\n",
      "('antibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MAR', 'O', 2)\n",
      "('index', 'O', 2)\n",
      "('=', 'O', 2)\n",
      "('0.89', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Intracellular', 'O', -100)\n",
      "('antibodies', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('IL-4', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('11B11', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('IL-17A', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('TC11-18H10.1', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('IFNγ', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('XMG1.2', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Biolegend', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('IL-5', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('TRFK5', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('IL-13', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ebio13A', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Foxp3', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('FJK-16', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('eBioscience', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Two', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('VH3073', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('VH4549', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 0)\n",
      "('well', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('C.', 'B', 2)\n",
      "('urealyticum', 'I', 2)\n",
      "('DSM', 'I', 2)\n",
      "('7109', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', 0)\n",
      "('resistant', 'O', -100)\n",
      "('to', 'O', 1)\n",
      "('rifampicin', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('MIC', 'O', -100)\n",
      "('>', 'O', 1)\n",
      "('32', 'O', 1)\n",
      "('mg', 'O', -100)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('novel', 'O', 2)\n",
      "('triscatecholate', 'O', 2)\n",
      "('siderophore', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('turnerbactin', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('supernatant', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('iron-limited', 'O', -100)\n",
      "('T.', 'B', -100)\n",
      "('turnerae', 'I', -100)\n",
      "('T7901', 'I', 2)\n",
      "('cultures', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('first', 'O', 2)\n",
      "('time', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('determinants', 'O', 2)\n",
      "('commonly', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('clinically', 'O', 2)\n",
      "('important', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('detected', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Nocardia', 'O', 2)\n",
      "('sp.', 'O', 2)\n",
      "('sul1', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('sul2', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('erm', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('tet', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('prevalent', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('SXT-resistant', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('Bt', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('LLP29', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('leaves', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Mognolia', 'O', 2)\n",
      "('denudate', 'O', 2)\n",
      "('Desr', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('robust', 'O', 2)\n",
      "('induction', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('mRFP1', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('while', 'O', -100)\n",
      "('encased', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('pellicle', 'O', 2)\n",
      "('indicates', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('despite', 'O', 2)\n",
      "('entrapped', 'O', -100)\n",
      "('within', 'O', 2)\n",
      "('nanocellulose', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('K.', 'B', 2)\n",
      "('intermedius', 'I', 2)\n",
      "('ENS15', 'I', 2)\n",
      "('cells', 'O', -100)\n",
      "('can', 'O', 2)\n",
      "('effectively', 'O', 2)\n",
      "('receive', 'O', 0)\n",
      "('signals', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('their', 'O', -100)\n",
      "('environment', 'O', 1)\n",
      "('enabling', 'O', -100)\n",
      "('its', 'O', 2)\n",
      "('applicability', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('applications', 'O', 2)\n",
      "('requiring', 'O', 2)\n",
      "('long-term', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('survival', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('tolerance', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('toxic', 'O', 2)\n",
      "('compounds', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('5', 'O', -100)\n",
      "('d', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('e', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Vibrio', 'B', -100)\n",
      "('qinghaiensis', 'I', 0)\n",
      "('sp.-Q67', 'I', 1)\n",
      "('(', 'O', -100)\n",
      "('Vqin-Q67', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('is', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('luminous', 'O', -100)\n",
      "('bacterium', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('first', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Qinghai', 'O', -100)\n",
      "('Lake', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Qinghai', 'O', 2)\n",
      "('Province', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('aspect', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('proteolysis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('confirmed', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('extracellular', 'O', 2)\n",
      "('protease', 'O', 2)\n",
      "('activities', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('Kangiella', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('K.', 'B', -100)\n",
      "('profundi', 'I', 2)\n",
      "('FT102', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('K.', 'B', -100)\n",
      "('koreensis', 'I', 1)\n",
      "('DSM', 'I', -100)\n",
      "('16069', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('K.', 'B', -100)\n",
      "('aquimarina', 'I', -100)\n",
      "('DSM', 'I', 2)\n",
      "('16071', 'I', 0)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('K.', 'B', -100)\n",
      "('geojedonensis', 'I', 1)\n",
      "('KCTC', 'I', 1)\n",
      "('23420', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('For', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('reason', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('S9', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('85', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('each', 'O', 2)\n",
      "('representing', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('subcluster', 'O', 2)\n",
      "('containing', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('M.', 'O', -100)\n",
      "('ruminalis', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('HPA92^T', 'B', 2)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('subcluster', 'O', 2)\n",
      "('including', 'O', 0)\n",
      "('no', 'O', -100)\n",
      "('reference', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('characterized', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('16S', 'O', 2)\n",
      "('rRNA', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('sequencing', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Strain', 'O', -100)\n",
      "('CPCC', 'B', 2)\n",
      "('101082', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('could', 'O', 1)\n",
      "('hydrolyze', 'O', -100)\n",
      "('urea', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', -100)\n",
      "('CPCC', 'B', 2)\n",
      "('101083', 'I', 2)\n",
      "('^T', 'O', -100)\n",
      "('could', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('RHLS22-1', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('=CGMCC', 'B', -100)\n",
      "('1.10101', 'I', -100)\n",
      "('^T', 'O', -100)\n",
      "('=', 'O', 2)\n",
      "('NBRC', 'B', -100)\n",
      "('114045', 'I', 2)\n",
      "('^T', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('melt', 'O', -100)\n",
      "('water', 'O', 2)\n",
      "('sample', 'O', -100)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('supraglacial', 'O', -100)\n",
      "('zone', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('Hailuogou', 'O', -100)\n",
      "('glacier', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('Sichuan', 'O', 2)\n",
      "('province', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Several', 'O', -100)\n",
      "('conditions', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('induce', 'O', 2)\n",
      "('hyphal', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('C.', 'O', 2)\n",
      "('albicans', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('incubation', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Lee’s', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('pH', 'O', 2)\n",
      "('3.5', 'O', -100)\n",
      "('or', 'O', -100)\n",
      "('pH', 'O', -100)\n",
      "('6.5', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('well', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('containing', 'O', 2)\n",
      "('serum', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('isoamyl', 'O', -100)\n",
      "('alcohol', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('Bleocin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('tested', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('none', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('these', 'O', 2)\n",
      "('triggered', 'O', 2)\n",
      "('filamentous', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('South', 'O', 2)\n",
      "('Asian', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('clade', 'O', 2)\n",
      "('I', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('auris', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('UACa11', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Yellow', 'O', -100)\n",
      "('zone', 'O', 2)\n",
      "('surrounding', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('colony', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('cellulose', 'O', 2)\n",
      "('agars', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('observed', 'O', -100)\n",
      "('upon', 'O', 2)\n",
      "('inoculation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Bacillus', 'B', 2)\n",
      "('sp.', 'I', 2)\n",
      "('strains', 'I', 0)\n",
      "('H1a', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('H1m', 'B', 1)\n",
      "(',', 'O', 1)\n",
      "('Sphingobacterium', 'B', -100)\n",
      "('sp.', 'I', 2)\n",
      "('strain', 'I', 0)\n",
      "('H1ai', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('Pseudomonas', 'B', 0)\n",
      "('sp.', 'I', -100)\n",
      "('strains', 'I', 1)\n",
      "('H1h', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('PH1b', 'B', 1)\n",
      "(',', 'O', -100)\n",
      "('M.', 'B', 2)\n",
      "('odoratimimus', 'I', 0)\n",
      "('strain', 'I', 1)\n",
      "('H1bi', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('S.', 'B', 1)\n",
      "('marcescens', 'I', -100)\n",
      "('strain', 'I', 2)\n",
      "('PH1a', 'I', 0)\n",
      "('and', 'O', -100)\n",
      "('M.', 'B', 2)\n",
      "('paraoxydans', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('DH1b', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('indicating', 'O', -100)\n",
      "('cellulolytic', 'O', -100)\n",
      "('activity', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('Vibrio', 'O', -100)\n",
      "('fujianensis', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('recently', 'O', -100)\n",
      "('identified', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('novel', 'O', 2)\n",
      "('Vibrio', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('FJ201301', 'B', 2)\n",
      "('^T', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('aquaculture', 'O', 0)\n",
      "('water', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Fujian', 'O', -100)\n",
      "('Province', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('2013', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('11', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('include', 'O', 2)\n",
      "('integral', 'O', 2)\n",
      "('membrane', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('delivered', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('merozoite', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('micronemes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AMA1', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('EBA175', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('GPI-anchored', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('merozoite', 'O', 2)\n",
      "('membrane', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MSP1', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('MSP4', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('MSP5', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('extrinsically', 'O', 2)\n",
      "('associated', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('merozoite', 'O', -100)\n",
      "('surface', 'O', 2)\n",
      "('during', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('maturation', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('PV', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MSP3', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('MSP6', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('soluble', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('secreted', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('parasite-host', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('interface', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RAP1', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('RAP2', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('RAP3', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Conclusions', 'O', -100)\n",
      "(':', 'O', 2)\n",
      "('The', 'O', 2)\n",
      "('fluorinated', 'O', 2)\n",
      "('compound', 'O', 2)\n",
      "('G29', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('moderate', 'O', -100)\n",
      "('activity', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('HAdV-5', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('several', 'O', -100)\n",
      "('mechanisms', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('These', 'O', -100)\n",
      "('results', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('consistent', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Kirby-Bauer', 'O', 2)\n",
      "('disk', 'O', 2)\n",
      "('diffusion', 'O', -100)\n",
      "('test', 'O', -100)\n",
      "('showing', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('KM1', 'B', 2)\n",
      "('strain', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('penicillin', 'O', 0)\n",
      "('G', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('vancomycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('bacitracin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('fosfomycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('rifampicin', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', -100)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('T81', 'B', -100)\n",
      "(',', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('heterotrophic', 'O', 2)\n",
      "('cellulolytic', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('sporulating', 'O', 2)\n",
      "('thermophile', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('we', 'O', -100)\n",
      "('previously', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('geothermal', 'O', 2)\n",
      "('soils', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Tikitere', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('New', 'O', 2)\n",
      "('Zealand', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('11', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('40', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('blood', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('BALB', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('c', 'O', 2)\n",
      "('mice', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('B31-5A4', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('B408', 'B', -100)\n",
      "('yielded', 'O', -100)\n",
      "('significantly', 'O', -100)\n",
      "('greater', 'O', -100)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('bacterial', 'O', -100)\n",
      "('burdens', 'O', -100)\n",
      "('than', 'O', 2)\n",
      "('B379', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('negative-control', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('B31A', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('high-passage', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('can', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('efficiently', 'O', 2)\n",
      "('survive', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('vertebrate', 'O', -100)\n",
      "('bloodstream', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('31', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('3A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Comment', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('prayer', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('healing', 'O', -100)\n",
      "(':', 'O', 2)\n",
      "('A', 'O', 2)\n",
      "('medical', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('scientific', 'O', 2)\n",
      "('perspective', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('randomized', 'O', 2)\n",
      "('controlled', 'O', 2)\n",
      "('trials', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('distincta', 'O', 2)\n",
      "('share', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('phenotypic', 'O', -100)\n",
      "('traits', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('closest', 'O', 2)\n",
      "('relative', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Pseudoalteromonas', 'B', 2)\n",
      "('arctica', 'I', 2)\n",
      "('A', 'I', 2)\n",
      "('37-1-2', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('including', 'O', -100)\n",
      "('carotenoid-like', 'O', 1)\n",
      "('pigmentation', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Table', 'O', 1)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('16S', 'O', 2)\n",
      "('rRNA', 'O', 2)\n",
      "('sequence', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('100', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('identity', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('distincta', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('S1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Inoculated', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('included', 'O', 2)\n",
      "('Aeromonas', 'B', 2)\n",
      "('veronii', 'I', 0)\n",
      "('strain', 'I', 1)\n",
      "('ZOR0001', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('Enterobacter', 'B', 1)\n",
      "('cloacae', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('ZOR0014', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Staphylococcus', 'B', 0)\n",
      "('sp.', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('ZWU0021', 'I', 1)\n",
      "('[', 'O', 1)\n",
      "('47', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Here', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('report', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('complete', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Pseudomonas', 'O', 2)\n",
      "('lundensis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('M101', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('M105', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('which', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('pasteurized', 'O', 2)\n",
      "('milk', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('immune', 'O', 2)\n",
      "('competent', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('fungal', 'O', 2)\n",
      "('bronchopneumonia', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('myeloid-derived', 'O', -100)\n",
      "('HIF-1α', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('critical', 'O', 2)\n",
      "('for', 'O', -100)\n",
      "('neutrophil', 'O', -100)\n",
      "('recruitment', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('ultimately', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('survival', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('mice', 'O', 2)\n",
      "('challenged', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('fumigatus', 'O', 2)\n",
      "('CEA10', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('analysing', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('antimicrobial', 'O', 2)\n",
      "('properties', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('titanium', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('silver', 'O', 2)\n",
      "('coatings', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('exposed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('gram', 'O', 2)\n",
      "('positive', 'O', 2)\n",
      "('Staphylococcus', 'B', 2)\n",
      "('epidermidis', 'I', 0)\n",
      "(',', 'I', 1)\n",
      "('strain', 'I', 1)\n",
      "('RP62A', 'I', 1)\n",
      "('(', 'O', 1)\n",
      "('ATCC', 'B', -100)\n",
      "('35984', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "('or', 'O', 0)\n",
      "('gram', 'O', 1)\n",
      "('negative', 'O', -100)\n",
      "('Klebsiella', 'B', -100)\n",
      "('pneumoniae', 'I', 2)\n",
      "(',', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('3091', 'I', 2)\n",
      "('[', 'O', 0)\n",
      "('20', 'O', 1)\n",
      "(']', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('RAW', 'O', -100)\n",
      "('264.7', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('line', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('breast', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MCF7', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('MDA-MB-231', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('acquired', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('American', 'O', -100)\n",
      "('Culture', 'O', -100)\n",
      "('Collection', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('ATCC', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Manassas', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('VA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('USA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('degradation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('phytosiderophores', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('SBW25', 'B', -100)\n",
      "('implies', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('strain', 'O', 0)\n",
      "('possesses', 'O', -100)\n",
      "('transporters', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('enzymes', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('taking', 'O', 2)\n",
      "('up', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('metabolizing', 'O', 2)\n",
      "('mugineic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('deoxymugineic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('integration', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('proteomic', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('physiological', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('transcriptional', 'O', 2)\n",
      "('profiling', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('OsiSh-2', 'B', 2)\n",
      "('induced', 'O', 0)\n",
      "('rice', 'O', -100)\n",
      "('defense', 'O', -100)\n",
      "('priming', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('controlled', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('energy-consuming', 'O', 2)\n",
      "('defense-related', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('thus', 'O', 2)\n",
      "('increasing', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('defense', 'O', 2)\n",
      "('capability', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('minimized', 'O', 2)\n",
      "('costs', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('immunity', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Citrobacter', 'B', -100)\n",
      "('werkmanii', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 1)\n",
      "('increased', 'O', 1)\n",
      "('CAT', 'O', -100)\n",
      "('activity', 'O', 2)\n",
      "('by', 'O', 0)\n",
      "('30', 'O', 1)\n",
      "('%', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('26', 'O', 1)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('when', 'O', 2)\n",
      "('applied', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('consortium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('33', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('6.7', 'O', 2)\n",
      "('EU', 'O', 2)\n",
      "('mg^-1', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('uninoculated', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('Bifidobacterium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Blautia', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Streptococcus', 'O', -100)\n",
      "('showed', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('abundances', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CPP', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Genus', 'O', 2)\n",
      "('level', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1F-H', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CFU-S12', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('for', 'O', -100)\n",
      "('X-RARα-positive', 'O', -100)\n",
      "('HPCs', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('reduced', 'O', 2)\n",
      "('upon', 'O', 2)\n",
      "('exposure', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('SSi', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', -100)\n",
      "('6B-C', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Three', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('acidophile', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('Leptospirillum', 'O', 2)\n",
      "('ferriphilum', 'O', 2)\n",
      "('^T', 'O', 2)\n",
      "('DSM', 'B', -100)\n",
      "('14647', 'I', -100)\n",
      "('[', 'O', -100)\n",
      "('20', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Sulfobacillus', 'O', -100)\n",
      "('thermosulfidooxidans', 'O', -100)\n",
      "('^T', 'O', 2)\n",
      "('DSM', 'B', -100)\n",
      "('9293', 'I', 0)\n",
      "('[', 'O', 1)\n",
      "('21', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Acidithiobacillus', 'O', 2)\n",
      "('caldus', 'O', 2)\n",
      "('^T', 'O', -100)\n",
      "('DSM', 'B', -100)\n",
      "('8584', 'I', -100)\n",
      "('[', 'O', -100)\n",
      "('22', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('occur', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('juniper', 'O', 2)\n",
      "('essential', 'O', 2)\n",
      "('oils', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Identification', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AB211', 'B', 2)\n",
      "('on', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('basis', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('source', 'O', 2)\n",
      "('utilization', 'O', 2)\n",
      "('pattern', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('GEN', 'O', 2)\n",
      "('III', 'O', 2)\n",
      "('MicroPlate', 'O', 2)\n",
      "('^TM', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('AB211', 'B', -100)\n",
      "('could', 'O', 2)\n",
      "('utilize', 'O', 2)\n",
      "('several', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('carbohydrates', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('amino', 'O', -100)\n",
      "('acids', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('grow', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('high', 'O', 2)\n",
      "('salt', 'O', 2)\n",
      "('concentration', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('several', 'O', 2)\n",
      "('antibiotics', 'O', 2)\n",
      "('like-', 'O', 2)\n",
      "('rifamycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('troleandomycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('lincomycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('nalidixic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('aztreonam', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Supplementary', 'O', 2)\n",
      "('Figure', 'O', -100)\n",
      "('S6B', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('has', 'O', 2)\n",
      "('implications', 'O', 2)\n",
      "('specifically', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('23S', 'O', 2)\n",
      "('rRNA', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('pertactin', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('mutation', 'O', 2)\n",
      "('studies', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('IMP', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('almost', 'O', 2)\n",
      "('exclusively', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('aeruginosa', 'O', 2)\n",
      "('isolates', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('showing', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('importance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('dissemination', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('carbapenemase', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('associated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('clusters', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('single', 'O', 2)\n",
      "('hospital', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('each', 'O', 2)\n",
      "('variant', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('associated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('specific', 'O', 2)\n",
      "('clone', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('IMP-8', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ST348', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('IMP-23', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('ST175', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('IMP-16', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('ST253', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('While', 'O', -100)\n",
      "('T8', 'B', 2)\n",
      "('predominantly', 'O', 0)\n",
      "('produces', 'O', -100)\n",
      "('lantibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('mutacin', 'O', 2)\n",
      "('II', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('UA140', 'B', 2)\n",
      "('produces', 'O', 2)\n",
      "('both', 'O', -100)\n",
      "('lantibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('mutacin', 'O', 2)\n",
      "('I', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('nonlantibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('mutacin', 'O', -100)\n",
      "('IV', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "('-', 'O', -100)\n",
      "('9', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('SA29-B', 'B', -100)\n",
      "('T', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('YU21-B', 'B', -100)\n",
      "('grew', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('15-40°C', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('pH', 'O', -100)\n",
      "('6.0-12', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('contained', 'O', -100)\n",
      "('C', 'O', 2)\n",
      "('16:0', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('dominant', 'O', -100)\n",
      "('cell', 'O', -100)\n",
      "('fatty', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', -100)\n",
      "('KA5-B', 'B', -100)\n",
      "('T', 'O', -100)\n",
      "('showed', 'O', -100)\n",
      "('no', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('40°C', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('pH', 'O', -100)\n",
      "('12', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('contained', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('moderate', 'O', 2)\n",
      "('amount', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "('16:0', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('inoculation', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('nicotianae', 'I', 0)\n",
      "('AFI2', 'I', -100)\n",
      "('slightly', 'O', 1)\n",
      "('reduced', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('Ni', 'O', -100)\n",
      "('content', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('shoots', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Leningrad', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('unreimbursed', 'O', 2)\n",
      "('time', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('resources', 'O', -100)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('treating', 'O', 2)\n",
      "('patients', 'O', 2)\n",
      "('through', 'O', 2)\n",
      "('expanded', 'O', 2)\n",
      "('access', 'O', 2)\n",
      "('can', 'O', 2)\n",
      "('potentially', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('offset', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('opportunities', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('possibility', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('earlier', 'O', 2)\n",
      "('conversations', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('formulary', 'O', 2)\n",
      "('committees', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('ability', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('capture', 'O', 2)\n",
      "('valuable', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('particularly', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('subtypes', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('included', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('clinical', 'O', 2)\n",
      "('trials', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('potentially', 'O', 2)\n",
      "('leading', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('broader', 'O', 2)\n",
      "('indications', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Host', 'O', -100)\n",
      "('cell', 'O', 2)\n",
      "('contact-dependent', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('Yersinia', 'O', -100)\n",
      "('virulence', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('assessed', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Y.', 'B', 2)\n",
      "('pseudotuberculosis', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('YPIII', 'I', 0)\n",
      "('and', 'O', -100)\n",
      "('isogenic', 'O', 1)\n",
      "('mutant', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('YP53', 'B', 1)\n",
      "('ΔcsrA', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('without', 'O', 2)\n",
      "('contact', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('HEp-2', 'B', 0)\n",
      "('cells', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('ATCC', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('USA', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('situ', 'O', 2)\n",
      "('monitoring', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('gfp', 'O', -100)\n",
      "('reporter', 'O', 2)\n",
      "('fusions', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('fluorescence', 'O', 2)\n",
      "('microscopy', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('b', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('situ', 'O', 2)\n",
      "('quantification', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('light', 'O', 2)\n",
      "('emission', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('luxCDABE', 'O', 2)\n",
      "('reporter', 'O', 2)\n",
      "('fusions', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('c', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('detection', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('quantification', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('LcrF', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('YadA', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('YopE', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Among', 'O', -100)\n",
      "('rare', 'O', 2)\n",
      "('taxa', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('sex-dependent', 'O', 2)\n",
      "('associations', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('NAFLD', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('order', 'O', 2)\n",
      "('RF39', 'B', 2)\n",
      "('has', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('positively', 'O', 0)\n",
      "('associated', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('healthy', 'O', 2)\n",
      "('diets', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('16,52', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('negatively', 'O', 2)\n",
      "('associated', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('BMI', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('blood', 'O', 2)\n",
      "('triglycerides', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('frailty', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('older', 'O', 2)\n",
      "('adults', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('52,53,54,55,56', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('suggesting', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('health', 'O', 2)\n",
      "('benefits', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('This', 'O', -100)\n",
      "('study', 'O', 2)\n",
      "('contributes', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('taxonomical', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('clinical', 'O', -100)\n",
      "('knowledge', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('genus', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('describing', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('novel', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('Marseille-Q3630', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('as', 'O', -100)\n",
      "('part', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('microbiological', 'O', -100)\n",
      "('workup', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('IHU-Méditerranée', 'O', 2)\n",
      "('Infection', 'O', 2)\n",
      "('Institute', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Marseille', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('France', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Xie', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "('extracted', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('polysaccharide', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('molecular', 'O', 2)\n",
      "('weight', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('11.68', 'O', 2)\n",
      "('kDa', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('brown', 'O', 2)\n",
      "('alga', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('inhibited', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('proliferation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('MKN45', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('blocking', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('cycle', 'O', -100)\n",
      "('during', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('G2', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('M', 'O', 2)\n",
      "('phase', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('94,95', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Likewise', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('half-life', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('endogenous', 'O', -100)\n",
      "('β-TrCP1', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('shortened', 'O', -100)\n",
      "('upon', 'O', -100)\n",
      "('SAG', 'O', -100)\n",
      "('transfection', 'O', -100)\n",
      "('into', 'O', 2)\n",
      "('A427', 'B', 2)\n",
      "('cells', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('expressed', 'O', 2)\n",
      "('high', 'O', 2)\n",
      "('β-TrCP1', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('low', 'O', 0)\n",
      "('SAG', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('2G', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Further', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('xopAQ', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('predicted', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('X.', 'O', 2)\n",
      "('hortorum', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('radicchio', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('X.', 'O', 2)\n",
      "('hortorum', 'O', -100)\n",
      "('pv.', 'O', -100)\n",
      "('pelargonii', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('X.', 'O', -100)\n",
      "('hortorum', 'O', -100)\n",
      "('pv.', 'O', -100)\n",
      "('cynarae', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('X.', 'O', 2)\n",
      "('campestris', 'O', -100)\n",
      "('pv.', 'O', -100)\n",
      "('coriandri', 'O', 2)\n",
      "('but', 'O', -100)\n",
      "('not', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('X.', 'O', -100)\n",
      "('hortorum', 'O', -100)\n",
      "('pv.', 'O', 2)\n",
      "('hederae', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('X.', 'O', 2)\n",
      "('hortorum', 'O', -100)\n",
      "('pv.', 'O', -100)\n",
      "('taraxaci', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('X.', 'B', -100)\n",
      "('hortorum', 'I', -100)\n",
      "('pv.', 'I', 2)\n",
      "('carotae', 'I', 2)\n",
      "('M081', 'I', 2)\n",
      "('.', 'O', -100)\n",
      "('One', 'O', -100)\n",
      "('auxotroph', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('isolate', 'O', -100)\n",
      "('No.1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('recently', 'O', 2)\n",
      "('enrolled', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('PCC.No.499', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('00', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('A5', 'O', -100)\n",
      "('CM', 'O', 2)\n",
      "('inhibited', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('migration', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('tumor', 'O', 2)\n",
      "('spheroids', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('green', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('toward', 'O', 2)\n",
      "('A5', 'O', 2)\n",
      "('spheroids', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('red', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('inhibition', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('suppressed', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('FFCM', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('Pa', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('ATCC', 'O', -100)\n",
      "('33913', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('NP_669902.1', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Yersinia', 'B', 2)\n",
      "('pestis', 'I', -100)\n",
      "('KIM', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('NP_828458.1', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Streptomyces', 'B', 2)\n",
      "('avermitilis', 'I', 0)\n",
      "('MA-4680', 'I', 1)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('CAD61030.1', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Arthrobacter', 'O', 2)\n",
      "('ilicis', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('ZP_00227861.1', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Kineococcus', 'B', 2)\n",
      "('radiotolerans', 'I', 0)\n",
      "('SRS30216', 'I', 1)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('while', 'O', 1)\n",
      "('many', 'O', -100)\n",
      "('other', 'O', -100)\n",
      "('proteins', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('NP_405150.1', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Yersinia', 'B', -100)\n",
      "('pestis', 'I', -100)\n",
      "('CO92', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('NP_437232.1', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Sinorhizobium', 'O', -100)\n",
      "('meliloti', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('NP_533476.1', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Agrobacterium', 'O', 2)\n",
      "('tumefaciens', 'O', 2)\n",
      "('str', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Individual', 'O', -100)\n",
      "('level', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('VA', 'O', 2)\n",
      "('interviews', 'O', 2)\n",
      "('comprising', 'O', 2)\n",
      "('details', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('any', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('illness', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('occurred', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('duration', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('final', 'O', 2)\n",
      "('illness', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('acute', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('<', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('weeks', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('chronic', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('≥2', 'O', 2)\n",
      "('weeks', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('cumulative', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('deaths', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('household', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('death', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('deaths', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('more', 'O', 2)\n",
      "('deaths', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('household', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('time', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1992-1996', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1997-2001', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2002-2006', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('2007-2011', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('place', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('death', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('home', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('hospital', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('health', 'O', 2)\n",
      "('facility', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('recorded', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Based', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('surveillance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('agricultural', 'O', 2)\n",
      "('sites', 'O', 2)\n",
      "('across', 'O', 2)\n",
      "('Canada', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('prevalence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('non-O157', 'B', 2)\n",
      "(':', 'I', 0)\n",
      "('H7', 'I', -100)\n",
      "('strains', 'O', -100)\n",
      "('now', 'O', 1)\n",
      "('exceeds', 'O', 1)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('O157', 'B', 2)\n",
      "(':', 'I', 2)\n",
      "('H7', 'I', 2)\n",
      "('by', 'O', 0)\n",
      "('about', 'O', 1)\n",
      "('7', 'O', 1)\n",
      "('%', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Public', 'O', 2)\n",
      "('Health', 'O', 2)\n",
      "('Agency', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Canada', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2015', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Klebsiella', 'B', -100)\n",
      "('pneumoniae', 'I', 0)\n",
      "('HSL4', 'I', 1)\n",
      "('is', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('1,3-propanediol-producing', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('mangrove', 'O', -100)\n",
      "('sediment', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('our', 'O', 2)\n",
      "('preliminary', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('LED', 'O', 2)\n",
      "('light', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('adults', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('T.', 'O', 2)\n",
      "('kuznetsovi', 'O', 2)\n",
      "('showed', 'O', -100)\n",
      "('clear', 'O', 2)\n",
      "('negative', 'O', -100)\n",
      "('phototaxis', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('UV', 'O', 2)\n",
      "('light', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('probably', 'O', 2)\n",
      "('also', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('green', 'O', -100)\n",
      "('light', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Initially', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('approximately', 'O', 2)\n",
      "('500-700', 'O', 2)\n",
      "('bp', 'O', 2)\n",
      "('regions', 'O', -100)\n",
      "('upstream', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('antA', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('benA', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('amplified', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cloned', 'O', 2)\n",
      "('upstream', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('lacZ', 'O', 2)\n",
      "('reporter', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('pDOW1017', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Liquid', 'O', -100)\n",
      "('LB', 'O', 2)\n",
      "('media', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('until', 'O', 2)\n",
      "('OD', 'O', 2)\n",
      "('0.6', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('E', 'B', -100)\n",
      "('.', 'I', -100)\n",
      "('coli', 'I', 2)\n",
      "('OP50', 'I', 0)\n",
      "('or', 'O', 1)\n",
      "('HT115', 'B', 1)\n",
      "('were', 'O', 1)\n",
      "('diluted', 'O', -100)\n",
      "('1:125', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('LB', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('incubated', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37', 'O', 2)\n",
      "('°C', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('shaking', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('N', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('identified', 'O', 2)\n",
      "('earlier', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('year', 'O', 2)\n",
      "('another', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('family', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('HCoV-NL63', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('6', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('variant', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('belongs', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('together', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('HCoV-229E', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('PEDV', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Four', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('low', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('intermediate', 'O', 2)\n",
      "('virulence', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('FR', 'B', 2)\n",
      "('27.1', 'I', 0)\n",
      "(',', 'O', 1)\n",
      "('CH', 'B', -100)\n",
      "('45.1', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('NO', 'B', 0)\n",
      "('764-5B', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('NO', 'B', -100)\n",
      "('765-6B', 'I', 2)\n",
      "(')', 'O', 0)\n",
      "('reached', 'O', 1)\n",
      "('final', 'O', -100)\n",
      "('densities', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('high', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('those', 'O', 0)\n",
      "('measured', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('high', 'O', -100)\n",
      "('virulence', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CH', 'B', 2)\n",
      "('49.3', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('CH', 'B', 2)\n",
      "('21.1', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('CH', 'B', 2)\n",
      "('60', 'I', 2)\n",
      "(';', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Table', 'O', 0)\n",
      "('.', 'O', 1)\n",
      "('Treating', 'O', -100)\n",
      "('HEKn', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('CUR', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('≥5', 'O', 2)\n",
      "('µM', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('CUR', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('induced', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('loss', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('metabolic', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('visible', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('MTT', 'O', 2)\n",
      "('assay', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('p', 'O', 2)\n",
      "('<', 'O', 2)\n",
      "('0.05', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('are', 'O', 2)\n",
      "('interested', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('NAP', 'O', 2)\n",
      "('degradation', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('Rhodococcus', 'B', 2)\n",
      "('opacus', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('M213', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 1)\n",
      "('was', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('fuel', 'O', 2)\n",
      "('oil-contaminated', 'O', 2)\n",
      "('soil', 'O', 2)\n",
      "('sample', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Toxins', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('antitoxins', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('BrnT', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('A', 'O', 2)\n",
      "('protein', 'O', -100)\n",
      "('family', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('C-107', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('NS58', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('AFC132', 'B', -100)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('acr3', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('not', 'O', 2)\n",
      "('frequent', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('genus', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('S17', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('SH31', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('both', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('Altiplano', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('pancreas-specific', 'O', 2)\n",
      "('PTF1A', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('master', 'O', 2)\n",
      "('regulator', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('pancreas', 'O', 2)\n",
      "('development', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('acinar', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('fate', 'O', 2)\n",
      "('specification', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('highly', 'O', 2)\n",
      "('expressed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('normal', 'O', 2)\n",
      "('AR42J-B13', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('reduced', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('almost', 'O', -100)\n",
      "('undetectable', 'O', -100)\n",
      "('levels', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('experimental', 'O', 2)\n",
      "('groups', 'O', 2)\n",
      "('exposed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('differentiation', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Thirty', 'O', -100)\n",
      "('days', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('PcPCL1606', 'B', 2)\n",
      "('preventive', 'O', 0)\n",
      "('inoculations', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('T1', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('20', 'O', 2)\n",
      "('days', 'O', -100)\n",
      "('before', 'O', 2)\n",
      "('inoculation', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('R.', 'O', 2)\n",
      "('necatrix', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('counts', 'O', 2)\n",
      "('maintained', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('values', 'O', -100)\n",
      "('among', 'O', -100)\n",
      "('soil', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('rhizosphere', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('treatments', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('values', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('total', 'O', 2)\n",
      "('heterotrophic', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('approximately', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('^6', 'O', 2)\n",
      "('cfu', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('slightly', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('values', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('Pseudomonas-like', 'O', -100)\n",
      "('counts', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('mainly', 'O', 2)\n",
      "('below', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('^5', 'O', 2)\n",
      "('cfu', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('review', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('describe', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('advances', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('made', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('recent', 'O', 2)\n",
      "('years', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('understanding', 'O', 2)\n",
      "('how', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('pLS20', 'O', 2)\n",
      "('conjugation', 'O', 2)\n",
      "('genes', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('regulated', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('identify', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('similarities', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('differences', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('conjugative', 'O', 2)\n",
      "('systems', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Gram+', 'O', -100)\n",
      "('origin', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('From', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('metabolic', 'O', 2)\n",
      "('flux', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('glycolytic', 'O', 2)\n",
      "('intermediates', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('G6P', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('F6P', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('F1,6P', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('G3P', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('PGP', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('3PG', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2PG', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('PEP', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Pyr', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('increased', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('BL21', 'B', -100)\n",
      "('/', 'I', 2)\n",
      "('pOri1', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('BL21', 'B', 2)\n",
      "('/', 'I', 2)\n",
      "('pOri2', 'I', 2)\n",
      "('relative', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('BL21', 'B', 2)\n",
      "('during', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('early', 'O', 1)\n",
      "('stationary', 'O', 1)\n",
      "('phase', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 0)\n",
      "('3A', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "('.', 'O', 1)\n",
      "('In', 'O', -100)\n",
      "('NOA16', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('molecularly', 'O', -100)\n",
      "('matched', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('cohort', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('PsPD', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('mainly', 'O', -100)\n",
      "('indicate', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('intratumoral', 'O', 2)\n",
      "('inflammatory', 'O', 2)\n",
      "('reaction', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('defined', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('size', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('tumour', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('T2-FLAIR', 'O', 2)\n",
      "('MRI', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('novel', 'O', -100)\n",
      "('appearance', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('enlargement', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('contrast-enhancing', 'O', 2)\n",
      "('lesions', 'O', 2)\n",
      "('followed', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('stabilization', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('regression', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('follow-up', 'O', -100)\n",
      "('MRI', 'O', -100)\n",
      "('up', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('months', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('initiation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('SOC', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('immunotherapy', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('14', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('research', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('coding', 'O', 2)\n",
      "('DNA', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('shared', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('related', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Sphaeropleales', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', -100)\n",
      "('revealed', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('ITBB-AG6', 'B', 2)\n",
      "('is', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('closely', 'O', 2)\n",
      "('related', 'O', 0)\n",
      "('to', 'O', -100)\n",
      "('M.', 'O', -100)\n",
      "('neglectum', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('families', 'O', 2)\n",
      "('Selenastraceae', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Scenedesmaceae', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('monophyletic', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('Neochloridaceae', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('polyphyletic', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('geometry', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('part', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('flasks', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('contact', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('rotating', 'O', 2)\n",
      "('culture', 'O', 2)\n",
      "('liquid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('identical', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('assures', 'O', 2)\n",
      "('comparability', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cultures', 'O', -100)\n",
      "('between', 'O', 2)\n",
      "('standard', 'O', -100)\n",
      "('shake', 'O', 2)\n",
      "('flasks', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('RAMOS', 'O', 2)\n",
      "('shake', 'O', 2)\n",
      "('flasks', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('L-tyrosine', 'O', 2)\n",
      "('degradation', 'O', 2)\n",
      "('pathway', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('I', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('lead', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('melanin', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('distincta', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('eumelanin', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('pyomelanin', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('exception', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('non-pigmented', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('P.', 'B', -100)\n",
      "('distincta', 'I', -100)\n",
      "('16-SW-7', 'I', 2)\n",
      "('[', 'O', 2)\n",
      "('26', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('endowment', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('terms', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('copper', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('gold', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('factors', 'O', 2)\n",
      "('should', 'O', 2)\n",
      "('allow', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('metallidurans', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('BS1', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('just', 'O', 2)\n",
      "('like', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('CH34', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('to', 'O', -100)\n",
      "('survive', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('high', 'O', 2)\n",
      "('copper', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('gold', 'O', -100)\n",
      "('contaminated', 'O', 2)\n",
      "('environments', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Also', 'O', -100)\n",
      "('missing', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('above', 'O', 2)\n",
      "('mentioned', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('O3.65', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('located', 'O', -100)\n",
      "('next', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('subunit', 'O', 2)\n",
      "('pcaIJ', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('regulatory', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('coded', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('pcaR', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('2609025149', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('TRIHO_43610', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('needed', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('functionality', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('enzyme', 'O', -100)\n",
      "('3-oxiadipate', 'O', 2)\n",
      "('CoA', 'O', 2)\n",
      "('transferase', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('For', 'O', -100)\n",
      "('example', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('ORFs', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('FO-36b', 'O', 2)\n",
      "('locus', 'O', 2)\n",
      "('tag', 'O', -100)\n",
      "('RS87_09285', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('shared', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('B.', 'B', -100)\n",
      "('safensis', 'I', 2)\n",
      "('MROC1', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('feces', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Gallus', 'O', 0)\n",
      "('gallus', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('B.', 'B', -100)\n",
      "('safensis', 'I', 1)\n",
      "('RP10', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('soils', 'O', 2)\n",
      "('contaminated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('heavy', 'O', 2)\n",
      "('metals', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('Chile', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('quinone', 'O', 2)\n",
      "('system', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('P4023', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('composed', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('94.3', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('menaquinone', 'O', 2)\n",
      "('MK-6', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('5.4', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('MK-5', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('0.2', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('MK-7', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('P7388', 'B', -100)\n",
      "('^T', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('90.2', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('MK-6', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('9.6', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('MK-5', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('0.2', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('MK-7', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('unique', 'O', 2)\n",
      "('inversion', 'O', 2)\n",
      "('blocks', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('wMelPlus', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('some', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('them', 'O', 2)\n",
      "('known', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('members', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('already', 'O', 2)\n",
      "('known', 'O', 2)\n",
      "('co-expressed', 'O', 2)\n",
      "('classes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('might', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('induction', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('stress-resistant', 'O', 2)\n",
      "('phenotype', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('are', 'O', -100)\n",
      "('worth', 'O', 2)\n",
      "('investigating', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('future', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('particular', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('combination', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('5‐FU', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('autophagy', 'O', -100)\n",
      "('inhibitors', 'O', -100)\n",
      "('enhanced', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('anticancer', 'O', 2)\n",
      "('effects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('5‐FU', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('HCT116R^F10', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Strain', 'O', -100)\n",
      "('ERB122', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('generated', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('mating', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('ERB039', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('KN99a', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('dissection', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('resulting', 'O', 0)\n",
      "('spores', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('71', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('conducted', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('paralysis', 'O', 2)\n",
      "('assay', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('elegans', 'O', 2)\n",
      "('fed', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('vitamin', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('EGb', 'O', 2)\n",
      "('761', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('Ginkgo', 'O', 2)\n",
      "('biloba', 'O', -100)\n",
      "('leaf', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('While', 'O', -100)\n",
      "('FORC_019', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('FORC_051', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('FORC_052', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('FORC_056', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('FORC_074', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('FORC_078', 'B', -100)\n",
      "('formed', 'O', 2)\n",
      "('RDAR', 'O', 0)\n",
      "('colonies', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('FORC_075', 'B', -100)\n",
      "('formed', 'O', -100)\n",
      "('SAW', 'O', 2)\n",
      "('colonies', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('significantly', 'O', -100)\n",
      "('distinct', 'O', -100)\n",
      "('colony', 'O', 2)\n",
      "('morphology', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('those', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('other', 'O', -100)\n",
      "('FORC', 'O', 2)\n",
      "('strains', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', -100)\n",
      "('2b', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Bdellovibrio', 'B', -100)\n",
      "('bacteriovorus', 'I', 0)\n",
      "('HD100', 'I', -100)\n",
      "('are', 'O', -100)\n",
      "('terrestrial', 'O', -100)\n",
      "('bacteria', 'O', -100)\n",
      "('which', 'O', 1)\n",
      "('show', 'O', -100)\n",
      "('natural', 'O', -100)\n",
      "('control', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('Pseudomonas', 'O', 2)\n",
      "('tolaasii', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('spoilage', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('mushroom', 'O', 2)\n",
      "('crops', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('non-sterile', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('biotic', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('mushroom', 'O', 2)\n",
      "('pileus', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Another', 'O', -100)\n",
      "('well-studied', 'O', 2)\n",
      "('mucin', 'O', 2)\n",
      "('degrader', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Akkermansia', 'B', 2)\n",
      "('muciniphila', 'I', 2)\n",
      "('ATCC', 'I', -100)\n",
      "('BAA-835', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('belongs', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('Verrucomicrobium', 'O', -100)\n",
      "('phylum', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('is', 'O', -100)\n",
      "('also', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('prominent', 'O', 1)\n",
      "('member', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('human', 'O', -100)\n",
      "('microbiota', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Also', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('authors', 'O', 2)\n",
      "('do', 'O', 2)\n",
      "(\"n't\", 'O', 2)\n",
      "('explain', 'O', 2)\n",
      "('why', 'O', -100)\n",
      "('HT115', 'B', -100)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('particularly', 'O', 0)\n",
      "('high', 'O', -100)\n",
      "('producer', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('GABA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('approximatelly', 'O', 2)\n",
      "('8-fold', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('when', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('BW25113', 'B', -100)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('another', 'O', -100)\n",
      "('derivative', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('parental', 'O', 2)\n",
      "('MG1665', 'B', 2)\n",
      "('K-12', 'I', 0)\n",
      "('strain', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Based', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('BLAST', 'O', 2)\n",
      "('search', 'O', 2)\n",
      "('program', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('comparison', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('amino', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('purified', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('D2-3', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('known', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RNA', 'O', 2)\n",
      "('polymerase', 'O', 2)\n",
      "('subunit', 'O', 2)\n",
      "('sigma-24', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('presented', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('Figure', 'O', -100)\n",
      "('9', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('FORC_075', 'B', 2)\n",
      "('showed', 'O', 0)\n",
      "('impaired', 'O', -100)\n",
      "('biofilm-', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('RDAR', 'O', -100)\n",
      "('colony-forming', 'O', 2)\n",
      "('abilities', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('distinct', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('other', 'O', 2)\n",
      "('FORC', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('including', 'O', 2)\n",
      "('FORC_078', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Figure', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "('illustrates', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('schematic', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('cysteine', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('methionine', 'O', 2)\n",
      "('biosynthesis', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('paracasei', 'I', 2)\n",
      "('FAM18149', 'I', 0)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('impact', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('150', 'O', 2)\n",
      "('fungal', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('development', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('wild-type', 'O', 2)\n",
      "('Arabidopsis', 'O', 2)\n",
      "('thaliana', 'O', -100)\n",
      "('Columbia-0', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Col-0', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('tested', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('fungus-plant', 'O', 2)\n",
      "('interaction', 'O', -100)\n",
      "('assays', 'O', -100)\n",
      "('performed', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('follows', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('data', 'O', 2)\n",
      "('support', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('notion', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('growing', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('lab', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('planktonically', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('LB', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('viable', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('system', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('studying', 'O', 2)\n",
      "('many', 'O', 2)\n",
      "('aspects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('CF', 'O', 2)\n",
      "('lung', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('many', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('aeruginosa', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('do', 'O', 2)\n",
      "('not', 'O', -100)\n",
      "('vary', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('regardless', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genotype', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('environment', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('DobHLH13', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('DobHLH16', 'O', -100)\n",
      "('had', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('most', 'O', -100)\n",
      "('abundant', 'O', 2)\n",
      "('transcripts', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('green', 'O', -100)\n",
      "('root', 'O', -100)\n",
      "('tip', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('FPKM', 'O', 2)\n",
      "('values', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('122.75', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('140.77', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Here', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('report', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('crystal', 'O', 2)\n",
      "('structures', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Mycobacterium', 'O', 2)\n",
      "('smegmatis', 'O', 2)\n",
      "('FadD32', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('apo', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('ATP-bound', 'O', -100)\n",
      "('states', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('2.4', 'O', 2)\n",
      "('Å', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('2.25', 'O', -100)\n",
      "('Å', 'O', -100)\n",
      "('resolution', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Since', 'O', -100)\n",
      "('H_2O_2', 'O', 2)\n",
      "('treatment', 'O', 2)\n",
      "('only', 'O', -100)\n",
      "('slightly', 'O', -100)\n",
      "('increased', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Asp', 'O', 2)\n",
      "('+', 'O', 2)\n",
      "('Asn', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Y300A', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CtCBM22A_Y300A', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('S1', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2-oxo-histidine', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('probably', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('major', 'O', -100)\n",
      "('product', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('oxidation', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('There', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('17', 'O', 2)\n",
      "('ARGs', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('newly', 'O', 2)\n",
      "('assembled', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('derived', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('tilapia', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('n', 'O', 2)\n",
      "('=', 'O', -100)\n",
      "('5', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('NK01', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('NK02', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('NK07', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('UDRT09', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('CNRT12', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('McClain', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('colleagues', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('19', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('described', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('antigenic', 'O', 2)\n",
      "('regions', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('SmD3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('antigenic', 'O', -100)\n",
      "('region', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "('encompasses', 'O', 2)\n",
      "('amino', 'O', 2)\n",
      "('acids', 'O', 2)\n",
      "('104-126', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('HGT5', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('fig', 'O', -100)\n",
      "('4', 'O', -100)\n",
      "('E', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('unique', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('apis', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('A29', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('B8', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('C6', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('strains', 'O', 0)\n",
      "('that', 'O', -100)\n",
      "('had', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', 2)\n",
      "('honey', 'O', 2)\n",
      "('bee', 'O', 2)\n",
      "('larvae', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('At', 'O', -100)\n",
      "('day', 'O', 2)\n",
      "('7', 'O', 2)\n",
      "('PW', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('Prevotella', 'O', 2)\n",
      "('affiliated', 'O', 2)\n",
      "('amplicons', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('ASV28', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('ASV33', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ASV50', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('well', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('Mitsuokella', 'O', -100)\n",
      "('genus', 'O', 2)\n",
      "('ASV17', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('differentially', 'O', 2)\n",
      "('abundant', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('when', 'O', -100)\n",
      "('compared', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('those', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('group', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('XCC0613', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('XC_3621', 'O', -100)\n",
      "('ORF', 'O', -100)\n",
      "('has', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('G+C', 'O', 2)\n",
      "('content', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('53.5', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('substantially', 'O', 2)\n",
      "('lower', 'O', -100)\n",
      "('than', 'O', -100)\n",
      "('average', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('xanthomonad', 'O', -100)\n",
      "('genes', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('suggestive', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('acquisition', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('HGT', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('evidence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('strong', 'O', 2)\n",
      "('redox', 'O', 2)\n",
      "('homeostasis', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('consistent', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('relatively', 'O', 2)\n",
      "('low', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('positive', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('nuclear', 'O', 2)\n",
      "('8-hydroxy-2-deoxyguanosine', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('DNA', 'O', -100)\n",
      "('damage', 'O', -100)\n",
      "('foci', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('established', 'O', -100)\n",
      "('PEsen', 'O', -100)\n",
      "('cells2325', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('although', 'O', 2)\n",
      "('there', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('numerous', 'O', 2)\n",
      "('articles', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('antistaphylococcal', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('O.', 'O', -100)\n",
      "('vulgare', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('T.', 'O', 2)\n",
      "('vulgaris', 'O', 2)\n",
      "('EOs', 'O', 2)\n",
      "('tested', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('broth', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('agar', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('substantially', 'O', 2)\n",
      "('fewer', 'O', 2)\n",
      "('articles', 'O', 2)\n",
      "('dealing', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('antibacterial', 'O', 2)\n",
      "('effects', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('aureus', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('published', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('vapor', 'O', 2)\n",
      "('phase', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('36,37', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('instance', 'O', 2)\n",
      "('Fan', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Wang', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Song', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Ding', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Wu', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Wu', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Gao', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Borriss', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('10', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('that', 'O', -100)\n",
      "('B.', 'B', 2)\n",
      "('velezensis', 'I', 2)\n",
      "('strain', 'I', -100)\n",
      "('FZB42', 'I', -100)\n",
      "('induces', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('roots', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('corn', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('mouse-ear', 'O', -100)\n",
      "('cress', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', 1)\n",
      "('duckweed', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Comparative', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('analysis', 'O', 2)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('ECU32', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('in', 'O', 1)\n",
      "('scaffold21xsize74584', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('carried', 'O', 2)\n",
      "('ORFs', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('both', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('ferric', 'O', -100)\n",
      "('enterobactin', 'O', -100)\n",
      "('ABC', 'O', -100)\n",
      "('transporter', 'O', -100)\n",
      "('system', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('fepA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('fepB', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('fepC', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('fepD', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('fepG', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('fes', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('enterobactin', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('siderophore', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('biosynthesis', 'O', 2)\n",
      "('exporter', 'O', -100)\n",
      "('system', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('entB', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('entC', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('entD', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('entE', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('entF', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('entS', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('G.', 'B', 2)\n",
      "('alkanivorans', 'I', 0)\n",
      "('135', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('alk', 'O', -100)\n",
      "('B', 'O', -100)\n",
      "('gene', 'O', -100)\n",
      "('was', 'O', 1)\n",
      "('not', 'O', 2)\n",
      "('detected', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('purple', 'O', 2)\n",
      "('non-sulfur', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Rhodopseudomonas', 'B', -100)\n",
      "('palustris', 'I', 2)\n",
      "('nifA^∗', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('McKinlay', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Harwood', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2010', 'O', 1)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('grown', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('defined', 'O', -100)\n",
      "('mineral', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('containing', 'O', -100)\n",
      "('Na_2HPO_4', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('1.77', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('L', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('KH_2PO_4', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1.77', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('0.1', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('v', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('v', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('concentrated', 'O', 2)\n",
      "('base', 'O', -100)\n",
      "('solution', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('0.5', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('v', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('v', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('Wolfe’s', 'O', -100)\n",
      "('vitamins', 'O', -100)\n",
      "('solution', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('mM', 'O', -100)\n",
      "('sodium', 'O', -100)\n",
      "('acetate', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('source', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('same', 'O', 2)\n",
      "('betaproteobacterial', 'O', 2)\n",
      "('OTU', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('CSW1.1', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('OTU001', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('made', 'O', 2)\n",
      "('up', 'O', 2)\n",
      "('28.5', 'O', -100)\n",
      "('±', 'O', -100)\n",
      "('12.9', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('QV1.1', 'O', -100)\n",
      "('community', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('remaining', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('taxa', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('rare', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('accounting', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('24.3', 'O', 2)\n",
      "('±', 'O', 2)\n",
      "('1.6', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('community', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('L.', 'B', -100)\n",
      "('rhamnosus', 'I', 0)\n",
      "('CA15', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('DSM', 'B', -100)\n",
      "('33960', 'I', 1)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('tested', 'O', 0)\n",
      "('for', 'O', 1)\n",
      "('its', 'O', -100)\n",
      "('ability', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('displace', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('inhibit', 'O', 2)\n",
      "('adhesion', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Streptococcus', 'B', 2)\n",
      "('agalactiae', 'I', -100)\n",
      "('DSM', 'I', 2)\n",
      "('2134', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('Enterobacter', 'B', 2)\n",
      "('cloacae', 'I', 0)\n",
      "('DSM', 'I', 1)\n",
      "('30054', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('Staphylococcus', 'B', -100)\n",
      "('aureus', 'I', 1)\n",
      "('DSM', 'I', 1)\n",
      "('1104', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('Candida', 'B', 0)\n",
      "('glabrata', 'I', 1)\n",
      "('ATCC', 'I', -100)\n",
      "('90030', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('Pseudomonas', 'B', -100)\n",
      "('aeruginosa', 'I', 2)\n",
      "('DSM', 'I', 0)\n",
      "('1117', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('Enterococcus', 'B', 1)\n",
      "('faecalis', 'I', -100)\n",
      "('DSM', 'I', 2)\n",
      "('2570', 'I', 0)\n",
      "(',', 'O', 1)\n",
      "('Gardnerella', 'B', 1)\n",
      "('vaginalis', 'I', 1)\n",
      "('DSM', 'I', -100)\n",
      "('14018', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('Candida', 'B', 1)\n",
      "('parapsilosis', 'I', 1)\n",
      "('ATCC', 'I', 1)\n",
      "('90018', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('Candida', 'B', 1)\n",
      "('albicans', 'I', 1)\n",
      "('ATCC10231', 'I', 1)\n",
      "('.', 'O', -100)\n",
      "('Methylobacterium', 'O', -100)\n",
      "('dichloromethanicum', 'O', 2)\n",
      "('subsp.', 'O', -100)\n",
      "('dichloromethanicum', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('DM4', 'B', -100)\n",
      "('was', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('only', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('capable', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('utilizing', 'O', -100)\n",
      "('dichloromethane', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Based', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('physiology', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('Caldisericum', 'O', 2)\n",
      "('exile', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('thermophilic', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('anaerobic', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('thiosulfate-reducing', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('affiliated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('OP5', 'O', 2)\n",
      "('clones', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Mori', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2008', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2009', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('upon', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('OP5', 'O', 2)\n",
      "('occurrence', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('sediments', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('sulfur-rich', 'O', 2)\n",
      "('environments', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Hugenholtz', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1998', 'O', 2)\n",
      "(';', 'O', -100)\n",
      "('Teske', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2002', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('it', 'O', 2)\n",
      "('can', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('assumed', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('OP5', 'O', -100)\n",
      "('members', 'O', 2)\n",
      "('could', 'O', 2)\n",
      "('be', 'O', -100)\n",
      "('also', 'O', 2)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('sulfur', 'O', 2)\n",
      "('cycle', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Recombinant', 'O', -100)\n",
      "('his_6-tagged', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('pantotrophus', 'O', -100)\n",
      "('SoxYZ', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('produced', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('purified', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('described', 'O', -100)\n",
      "('previously', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('6', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('following', 'O', 2)\n",
      "('modifications.', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('TCEP', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('tris', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('2-carboxyethyl', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('phosphine', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('added', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('all', 'O', -100)\n",
      "('buffers', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('end', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('immobilized', 'O', -100)\n",
      "('metal', 'O', 2)\n",
      "('affinity', 'O', 2)\n",
      "('chromatography', 'O', -100)\n",
      "('step', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('two', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('aeruginosa', 'O', 2)\n",
      "('differ', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('phenazine', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('different', 'O', -100)\n",
      "('conditions', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('where', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('produces', 'O', 2)\n",
      "('4-fold', 'O', 2)\n",
      "('higher', 'O', 0)\n",
      "('pyocyanin', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('PYO', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('than', 'O', -100)\n",
      "('does', 'O', 2)\n",
      "('PA14', 'B', 2)\n",
      "('in', 'O', -100)\n",
      "('medium', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('mimics', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('cystic', 'O', 2)\n",
      "('fibrotic', 'O', 2)\n",
      "('lung', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('77', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('PA14', 'B', 2)\n",
      "('produces', 'O', 2)\n",
      "('10-fold', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('pyocyanin', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('in', 'O', 2)\n",
      "('Luria-Bertani', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('LB', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('medium', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('57', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('sequences', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('sequenced', 'O', 2)\n",
      "('wsp', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('for', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('Wolbachia', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Genbank', 'O', 2)\n",
      "('accession', 'O', 2)\n",
      "('numbers', 'O', 2)\n",
      "('AF020070', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('AF020059', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('AJ252062', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Strain', 'B', -100)\n",
      "('159469^T', 'I', 0)\n",
      "('clustered', 'O', 1)\n",
      "('close', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('L.', 'B', -100)\n",
      "('garvieae', 'I', -100)\n",
      "('JCM', 'I', 2)\n",
      "('10343^T', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('=ATCC', 'B', 0)\n",
      "('43921^T', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('maximum-likelihood', 'O', -100)\n",
      "('tree', 'O', 1)\n",
      "('reconstructed', 'O', -100)\n",
      "('on', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('basis', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('16S', 'O', 2)\n",
      "('rRNA', 'O', 0)\n",
      "('gene', 'O', -100)\n",
      "('sequences', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('RAxML', 'O', -100)\n",
      "('v.', 'O', -100)\n",
      "('8', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('general', 'O', 2)\n",
      "('time-reversible', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('GTRGAMMAI', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('substitution', 'O', 2)\n",
      "('model', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('1000', 'O', 2)\n",
      "('bootstrap', 'O', 2)\n",
      "('repetitions', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('DNA', 'O', 2)\n",
      "('repair', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Escherichia', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('AlkB', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('induced', 'O', -100)\n",
      "('by', 'O', 2)\n",
      "('responding', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('alkylating', 'O', 2)\n",
      "('agents', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('51', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Kocuria_palustris_1', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('first', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('cattails', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('river', 'O', -100)\n",
      "('tributary', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Kovács', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1999', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('subsequently', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('rice', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Kaga', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2009', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('soil', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Caliz', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2011', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('marine', 'O', -100)\n",
      "('sponges', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Martín', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2013', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Plants', 'O', -100)\n",
      "('applied', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', 1)\n",
      "('showed', 'O', -100)\n",
      "('an', 'O', -100)\n",
      "('increase', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('75', 'O', 1)\n",
      "('%', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('16.2', 'O', 2)\n",
      "('EU', 'O', 2)\n",
      "('mg^-1', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('Enterobacter', 'B', 2)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 2)\n",
      "('raised', 'O', 2)\n",
      "('SOD', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('51', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('consortium', 'O', 0)\n",
      "('enhanced', 'O', 1)\n",
      "('SOD', 'O', -100)\n",
      "('activity', 'O', 1)\n",
      "('by', 'O', 1)\n",
      "('42', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('RCC', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('EphA2', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('detectable', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('variable', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('EphA2', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('HEK-293', 'O', -100)\n",
      "('cell', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('very', 'O', 2)\n",
      "('low', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('title', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('abstract', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('each', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('2469', 'O', 2)\n",
      "('articles', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('exported', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('.csv', 'O', -100)\n",
      "('file', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('screened', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Life', 'O', -100)\n",
      "('styles', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Colletotrichum', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('implications', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('plant', 'O', 2)\n",
      "('biosecurity', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Although', 'O', -100)\n",
      "('producers', 'O', 2)\n",
      "('try', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('partially', 'O', 2)\n",
      "('compensate', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('lack', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('nutrients', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('specific', 'O', 2)\n",
      "('molasses', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('this', 'O', -100)\n",
      "('does', 'O', -100)\n",
      "('not', 'O', 2)\n",
      "('seem', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('fully', 'O', 2)\n",
      "('compensate', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('differences', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('nutrient', 'O', 2)\n",
      "('portfolio', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('batches', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('yeast', 'O', -100)\n",
      "('extract', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('28', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('29', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('M.', 'O', -100)\n",
      "('faveolata', 'O', 2)\n",
      "('larvae', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('inoculated', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('SSPe', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('Mf10.14b', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('704', 'B', 0)\n",
      "('(', 'O', -100)\n",
      "('Clade', 'O', 2)\n",
      "('B', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('gorgonian', 'O', -100)\n",
      "('Pseudopterogorgia', 'O', 2)\n",
      "('elisabethae', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('M.', 'O', -100)\n",
      "('faveolata', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('gorgonian', 'O', 2)\n",
      "('Plexaura', 'O', 2)\n",
      "('kuna', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BS1', 'B', 2)\n",
      "('another', 'O', 0)\n",
      "('homologous', 'O', -100)\n",
      "('czc', 'O', 2)\n",
      "('determinant', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('identified', 'O', -100)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('chromid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('czc', 'O', 2)\n",
      "('determinant', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('pMOL30', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('CH34', 'B', -100)\n",
      "('.', 'O', 2)\n",
      "('Hence', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('investigated', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('mode', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('action', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('polyphenol-sensitized', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CC', 'O', -100)\n",
      "('treated', 'O', -100)\n",
      "('MCF-7', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('MDA', 'O', 2)\n",
      "('MB-231', 'O', 2)\n",
      "('cells', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('cell', 'O', 2)\n",
      "('cycle', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('redox', 'O', -100)\n",
      "('homeostasis', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('activated', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('kinase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('SAPKs', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('JNK', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('p38', 'O', 2)\n",
      "('MAPK', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('downstream', 'O', -100)\n",
      "('apoptosis', 'O', 2)\n",
      "('effectors', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('After', 'O', -100)\n",
      "('treatment', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('HMs', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('peaks', 'O', 2)\n",
      "('shifted', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('3423', 'O', 2)\n",
      "('cm^-1', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('case', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('Citrobacter', 'B', -100)\n",
      "('werkmanii', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('WWN1', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('3421', 'O', -100)\n",
      "('cm^-1', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('case', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('Enterobacter', 'B', 1)\n",
      "('cloacae', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 2)\n",
      "('treated', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Transformed', 'O', -100)\n",
      "('BL21', 'B', 2)\n",
      "('E.coli', 'I', 0)\n",
      "('cells', 'O', -100)\n",
      "('were', 'O', 1)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Luria-Bertani', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('media', 'O', 2)\n",
      "('until', 'O', -100)\n",
      "('reaching', 'O', -100)\n",
      "('an', 'O', -100)\n",
      "('optical', 'O', -100)\n",
      "('density', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('0.6-0.8', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('then', 'O', 2)\n",
      "('induced', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('0.5', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('IPTG', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('three', 'O', 2)\n",
      "('V.', 'O', 2)\n",
      "('fluvialis', 'O', 2)\n",
      "('draft', 'O', -100)\n",
      "('genomes', 'O', 2)\n",
      "('include', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('V.', 'B', 2)\n",
      "('fluvialis', 'I', 2)\n",
      "('NCDO', 'I', 2)\n",
      "('2497', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('Accession', 'O', -100)\n",
      "('no', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('collected', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('aeruginosa', 'I', 0)\n",
      "('KT1115', 'I', -100)\n",
      "('with', 'O', 1)\n",
      "('rugose', 'O', 1)\n",
      "('small', 'O', -100)\n",
      "('colony', 'O', -100)\n",
      "('variant', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RSCV', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('phenotype', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('soil', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('dynamic', 'O', 2)\n",
      "('regulation', 'O', 2)\n",
      "('mechanism', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('polysaccharide', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('RLs', 'O', 2)\n",
      "('synthesis', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('LTR', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('then', 'O', 2)\n",
      "('substituted', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('PstI', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('HincII', 'O', -100)\n",
      "('restriction', 'O', 2)\n",
      "('sites', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('homologous', 'O', -100)\n",
      "('sequences', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('naturally', 'O', 2)\n",
      "('occurring', 'O', 2)\n",
      "('LTR', 'O', 2)\n",
      "('containing', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('21-bp', 'O', 2)\n",
      "('elements', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('Chandhasin', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('manuscript', 'O', 2)\n",
      "('submitted', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('FeLV-945', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('contains', 'O', -100)\n",
      "('three', 'O', 2)\n",
      "('21-bp', 'O', 2)\n",
      "('elements', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Thus', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('postulate', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Desulfuromonadaceae', 'B', 2)\n",
      "('strain', 'I', 0)\n",
      "('DM1', 'I', -100)\n",
      "('may', 'O', -100)\n",
      "('have', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('mechanism', 'O', -100)\n",
      "('to', 'O', 1)\n",
      "('‘sense’', 'O', 1)\n",
      "('surface', 'O', -100)\n",
      "('potentials', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('adapt', 'O', 2)\n",
      "('accordingly', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('rapidly', 'O', 2)\n",
      "('reconstruct', 'O', -100)\n",
      "('its', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('profiles', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('central', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('metabolism', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('EET-associated', 'O', 2)\n",
      "('MH-cytCs', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Injection', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('dsRNA', 'O', 2)\n",
      "('specific', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('WSSV', 'O', 2)\n",
      "('PK1', 'O', 2)\n",
      "('decreases', 'O', -100)\n",
      "('cumulative', 'O', 2)\n",
      "('mortality', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('showing', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('disrupting', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "('iron', 'O', 2)\n",
      "('withholding', 'O', 2)\n",
      "('mechanisms', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('important', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('successful', 'O', 2)\n",
      "('WSSV', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('166', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('our', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 1)\n",
      "('treated', 'O', 1)\n",
      "('plants', 'O', -100)\n",
      "('showed', 'O', 2)\n",
      "('lower', 'O', 0)\n",
      "('proline', 'O', 1)\n",
      "('content', 'O', -100)\n",
      "('under', 'O', 1)\n",
      "('HM', 'O', 1)\n",
      "('stress', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('comparison', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('uninoculated', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('RT885', 'B', -100)\n",
      "('was', 'O', 0)\n",
      "('classified', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('antimicrobial', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('clindamycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('RT883', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('RT884', 'B', 2)\n",
      "('were', 'O', 0)\n",
      "('classified', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('susceptible', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('intermediately', 'O', -100)\n",
      "('susceptible', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('CECT', 'B', 2)\n",
      "('4043', 'I', 0)\n",
      "('and', 'O', -100)\n",
      "('CECT', 'B', 1)\n",
      "('539', 'I', -100)\n",
      "('used', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('this', 'O', -100)\n",
      "('experiment', 'O', 1)\n",
      "('have', 'O', -100)\n",
      "('proven', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('bile', 'O', 2)\n",
      "('salts', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vitro', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('mimic', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('animal', 'O', 2)\n",
      "('GIT', 'O', 2)\n",
      "('environment', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Meanwhile', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('OsiSh-2', 'B', 2)\n",
      "('improved', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('chloroplast', 'O', -100)\n",
      "('development', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('optimally', 'O', 2)\n",
      "('maintained', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('related', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('thus', 'O', 2)\n",
      "('promoting', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('crop', 'O', 2)\n",
      "('yield', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('Geobacillus', 'B', 2)\n",
      "('icigianus', 'I', 0)\n",
      "('G1w1T', 'I', -100)\n",
      "('is', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('new', 'O', -100)\n",
      "('thermophilic', 'O', -100)\n",
      "('strain', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('genus', 'O', -100)\n",
      "('Geobacillus', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('firstly', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('sludge', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('hydrothermal', 'O', -100)\n",
      "('vent', 'O', 2)\n",
      "('located', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Valley', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Geysers', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Kamchatka', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Russia', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('IMR-32', 'B', -100)\n",
      "(',', 'O', 0)\n",
      "('KP-N-RT-BM-1', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('NB-1', 'B', 2)\n",
      "('human', 'O', 0)\n",
      "('neuroblastoma', 'O', -100)\n",
      "('cells', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('cultured', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('RPMI', 'O', -100)\n",
      "('1640', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Nacalai', 'O', 2)\n",
      "('Tesque', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('supplemented', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('FBS', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Pairwise', 'O', -100)\n",
      "('distances', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('luminescent', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('reporters', 'O', 2)\n",
      "('show', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('effects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('anthrose', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('decoyinine', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('expression', 'O', 2)\n",
      "('profiles', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('atxA', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('lef', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('pagA', 'O', 2)\n",
      "('promoters', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('WT', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Δ', 'O', -100)\n",
      "('antC', 'O', 2)\n",
      "('Sterne', 'B', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', -100)\n",
      "('S6', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Bacterial', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('72', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('determine', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('specificity', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('sensitivity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('PCR', 'O', 2)\n",
      "('assays', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('National', 'O', 2)\n",
      "('Animal', 'O', 2)\n",
      "('Disease', 'O', 2)\n",
      "('Center', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ames', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('IA', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('cultured', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('Luria-Bertani', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('broth', 'O', -100)\n",
      "('agar', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Sorbitol', 'O', -100)\n",
      "('MacConkey', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('SMAC', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('plates', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Testing', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('fully', 'O', 2)\n",
      "('susceptible', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('H37Rv', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('ten', 'O', -100)\n",
      "('clinical', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('patients', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('drug-resistant', 'O', 2)\n",
      "('TB', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('YMA4', 'B', -100)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('sediment', 'O', 2)\n",
      "('sample', 'O', 2)\n",
      "('collected', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('open', 'O', 2)\n",
      "('ocean', 'O', 2)\n",
      "('off', 'O', 2)\n",
      "('Lamay', 'O', 2)\n",
      "('island', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Taiwan', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('incomplete', 'O', 2)\n",
      "('nucleotide', 'O', 2)\n",
      "('synthesis', 'O', 2)\n",
      "('pathway', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('existed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Rickettsia', 'O', 2)\n",
      "('prowazekii', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('64', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('endocellular', 'O', 2)\n",
      "('lifestyle', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('then', 'O', 2)\n",
      "('assessed', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('capacities', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('mature', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('during', 'O', 2)\n",
      "('18', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BHI', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('sucrose', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('observing', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('robust', 'O', 2)\n",
      "('biomass', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('formed', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('defective', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('initiation', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('SK115', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('SK678', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('SK1056', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('trait', 'O', -100)\n",
      "('also', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('SK160', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('SK330', 'B', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Hence', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('results', 'O', 2)\n",
      "('revealed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('ZH07', 'B', 2)\n",
      "('could', 'O', 0)\n",
      "('efficiently', 'O', -100)\n",
      "('degrade', 'O', -100)\n",
      "('multiple', 'O', 2)\n",
      "('autotoxins', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('indicating', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('mitigation', 'O', -100)\n",
      "('effects', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('ZH07', 'B', 2)\n",
      "('on', 'O', 2)\n",
      "('autotoxin', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('peanuts', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Salicylic', 'O', -100)\n",
      "('acid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('phenolic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('meadowsweet', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('38', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('also', 'O', 2)\n",
      "('detected', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Koskue', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Macrococcus', 'O', -100)\n",
      "('caseolyticus', 'O', 2)\n",
      "('DNA', 'O', -100)\n",
      "('gyrase', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('included', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('an', 'O', -100)\n",
      "('outgroup', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('One', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('metabolites', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('overabundant', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('HT115', 'B', -100)\n",
      "('compared', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('OP50', 'B', 2)\n",
      "('is', 'O', 0)\n",
      "('GABA', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('wadsworthia', 'I', 0)\n",
      "('3_1_6', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('Tpa', 'O', -100)\n",
      "('occurs', 'O', -100)\n",
      "('immediately', 'O', -100)\n",
      "('downstream', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('AlaDH', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', -100)\n",
      "('4c', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('our', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('there', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('significant', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('POD', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WWN1', 'I', 1)\n",
      "('and', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('JWM6', 'I', 1)\n",
      "('inoculated', 'O', 1)\n",
      "('wheat', 'O', -100)\n",
      "('plants', 'O', 2)\n",
      "('as', 'O', 0)\n",
      "('compared', 'O', 1)\n",
      "('to', 'O', -100)\n",
      "('uninoculated', 'O', 1)\n",
      "('plants', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('RTK17.1', 'B', -100)\n",
      "('were', 'O', 0)\n",
      "('prepared', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('V4', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('pH', 'O', 2)\n",
      "('2.5', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('methane-grown', 'O', 2)\n",
      "('cultures', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('final', 'O', -100)\n",
      "('OD', 'O', -100)\n",
      "('_600nm', 'O', 2)\n",
      "('0.8-1.0', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('previously', 'O', -100)\n",
      "('described', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Carere', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2017', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('ZYF612', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('this', 'O', -100)\n",
      "('gene', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('most', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Labrenzia', 'O', 2)\n",
      "('suaedae', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('93', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('amino-acid', 'O', -100)\n",
      "('identity', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('plant', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Suaeda', 'O', 2)\n",
      "('maritima', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('grows', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('tidal', 'O', 2)\n",
      "('flats', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Bibi', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2014', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('mitochondrial', 'O', 2)\n",
      "('calcium', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('ES2', 'B', 2)\n",
      "('cells', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('increased', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('1.5', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('3.5', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('4.0', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('5', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('20', 'O', -100)\n",
      "('μg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('mL', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('stigmasterol', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('case', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('vehicle-treated', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1.1', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('3C', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('finding', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('compatible', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Gothwal', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Iwata', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "('who', 'O', -100)\n",
      "('described', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('nitrogen', 'O', -100)\n",
      "('activity', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('Azotobacter', 'O', 2)\n",
      "('beijerinckii', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Lysobacter', 'B', 2)\n",
      "('enzymogenes', 'I', 2)\n",
      "('DMS', 'I', 2)\n",
      "('2043T', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('rhizospheric', 'O', 2)\n",
      "('bacterial', 'O', -100)\n",
      "('isolates', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('screened', 'O', -100)\n",
      "('by', 'O', 2)\n",
      "('changing', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('color', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('BTB', 'O', 0)\n",
      "('containing', 'O', -100)\n",
      "('nitrogen', 'O', 1)\n",
      "('free', 'O', -100)\n",
      "('media', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('these', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('color', 'O', -100)\n",
      "('change', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('BTB', 'O', 2)\n",
      "('containing', 'O', 2)\n",
      "('media', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('suggesting', 'O', 2)\n",
      "('excretion', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('ammonia', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('15', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('16', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('mushroom', 'O', 2)\n",
      "('pathogens', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('non-pathogenic', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('NS1', 'B', -100)\n",
      "('had', 'O', -100)\n",
      "('between', 'O', 2)\n",
      "('one', 'O', 0)\n",
      "('to', 'O', -100)\n",
      "('three', 'O', 2)\n",
      "('T3SS', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('elements', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('broken', 'O', -100)\n",
      "('up', 'O', 2)\n",
      "('T3SS', 'O', 2)\n",
      "('clusters', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('being', 'O', 2)\n",
      "('evident', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('genomes', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Regarding', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('experimental', 'O', 2)\n",
      "('procedure', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('method', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('evaluate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('polysaccharide-degrading', 'O', 2)\n",
      "('ability', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('KA5-B', 'B', 2)\n",
      "('^T', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('used', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('Sakai', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2014', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Interestingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('despite', 'O', 2)\n",
      "('being', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('symptomatic', 'O', 2)\n",
      "('foliar', 'O', 2)\n",
      "('tissue', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('20GA0206', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('21GA0654', 'B', -100)\n",
      "('were', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('able', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('induce', 'O', -100)\n",
      "('foliar', 'O', -100)\n",
      "('symptoms', 'O', -100)\n",
      "('upon', 'O', 2)\n",
      "('inoculation', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('greenhouse', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('All', 'O', -100)\n",
      "('qPCR', 'O', 2)\n",
      "('experiments', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('repeated', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('times', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('reproducibility', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Initial', 'O', -100)\n",
      "('research', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('V_2O_5', 'O', 2)\n",
      "('photocatalytic', 'O', 2)\n",
      "('raw', 'O', -100)\n",
      "('material', 'O', -100)\n",
      "('mainly', 'O', -100)\n",
      "('extracted', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('vanadium', 'O', 2)\n",
      "('ore', 'O', 2)\n",
      "('directly', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('photocatalytic', 'O', 2)\n",
      "('performance', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('low', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('All', 'O', -100)\n",
      "('whisky', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('three', 'O', -100)\n",
      "('milk', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CAUH18', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('H10', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('R0052', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('possessed', 'O', 2)\n",
      "('a', 'O', 0)\n",
      "('single', 'O', -100)\n",
      "('GH32-1', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('no', 'O', -100)\n",
      "('milk', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('included', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('vitro', 'O', -100)\n",
      "('carbohydrate', 'O', -100)\n",
      "('metabolic', 'O', 2)\n",
      "('tests', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('enzyme', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Among', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('18', 'O', 2)\n",
      "('BGCs', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('S.', 'B', -100)\n",
      "('coelicoflavus', 'I', 2)\n",
      "('NBRC', 'I', 0)\n",
      "('15399^T', 'I', -100)\n",
      "(',', 'O', 1)\n",
      "('13', 'O', -100)\n",
      "('are', 'O', -100)\n",
      "('present', 'O', -100)\n",
      "('also', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('S.', 'B', -100)\n",
      "('coelicoflavus', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('ZG0656', 'I', -100)\n",
      "('whereas', 'O', -100)\n",
      "('three', 'O', 2)\n",
      "('lantipeptide', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('terpene', 'O', 2)\n",
      "('BGCs', 'O', 2)\n",
      "('are', 'O', 0)\n",
      "('not', 'O', -100)\n",
      "('.', 'O', 1)\n",
      "('Pectobacterium', 'O', -100)\n",
      "('brasiliense', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Pbr', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('worldwide-distributed', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('pathogen', 'O', 2)\n",
      "('causing', 'O', 2)\n",
      "('soft', 'O', -100)\n",
      "('rot', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('wide', 'O', 2)\n",
      "('range', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('economically', 'O', 2)\n",
      "('important', 'O', 2)\n",
      "('crops', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('selected', 'O', 2)\n",
      "('PfNPNA-1', 'O', 2)\n",
      "('V_H', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('κ', 'O', -100)\n",
      "('gene', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('excised', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('rf', 'O', 2)\n",
      "('JC-M13-88', 'O', 2)\n",
      "('DNA', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('digestion', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('XbaI', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('HindIII', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('ligated', 'O', -100)\n",
      "('into', 'O', -100)\n",
      "('similarly', 'O', -100)\n",
      "('digested', 'O', -100)\n",
      "('pAbHIS', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('functional', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cloned', 'O', 2)\n",
      "('fliC-H4', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('JM109', 'B', -100)\n",
      "('derivative', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('TPE1976', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('fliC-H4_U9-41', 'O', -100)\n",
      "('clone', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('TPE1978', 'B', -100)\n",
      "('(', 'O', -100)\n",
      "('fliC-H4_P12b', 'O', -100)\n",
      "('clone', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('analyzed', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('tube', 'O', -100)\n",
      "('agglutination', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('H4', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('H48', 'O', 2)\n",
      "('antisera', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Transfected', 'O', -100)\n",
      "('COS7', 'B', 2)\n",
      "('cells', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('washed', 'O', 2)\n",
      "('twice', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('20', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('N-ethylmaleimide', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('PBS', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('solubilized', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('15', 'O', 2)\n",
      "('min', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('4°C', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('3-', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('3-Cholamidopropyl', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('dimethylammonio', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('-1-propanesulfonic', 'O', -100)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CHAPS', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('HEPES-buffered', 'O', -100)\n",
      "('saline', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('HBS', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('buffer', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('10', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('HEPES-NaOH', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('pH', 'O', -100)\n",
      "('7.4', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('150', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('NaCl', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('EDTA', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('1', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('CHAPS', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('0.4', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('PMSF', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "('µM', 'O', 2)\n",
      "('pepstatin', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('sequenced', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('series', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('44', 'O', 2)\n",
      "('colorectal', 'O', 2)\n",
      "('tumours', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('normal', 'O', 2)\n",
      "('controls', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('MSI', 'O', 2)\n",
      "('loci', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BAT25', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('BAT26', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('BAT34c4', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('D18S55', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('D5S346', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('second', 'O', 2)\n",
      "('series', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('six', 'O', -100)\n",
      "('colorectal', 'O', 2)\n",
      "('tumours', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('no', 'O', -100)\n",
      "('control', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('mononucleotide', 'O', 2)\n",
      "('loci', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BAT25', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('BAT26', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Genome', 'O', -100)\n",
      "('Sequencing', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Kocuria', 'B', 2)\n",
      "('varians', 'I', 0)\n",
      "('Strain', 'I', -100)\n",
      "('80', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('Isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('Starter', 'O', 1)\n",
      "('Culture', 'O', 1)\n",
      "('for', 'O', 2)\n",
      "('Dry', 'O', 2)\n",
      "('Sausage', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('summary', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('assessment', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('airways', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('patients', 'O', 2)\n",
      "('reveals', 'O', 2)\n",
      "('distinct', 'O', 2)\n",
      "('immunological', 'O', 2)\n",
      "('features', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('COVID-19', 'O', 2)\n",
      "('ARDS', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('case', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('ppm', 'O', 2)\n",
      "('HM', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('APX', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('ranged', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('1.16', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('2.95', 'O', 2)\n",
      "('EU', 'O', -100)\n",
      "('mg^-1', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Citrobacter', 'B', -100)\n",
      "('werkmanii', 'I', -100)\n",
      "('strain', 'I', 2)\n",
      "('WWN1', 'I', 2)\n",
      "(',', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', 2)\n",
      "('JWM6', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('their', 'O', 1)\n",
      "('consortium', 'O', -100)\n",
      "('increased', 'O', -100)\n",
      "('shoot', 'O', -100)\n",
      "('APX', 'O', -100)\n",
      "('by', 'O', 1)\n",
      "('150', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('153', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', 1)\n",
      "('152', 'O', -100)\n",
      "('%', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('respectively', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('respective', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('Background', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('Strains', 'O', 2)\n",
      "('HL1', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('M1', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('isolated', 'O', 0)\n",
      "('from', 'O', 2)\n",
      "('kefir', 'O', 2)\n",
      "('grains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('have', 'O', -100)\n",
      "('been', 'O', -100)\n",
      "('tentatively', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('partial', 'O', 2)\n",
      "('16S', 'O', 2)\n",
      "('rRNA', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('Lactobacillus', 'O', 2)\n",
      "('kefiranofaciens', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('China', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('SFGR', 'O', 2)\n",
      "('genotypes', 'O', 2)\n",
      "('have', 'O', -100)\n",
      "('been', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('causative', 'O', 2)\n",
      "('agents', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('rickettsiosis', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('R.', 'O', 2)\n",
      "('heilongjiangensis', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('R.', 'O', 2)\n",
      "('sibirica', 'O', -100)\n",
      "('subsp.', 'O', 2)\n",
      "('sibirica', 'O', -100)\n",
      "('BJ-90', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Candidatus', 'O', -100)\n",
      "('Rickettsia', 'O', 2)\n",
      "('tarasevichiae', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('R.', 'O', 2)\n",
      "('raoultii', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Rickettsia', 'O', -100)\n",
      "('sp', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('show', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('supernatants', 'O', 2)\n",
      "('derived', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('L.', 'B', 2)\n",
      "('reuteri', 'I', 0)\n",
      "('AN417', 'I', -100)\n",
      "('cultures', 'O', 1)\n",
      "('are', 'O', -100)\n",
      "('able', 'O', -100)\n",
      "('to', 'O', 1)\n",
      "('suppress', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('growth', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('formation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('oral', 'O', 2)\n",
      "('pathogenic', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('B.', 'B', -100)\n",
      "('anthracis', 'I', 0)\n",
      "('Sterne', 'I', -100)\n",
      "('34F2', 'I', 1)\n",
      "('was', 'O', -100)\n",
      "('obtained', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('Colorado', 'O', 1)\n",
      "('Serum', 'O', -100)\n",
      "('Company', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('USA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('grown', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('BHI', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Becton', 'O', 2)\n",
      "('Dickinson', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('Franklin', 'O', 2)\n",
      "('Lakes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('NJ', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('USA', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('heart', 'O', -100)\n",
      "('infusion', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('HIB', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Research', 'O', 2)\n",
      "('Products', 'O', 2)\n",
      "('International', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('Mount', 'O', 2)\n",
      "('Prospect', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('IL', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('USA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Cells', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('A.', 'B', 2)\n",
      "('profundus', 'I', 0)\n",
      "('AV18^T', 'I', -100)\n",
      "('are', 'O', 1)\n",
      "('reported', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('Gram', 'O', 1)\n",
      "('stain-negative', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('highly', 'O', -100)\n",
      "('irregular', 'O', 2)\n",
      "('cocci', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('occurring', 'O', 2)\n",
      "('singly', 'O', 2)\n",
      "('or', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('pairs', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Meanwhile', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('acetolactate', 'O', 2)\n",
      "('synthase', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('Bacillus', 'B', -100)\n",
      "('subtilis', 'I', -100)\n",
      "('168', 'I', 2)\n",
      "('eliminates', 'O', 2)\n",
      "('pyruvate', 'O', 0)\n",
      "('and', 'O', 1)\n",
      "('alanine', 'O', 1)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('by-products', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('aspartate', 'O', 2)\n",
      "('transamination', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('First', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('overlay', 'O', 2)\n",
      "('assay', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('Mcc1229', 'O', 2)\n",
      "('inhibited', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('susceptible', 'O', -100)\n",
      "('E.', 'B', -100)\n",
      "('coli', 'I', 2)\n",
      "('O157', 'I', 2)\n",
      "(':', 'I', 2)\n",
      "('H7', 'I', 0)\n",
      "('strain', 'O', -100)\n",
      "('PA2', 'B', 1)\n",
      "(',', 'O', 1)\n",
      "('creating', 'O', 1)\n",
      "('a', 'O', 1)\n",
      "('zone', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('clearing', 'O', -100)\n",
      "('around', 'O', 2)\n",
      "('Mcc1229-producing', 'O', 2)\n",
      "('colonies', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('S2', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('HLT2-23^T', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('=CGMCC', 'B', -100)\n",
      "('1.11085^T', 'I', -100)\n",
      "('=', 'O', -100)\n",
      "('NBRC', 'B', -100)\n",
      "('114046^T', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('ice', 'O', -100)\n",
      "('sample', 'O', -100)\n",
      "('obtained', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('supraglacial', 'O', 2)\n",
      "('zone', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('Hailuogou', 'O', -100)\n",
      "('glacier', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('Sichuan', 'O', -100)\n",
      "('province', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('China', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Isolates', 'O', -100)\n",
      "('UCD_MED3', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('UCD_MED7', 'B', -100)\n",
      "('were', 'O', -100)\n",
      "('subcultured', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('Petri', 'O', 2)\n",
      "('plates', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('surfaces', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('agar', 'O', 2)\n",
      "('deep', 'O', 2)\n",
      "('tubes', 'O', -100)\n",
      "('containing', 'O', 2)\n",
      "('Mueller', 'O', 2)\n",
      "('Hinton', 'O', -100)\n",
      "('agar', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MHA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('promote', 'O', 2)\n",
      "('pyomelanin', 'O', -100)\n",
      "('production', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('some', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('9', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('10', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Microbulbifer', 'B', -100)\n",
      "('mangrovi', 'I', 0)\n",
      "('DD13', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('KCTC', 'B', -100)\n",
      "('23483', 'I', 1)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Alcanivorax', 'B', 1)\n",
      "('xenomutans', 'I', -100)\n",
      "('JC109', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('KCTC', 'B', -100)\n",
      "('23751', 'I', 1)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Alteromonas', 'B', 2)\n",
      "('lipolytica', 'I', 0)\n",
      "('JW12', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('KCTC', 'B', -100)\n",
      "('52408', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('Alcanivorax', 'B', -100)\n",
      "('dieselolei', 'I', 1)\n",
      "('B‐5', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('DSM', 'B', 0)\n",
      "('16502', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('Aestuariibacter', 'B', 2)\n",
      "('aggregatus', 'I', 2)\n",
      "('WH169', 'I', 0)\n",
      "('(', 'O', -100)\n",
      "('DSM', 'B', 1)\n",
      "('23094', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Pseudoalteromonas', 'B', 1)\n",
      "('gelatinilytica', 'I', -100)\n",
      "('NH153', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('DSM', 'B', 0)\n",
      "('100951', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "(']', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('routinely', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('Marine', 'O', -100)\n",
      "('Broth', 'O', -100)\n",
      "('or', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('Marine', 'O', 1)\n",
      "('Agar', 'O', -100)\n",
      "('plates', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('BD', 'O', -100)\n",
      "('Difco™', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Further', 'O', -100)\n",
      "('research', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('SRS89', 'B', 2)\n",
      "('strain', 'O', 0)\n",
      "('should', 'O', -100)\n",
      "('focus', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('optimizing', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('industrial-scale', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Reb', 'O', -100)\n",
      "('A', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('fermenter', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('mRNA', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('A3H', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('CEM-SS', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('CEM.NKR', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('these', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('clones', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('100', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('1000-fold', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('those', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('H9', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('clone', 'O', -100)\n",
      "('#', 'O', -100)\n",
      "('7', 'O', 2)\n",
      "('expressed', 'O', 2)\n",
      "('slightly', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('A3H', 'O', 2)\n",
      "('mRNA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('3C', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Mc-6', 'O', -100)\n",
      "('produced', 'O', 2)\n",
      "('one', 'O', -100)\n",
      "('major', 'O', -100)\n",
      "('compound', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('3-nitropropionic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('has', 'O', -100)\n",
      "('been', 'O', -100)\n",
      "('previously', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('Phomopsis', 'O', 2)\n",
      "('sp.', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('evaluated', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('biological', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('against', 'O', -100)\n",
      "('15', 'O', -100)\n",
      "('different', 'O', -100)\n",
      "('mycobacteria', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('9', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('infectivity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('MM1', 'B', 2)\n",
      "('strain', 'O', 0)\n",
      "('is', 'O', -100)\n",
      "('comparable', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('s.s.', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('have', 'O', -100)\n",
      "('been', 'O', -100)\n",
      "('studied', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('laboratory', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('B31', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('N40', 'B', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('Ixodes', 'O', 2)\n",
      "('scapularis', 'O', 2)\n",
      "('tick', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('NCH-1', 'B', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('human', 'O', -100)\n",
      "('skin', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('297', 'B', 2)\n",
      "('(', 'O', 0)\n",
      "('hamster', 'O', -100)\n",
      "('passaged', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('cerebrospinal', 'O', 2)\n",
      "('fluid', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('CPCC', 'B', 2)\n",
      "('101082', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('=NBRC', 'B', -100)\n",
      "('113513', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "('=KCTC', 'B', -100)\n",
      "('62853', 'I', 2)\n",
      "('^T', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('hinterland', 'O', -100)\n",
      "('area', 'O', 2)\n",
      "('characterized', 'O', -100)\n",
      "('by', 'O', 0)\n",
      "('cyanobacteria-dominated', 'O', -100)\n",
      "('crusts', 'O', -100)\n",
      "('collected', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('Badain', 'O', -100)\n",
      "('Jaran', 'O', 2)\n",
      "('desert', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('4,5-Dichloropyrrolyl-S-carrier', 'O', -100)\n",
      "('protein', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('found', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('starter', 'O', -100)\n",
      "('unit', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('pyoluteorin', 'O', -100)\n",
      "('biosynthetic', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('Pseudomonas', 'B', 2)\n",
      "('fluorescens', 'I', 2)\n",
      "('Pf-5', 'I', 2)\n",
      "('[', 'O', 2)\n",
      "('44', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('AMMD', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('no', 'O', -100)\n",
      "('cognate', 'O', 2)\n",
      "('luxR', 'O', 2)\n",
      "('-type', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('identified', 'O', 2)\n",
      "('associated', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('cepI2', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Choudhary', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2013', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('diazoefficiens', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('110spc4', 'I', 1)\n",
      "('as', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('model', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('as', 'O', 1)\n",
      "('a', 'O', 1)\n",
      "('wealth', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('functional', 'O', -100)\n",
      "('genomics', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('compiled', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('can', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('rule', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('possibility', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('organisms', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('UCD_MED5', 'B', 2)\n",
      "('could', 'O', 0)\n",
      "('have', 'O', -100)\n",
      "('contributed', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('discoloration', 'O', -100)\n",
      "('considering', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('coculture', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('UCD_MED5', 'B', -100)\n",
      "('and', 'O', 2)\n",
      "('UCD_MED7', 'B', 2)\n",
      "('formed', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('purple', 'O', 2)\n",
      "('/', 'O', 0)\n",
      "('lavender', 'O', -100)\n",
      "('biofilm', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('bacterial', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('hemolymph', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('HIM3', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('macerate', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('IJs', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('NA04', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('E.', 'B', -100)\n",
      "('coli', 'I', 2)\n",
      "('DH5α', 'I', 0)\n",
      "('(', 'O', -100)\n",
      "('negative', 'O', -100)\n",
      "('control', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('individually', 'O', 0)\n",
      "('inoculated', 'O', -100)\n",
      "('in', 'O', 1)\n",
      "('100', 'O', 1)\n",
      "('mL', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('LB', 'O', 2)\n",
      "('liquid', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('overnight', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('150', 'O', 2)\n",
      "('rpm', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Our', 'O', -100)\n",
      "('results', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Na^+', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('K^+', 'O', -100)\n",
      "('ratio', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('shoot', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('root', 'O', 2)\n",
      "('tissues', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('OsEXPA7-OX', 'B', 2)\n",
      "('line', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('than', 'O', 0)\n",
      "('that', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('shoots', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('roots', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('WT', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('5C', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('F', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('002161215', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Unassigned', 'O', -100)\n",
      "('Anaerofilum', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Unassigned', 'O', 2)\n",
      "('Faecalibacterium', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Fournierella', 'B', -100)\n",
      "('sp.', 'I', 2)\n",
      "('004558145', 'I', -100)\n",
      "(';', 'O', -100)\n",
      "('bacteria', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('genera', 'O', -100)\n",
      "('Anaerobutyricum', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('43', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Dorea', 'O', 2)\n",
      "('[', 'O', 0)\n",
      "('4', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Faecalibacterium', 'O', -100)\n",
      "('[', 'O', 1)\n",
      "('44', 'O', -100)\n",
      "(']', 'O', 1)\n",
      "('are', 'O', -100)\n",
      "('all', 'O', -100)\n",
      "('known', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('SCFA', 'O', 2)\n",
      "('producers', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('E.', 'O', -100)\n",
      "('ulmoides', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('tree', 'O', -100)\n",
      "('cultivated', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('China', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('gutta-percha', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('116', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('order', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('untangle', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('relationships', 'O', -100)\n",
      "('between', 'O', -100)\n",
      "('these', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('acquired', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('S', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('somaliensis', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('S', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('sudanensis', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('DSMZ', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('DSM', 'B', 2)\n",
      "('40738', 'I', 2)\n",
      "('and', 'O', -100)\n",
      "('DSM', 'B', -100)\n",
      "('41923', 'I', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('sequenced', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('assembled', 'O', 0)\n",
      "('their', 'O', 1)\n",
      "('genomes', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Although', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('distincta', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('P.', 'B', 2)\n",
      "('arctica', 'I', -100)\n",
      "('A', 'I', 2)\n",
      "('37-1-2', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('strains', 'O', 1)\n",
      "('were', 'O', -100)\n",
      "('unable', 'O', 1)\n",
      "('to', 'O', 1)\n",
      "('degrade', 'O', -100)\n",
      "('agar', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('dbCAN', 'O', 2)\n",
      "('discovered', 'O', -100)\n",
      "('GH50', 'O', 2)\n",
      "('agarase', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('arctica', 'I', 2)\n",
      "('A', 'I', 2)\n",
      "('37-1-2', 'I', 2)\n",
      "('^T', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', -100)\n",
      "('8', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('shared', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('UM_RHS', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('UM_NYF', 'B', -100)\n",
      "('were', 'O', -100)\n",
      "('higher', 'O', -100)\n",
      "('compared', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('that', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('UM_RHS', 'B', -100)\n",
      "('and', 'O', 2)\n",
      "('M', 'B', 2)\n",
      "('.', 'I', 2)\n",
      "('cosmeticum', 'I', 2)\n",
      "('DSM44829', 'I', 2)\n",
      "('or', 'O', 2)\n",
      "('UM_NYF', 'B', 0)\n",
      "('and', 'O', -100)\n",
      "('M', 'B', -100)\n",
      "('.', 'I', -100)\n",
      "('cosmeticum', 'I', 2)\n",
      "('DSM44829', 'I', 0)\n",
      "(',', 'O', 1)\n",
      "('probably', 'O', 1)\n",
      "('reflecting', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('fact', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('both', 'O', -100)\n",
      "('UM_RHS', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('UM_NYF', 'B', -100)\n",
      "('are', 'O', -100)\n",
      "('highly', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('because', 'O', 0)\n",
      "('they', 'O', 1)\n",
      "('were', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('very', 'O', -100)\n",
      "('close', 'O', -100)\n",
      "('geographical', 'O', -100)\n",
      "('regions', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Indonesia', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Malaysia', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('whereas', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('M', 'B', -100)\n",
      "('.', 'I', 2)\n",
      "('cosmeticum', 'I', 0)\n",
      "('DSM44829', 'I', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('granulomatous', 'O', 2)\n",
      "('lesion', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('female', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Venezuela', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('https', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('www.dsmz.de', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('catalogues', 'O', 2)\n",
      "('/', 'O', 0)\n",
      "('details', 'O', 1)\n",
      "('/', 'O', 1)\n",
      "('culture', 'O', -100)\n",
      "('/', 'O', 1)\n",
      "('DSM-44829.html', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('C.', 'B', -100)\n",
      "('maritimus', 'I', 0)\n",
      "('KKC1', 'I', -100)\n",
      "('can', 'O', 1)\n",
      "('grow', 'O', -100)\n",
      "('anaerobically', 'O', -100)\n",
      "('on', 'O', 1)\n",
      "('100', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('CO', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('producing', 'O', 2)\n",
      "('H', 'O', -100)\n",
      "('_2', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CO', 'O', 2)\n",
      "('_2', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('without', 'O', 2)\n",
      "('additional', 'O', 2)\n",
      "('electron', 'O', -100)\n",
      "('acceptors', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('thiosulfate', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('sulfite', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ferric', 'O', 2)\n",
      "('citrate', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('amorphous', 'O', 2)\n",
      "('Fe', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('III', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('oxide', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Fe', 'O', -100)\n",
      "('_2', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "('_3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('fumarate', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('organic', 'O', 2)\n",
      "('compounds', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('pyruvate', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('thiosulfate', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Table', 'O', -100)\n",
      "('1', 'O', 2)\n",
      "('shows', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('chemical', 'O', 2)\n",
      "('composition', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('white', 'O', 2)\n",
      "('soft', 'O', 2)\n",
      "('cheese', 'O', 2)\n",
      "('made', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('L.', 'O', 2)\n",
      "('helveticus', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('L.', 'O', 2)\n",
      "('rhamnosus', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('S.', 'B', -100)\n",
      "('thermophilus', 'I', 2)\n",
      "('S_3855', 'I', 2)\n",
      "('during', 'O', -100)\n",
      "('pickling', 'O', 2)\n",
      "('periods', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('stored', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('refrigerator', 'O', -100)\n",
      "('for', 'O', 1)\n",
      "('up', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('28', 'O', -100)\n",
      "('days', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Systematics', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('root‐knot', 'O', -100)\n",
      "('nematodes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Nematoda', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('Meloidogynidae', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('SdrF-defective', 'O', -100)\n",
      "('S.', 'B', 2)\n",
      "('epidermidis', 'I', -100)\n",
      "('strains', 'I', -100)\n",
      "('9', 'I', -100)\n",
      "('and', 'O', -100)\n",
      "('K28', 'B', 0)\n",
      "('showed', 'O', -100)\n",
      "('high', 'O', 1)\n",
      "('adherence', 'O', 1)\n",
      "('levels', 'O', 1)\n",
      "('to', 'O', 2)\n",
      "('Dacron', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('K28', 'O', 2)\n",
      "('level', 'O', 2)\n",
      "('being', 'O', 2)\n",
      "('even', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('than', 'O', -100)\n",
      "('9491', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Specifically', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('calculated', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('log', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('ratio', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('intensity', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('individual', 'O', 2)\n",
      "('SNL', 'O', 2)\n",
      "('operated', 'O', 2)\n",
      "('DRGs', 'O', -100)\n",
      "('vs.', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('average', 'O', -100)\n",
      "('expression', 'O', 2)\n",
      "('intensity', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('corresponding', 'O', 2)\n",
      "('DRGs', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sham', 'O', 2)\n",
      "('operated', 'O', 2)\n",
      "('mice', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Resolver', 'O', 2)\n",
      "('v5.1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('behaviour', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('Clostridium', 'B', 2)\n",
      "('tetani', 'I', 0)\n",
      "('34', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('sporulating', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('anaerobic', 'O', 2)\n",
      "('bacterium', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('resides', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('soil', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('pathogenic', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('humans', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('animals', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Environmental', 'O', -100)\n",
      "('Haitian', 'O', 2)\n",
      "('V.', 'O', 2)\n",
      "('cholerae', 'O', -100)\n",
      "('O1', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('nalidixic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('SXT', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('cotrimazole', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('reduced', 'O', 2)\n",
      "('susceptibility', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('ampicillin', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('chloramphenicol', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('instance', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('168', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('makes', 'O', 2)\n",
      "('polymers', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('glycerol', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('another', 'O', 2)\n",
      "('frequently', 'O', 2)\n",
      "('studied', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('W23', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('makes', 'O', -100)\n",
      "('polymers', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('ribitol', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('results', 'O', 2)\n",
      "('indicate', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('M.', 'O', 2)\n",
      "('catarrhalis', 'O', 2)\n",
      "('OMVs', 'O', -100)\n",
      "('significantly', 'O', 2)\n",
      "('activate', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('hBD-2', 'O', 2)\n",
      "('promoter', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('A549', 'B', 2)\n",
      "('cells', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('presence', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('IL-1β', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('flax', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Linum', 'O', -100)\n",
      "('usitatissimum', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('L6', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('typical', 'O', -100)\n",
      "('TNL', 'O', -100)\n",
      "('protein', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('directly', 'O', 2)\n",
      "('recognizes', 'O', 2)\n",
      "('variants', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('biotrophic', 'O', 2)\n",
      "('flax', 'O', -100)\n",
      "('rust', 'O', 2)\n",
      "('fungus', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Melampsora', 'O', 2)\n",
      "('lini', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('effector', 'O', 2)\n",
      "('AvrL567', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('20', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Interactions', 'O', -100)\n",
      "('between', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('microenvironment', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('emerging', 'O', 2)\n",
      "('issue', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('tumor', 'O', 2)\n",
      "('progression', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('They', 'O', -100)\n",
      "('included', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('stutzeri', 'O', 2)\n",
      "('Cr', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('VI', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('-', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('nitrate-reducing', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('RCH2', 'B', 2)\n",
      "('(', 'O', 2)\n",
      "('Chakraborty', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "('2017', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('naphthalene-degrading', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('19SMN4', 'B', -100)\n",
      "('(', 'O', -100)\n",
      "('Rosselló-Mora', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "('1994', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('naphthalene-degrading', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('CCUG', 'B', 2)\n",
      "('29243', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('equivalent', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('AN10', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Brunet-Galmés', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "('2012', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('petroleum-emulsifying', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SLG510A3-8', 'B', -100)\n",
      "('(', 'O', 2)\n",
      "('Hu', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "('2015', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('nitrogen-fixing', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('rhizosphere-associated', 'O', 1)\n",
      "('strain', 'O', -100)\n",
      "('DSM', 'B', 2)\n",
      "('4166', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('Yu', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "('2011', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('nitrogen-fixing', 'O', -100)\n",
      "('root-associated', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('A1501', 'B', -100)\n",
      "('(', 'O', -100)\n",
      "('Yan', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', -100)\n",
      "('2008', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('HW08', 'B', -100)\n",
      "('(', 'O', 0)\n",
      "('IEHW08', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('biologically', 'O', 0)\n",
      "('safe', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('mice', 'O', -100)\n",
      "('while', 'O', -100)\n",
      "('keeping', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('capacity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('degrading', 'O', 2)\n",
      "('SW', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Growth', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('BUN14', 'B', 2)\n",
      "('on', 'O', 0)\n",
      "('plates', 'O', -100)\n",
      "('containing', 'O', 2)\n",
      "('TSA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Trypticase', 'O', 2)\n",
      "('soy', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('medium', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('formed', 'O', 2)\n",
      "('circular', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('opaque', 'O', 2)\n",
      "('yellow', 'O', 2)\n",
      "('colonies', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Overall', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('included', 'O', 2)\n",
      "('65', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('pumilus', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('B.', 'O', 2)\n",
      "('safensis', 'O', -100)\n",
      "('genomes', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('including', 'O', 2)\n",
      "('those', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('FO-36b', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('MERTA', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('SAFR-032', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('ATCC7061', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('four', 'O', -100)\n",
      "('representative', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('B.', 'O', -100)\n",
      "('altitudinis', 'O', 2)\n",
      "('complex', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('viz.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('B.', 'B', 2)\n",
      "('aerophilus', 'I', 0)\n",
      "('C772', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('B.', 'B', 2)\n",
      "('altitudinis', 'I', 2)\n",
      "('41KF2b', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('cellulasensis', 'I', 2)\n",
      "('NIO-1130', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('T', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('stratosphericus', 'I', 2)\n",
      "('LAMA', 'I', -100)\n",
      "('585', 'I', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('milk', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('possessing', 'O', 2)\n",
      "('six', 'O', 2)\n",
      "('GH13', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('R0052', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('CAUH18', 'B', 0)\n",
      "('.', 'O', -100)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('confirmed', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('rod-shaped', 'O', 2)\n",
      "('morphology', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('A.', 'B', -100)\n",
      "('caecimuris', 'I', 2)\n",
      "('DSM', 'I', 2)\n",
      "('21839', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Fig.', 'O', -100)\n",
      "('1b', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('A.', 'B', 1)\n",
      "('equolifaciens', 'I', 1)\n",
      "('subsp.', 'I', -100)\n",
      "('celatus', 'I', 2)\n",
      "('DSM', 'I', -100)\n",
      "('18785', 'I', 2)\n",
      "('^T', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Fig.', 'O', 2)\n",
      "('1c', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('A.', 'B', -100)\n",
      "('mucosicola', 'I', 1)\n",
      "('DSM', 'I', -100)\n",
      "('19490', 'I', -100)\n",
      "('^T', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Fig.', 'O', 1)\n",
      "('1e', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('A.', 'B', 1)\n",
      "('muris', 'I', 1)\n",
      "('DSM', 'I', -100)\n",
      "('29508', 'I', 2)\n",
      "('^T', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Fig.', 'O', 2)\n",
      "('1f', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('nucleus', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('relaxed', 'O', 2)\n",
      "('circular', 'O', 2)\n",
      "('DNA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('rcDNA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('released', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('viral', 'O', 2)\n",
      "('capsids', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('modified', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('covalently', 'O', 2)\n",
      "('closed', 'O', 2)\n",
      "('circular', 'O', 2)\n",
      "('DNA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('cccDNA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "('polymerases', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('ligases', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('topoisomerase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('persists', 'O', 2)\n",
      "('episomically', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('nucleus', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('infected', 'O', 2)\n",
      "('hepatocytes', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('11,12,13,14', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('cccDNA', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('relatively', 'O', 2)\n",
      "('stable', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('quiescent', 'O', -100)\n",
      "('hepatocytes', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('mechanisms', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('regulate', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('stability', 'O', -100)\n",
      "('are', 'O', 2)\n",
      "('complex', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('fully', 'O', 2)\n",
      "('understood', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('include', 'O', 2)\n",
      "('immune', 'O', 2)\n",
      "('responses', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('cytokine', 'O', 2)\n",
      "('stimuli', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Escherichia', 'B', -100)\n",
      "('coli', 'I', 0)\n",
      "('BL21*', 'I', 1)\n",
      "('(', 'O', 1)\n",
      "('DE3', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('harbouring', 'O', 2)\n",
      "('recombinant', 'O', 0)\n",
      "('pGEX-6P-1', 'O', -100)\n",
      "('plasmids', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('cultured', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "('broth', 'O', -100)\n",
      "('supplemented', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('1', 'O', -100)\n",
      "('per', 'O', -100)\n",
      "('cent', 'O', -100)\n",
      "('glucose', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('ampicillin', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('50', 'O', 2)\n",
      "('μg', 'O', 2)\n",
      "('ml^−1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Methods.L.', 'B', -100)\n",
      "('kunkeei', 'I', 0)\n",
      "('MP2', 'I', -100)\n",
      "('was', 'O', -100)\n",
      "('originally', 'O', -100)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('guts', 'O', -100)\n",
      "('of', 'O', 1)\n",
      "('Chilean', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('mellifera', 'O', 2)\n",
      "('individuals', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('So', 'O', -100)\n",
      "('far', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('no', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('components', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('novel', 'O', 2)\n",
      "('EEU', 'O', 2)\n",
      "('pathway', 'O', 2)\n",
      "('have', 'O', -100)\n",
      "('been', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AB26', 'B', 2)\n",
      "('other', 'O', 2)\n",
      "('than', 'O', 0)\n",
      "('EeuP', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Coxiella', 'O', -100)\n",
      "('burnetii', 'O', 2)\n",
      "('DNA', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('detected', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('12', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cattle', 'O', 2)\n",
      "('meat', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Cholesterol', 'O', -100)\n",
      "('can', 'O', 2)\n",
      "('diffuse', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('membranes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('liposomes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('emulsions', 'O', 2)\n",
      "('through', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('aqueous', 'O', 2)\n",
      "('phase', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Citrate', 'O', -100)\n",
      "('uptake', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('metabolism', 'O', 2)\n",
      "('operon', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('encoded', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('cane', 'O', 2)\n",
      "('juice', 'O', 2)\n",
      "('isolates', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('FDAARGOS_1003', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('LMG', 'B', 0)\n",
      "('11482', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('NCDO', 'B', -100)\n",
      "('768', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('sourdough', 'O', 2)\n",
      "('isolate', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('TR070', 'B', 1)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('unknown', 'O', 0)\n",
      "('isolate', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('LMG', 'B', 1)\n",
      "('11483', 'I', -100)\n",
      "('.', 'O', 2)\n",
      "('E.', 'O', -100)\n",
      "('coli', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('ATCC', 'B', 2)\n",
      "('25922', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('stx1-', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('stx2-', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('eae-', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('saa-', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ehxA-', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('rfbO157-', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('EDL', 'B', 2)\n",
      "('933', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('O157', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('H7', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('stx1+', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('stx2+', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('eae+', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('saa-', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('ehxA+', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('rfbO157+', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('UNCPBA', 'B', 2)\n",
      "('O91', 'I', 2)\n",
      "(':', 'I', 2)\n",
      "('H21', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('stx1+', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('stx2+', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('eae-', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('saa+', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('ehxA+', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('rfbO157-', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('used', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('controls', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('Gossypium', 'O', 2)\n",
      "('hirsutum', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('GbMYB5', 'O', 2)\n",
      "('could', 'O', -100)\n",
      "('increase', 'O', -100)\n",
      "('proline', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('antioxidant', 'O', -100)\n",
      "('enzyme', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('malondialdehyde', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('MDA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('resist', 'O', 2)\n",
      "('drought', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Chen', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2015', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Classical', 'O', -100)\n",
      "('biochemical', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('evidenced', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('AB211', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('positive', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('MR-', 'O', 2)\n",
      "('VP', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('indole', 'O', -100)\n",
      "('production', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('negative', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('catalase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('amylase', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('gelatinase', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Supplementary', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('S6A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('test', 'O', 2)\n",
      "('whether', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Sli', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('efficient', 'O', 2)\n",
      "('pollen', 'O', 2)\n",
      "('SI-breaker', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('RH', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('E172', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('used', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Sli', 'O', 2)\n",
      "('gene', 'O', -100)\n",
      "('donor', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('pollinate', 'O', 2)\n",
      "('125', 'O', 2)\n",
      "('self-incompatible', 'O', 2)\n",
      "('diploid', 'O', -100)\n",
      "('lines', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('belong', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('six', 'O', -100)\n",
      "('groups', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('50', 'O', -100)\n",
      "('S.', 'O', -100)\n",
      "('stenotomum', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('37', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('goniocalyx', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('25', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('phureja11', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('tuberosom', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('S.', 'O', -100)\n",
      "('ajanhuiri', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('S.', 'O', 2)\n",
      "('curtilobum', 'O', -100)\n",
      "('lines', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Fig', 'O', 2)\n",
      "('2b', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Supplementary', 'O', -100)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('pellet', 'O', 2)\n",
      "('morphology', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('physiology', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('rebeccamycin', 'O', 2)\n",
      "('producing', 'O', 2)\n",
      "('filamentous', 'O', -100)\n",
      "('actinomycete', 'O', -100)\n",
      "('Lentzea', 'O', -100)\n",
      "('aerocolonigenes', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('investigated', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('microscopy', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('flow', 'O', -100)\n",
      "('cytometry', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('studied', 'O', 2)\n",
      "('acr3', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('are', 'O', -100)\n",
      "('under', 'O', 2)\n",
      "('negative', 'O', 2)\n",
      "('selection', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('dN', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('dS', 'O', 2)\n",
      "('ratios', 'O', 2)\n",
      "('ranging', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('0.1202', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('0.1403', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('diverging', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('S17', 'B', 2)\n",
      "('sequence', 'O', 2)\n",
      "('which', 'O', -100)\n",
      "('seems', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('be', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('an', 'O', -100)\n",
      "('independent', 'O', 2)\n",
      "('branch', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Preclinical', 'O', -100)\n",
      "('studies', 'O', 2)\n",
      "('supported', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('CT1812', 'O', 2)\n",
      "('impacting', 'O', 2)\n",
      "('Aβ', 'O', -100)\n",
      "('oligomers', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('AD', 'O', 2)\n",
      "('transgenic', 'O', 2)\n",
      "('mice', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('K.', 'B', -100)\n",
      "('koreensis', 'I', 0)\n",
      "('SW-125T', 'I', -100)\n",
      "('is', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('Gram-negative', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('non-motile', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('non-spore-forming', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('tidal', 'O', -100)\n",
      "('flat', 'O', 2)\n",
      "('sediments', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('Daepo', 'O', -100)\n",
      "('Beach', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Yellow', 'O', -100)\n",
      "('Sea', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Korea', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Kytococcus', 'B', -100)\n",
      "('sedentarius', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('541^T', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('DSM20547', 'B', -100)\n",
      "(',', 'O', 1)\n",
      "('was', 'O', -100)\n",
      "('grown', 'O', -100)\n",
      "('in', 'O', 1)\n",
      "('DSMZ', 'O', 1)\n",
      "('medium', 'O', -100)\n",
      "('92', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('3', 'O', 2)\n",
      "('%', 'O', 0)\n",
      "('trypticase', 'O', -100)\n",
      "('soy', 'O', -100)\n",
      "('broth', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('0.3', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('at', 'O', 2)\n",
      "('30°C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('spermidine', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('putrescine-binding', 'O', 2)\n",
      "('periplasmic', 'O', 2)\n",
      "('protein', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('PotD', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('among', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('dairy-niche', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('i.e.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('CAUH35', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('FAM18149', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('JCM8130', 'B', 2)\n",
      "(',', 'O', -100)\n",
      "('KL1', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('N1115', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('TMW1_1434', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Both', 'O', -100)\n",
      "('symbionts', 'O', 2)\n",
      "('exhibited', 'O', 2)\n",
      "('low', 'O', -100)\n",
      "('GC', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('typical', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('obligate', 'O', 2)\n",
      "('symbioses', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('Sulcia-Md', 'B', 2)\n",
      "('had', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('GC', 'O', -100)\n",
      "('content', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('24', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('while', 'O', -100)\n",
      "('Nasuia-Md', 'B', 2)\n",
      "('had', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('GC', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('16', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 0)\n",
      "('1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('OXA', 'O', 2)\n",
      "('-12', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('observed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('tilapia', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('NK02', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('NK09', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('but', 'O', -100)\n",
      "('abundant', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('fish', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('14', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('reported', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('participants', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('BMI', 'O', 2)\n",
      "('≥', 'O', 2)\n",
      "('30', 'O', 2)\n",
      "('kg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('m^2', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('significantly', 'O', -100)\n",
      "('higher', 'O', -100)\n",
      "('risk', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('functional', 'O', 2)\n",
      "('impairment', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('SPPB', 'O', 2)\n",
      "('≤', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "('points', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('OR', 'O', 2)\n",
      "('4.47', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('95', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('CI', 'O', 2)\n",
      "('1.30-15.33', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('p', 'O', 2)\n",
      "('=', 'O', 2)\n",
      "('0.017', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('falls', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('TUG', 'O', -100)\n",
      "('<', 'O', -100)\n",
      "('1', 'O', -100)\n",
      "('point', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('OR', 'O', 2)\n",
      "('2.58', 'O', 2)\n",
      "(';', 'O', -100)\n",
      "('95', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('CI', 'O', 2)\n",
      "('1.15-5.79', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('p', 'O', 2)\n",
      "('=', 'O', -100)\n",
      "('0.022', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('probability', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('achieve', 'O', -100)\n",
      "('poor', 'O', -100)\n",
      "('performance', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('MMSE', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('≤', 'O', -100)\n",
      "('23', 'O', -100)\n",
      "('points', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('OR', 'O', -100)\n",
      "('0.50', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('95', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('CI', 'O', -100)\n",
      "('0.28-0.87', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('p', 'O', 2)\n",
      "('=', 'O', 2)\n",
      "('0.015', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('further', 'O', 2)\n",
      "('explore', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('RLs', 'O', 2)\n",
      "('synthesis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('RSCV', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('KT1115', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('comparison', 'O', 0)\n",
      "('experiments', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('conducted', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('KT1115', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('in', 'O', 2)\n",
      "('1-L', 'O', 0)\n",
      "('shake', 'O', -100)\n",
      "('flasks', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('RON', 'O', 0)\n",
      "('medium', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('After', 'O', -100)\n",
      "('48', 'O', 2)\n",
      "('h', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('animals', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('groups', 'O', 2)\n",
      "('I', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('IV', 'O', 2)\n",
      "('received', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('μL', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('containing', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('^4', 'O', 2)\n",
      "('colony', 'O', 2)\n",
      "('forming', 'O', -100)\n",
      "('unit', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CFU', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('spores', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RT027', 'B', 2)\n",
      "('/', 'O', 2)\n",
      "('CD196', 'B', 0)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('RT883', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('RT884', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('RT885', 'B', 0)\n",
      "('strains', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('esophageal', 'O', 2)\n",
      "('gavage', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Briefly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('ulmoides', 'O', 2)\n",
      "('Oliver', 'O', -100)\n",
      "('cortex', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('100', 'O', -100)\n",
      "('g', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('treated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('80', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('ethanol', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('boiled', 'O', 2)\n",
      "('twice', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('reflux', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('h', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Specifically', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('PA14', 'B', 2)\n",
      "('grown', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('BM2', 'O', 2)\n",
      "('0.4', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('fructose', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('carbon', 'O', -100)\n",
      "('source', 'O', 2)\n",
      "('yielded', 'O', 2)\n",
      "('robust', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('numerous', 'O', 2)\n",
      "('large', 'O', 2)\n",
      "('mushroom', 'O', 2)\n",
      "('shaped', 'O', 2)\n",
      "('biofilms', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('5B', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('PCC7120', 'B', -100)\n",
      "('under', 'O', 0)\n",
      "('UV-B', 'O', -100)\n",
      "('stress', 'O', -100)\n",
      "('might', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('involved', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('nullifying', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('damaging', 'O', 2)\n",
      "('effect', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('UV', 'O', 2)\n",
      "('radiation', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('combined', 'O', 2)\n",
      "('ATP-independent', 'O', 2)\n",
      "('repair', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('PfpI', 'O', 2)\n",
      "('protease', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('ferritin', 'O', -100)\n",
      "('domain', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Shrivastava', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2015', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('CT3', 'B', -100)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('activated', 'O', 2)\n",
      "('sludge', 'O', 2)\n",
      "('treatment', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('1994', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Bari', 'O', 2)\n",
      "('West', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Italy', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('41°06′41″', 'O', 2)\n",
      "('N', 'O', 2)\n",
      "('16°51′19″', 'O', 2)\n",
      "('E', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('30', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Deinococcus', 'B', 2)\n",
      "('radiodurans', 'I', 0)\n",
      "('R1', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('tsaB', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('tsaD', 'O', 1)\n",
      "('are', 'O', -100)\n",
      "('non-essential', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('in', 'O', 2)\n",
      "('Synechocystis', 'O', 2)\n",
      "('sp', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('On', 'O', -100)\n",
      "('comparing', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('baumannii', 'O', 2)\n",
      "('DR1', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('A.', 'O', -100)\n",
      "('baumannii', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('DR2', 'B', 2)\n",
      "('and', 'O', -100)\n",
      "('AB067', 'B', 2)\n",
      "('strains', 'O', 2)\n",
      "('displayed', 'O', 0)\n",
      "('gained', 'O', -100)\n",
      "('phenotypic', 'O', 2)\n",
      "('resistance', 'O', 0)\n",
      "('to', 'O', -100)\n",
      "('compounds', 'O', -100)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('polymyxin', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('doxycycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('chlortetracycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('minocycline', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('rolitetracycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('cloxacillin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('nafcillin', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('oxacillin', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('phenethicillin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('erythromycin', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('D', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('L', 'O', 2)\n",
      "('-serine', 'O', 2)\n",
      "('hydroxamate', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', -100)\n",
      "('5C', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Here', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('describe', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('new', 'O', 2)\n",
      "('methanotrophic', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('that', 'O', -100)\n",
      "('shares', 'O', 2)\n",
      "('phenotypic', 'O', 2)\n",
      "('characteristics', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('phylogenetic', 'O', 2)\n",
      "('relatedness', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('marine', 'O', 2)\n",
      "('methanotroph', 'O', 2)\n",
      "('Methylomarinum', 'O', 2)\n",
      "('vadi', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('These', 'O', -100)\n",
      "('studies', 'O', 2)\n",
      "('demonstrate', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('tropical', 'O', 2)\n",
      "('ecosystems', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('droughts', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('not', 'O', -100)\n",
      "('well', 'O', 2)\n",
      "('understood', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('varies', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('regions', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('28', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('We', 'O', -100)\n",
      "('hypothesized', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('Orientalis', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('MG05-1020', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('IP275', 'B', 0)\n",
      "('and', 'O', -100)\n",
      "('FV-1', 'B', -100)\n",
      "('came', 'O', -100)\n",
      "('into', 'O', -100)\n",
      "('being', 'O', 2)\n",
      "('by', 'O', 0)\n",
      "('losing', 'O', -100)\n",
      "('certain', 'O', -100)\n",
      "('DFRs', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('genomovar09', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('found', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('altered', 'O', 2)\n",
      "('colony', 'O', 2)\n",
      "('morphology', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('did', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('detect', 'O', 2)\n",
      "('any', 'O', 2)\n",
      "('novel', 'O', 2)\n",
      "('apolar', 'O', 2)\n",
      "('lipids', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('our', 'O', -100)\n",
      "('analysis', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('M.', 'O', 2)\n",
      "('smegmatis', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('lsr2', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "(':Tn5370', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('NJS20.1', 'B', 2)\n",
      "('or', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('Δ', 'O', 2)\n",
      "('lsr2', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('NJS22', 'B', -100)\n",
      "('.', 'O', 2)\n",
      "('Antibacterial', 'O', -100)\n",
      "('activity', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('investigated', 'O', 2)\n",
      "('through', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('application', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('five', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('10^6-10^7', 'O', 2)\n",
      "('cfu', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('mL', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(':', 'O', -100)\n",
      "('Escherichia', 'B', -100)\n",
      "('coli', 'I', -100)\n",
      "('CCM', 'I', 2)\n",
      "('4517', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('Staphylococcus', 'B', 2)\n",
      "('aureus', 'I', 2)\n",
      "('CCM', 'I', 0)\n",
      "('4516', 'I', 1)\n",
      "(',', 'O', 1)\n",
      "('Klebsiella', 'B', -100)\n",
      "('pneumoniae', 'I', 1)\n",
      "('CCM', 'I', -100)\n",
      "('4415', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('Pseudomonas', 'B', 1)\n",
      "('aeruginosa', 'I', 1)\n",
      "('CM', 'I', -100)\n",
      "('1961', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Enterococcus', 'B', 0)\n",
      "('faecalis', 'I', 1)\n",
      "('CCM', 'I', 1)\n",
      "('3956', 'I', -100)\n",
      "('.', 'O', 1)\n",
      "('Our', 'O', -100)\n",
      "('study', 'O', 2)\n",
      "('shows', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('EPR9N', 'B', 2)\n",
      "('contains', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('variety', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('toxin', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('toxin-antitoxin', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('are', 'O', -100)\n",
      "('also', 'O', -100)\n",
      "('found', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('thermophilus', 'O', 2)\n",
      "('symbiont', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('M.', 'B', -100)\n",
      "('ciceri', 'I', 0)\n",
      "('strain', 'I', -100)\n",
      "('WSM1271^T', 'I', 1)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('motile', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('Gram-negative', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('non-spore-forming', 'O', -100)\n",
      "('rod', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('2', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('order', 'O', -100)\n",
      "('Rhizobiales', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('class', 'O', 2)\n",
      "('Alphaproteobacteria', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Interestingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('inactivation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('piscibactin', 'O', 2)\n",
      "('synthesis', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('V.', 'B', -100)\n",
      "('anguillarum', 'I', -100)\n",
      "('RV22', 'I', -100)\n",
      "('produced', 'O', 2)\n",
      "('just', 'O', 2)\n",
      "('a', 'O', 0)\n",
      "('small', 'O', -100)\n",
      "('decrease', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('growth', 'O', -100)\n",
      "('ability', 'O', 1)\n",
      "('under', 'O', -100)\n",
      "('iron', 'O', 2)\n",
      "('restriction', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('inoculation', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('carried', 'O', 2)\n",
      "('out', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('adding', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('equal', 'O', 2)\n",
      "('part', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('pelleted', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('glass', 'O', 2)\n",
      "('Erlenmeyer’s', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('mL', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('liquid', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('BG11_0', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Z8', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Kotai', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1972', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(']', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('b', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('solid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Z8', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('BG11_0', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('plates', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('allowed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('grow', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('19°C', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('under', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('12:12', 'O', 2)\n",
      "('h', 'O', -100)\n",
      "('light', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('20-30', 'O', 2)\n",
      "('μmol', 'O', 2)\n",
      "('m^−2', 'O', 2)\n",
      "('s^−1', 'O', 2)\n",
      "('photon', 'O', 2)\n",
      "('irradiance', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('dark', 'O', 2)\n",
      "('cycle', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('absence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('OmpC', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 2)\n",
      "('B', 'I', 0)\n",
      "(',', 'O', -100)\n",
      "('T4', 'O', 1)\n",
      "('attaches', 'O', 1)\n",
      "('to', 'O', 2)\n",
      "('LPS', 'O', 2)\n",
      "('chains', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('exposed', 'O', 2)\n",
      "('terminal', 'O', 2)\n",
      "('glucose', 'O', 2)\n",
      "('residue', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Overnight-grown', 'O', -100)\n",
      "('V.', 'B', 2)\n",
      "('cholerae', 'I', -100)\n",
      "('N16961', 'I', -100)\n",
      "(',', 'O', 0)\n",
      "('Δ', 'O', -100)\n",
      "('manP', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('Δ', 'O', -100)\n",
      "('manR', 'O', 2)\n",
      "('cultures', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('diluted', 'O', -100)\n",
      "('100-fold', 'O', 2)\n",
      "('into', 'O', 2)\n",
      "('fresh', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "('medium', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('cultured', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37', 'O', 2)\n",
      "('°C', 'O', -100)\n",
      "('until', 'O', -100)\n",
      "('OD', 'O', 2)\n",
      "('_600', 'O', 2)\n",
      "('reached', 'O', 2)\n",
      "('0.5', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('order', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('address', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('question', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('how', 'O', 2)\n",
      "('Kp36-117R', 'B', 2)\n",
      "('becomes', 'O', 0)\n",
      "('resistant', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('phage', 'O', -100)\n",
      "('117', 'O', -100)\n",
      "('mechanistically', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('quantified', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('rate', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('adsorption', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('phage', 'O', 2)\n",
      "('117', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('Kp36', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('Kp36-117R', 'B', 2)\n",
      "('to', 'O', 2)\n",
      "('investigate', 'O', 2)\n",
      "('whether', 'O', 0)\n",
      "('strain', 'O', -100)\n",
      "('K36-117R', 'B', 2)\n",
      "('resistance', 'O', 0)\n",
      "('could', 'O', -100)\n",
      "('be', 'O', -100)\n",
      "('explained', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('preventing', 'O', 2)\n",
      "('phage', 'O', 2)\n",
      "('adsorption', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('L.', 'O', -100)\n",
      "('kefiranofaciens', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('M1', 'B', 2)\n",
      "('and', 'O', -100)\n",
      "('HL1', 'B', -100)\n",
      "('were', 'O', -100)\n",
      "('previously', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('kefir', 'O', 2)\n",
      "('grains', 'O', 0)\n",
      "('[', 'O', -100)\n",
      "('6', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('reversed-phase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('RP', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('-HPLC', 'O', 2)\n",
      "('analyses', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('SRS89', 'B', 2)\n",
      "('of', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('vagans', 'O', 2)\n",
      "('contains', 'O', 0)\n",
      "('Reb', 'O', -100)\n",
      "('A', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('demonstrated', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('time', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('interaction', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('HTLV-1', 'O', 2)\n",
      "('Tax', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('histone', 'O', 2)\n",
      "('methyltransferase', 'O', 2)\n",
      "('SUV39H1', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('recent', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('excellent', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('inter-rhizosphere', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('B.', 'B', -100)\n",
      "('aryabhattai', 'I', 2)\n",
      "('LAD', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('maize', 'O', -100)\n",
      "('rhizosphere', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('impact', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('LAD', 'B', 2)\n",
      "('on', 'O', 2)\n",
      "('rhizosphere', 'O', 2)\n",
      "('microbial', 'O', 2)\n",
      "('structure', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('explored', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('previous', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Deng', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2022', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Since', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('KA5-B', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('exhibited', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('ability', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('degrade', 'O', -100)\n",
      "('not', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('various', 'O', 2)\n",
      "('polysaccharides', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('degrading', 'O', 2)\n",
      "('abilities', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('among', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Monacolin', 'O', -100)\n",
      "('J', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('L', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('characterized', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('cultures', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('M.', 'B', 2)\n",
      "('ruber', 'I', 2)\n",
      "('strain3426', 'I', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('pathway', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('dTDP-beta-L-rhamnose', 'O', 2)\n",
      "('biosynthesis', 'O', 2)\n",
      "('occurred', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('P.', 'O', -100)\n",
      "('distincta', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('U2A', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('SM1926', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('AC163', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('Erwinia', 'O', -100)\n",
      "('amylovora', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('plant', 'O', 2)\n",
      "('pathogen', 'O', -100)\n",
      "('that', 'O', -100)\n",
      "('causes', 'O', 2)\n",
      "('fire', 'O', 2)\n",
      "('blight', 'O', 2)\n",
      "('disease', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Rosaceous', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('pear', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('apple', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('This', 'O', -100)\n",
      "('GII.17', 'B', 2)\n",
      "('strain', 'O', 0)\n",
      "('also', 'O', -100)\n",
      "('caused', 'O', -100)\n",
      "('many', 'O', 2)\n",
      "('outbreaks', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Japan', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('sporadically', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('parts', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('world', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('3,4', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Two', 'O', -100)\n",
      "('main', 'O', 2)\n",
      "('pathways', 'O', 2)\n",
      "('involving', 'O', 2)\n",
      "('several', 'O', 2)\n",
      "('enzymes', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('responsible', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('C-1', 'O', 2)\n",
      "('transfer', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('AM1', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('relying', 'O', 0)\n",
      "('on', 'O', -100)\n",
      "('H', 'O', 2)\n",
      "('_4', 'O', 2)\n",
      "('F', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('dH', 'O', 2)\n",
      "('_4', 'O', 2)\n",
      "('MPT', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C-1', 'O', 2)\n",
      "('carrier', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Vorholt', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2002', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('MDT1-3', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('RHLT2-21', 'B', -100)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Sr47', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('contained', 'O', -100)\n",
      "('MK-11', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('MK-12', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('primary', 'O', 2)\n",
      "('menaquinones', 'O', 2)\n",
      "('with', 'O', 0)\n",
      "('minor', 'O', -100)\n",
      "('amounts', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('MK-10', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('simulated', 'O', 2)\n",
      "('sunlight', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('illuminance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('56', 'O', -100)\n",
      "('klx', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('moderate', 'O', 2)\n",
      "('impact', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('interfering', 'O', 2)\n",
      "('substances', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('survival', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('SARS-CoV-2', 'B', 2)\n",
      "('dried', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('stainless', 'O', 2)\n",
      "('steel', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('fact', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('pair', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('hydrophila', 'O', 2)\n",
      "('subsp.', 'O', -100)\n",
      "('dhakensis-A.', 'O', 2)\n",
      "('aquariorum', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', -100)\n",
      "('shared', 'O', 2)\n",
      "('an', 'O', -100)\n",
      "('identical', 'O', -100)\n",
      "('phenotype', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('separated', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('our', 'O', -100)\n",
      "('study', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('hydrophila', 'O', 2)\n",
      "('subsp.', 'O', 2)\n",
      "('hydrophila', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('inability', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('produce', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('l-arabinose', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('l-fucose', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('its', 'O', -100)\n",
      "('use', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('urocanic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('further', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('ability', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('hydrolyze', 'O', -100)\n",
      "('sodium', 'O', -100)\n",
      "('dodecyl', 'O', 2)\n",
      "('sulfate', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('SDS', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('i.e.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('alkyl', 'O', 2)\n",
      "('sulfatase', 'O', 2)\n",
      "('activity', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('pan', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('core', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('difficile', 'O', 2)\n",
      "('comprising', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('RT883', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('RT884', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('RT885', 'B', 2)\n",
      "('and', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('eight', 'O', -100)\n",
      "('reference', 'O', 2)\n",
      "('strains', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('obtained', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('Roary', 'O', 2)\n",
      "('v3.13.0', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('45', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('results', 'O', 2)\n",
      "('show', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('auto-aggregation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('KT1115', 'B', -100)\n",
      "('cells', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('visible', 'O', 0)\n",
      "('at', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('macroscopic', 'O', 2)\n",
      "('level', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('PAO1', 'B', 2)\n",
      "('did', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('show', 'O', 2)\n",
      "('any', 'O', 2)\n",
      "('auto-aggregation', 'O', 0)\n",
      "('under', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figures', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('C', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('S1', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('methanol', 'O', 2)\n",
      "('fractions', 'O', 2)\n",
      "('132-F30', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('132-F50', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('132-F70', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('132-F100', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('inhibited', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Escherichia', 'O', -100)\n",
      "('coli', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('ATCC', 'B', 2)\n",
      "('25922', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Staphylococcus', 'O', -100)\n",
      "('aureus', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('MRSA', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vitro', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('most', 'O', 0)\n",
      "('effectively', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Further', 'O', -100)\n",
      "('validation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CN-BOT', 'O', 2)\n",
      "('tag-switch', 'O', 2)\n",
      "('assay', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('experiments', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CSE', 'O', 2)\n",
      "('inhibitor', 'O', 2)\n",
      "('propargylglycine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('PG', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('25', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('CBS', 'O', -100)\n",
      "('inhibitor', 'O', -100)\n",
      "('aminooxyacetic', 'O', -100)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AOAA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('used.35', 'O', 2)\n",
      "('Both', 'O', 2)\n",
      "('inhibitors', 'O', 2)\n",
      "('decreased', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('signal', 'O', 2)\n",
      "('intensity', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('bovine', 'O', -100)\n",
      "('arterial', 'O', 2)\n",
      "('endothelial', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BAE', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('SH-SY5Y', 'B', 2)\n",
      "('neuroblastoma', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('albeit', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('completely', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', -100)\n",
      "('3C', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('S8C', 'O', 0)\n",
      "('in', 'O', -100)\n",
      "('ESI†', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('incubation', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Na_2S', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('d-cysteine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('alternative', 'O', 2)\n",
      "('substrate', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('MST-catalyzed', 'O', 2)\n",
      "('H_2S', 'O', -100)\n",
      "('generation', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('36', 'O', 2)\n",
      "('increased', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('fluorescence', 'O', 2)\n",
      "('signals', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Among', 'O', -100)\n",
      "('cellulolytic', 'O', 2)\n",
      "('enzymes', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('only', 'O', 2)\n",
      "('GH3', 'O', 2)\n",
      "('β-1,4-glucosidase', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('abundantly', 'O', -100)\n",
      "('produced', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('fungal', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('under', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('induction', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Nevertheless', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cognate', 'O', 2)\n",
      "('substrate', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('family', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('enzymes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('carboxymuconolactone', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('is', 'O', -100)\n",
      "('not', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('hemiacetal', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('thus', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('involvement', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('metaldehyde', 'O', 2)\n",
      "('degradation', 'O', 2)\n",
      "('pathway', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Sphingobium', 'B', 2)\n",
      "('CMET-H', 'I', 2)\n",
      "('is', 'O', 2)\n",
      "('speculative', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('would', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('corroborated', 'O', 0)\n",
      "('experimentally', 'O', -100)\n",
      "('.', 'O', 1)\n",
      "('In', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('P.', 'B', 2)\n",
      "('agarivorans', 'I', 0)\n",
      "('Hao', 'I', -100)\n",
      "('2018', 'I', 1)\n",
      "('strain', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('high-yield', 'O', 1)\n",
      "('producer', 'O', -100)\n",
      "('of', 'O', 1)\n",
      "('EPS', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('high', 'O', 2)\n",
      "('biological', 'O', -100)\n",
      "('activity', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('terms', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('anti-oxidation', 'O', 2)\n",
      "('properties', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('moisture', 'O', 2)\n",
      "('absorption', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('moisturization', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('adsorption', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('heavy', 'O', -100)\n",
      "('metal', 'O', 2)\n",
      "('ions', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vitro', 'O', 2)\n",
      "('flocculation', 'O', -100)\n",
      "('[', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('14', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Approximately', 'O', -100)\n",
      "('74', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('encoded', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('CENA596', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('70', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('CCY9414', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('71', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('UHCC', 'B', -100)\n",
      "('0039', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('67', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('NIES-3585', 'B', 0)\n",
      "('were', 'O', -100)\n",
      "('orthologs', 'O', 1)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('3122', 'O', 2)\n",
      "('proteins', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', -100)\n",
      "('3', 'O', -100)\n",
      "('A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Vitamin', 'O', -100)\n",
      "('D', 'O', 2)\n",
      "('deficiency', 'O', 2)\n",
      "('results', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('abnormalities', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('calcium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('phosphorus', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('bone', 'O', 2)\n",
      "('metabolism', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('order', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('obtain', 'O', 2)\n",
      "('highly', 'O', 2)\n",
      "('purified', 'O', 2)\n",
      "('LPS', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('use', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('endotoxin', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('mallei', 'O', 2)\n",
      "('GM3773', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('CPS', 'O', -100)\n",
      "('mutant', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('well', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('ATCC', 'B', 2)\n",
      "('23344', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('wild-type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "('using', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('modified', 'O', 2)\n",
      "('enzyme', 'O', 2)\n",
      "('hot', 'O', -100)\n",
      "('aqueous-phenol', 'O', -100)\n",
      "('procedure', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Lastly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('n', 'O', 2)\n",
      "('-caproate', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('electron', 'O', -100)\n",
      "('acceptor', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('initial', 'O', 2)\n",
      "('total', 'O', 2)\n",
      "('concentration', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('102.4', 'O', 2)\n",
      "('±', 'O', 2)\n",
      "('0.5', 'O', 2)\n",
      "('mmol', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "('L', 'O', -100)\n",
      "('^-1', 'O', -100)\n",
      "('resulted', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('an', 'O', -100)\n",
      "('undissociated', 'O', -100)\n",
      "('n', 'O', 2)\n",
      "('-caproic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('concentration', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('∼19.8', 'O', -100)\n",
      "('mmol', 'O', 2)\n",
      "('C', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('^-1', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('∼3.3', 'O', -100)\n",
      "('mM', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('pH', 'O', -100)\n",
      "('value', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('5.5', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('completely', 'O', -100)\n",
      "('inhibited', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('metabolism', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'B', 2)\n",
      "('7D4C2', 'I', 2)\n",
      "('(', 'O', -100)\n",
      "('Figures', 'O', -100)\n",
      "('3A', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('G', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Supplementary', 'O', -100)\n",
      "('Figure', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Supplementary', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Qualitative', 'O', -100)\n",
      "('differences', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('iso-C', 'O', 2)\n",
      "('_15:0', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('iso-C', 'O', -100)\n",
      "('_17:0', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('C', 'O', -100)\n",
      "('_20:0', 'O', -100)\n",
      "('clearly', 'O', 2)\n",
      "('distinguish', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('CCM', 'B', -100)\n",
      "('9025', 'I', 2)\n",
      "('^T', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('S.', 'O', -100)\n",
      "('chromogenes', 'O', 2)\n",
      "('CCM', 'B', 2)\n",
      "('3387', 'I', 2)\n",
      "('^T', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('lower', 'O', 2)\n",
      "('amounts', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('anteiso-C', 'O', 0)\n",
      "('_17:0', 'O', -100)\n",
      "('are', 'O', 1)\n",
      "('specific', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('S.', 'B', -100)\n",
      "('agnetis', 'I', 2)\n",
      "('CCM', 'I', 2)\n",
      "('8869', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('global', 'O', 2)\n",
      "('basal', 'O', 2)\n",
      "('suppression', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('ISGs', 'O', 2)\n",
      "('indicated', 'O', 2)\n",
      "('possible', 'O', -100)\n",
      "('dysregulation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('T1IFN', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('autocrine', 'O', -100)\n",
      "('signaling', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Furthermore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cell-free', 'O', 2)\n",
      "('supernatants', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('RV22', 'B', -100)\n",
      "('cultures', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('iron', 'O', 0)\n",
      "('restriction', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('addition', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('GaBr', 'O', 2)\n",
      "('_3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('order', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('stabilize', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('siderophore', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('corresponding', 'O', 2)\n",
      "('complex', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('detected', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('characteristic', 'O', 2)\n",
      "('piscibactin-Ga', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('III', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('complex', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('t', 'O', 2)\n",
      "('_R', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('9.56', 'O', -100)\n",
      "('min', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Figure', 'O', 2)\n",
      "('2D', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Spectroscopic', 'O', -100)\n",
      "('data', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('compound', 'O', 2)\n",
      "('8', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('quite', 'O', 2)\n",
      "('similar', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('those', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('methyl', 'O', 2)\n",
      "('ester', 'O', 2)\n",
      "('derivative', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('resin', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Acacia', 'O', 2)\n",
      "('sp', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('absorbance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('measured', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('microplate', 'O', 2)\n",
      "('reader', 'O', 2)\n",
      "('at', 'O', -100)\n",
      "('490', 'O', -100)\n",
      "('nm', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('any', 'O', 2)\n",
      "('case', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Maribacter', 'O', 2)\n",
      "('sp.', 'O', 2)\n",
      "('induces', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('specific', 'O', -100)\n",
      "('rhizoid', 'O', 2)\n",
      "('formation', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('both', 'O', 2)\n",
      "('morphotypes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('‘slender’', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('wild', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('according', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('intrinsic', 'O', -100)\n",
      "('program', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('demonstrated', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('experiments', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Spoerner', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2012', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('Vesty', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2015', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Bordetella', 'O', -100)\n",
      "('spp.', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Achromobacter', 'O', -100)\n",
      "('spp', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('S.', 'B', -100)\n",
      "('mutans', 'I', 0)\n",
      "('UA159', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('ATCC', 'B', 1)\n",
      "('700610', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('cultured', 'O', 0)\n",
      "('in', 'O', 1)\n",
      "('Todd-Hewitt', 'O', -100)\n",
      "('broth', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('63', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('0.3', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('yeast', 'O', -100)\n",
      "('extract', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('THY', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('liquid', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('agar', 'O', 2)\n",
      "('plates', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Interestingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('117', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('these', 'O', 2)\n",
      "('132', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('contribute', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('survival', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('tomato', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('are', 'O', 2)\n",
      "('located', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('chromosome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('R.', 'O', 2)\n",
      "('solanacearum', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('GMI1000', 'B', 2)\n",
      "('.', 'O', -100)\n",
      "('V.', 'B', -100)\n",
      "('fujianensis', 'I', 0)\n",
      "('FJ201301', 'I', -100)\n",
      "('^T', 'O', 1)\n",
      "('carried', 'O', -100)\n",
      "('17', 'O', -100)\n",
      "('kinds', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('potential', 'O', 1)\n",
      "('virulence', 'O', -100)\n",
      "('factors', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('106', 'O', -100)\n",
      "('related', 'O', 2)\n",
      "('genes', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('From', 'O', -100)\n",
      "('both', 'O', 2)\n",
      "('fittings', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('closest', 'O', 2)\n",
      "('distance', 'O', 2)\n",
      "('between', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('ACE', 'O', 2)\n",
      "('spacecraft', 'O', 2)\n",
      "('path', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('rope', 'O', 2)\n",
      "('axis', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('about', 'O', 2)\n",
      "('0.3R', 'O', -100)\n",
      "('_c', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('where', 'O', 2)\n",
      "('R', 'O', 2)\n",
      "('_c', 'O', -100)\n",
      "('~', 'O', -100)\n",
      "('0.2', 'O', 2)\n",
      "('AU', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('radius', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('MFR’s', 'O', 2)\n",
      "('cross', 'O', 2)\n",
      "('section', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Draft', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('cold-adapted', 'O', 2)\n",
      "('phosphorous-solubilizing', 'O', 2)\n",
      "('Pseudomonas', 'B', -100)\n",
      "('koreensis', 'I', -100)\n",
      "('P2', 'I', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Sela', 'O', -100)\n",
      "('Lake', 'O', -100)\n",
      "(',', 'O', 0)\n",
      "('India', 'O', 1)\n",
      "('All', 'O', -100)\n",
      "('fecal', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('underwent', 'O', 2)\n",
      "('nutrient', 'O', 2)\n",
      "('enrichment', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('aerobic', 'O', 2)\n",
      "('culture', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('tetrathionate', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('9:1', 'O', -100)\n",
      "('broth', 'O', -100)\n",
      "('volume', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('fecal', 'O', 2)\n",
      "('mass', 'O', -100)\n",
      "(';', 'O', -100)\n",
      "('Difco', 'O', 2)\n",
      "('Laboratories', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Sparks', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('MD', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('United', 'O', -100)\n",
      "('States', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Rappaport-Vassiliadis', 'O', -100)\n",
      "('R10', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Difco', 'O', 2)\n",
      "('Laboratories', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('streaked', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('isolation', 'O', -100)\n",
      "('on', 'O', -100)\n",
      "('xylose-lysine-tergitol', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('XLT-4', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('agar', 'O', -100)\n",
      "('plates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BD', 'O', 2)\n",
      "('Diagnostic', 'O', -100)\n",
      "('Systems', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Sparks', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('MD', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('United', 'O', 2)\n",
      "('States', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Lactobacillus', 'B', -100)\n",
      "('acidophilus', 'I', 0)\n",
      "('NCFM', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('Lacticaseibacillus', 'B', 1)\n",
      "('rhamnosus', 'I', -100)\n",
      "('GG', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('Lactobacillus', 'B', -100)\n",
      "('crispatus', 'I', -100)\n",
      "('NCK1351', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Lactobacillus', 'B', 1)\n",
      "('gasseri', 'I', -100)\n",
      "('NCK1342', 'I', 1)\n",
      "('were', 'O', 2)\n",
      "('inoculated', 'O', 0)\n",
      "('from', 'O', 1)\n",
      "('−80°C', 'O', -100)\n",
      "('glycerol', 'O', -100)\n",
      "('stocks', 'O', -100)\n",
      "('into', 'O', 1)\n",
      "('de', 'O', -100)\n",
      "('Man-Rogosa-Sharpe', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('MRS', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('broth', 'O', 0)\n",
      "('(', 'O', 1)\n",
      "('Difco', 'O', -100)\n",
      "('Laboratories', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('Inc.', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('Detroit', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('MI', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('grown', 'O', -100)\n",
      "('overnight', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('water', 'O', 2)\n",
      "('bath', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Among', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('polysaccharides', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('A.', 'O', 2)\n",
      "('baumannii', 'O', 2)\n",
      "('biofilm', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('poly-β-', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('1-6', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('-', 'O', -100)\n",
      "('N', 'O', -100)\n",
      "('-acetylglucosamine', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('considered', 'O', -100)\n",
      "('an', 'O', -100)\n",
      "('important', 'O', 2)\n",
      "('factor', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('protects', 'O', 2)\n",
      "('bacteria', 'O', -100)\n",
      "('against', 'O', 2)\n",
      "('host', 'O', 2)\n",
      "('immune', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('biofilm', 'O', 2)\n",
      "('formation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('entire', 'O', 2)\n",
      "('locus', 'O', 2)\n",
      "('pgaABCD', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('NODE_2_length', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('NOXP01000002', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('NODE_15_length', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('NOXO01000015', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('scaffold', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('LZOC01000003', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('A.', 'O', 2)\n",
      "('baumannii', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('DR1', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('DR2', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('AB067', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('displayed', 'O', 2)\n",
      "('>', 'O', -100)\n",
      "('98.64', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('identity', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('homologs', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('A.', 'O', -100)\n",
      "('baumannii', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Test', 'O', -100)\n",
      "('microbial', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Staphylococcus', 'O', 2)\n",
      "('aureus', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ATCC', 'B', 2)\n",
      "('13565', 'I', 0)\n",
      "(',', 'O', 1)\n",
      "('enterotoxin', 'O', -100)\n",
      "('A', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Escherichia', 'B', 2)\n",
      "('coli', 'I', 2)\n",
      "('O157', 'I', 0)\n",
      "(':', 'I', 1)\n",
      "('H7', 'I', 1)\n",
      "('(', 'O', 1)\n",
      "('ATCC', 'B', 1)\n",
      "('11234', 'I', 2)\n",
      "(')', 'O', 0)\n",
      "(',', 'O', 1)\n",
      "('Candida', 'O', -100)\n",
      "('albicans', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ATCC', 'B', 2)\n",
      "('18804', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', 1)\n",
      "('Aspergillus', 'O', -100)\n",
      "('flavus', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('ATCC', 'B', 2)\n",
      "('22546', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('obtained', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('the', 'O', 1)\n",
      "('Korean', 'O', -100)\n",
      "('Culture', 'O', 2)\n",
      "('Center', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('Microorganisms', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('KCCM', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Seoul', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Korea', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('p441', 'O', 2)\n",
      "('plasmid', 'O', 2)\n",
      "('backbone', 'O', -100)\n",
      "('also', 'O', 2)\n",
      "('contains', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "('unique', 'O', 2)\n",
      "('3′', 'O', 2)\n",
      "('restriction', 'O', 2)\n",
      "('enzyme', 'O', 2)\n",
      "('sites', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('linearization', 'O', 2)\n",
      "('prior', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('cRNA', 'O', 2)\n",
      "('synthesis', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Escherichia', 'O', -100)\n",
      "('coli', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('more', 'O', 2)\n",
      "('genetically', 'O', 2)\n",
      "('diverse', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('S.', 'O', 2)\n",
      "('aureus', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('unsurprisingly', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', -100)\n",
      "('ST131', 'I', -100)\n",
      "('has', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('larger', 'O', 0)\n",
      "('pan-genome', 'O', -100)\n",
      "('than', 'O', 1)\n",
      "('S.', 'B', 1)\n",
      "('aureus', 'I', -100)\n",
      "('ST22', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "('806', 'O', 2)\n",
      "('genes', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('16', 'O', 2)\n",
      "('429', 'O', 0)\n",
      "('IGRs', 'O', -100)\n",
      "('(', 'O', 1)\n",
      "('Fig', 'O', 1)\n",
      "('2c', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Residues', 'O', -100)\n",
      "('25-139', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('PAS', 'O', -100)\n",
      "('domains', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('K.', 'O', 2)\n",
      "('pneumoniae', 'O', 2)\n",
      "('NifL', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AAB26403.1', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('residues', 'O', -100)\n",
      "('8-129', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('PAS', 'O', -100)\n",
      "('domain', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('Aer', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AYG17963.1', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('residues', 'O', 2)\n",
      "('85-329', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Methylococcus', 'O', 2)\n",
      "('capsulatus', 'O', -100)\n",
      "('MmoS', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('AAP80772.1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('aligned', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('residues', 'O', -100)\n",
      "('9-121', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('PAS', 'O', 2)\n",
      "('domain', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('A.', 'O', -100)\n",
      "('brasilense', 'O', -100)\n",
      "('Aer', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('COBALT', 'O', -100)\n",
      "('multiple', 'O', 2)\n",
      "('sequence', 'O', -100)\n",
      "('alignment', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('NCBI', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Papadopoulos', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Agarwala', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('2007', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('1.5', 'O', -100)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('l', 'O', -100)\n",
      "('NaNO_3', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('normal', 'O', 2)\n",
      "('BG11', 'O', 2)\n",
      "('medium', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('reduced', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('0.5', 'O', 2)\n",
      "('g', 'O', -100)\n",
      "('/', 'O', 2)\n",
      "('l', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('prepare', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('nitrogen-deficient', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('MolDiscovery', 'O', -100)\n",
      "('detected', 'O', 2)\n",
      "('lipopeptin', 'O', -100)\n",
      "('A-C', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('multiple', 'O', 2)\n",
      "('Streptomyces', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('including', 'O', -100)\n",
      "('S.', 'B', 2)\n",
      "('hygroscopicus', 'I', -100)\n",
      "('NRRL', 'I', -100)\n",
      "('B-1477', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('S.', 'B', 2)\n",
      "('rimosus', 'I', 2)\n",
      "('NRRL', 'I', 2)\n",
      "('WC3874', 'I', 2)\n",
      "(',', 'O', 0)\n",
      "('S.', 'B', -100)\n",
      "('rimosus', 'I', 1)\n",
      "('NRRL', 'I', -100)\n",
      "('WC3904', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('S.', 'B', -100)\n",
      "('rimosus', 'I', -100)\n",
      "('NRRL', 'I', 1)\n",
      "('B-2659', 'I', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('serotype', 'O', 2)\n",
      "('Ib', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ST10', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SF0680', 'B', -100)\n",
      "('and', 'O', 2)\n",
      "('serotype', 'O', 0)\n",
      "('III', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('ST1', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SF0942', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('exhibited', 'O', -100)\n",
      "('resistance', 'O', 2)\n",
      "('to', 'O', 0)\n",
      "('erythromycin', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('clindamycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('minocycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('erm', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('tet', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('M', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('while', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('serotype', 'O', 2)\n",
      "('III', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ST1', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SF1510', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('resistant', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('minocycline', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('had', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('bovine-associated', 'O', -100)\n",
      "('tet', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Under', 'O', -100)\n",
      "('200', 'O', 2)\n",
      "('ppm', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('chlorophyll', 'O', 2)\n",
      "('b', 'O', 2)\n",
      "('content', 'O', 2)\n",
      "('ranged', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('1.81', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('2.25', 'O', -100)\n",
      "('mg', 'O', -100)\n",
      "('/', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('FW', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('22', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('23', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('plants', 'O', 2)\n",
      "('inoculated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('Citrobacter', 'B', 2)\n",
      "('werkmanii', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('WWN1', 'I', 2)\n",
      "('and', 'O', 2)\n",
      "('Enterobacter', 'B', 0)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', 1)\n",
      "('JWM6', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Nonetheless', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('HT115', 'B', 2)\n",
      "('has', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('highly', 'O', -100)\n",
      "('increased', 'O', 2)\n",
      "('capacity', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('produce', 'O', 2)\n",
      "('GABA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('even', 'O', 2)\n",
      "('compared', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('K-12', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('shown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('7II', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('B', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('non-adherent', 'O', 2)\n",
      "('spherical', 'O', 2)\n",
      "('clusters', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('cells', 'O', -100)\n",
      "('formed', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('culturing', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('stem', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('progenitor', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('MDA-MB-231', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('serum-free', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('We', 'O', -100)\n",
      "('devised', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('nomenclature', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('sequence', 'O', 2)\n",
      "('types', 'O', 2)\n",
      "('based', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('combination', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('alleles', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('loci', 'O', 2)\n",
      "('bteA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('bopB', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('bopD', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('bopN', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('bsp22', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('these', 'O', -100)\n",
      "('are', 'O', -100)\n",
      "('referred', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('T3SS', 'O', -100)\n",
      "('sequence', 'O', 2)\n",
      "('types', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('T3ST', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('genome', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('NAN-27', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('approximately', 'O', -100)\n",
      "('12.02', 'O', -100)\n",
      "('Mb', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('5,700', 'O', -100)\n",
      "('genes', 'O', -100)\n",
      "('according', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('de', 'O', 2)\n",
      "('novo', 'O', -100)\n",
      "('sequencing', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('NAN-27', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Wang', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2017', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('That', 'O', -100)\n",
      "('phylotypes', 'O', 2)\n",
      "('related', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('C.', 'O', 2)\n",
      "('drakei', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('S1', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('C.', 'O', -100)\n",
      "('magnum', 'O', 2)\n",
      "('(', 'O', 0)\n",
      "('M227', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('repeatedly', 'O', -100)\n",
      "('observed', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('treatments', 'O', 2)\n",
      "('with', 'O', 0)\n",
      "('Carex', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Molinia', 'O', 2)\n",
      "('roots', 'O', 2)\n",
      "('suggests', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('clostridial', 'O', 2)\n",
      "('acetogens', 'O', 2)\n",
      "('colonize', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('roots', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('these', 'O', -100)\n",
      "('graminoids', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('fen', 'O', 2)\n",
      "('Schlöppnerbrunnen', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('This', 'O', -100)\n",
      "('shows', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('large', 'O', 2)\n",
      "('molecular', 'O', 2)\n",
      "('diversity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('wheat', 'O', 2)\n",
      "('accessions', 'O', 2)\n",
      "('hosted', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('genebank', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('IPK', 'O', -100)\n",
      "('Gatersleben', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('16S', 'O', 2)\n",
      "('rRNA', 'O', 2)\n",
      "('gene', 'O', 2)\n",
      "('identity', 'O', 2)\n",
      "('value', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('Desulfovibrio', 'B', 2)\n",
      "('desulfuricans', 'I', 0)\n",
      "('DSM', 'I', -100)\n",
      "('642^T', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('representing', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('type', 'O', -100)\n",
      "('species', 'O', 1)\n",
      "('of', 'O', 1)\n",
      "('the', 'O', -100)\n",
      "('genus', 'O', -100)\n",
      "('Desulfovibrio', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('88.6', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('therefore', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('below', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('proposed', 'O', -100)\n",
      "('cut-off', 'O', -100)\n",
      "('level', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('delineation', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('genera', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('usually', 'O', 2)\n",
      "('given', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('94.5', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Yarza', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2014', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Interestingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('failed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('identify', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('hemolysin', 'O', 2)\n",
      "('III', 'O', 2)\n",
      "('gene', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('JDM1', 'B', 2)\n",
      "('genome', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('E.', 'B', -100)\n",
      "('coli', 'I', 0)\n",
      "('DH5α', 'I', -100)\n",
      "('and', 'O', 1)\n",
      "('E.', 'B', 1)\n",
      "('coli', 'I', -100)\n",
      "('JM110', 'I', -100)\n",
      "('applied', 'O', 2)\n",
      "('in', 'O', 0)\n",
      "('this', 'O', -100)\n",
      "('work', 'O', 1)\n",
      "('were', 'O', 1)\n",
      "('used', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('construction', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('propagation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('plasmids', 'O', 2)\n",
      "('carrying', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('without', 'O', 2)\n",
      "('methylation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('mL', 'O', 2)\n",
      "('Luria-Bertani', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('5', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('10', 'O', -100)\n",
      "('g', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('L', 'O', -100)\n",
      "('tryptone', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('g', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('sodium', 'O', 2)\n",
      "('chloride', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('15', 'O', 2)\n",
      "('mL', 'O', 2)\n",
      "('tubes', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('37°C', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('200', 'O', 2)\n",
      "('rpm', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Therefore', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('present', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('high-quality', 'O', 2)\n",
      "('complete', 'O', 2)\n",
      "('genome', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('SOX', 'O', 2)\n",
      "('symbiont', 'O', 2)\n",
      "('associated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('specimen', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('B.', 'O', 2)\n",
      "('thermophilus', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('9', 'O', 2)\n",
      "('N', 'O', -100)\n",
      "('hydrothermal', 'O', 2)\n",
      "('vent', 'O', -100)\n",
      "('fields', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('East', 'O', 2)\n",
      "('Pacific', 'O', 2)\n",
      "('Rise', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('hereafter', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('symbiont', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('referred', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('EPR9N', 'B', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('In', 'O', -100)\n",
      "('summary', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('results', 'O', 2)\n",
      "('reported', 'O', 2)\n",
      "('here', 'O', 2)\n",
      "('illustrate', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('ability', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('B.', 'B', 2)\n",
      "('phytofirmans', 'I', 0)\n",
      "('PsJN', 'I', -100)\n",
      "('to', 'O', 1)\n",
      "('infect', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('colonize', 'O', -100)\n",
      "('responsive', 'O', -100)\n",
      "('switchgrass', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('Panicum', 'O', -100)\n",
      "('virgatum', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('cultivars', 'O', 2)\n",
      "('like', 'O', 2)\n",
      "('Alamo', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('promote', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('growth', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('knockout', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('α-pheromone', 'O', 2)\n",
      "('precursor', 'O', 2)\n",
      "('gene', 'O', -100)\n",
      "('Foc-PP1', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('Foc', 'O', 2)\n",
      "('TR4', 'O', 2)\n",
      "('decreased', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('pathogen', 'O', -100)\n",
      "('virulence', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('banana', 'O', -100)\n",
      "('host', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('suggesting', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('α-pheromones', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('play', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('previously', 'O', 2)\n",
      "('undiscovered', 'O', 2)\n",
      "('role', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('fungal', 'O', 2)\n",
      "('virulence', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('thereby', 'O', -100)\n",
      "('broadening', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('functional', 'O', 2)\n",
      "('activity', 'O', 2)\n",
      "('spectrum', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('α-pheromones', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('asexual', 'O', 2)\n",
      "('fungi', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Immunoprecipitation', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('pan-plectin', 'O', 2)\n",
      "('antibodies', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('protein', 'O', -100)\n",
      "('A', 'O', -100)\n",
      "('/', 'O', -100)\n",
      "('G', 'O', 2)\n",
      "('sepharose', 'O', 2)\n",
      "('beads', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Pierce', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('FO-36b', 'B', -100)\n",
      "('has', 'O', 0)\n",
      "('an', 'O', -100)\n",
      "('established', 'O', -100)\n",
      "('phenotype', 'O', -100)\n",
      "('showing', 'O', 2)\n",
      "('spore', 'O', 2)\n",
      "('resistance', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('peroxide', 'O', 2)\n",
      "('exceeding', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('JPL-CRF', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('13', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Compounds', 'O', -100)\n",
      "('1-5', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('tested', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('potential', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vitro', 'O', 2)\n",
      "('cytotoxicity', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('lung', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('line', 'O', 2)\n",
      "('A549', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('nasopharyngeal', 'O', 2)\n",
      "('carcinoma', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('line', 'O', 2)\n",
      "('CNE2', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('breast', 'O', 2)\n",
      "('adenocarcinoma', 'O', -100)\n",
      "('cell', 'O', 2)\n",
      "('line', 'O', 2)\n",
      "('MCF-7', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('hepatocarcinoma', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('cell', 'O', -100)\n",
      "('line', 'O', -100)\n",
      "('HepG2', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('normal', 'O', 2)\n",
      "('hepatic', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('line', 'O', 2)\n",
      "('HL7702', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('DQ12-45-1b', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('Dt', 'O', -100)\n",
      "('HtaA', 'O', -100)\n",
      "('distributes', 'O', -100)\n",
      "('both', 'O', -100)\n",
      "('at', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('cellular', 'O', 2)\n",
      "('mycomembrane', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('mMVs', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('abundance', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('indicates', 'O', 2)\n",
      "('that', 'O', -100)\n",
      "('Dt', 'O', -100)\n",
      "('HtaA', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('randomly', 'O', 2)\n",
      "('packaged', 'O', -100)\n",
      "('into', 'O', 2)\n",
      "('mMVs', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Previous', 'O', -100)\n",
      "('studies', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('healthy', 'O', 2)\n",
      "('individuals', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('tendency', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('constipation', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('frequencies', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('Bifidobacterium', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Lactobacillus', 'O', 2)\n",
      "('increase', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('frequency', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('genus', 'O', 2)\n",
      "('Enterobacter', 'O', 2)\n",
      "('decreases', 'O', 2)\n",
      "('after', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('consumption', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('KB290', 'B', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('MI', 'B', 2)\n",
      "('10-1553^T', 'I', 0)\n",
      "('(', 'O', 1)\n",
      "('=ATCC', 'B', -100)\n",
      "('TSD-55^T=CCOS', 'I', -100)\n",
      "('1900^T', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('ORS', 'B', 2)\n",
      "('1407^T', 'I', 0)\n",
      "('(', 'O', 1)\n",
      "('=CIP', 'B', -100)\n",
      "('109850^T=LMG', 'I', -100)\n",
      "('27395^T', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('Ovarian', 'O', -100)\n",
      "('cancer', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('BG-1', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('UCI-101', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('HEY', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('OVCA420', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('OVCA432', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('OVCA433', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('OVCAR2', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('OVCAR3', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('OVCAR5', 'B', -100)\n",
      "('cells', 'O', 2)\n",
      "('were', 'O', 0)\n",
      "('cultured', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('McCoy', 'O', -100)\n",
      "(\"'s\", 'O', 2)\n",
      "('5A', 'O', 0)\n",
      "('growth', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Invitrogen', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "('supplemented', 'O', -100)\n",
      "('with', 'O', -100)\n",
      "('10', 'O', 2)\n",
      "('%', 'O', 0)\n",
      "('fetal', 'O', -100)\n",
      "('bovine', 'O', -100)\n",
      "('serum', 'O', 2)\n",
      "('(', 'O', 0)\n",
      "('FBS', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('antibiotics', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('units', 'O', 2)\n",
      "('/', 'O', -100)\n",
      "('ml', 'O', 2)\n",
      "('penicillin', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('100', 'O', 2)\n",
      "('µg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "('streptomycin', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('attempt', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('conducted', 'O', 2)\n",
      "('experiments', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('whether', 'O', 2)\n",
      "('phage', 'O', 2)\n",
      "('117-resistance', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', -100)\n",
      "('Kp36-117R', 'B', -100)\n",
      "('would', 'O', 2)\n",
      "('affect', 'O', 2)\n",
      "('bacterial', 'O', 0)\n",
      "('adsorption', 'O', -100)\n",
      "('rates', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Differently', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('Clostridium', 'O', 2)\n",
      "('formicoaceticum', 'O', 2)\n",
      "('MTHFR', 'O', 2)\n",
      "('consists', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('only', 'O', 2)\n",
      "('MetFV', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('can', 'O', 2)\n",
      "('use', 'O', 2)\n",
      "('Fd_red^2−', 'O', -100)\n",
      "('or', 'O', 2)\n",
      "('FADH_2', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('electron', 'O', -100)\n",
      "('donor', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('24', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('CPT56D-587-MTF', 'B', -100)\n",
      "('was', 'O', 0)\n",
      "('grown', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('24', 'O', -100)\n",
      "('h', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('R2A', 'O', -100)\n",
      "('medium', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('30°C', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('200', 'O', 2)\n",
      "('rpm', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('present', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('aimed', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('investigating', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('role', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('novel', 'O', 2)\n",
      "('Clostridium', 'B', 2)\n",
      "('sp.', 'I', 0)\n",
      "('isolate', 'I', 1)\n",
      "('BL8', 'I', -100)\n",
      "('in', 'O', 1)\n",
      "('human', 'O', 1)\n",
      "('gut', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('genome', 'O', 2)\n",
      "('sequencing', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('tool', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Streptomyces', 'O', -100)\n",
      "('olivaceoviridis', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ATCC', 'B', -100)\n",
      "('23630', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('T.', 'O', 0)\n",
      "('fusca', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('ATCC', 'B', 2)\n",
      "('27730', 'I', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('GYM', 'O', 2)\n",
      "('Streptomyces', 'O', 0)\n",
      "('medium', 'O', 1)\n",
      "('(', 'O', -100)\n",
      "('0.4', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('glucose', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('0.4', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('1', 'O', -100)\n",
      "('%', 'O', -100)\n",
      "('malt', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('0.2', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('CaCO_3', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('28°C', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('TYG', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('0.3', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('tryptone', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('0.3', 'O', -100)\n",
      "('%', 'O', 2)\n",
      "('yeast', 'O', 2)\n",
      "('extract', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('0.3', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('glucose', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('0.1', 'O', 2)\n",
      "('%', 'O', 2)\n",
      "('K_2HPO_4', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('50°C', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('identify', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('traits', 'O', 2)\n",
      "('important', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('potato', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Solanum', 'O', 2)\n",
      "('tuberosum', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('tubers', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('examined', 'O', 2)\n",
      "('three', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('across', 'O', 2)\n",
      "('two', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('D.', 'B', 2)\n",
      "('dadantii', 'I', 2)\n",
      "('3937', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('Dda', 'B', 0)\n",
      "('3937', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('D.', 'B', -100)\n",
      "('dianthicola', 'I', 1)\n",
      "('ME23', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('Ddia', 'B', 0)\n",
      "('ME23', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('D.', 'B', 2)\n",
      "('dianthicola', 'I', 0)\n",
      "('67-19', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('Ddia', 'B', -100)\n",
      "('6719', 'I', -100)\n",
      "(')', 'O', 1)\n",
      "('.', 'O', -100)\n",
      "('Similar', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('streptococci', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('S.', 'B', 2)\n",
      "('oriscaviae', 'I', 0)\n",
      "('HKU75', 'I', -100)\n",
      "('^T', 'O', 1)\n",
      "('may', 'O', -100)\n",
      "('possess', 'O', -100)\n",
      "('virulence', 'O', -100)\n",
      "('factors', 'O', 1)\n",
      "('that', 'O', -100)\n",
      "('enable', 'O', -100)\n",
      "('it', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('colonize', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('oral', 'O', 2)\n",
      "('cavity', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('guinea', 'O', 2)\n",
      "('pigs', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('establish', 'O', -100)\n",
      "('bite', 'O', 2)\n",
      "('wound', 'O', 2)\n",
      "('infection', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Finally', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('V.', 'O', 2)\n",
      "('mimicus', 'O', 2)\n",
      "('produces', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('novel', 'O', -100)\n",
      "('hemolysin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('designated', 'O', 2)\n",
      "('HLX', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('but', 'O', 2)\n",
      "('little', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('known', 'O', -100)\n",
      "('regarding', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('function', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('11', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Strain', 'O', -100)\n",
      "('DSM', 'B', 2)\n",
      "('19163', 'I', 0)\n",
      "('was', 'O', 1)\n",
      "('acquired', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('freeze-dried', 'O', 2)\n",
      "('vial', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('German', 'O', -100)\n",
      "('Collection', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('Microorganisms', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Cell', 'O', 2)\n",
      "('Cultures', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('DSMZ', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('grown', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('TSA', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('solid', 'O', 2)\n",
      "('marine', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Transgenic', 'O', -100)\n",
      "('BrAN', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('functional', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Arabidopsis', 'O', 2)\n",
      "('an-t1', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('rescuing', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('mutant', 'O', 2)\n",
      "('phenotype', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('features', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('narrow', 'O', 2)\n",
      "('leaves', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('defects', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('polarity', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('abnormal', 'O', 2)\n",
      "('arrangement', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cortical', 'O', 2)\n",
      "('microtubules', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('MC', 'O', 2)\n",
      "('degrading', 'O', 2)\n",
      "('bacterium', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('termed', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('C-1', 'B', 2)\n",
      "('strain', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Lake', 'O', 2)\n",
      "('Hongfeng', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Guizhou', 'O', 2)\n",
      "('Province', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('China', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('enrichment', 'O', -100)\n",
      "('culture', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('MCLR', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('major', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('nitrogen', 'O', 2)\n",
      "('source', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('It', 'O', -100)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('common', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('virulence-associated', 'O', 2)\n",
      "('AHL', 'O', 2)\n",
      "('produced', 'O', -100)\n",
      "('by', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('number', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('gram-negative', 'O', 2)\n",
      "('bacteria', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('species', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('genera', 'O', -100)\n",
      "('Pseudomonas', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Aeromonas', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Serratia2324', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('shown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1A', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('S3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('ZH07', 'B', 2)\n",
      "('has', 'O', 0)\n",
      "('the', 'O', -100)\n",
      "('ability', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('grow', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('MSM', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('benzoic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('its', 'O', 2)\n",
      "('sole', 'O', 2)\n",
      "('carbon', 'O', 2)\n",
      "('source', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('BChl', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('molecules', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('970', 'B', 2)\n",
      "('LH1', 'O', 2)\n",
      "('are', 'O', 0)\n",
      "('aligned', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('RC', 'O', 2)\n",
      "('special', 'O', 2)\n",
      "('pair', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('BChl', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('_L', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('BChl', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('_M', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('same', 'O', 2)\n",
      "('plane', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('parallels', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('membrane', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1c', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('underwent', 'O', 2)\n",
      "('complex', 'O', 2)\n",
      "('recombination', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('prototype', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('QYYZ', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('mainly', 'O', -100)\n",
      "('provided', 'O', -100)\n",
      "('structural', 'O', 2)\n",
      "('protein', 'O', 2)\n",
      "('fragments', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('GP2-N', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('recombinant', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('“widespreadedness”', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('an', 'O', -100)\n",
      "('individual', 'O', -100)\n",
      "(\"'s\", 'O', -100)\n",
      "('pain', 'O', -100)\n",
      "('often', 'O', 2)\n",
      "('reflects', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('degree', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('their', 'O', 2)\n",
      "('pain', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('centralized', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('contrast', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('invading', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('significantly', 'O', 2)\n",
      "('increased', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('pcDNA3.1-Livin', 'O', 2)\n",
      "('transfected', 'O', 2)\n",
      "('SW480', 'B', -100)\n",
      "('and', 'O', -100)\n",
      "('DKO1', 'B', -100)\n",
      "('cells', 'O', -100)\n",
      "('compared', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('empty-pcDNA3.1', 'O', -100)\n",
      "('transfected', 'O', 2)\n",
      "('cells', 'O', 0)\n",
      "('(', 'O', -100)\n",
      "('p=0.024', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('p=0.012', 'O', 0)\n",
      "(',', 'O', -100)\n",
      "('respectively', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('1A', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Moreover', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('plant', 'O', 2)\n",
      "('growth-promoting', 'O', 2)\n",
      "('attributes', 'O', 2)\n",
      "('assay', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('ZH07', 'B', 2)\n",
      "('showed', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('capacities', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('nitrogen', 'O', -100)\n",
      "('fixation', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('phosphorus', 'O', 2)\n",
      "('solubilization', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('potassium', 'O', 2)\n",
      "('dissolution', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('siderophore', 'O', 2)\n",
      "('production', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('IAA', 'O', 2)\n",
      "('synthesis', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('From', 'O', -100)\n",
      "('this', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('obtained', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('unialgal', 'O', 2)\n",
      "('isolate', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('JSC-12', 'B', -100)\n",
      "('following', 'O', 2)\n",
      "('references', 'O', 2)\n",
      "('2', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('4', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('DH', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('medium', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('6', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('0.04', 'O', 2)\n",
      "('mM', 'O', 2)\n",
      "('FeCl', 'O', 2)\n",
      "('_3', 'O', 2)\n",
      "('·6H', 'O', 2)\n",
      "('_2', 'O', 2)\n",
      "('O', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('AB26', 'B', -100)\n",
      "('strains', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('grown', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('MB', 'O', 2)\n",
      "('broth', 'O', 2)\n",
      "('under', 'O', 2)\n",
      "('dark', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('shaking', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('200', 'O', 2)\n",
      "('rpm', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('aerobic', 'O', 2)\n",
      "('heterotrophic', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('human', 'O', 2)\n",
      "('stool-derived', 'O', 2)\n",
      "('Bilophila', 'O', 2)\n",
      "('wadsworthia', 'O', -100)\n",
      "('strain', 'O', -100)\n",
      "('caused', 'O', 2)\n",
      "('systemic', 'O', -100)\n",
      "('inflammation', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('specific-pathogen-free', 'O', -100)\n",
      "('mice', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('49', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('T81', 'B', -100)\n",
      "('(', 'O', 0)\n",
      "('class', 'O', -100)\n",
      "('Ktedonobacteria', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('cellulolytic', 'O', -100)\n",
      "('thermophilic', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('previously', 'O', 2)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('geothermal', 'O', 2)\n",
      "('soils', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Tikitere', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('New', 'O', 2)\n",
      "('Zealand', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('11', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('39', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('40', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Failure', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('find', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('transcripts', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('plastid', 'O', 2)\n",
      "('protein-coding', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('our', 'O', -100)\n",
      "('SL-based', 'O', 2)\n",
      "('Eutreptiella', 'B', 2)\n",
      "('sp.', 'I', 2)\n",
      "('454', 'I', 2)\n",
      "('transcriptome', 'O', -100)\n",
      "('dataset', 'O', -100)\n",
      "('could', 'O', 0)\n",
      "('be', 'O', -100)\n",
      "('due', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('these', 'O', 1)\n",
      "('plastid', 'O', -100)\n",
      "('genes', 'O', 1)\n",
      "('are', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('SL', 'O', 2)\n",
      "('trans-spliced', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('thus', 'O', 2)\n",
      "('absent', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('cDNA', 'O', 2)\n",
      "('libraries', 'O', 2)\n",
      "('prepared', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('Eut-SL', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('a', 'O', 2)\n",
      "('selective', 'O', 2)\n",
      "('primer', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('due', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('plastid-targeted', 'O', 2)\n",
      "('proteins', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', -100)\n",
      "('species', 'O', -100)\n",
      "('have', 'O', 2)\n",
      "('long', 'O', 2)\n",
      "('leader', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('sequences', 'O', 2)\n",
      "('we', 'O', 2)\n",
      "('got', 'O', 2)\n",
      "('for', 'O', -100)\n",
      "('these', 'O', -100)\n",
      "('genes', 'O', 2)\n",
      "('could', 'O', 2)\n",
      "('not', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('significant', 'O', 2)\n",
      "('hits', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('homologs', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('GenBank', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Mirroring', 'O', -100)\n",
      "('tR37', 'O', 2)\n",
      "('on', 'O', -100)\n",
      "('top', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('long', 'O', 2)\n",
      "('Stem', 'O', 2)\n",
      "('I', 'O', 2)\n",
      "(\"'s\", 'O', 2)\n",
      "('feature', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('functionally', 'O', 2)\n",
      "('critical', 'O', -100)\n",
      "('purine', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('A90', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Oceanobacillus', 'O', 2)\n",
      "('iheyensis', 'O', 2)\n",
      "('glyQ', 'O', 2)\n",
      "('T‐box', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('stacks', 'O', -100)\n",
      "('underneath', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('codon-anticodon', 'O', -100)\n",
      "('duplex', 'O', 2)\n",
      "('with', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('wobble', 'O', -100)\n",
      "('base', 'O', -100)\n",
      "('pair', 'O', 2)\n",
      "('involving', 'O', 2)\n",
      "('t34', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('thus', 'O', 2)\n",
      "('creating', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('stacking', 'O', 2)\n",
      "('“sandwich”', 'O', 2)\n",
      "('that', 'O', -100)\n",
      "('axially', 'O', -100)\n",
      "('stabilizes', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('duplex', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Figure', 'O', 2)\n",
      "('3b', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('c', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('Zhang', 'O', 2)\n",
      "('&', 'O', 2)\n",
      "(\"Ferré‐D'Amaré\", 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2013', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('support', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('E.', 'B', 2)\n",
      "('coli', 'I', 0)\n",
      "('asd', 'I', -100)\n",
      "('mutants', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('TSB', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('TSA', 'O', 2)\n",
      "('media', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('supplemented', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('400μg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "('diaminopimelic', 'O', 2)\n",
      "('acid', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('DAP', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Sigma', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('MO', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('BD', 'O', 2)\n",
      "('Accuri^TM', 'O', 2)\n",
      "('C6', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Becton-Dickinson', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('NJ', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('USA', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('used', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('evaluation', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('redox', 'O', 2)\n",
      "('biosensor', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('different', 'O', 2)\n",
      "('deletions', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Genes', 'O', -100)\n",
      "('for', 'O', 2)\n",
      "('natural', 'O', 2)\n",
      "('transformation', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('comA', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('exbB', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('pil', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('type', 'O', 2)\n",
      "('IV', 'O', 2)\n",
      "('pili', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('found', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('stutzeri', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('CCUG', 'B', 2)\n",
      "('29243', 'I', -100)\n",
      "('(', 'O', 2)\n",
      "('Brunet-Galmés', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "('2012', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('and', 'O', 1)\n",
      "('DSM', 'B', -100)\n",
      "('10701', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('model', 'O', -100)\n",
      "('organism', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('natural', 'O', -100)\n",
      "('transformation', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Chakraborty', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('biochemical', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('enzymatic', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('purified', 'O', 2)\n",
      "('putative', 'O', 2)\n",
      "('agarases', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('F.', 'B', -100)\n",
      "('faecale', 'I', 2)\n",
      "('WV33', 'I', 0)\n",
      "('^T', 'O', -100)\n",
      "('would', 'O', 1)\n",
      "('elucidate', 'O', -100)\n",
      "('the', 'O', 1)\n",
      "('mechanism', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('agar-depolymerization', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('GH4', 'O', -100)\n",
      "('maltose-6′-phosphate', 'O', 2)\n",
      "('glucosidase', 'O', -100)\n",
      "('was', 'O', 2)\n",
      "('conserved', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('all', 'O', -100)\n",
      "('whisky', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('milk', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('JCM', 'B', 2)\n",
      "('1120', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('CAUH18', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('H10', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('R0052', 'B', 2)\n",
      "('.', 'O', 2)\n",
      "('With', 'O', -100)\n",
      "('respect', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('performances', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('proteomics', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('bioinformatics', 'O', 2)\n",
      "('tools', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('work', 'O', 2)\n",
      "('successfully', 'O', 2)\n",
      "('explores', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('DUF326-like', 'O', 2)\n",
      "('domain', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('Enterobacter', 'B', -100)\n",
      "('cloacae', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('EC01', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('cadmium-tolerant', 'O', 0)\n",
      "('strain', 'O', 1)\n",
      "(')', 'O', -100)\n",
      "('has', 'O', 1)\n",
      "('been', 'O', 1)\n",
      "('found', 'O', -100)\n",
      "('to', 'O', 2)\n",
      "('play', 'O', 2)\n",
      "('imperative', 'O', -100)\n",
      "('roles', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('detoxification', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('cadmium', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('possibly', 'O', 2)\n",
      "('by', 'O', 2)\n",
      "('binding', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('cadmium', 'O', 2)\n",
      "('ions', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('SJCon^T', 'B', 2)\n",
      "('=MTCC', 'B', 0)\n",
      "('10104^T', 'I', -100)\n",
      "('=DSM', 'B', -100)\n",
      "('23165^T', 'I', -100)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', 1)\n",
      "('pesticide-contaminated', 'O', -100)\n",
      "('soil', 'O', -100)\n",
      "('sample', 'O', -100)\n",
      "('collected', 'O', -100)\n",
      "('in', 'O', 0)\n",
      "('Punjab', 'O', -100)\n",
      "('State', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('India', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('This', 'O', -100)\n",
      "('has', 'O', 2)\n",
      "('important', 'O', 2)\n",
      "('advantage', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('fight', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('terrorism', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('detection', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('suspicious', 'O', 2)\n",
      "('or', 'O', 2)\n",
      "('prohibited', 'O', 2)\n",
      "('substances', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('amputated', 'O', 2)\n",
      "('distal', 'O', 2)\n",
      "('part', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('penis', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('contaminated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('ants', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('fragile', 'O', 2)\n",
      "('dorsal', 'O', 2)\n",
      "('veins', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('addition', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('four', 'O', 2)\n",
      "('genes', 'O', 2)\n",
      "('comprising', 'O', 2)\n",
      "('key', 'O', 2)\n",
      "('regulatory', 'O', 2)\n",
      "('components', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('glycolytic', 'O', 2)\n",
      "('pathway', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('represented', 'O', 2)\n",
      "('amongst', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('class', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('biomarkers', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('including', 'O', 2)\n",
      "('pyruvate', 'O', 2)\n",
      "('kinase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CG12229', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('fructose-bisphosphate', 'O', -100)\n",
      "('aldolase', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('delilah', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('well', 'O', -100)\n",
      "('as', 'O', 2)\n",
      "('genes', 'O', -100)\n",
      "('that', 'O', 2)\n",
      "('act', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('peripheral', 'O', -100)\n",
      "('pathways', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('those', 'O', 2)\n",
      "('encoding', 'O', 2)\n",
      "('trehalose-phosphatase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CG5177', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('L-iditol', 'O', 2)\n",
      "('2-dehydrogenase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('CG4836', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Eighteen', 'O', -100)\n",
      "('tumours', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('T1', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('82', 'O', 2)\n",
      "('tumours', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('T2', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('tumours', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('T3', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('effect', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('erythritol', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('growth', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('S2', 'B', 2)\n",
      "('was', 'O', 2)\n",
      "('estimated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 0)\n",
      "('size', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('colonies', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('test', 'O', 2)\n",
      "('plates', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('comparison', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('plates', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('S2', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('S19', 'B', 2)\n",
      "('inoculated', 'O', 2)\n",
      "('on', 'O', 0)\n",
      "('TSA', 'O', 2)\n",
      "('plates', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('S19', 'B', 2)\n",
      "('inoculated', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('TSA', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ery', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('plates', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Intriguingly', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('only', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('Scalindua', 'O', 2)\n",
      "('genomes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('except', 'O', -100)\n",
      "('for', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('Scalindua', 'O', 2)\n",
      "('sp001828595', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('contained', 'O', 2)\n",
      "('an', 'O', 2)\n",
      "('operon', 'O', -100)\n",
      "('encoding', 'O', -100)\n",
      "('Fdp', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('rubredoxin', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Rd', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('electron', 'O', 2)\n",
      "('carrier', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('Fdp', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('reported', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('previous', 'O', 2)\n",
      "('experiments', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('21', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('wild-type', 'O', 2)\n",
      "('V52', 'B', 2)\n",
      "('yielded', 'O', -100)\n",
      "('~1000-fold', 'O', -100)\n",
      "('killing', 'O', 0)\n",
      "('of', 'O', -100)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('following', 'O', 2)\n",
      "('human', 'O', 2)\n",
      "('cell', 'O', 2)\n",
      "('lines', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('this', 'O', 2)\n",
      "('study', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('i', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('epidermoid', 'B', 2)\n",
      "('carcinoma', 'I', 0)\n",
      "('A-431', 'I', -100)\n",
      "('(', 'O', 1)\n",
      "('ATCC^®', 'O', 1)\n",
      "('CRL-1555', 'B', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('ii', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('colorectal', 'B', -100)\n",
      "('carcinoma', 'I', 0)\n",
      "('HCT116', 'I', -100)\n",
      "('(', 'O', -100)\n",
      "('ATCC^®', 'O', -100)\n",
      "('CCL-247', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('iii', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('colorectal', 'B', 0)\n",
      "('carcinoma', 'I', 1)\n",
      "('HCT116p53^−', 'I', 1)\n",
      "('/', 'O', -100)\n",
      "('−', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('iv', 'O', 0)\n",
      "(')', 'O', -100)\n",
      "('normal', 'B', -100)\n",
      "('fibroblasts', 'I', 2)\n",
      "('BJ', 'I', 2)\n",
      "('(', 'O', 2)\n",
      "('ATCC^®', 'O', 2)\n",
      "('CRL-2522', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "('.', 'O', 1)\n",
      "('JSC-12', 'B', -100)\n",
      "('is', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('filamentous', 'O', -100)\n",
      "('cyanobacterium', 'O', -100)\n",
      "('with', 'O', 2)\n",
      "('narrow', 'O', 2)\n",
      "('trichomes', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('0.8', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('1.8', 'O', -100)\n",
      "('μm', 'O', -100)\n",
      "('wide', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('resembling', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('morphological', 'O', 2)\n",
      "('features', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('Leptolyngbya', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('7', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('these', 'O', 2)\n",
      "('chemostat', 'O', 2)\n",
      "('cultures', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('biomass', 'O', 2)\n",
      "('yield', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('glucose', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('FS09322', 'B', 2)\n",
      "('was', 'O', 2)\n",
      "('lower', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('respiration', 'O', -100)\n",
      "('rates', 'O', -100)\n",
      "('were', 'O', -100)\n",
      "('consistently', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('reference', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('What', 'O', -100)\n",
      "('triggers', 'O', 2)\n",
      "('H.', 'O', 2)\n",
      "('fraxineus', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('switch', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('commensal', 'O', 2)\n",
      "('endophyte', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('Asian', 'O', 2)\n",
      "('ash', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('a', 'O', -100)\n",
      "('virulent', 'O', -100)\n",
      "('pathogen', 'O', 2)\n",
      "('on', 'O', 2)\n",
      "('European', 'O', 2)\n",
      "('ash', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('yet', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('unknown', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Lake', 'O', -100)\n",
      "('waters', 'O', 2)\n",
      "('at', 'O', 2)\n",
      "('sampling', 'O', 2)\n",
      "('point', 'O', 2)\n",
      "('IK-2', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('high', 'O', -100)\n",
      "('Al', 'O', -100)\n",
      "('contents', 'O', 2)\n",
      "('equal', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('75.2', 'O', 2)\n",
      "('µg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('L', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('Under', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('tested', 'O', 2)\n",
      "('conditions', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('most', 'O', 2)\n",
      "('promising', 'O', 2)\n",
      "('activities', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('α-glucosidase', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('Pseudomonas', 'B', -100)\n",
      "('extremaustralis', 'I', -100)\n",
      "('strains', 'I', 2)\n",
      "('2ASSA', 'I', 0)\n",
      "('and', 'O', 1)\n",
      "('2ASCA', 'B', -100)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('β-galactosidase', 'O', 1)\n",
      "(',', 'O', 1)\n",
      "('β-glucosidase', 'O', -100)\n",
      "('β-xylosidase', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('cellobiohydrolase', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('several', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('PMA', 'O', 2)\n",
      "('treated', 'O', 2)\n",
      "('samples', 'O', 2)\n",
      "('collected', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('GSE', 'O', 2)\n",
      "('surface', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('the', 'O', 2)\n",
      "('CRV1', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('dominated', 'O', 2)\n",
      "('by', 'O', -100)\n",
      "('reads', 'O', 2)\n",
      "('belonging', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('genera', 'O', 2)\n",
      "('Acidovorax', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Caulobacter', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Massilia', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('reclassified', 'O', 2)\n",
      "('as', 'O', -100)\n",
      "('Naxibacter', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('while', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('CRV2', 'O', 2)\n",
      "('GSE', 'O', -100)\n",
      "('surface', 'O', 2)\n",
      "('showed', 'O', 2)\n",
      "('Pseudomonas', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Bacillus', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('therapeutic', 'O', 2)\n",
      "('potential', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('intranasal', 'O', 2)\n",
      "('PC945', 'O', 2)\n",
      "('has', 'O', 2)\n",
      "('been', 'O', -100)\n",
      "('investigated', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('transiently', 'O', 2)\n",
      "('neutropenic', 'O', 2)\n",
      "('mice', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('invasive', 'O', 2)\n",
      "('pulmonary', 'O', 2)\n",
      "('aspergillosis', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('strains', 'O', 2)\n",
      "('16-SW-7', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('P.', 'B', -100)\n",
      "('distincta', 'I', -100)\n",
      "('KMM', 'I', -100)\n",
      "('638', 'I', -100)\n",
      "('^T', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('and', 'O', -100)\n",
      "('P.', 'B', 1)\n",
      "('paragorgicola', 'I', -100)\n",
      "('KMM', 'I', 1)\n",
      "('3548', 'I', -100)\n",
      "('^T', 'O', 1)\n",
      "('shared', 'O', -100)\n",
      "('many', 'O', 2)\n",
      "('common', 'O', -100)\n",
      "('phenotypic', 'O', 2)\n",
      "('features', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('such', 'O', -100)\n",
      "('as', 'O', 1)\n",
      "('respiratory', 'O', -100)\n",
      "('type', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('metabolism', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('motility', 'O', 1)\n",
      "('by', 'O', -100)\n",
      "('means', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('flagella', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('presence', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('catalase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('alkaline', 'O', 2)\n",
      "('phosphatase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('esterase', 'O', 2)\n",
      "('lipase', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('C8', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('leucine', 'O', 2)\n",
      "('arylamidase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('valine', 'O', 2)\n",
      "('arylamidase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('phosphatase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('naphthol-AS-BI-phosphohydrolase', 'O', 2)\n",
      "('activities', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Table', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('P', 'O', -100)\n",
      "('var.', 'O', 2)\n",
      "('depressus', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Ball', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('Maire', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('tested', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('five', 'O', -100)\n",
      "('microbial', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(':', 'O', 2)\n",
      "('one', 'O', 2)\n",
      "('gram-positive', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Staphylococcus', 'O', 2)\n",
      "('aureus', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('four', 'O', -100)\n",
      "('gram-negative', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ATB', 'B', 2)\n",
      "(':', 'I', 2)\n",
      "('57', 'I', -100)\n",
      "(')', 'O', -100)\n",
      "('B6N', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('E.', 'O', -100)\n",
      "('coli', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ATB', 'B', 0)\n",
      "(':', 'I', -100)\n",
      "('97', 'I', 1)\n",
      "(')', 'O', 1)\n",
      "('BGM', 'B', 2)\n",
      "(',', 'O', 0)\n",
      "('Pseudomonas', 'O', -100)\n",
      "('aeruginosa', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('Klebsiella', 'O', 2)\n",
      "('pneumonia', 'O', 2)\n",
      "(')', 'O', 0)\n",
      "('.', 'O', -100)\n",
      "('Strain', 'O', -100)\n",
      "('CENA596', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('shrimp', 'O', -100)\n",
      "('production', 'O', 2)\n",
      "('ponds', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('southern', 'O', 2)\n",
      "('region', 'O', 2)\n",
      "('of', 'O', -100)\n",
      "('Brazil', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('47', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('A.', 'O', 2)\n",
      "('baumannii', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('DR1', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('DR2', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('AB067', 'B', 0)\n",
      "('strains', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('assessed', 'O', 2)\n",
      "('for', 'O', 0)\n",
      "('antibiotic', 'O', -100)\n",
      "('susceptibility', 'O', -100)\n",
      "('using', 'O', 2)\n",
      "('Kirby', 'O', 2)\n",
      "('Bauer', 'O', 2)\n",
      "('disk', 'O', 2)\n",
      "('diffusion', 'O', 2)\n",
      "('assay', 'O', 2)\n",
      "(';', 'O', 2)\n",
      "('antibiotics', 'O', 2)\n",
      "('disks', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('μg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ml', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('HiMedia', 'O', 2)\n",
      "('Laboratories', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('India', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('doripenem', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('doxycycline', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('ertapenem', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('erythromycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('gentamicin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('imipenem', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('kanamycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('meropenem', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('methicillin', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('minocycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('oxacillin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('streptomycin', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('tetracycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('tigecycline', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('data', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('tabulated', 'O', 2)\n",
      "('.', 'O', -100)\n",
      "('The', 'O', -100)\n",
      "('remaining', 'O', 2)\n",
      "('serotype', 'O', 2)\n",
      "('Ib', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('ST10', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('SF0680', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('exhibiting', 'O', 0)\n",
      "('a', 'O', -100)\n",
      "('higher', 'O', -100)\n",
      "('penicillin', 'O', 2)\n",
      "('MIC', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('μg', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('mL', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('had', 'O', 2)\n",
      "('substitutions', 'O', 2)\n",
      "('V405A', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Q557E', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('G398A', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('PBP2X', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('As', 'O', -100)\n",
      "('depicted', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('5a', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('viral', 'O', 2)\n",
      "('titers', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('all', 'O', 2)\n",
      "('tissues', 'O', 2)\n",
      "('except', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('brain', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('MV', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('12', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('groups', 'O', 2)\n",
      "('JS', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('KR', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('07', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('viral', 'O', 2)\n",
      "('titers', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('higher', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('trachea', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('brain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('nasal', 'O', 2)\n",
      "('turbinate', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('soft', 'O', -100)\n",
      "('palate', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('JS', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('10', 'O', 2)\n",
      "('than', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('group', 'O', 2)\n",
      "('KR', 'O', 2)\n",
      "('/', 'O', 2)\n",
      "('07', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('For', 'O', -100)\n",
      "('example', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Phaeobacter', 'B', 2)\n",
      "('gallaeciensis', 'I', 0)\n",
      "('DSM', 'I', -100)\n",
      "('26640', 'I', -100)\n",
      "('produces', 'O', 1)\n",
      "('up', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('eight', 'O', 1)\n",
      "('different', 'O', 1)\n",
      "('AHLs', 'O', -100)\n",
      "('via', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('three', 'O', 2)\n",
      "('luxI', 'O', 2)\n",
      "('homologues', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Ziesche', 'O', -100)\n",
      "('et', 'O', 2)\n",
      "('al', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2015', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('C.', 'B', -100)\n",
      "('cadaveris', 'I', 0)\n",
      "('IFB3C5', 'I', -100)\n",
      "('was', 'O', 1)\n",
      "('isolated', 'O', -100)\n",
      "('from', 'O', 1)\n",
      "('a', 'O', -100)\n",
      "('cryopreserved', 'O', -100)\n",
      "('colon', 'O', -100)\n",
      "('adenocarcinoma', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('67-year-old', 'O', 2)\n",
      "('treatment-naive', 'O', 2)\n",
      "('female', 'O', 2)\n",
      "('colorectal', 'O', 2)\n",
      "('cancer', 'O', 2)\n",
      "('patient', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('originally', 'O', -100)\n",
      "('resected', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('1989', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('Seattle', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('WA', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A549', 'B', -100)\n",
      "('and', 'O', 0)\n",
      "('HeLa', 'B', 2)\n",
      "('cells', 'O', 0)\n",
      "('were', 'O', 2)\n",
      "('incubated', 'O', 2)\n",
      "('with', 'O', 2)\n",
      "('spores', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('2', 'O', 2)\n",
      "('hr', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('respectively', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('These', 'O', -100)\n",
      "('four', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('well', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('toxigenic', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "(':', 'O', -100)\n",
      "('TH510', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('TH1526', 'B', 0)\n",
      "('from', 'O', -100)\n",
      "('India', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('CD1791', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('CD2173', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('CD72', 'B', 2)\n",
      "(',', 'O', 2)\n",
      "('CD2225', 'B', 0)\n",
      "(',', 'O', -100)\n",
      "('CD5052', 'B', -100)\n",
      "(',', 'O', 2)\n",
      "('CD4728', 'B', 0)\n",
      "('from', 'O', -100)\n",
      "('Belarus', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('CD31A', 'B', 0)\n",
      "('from', 'O', -100)\n",
      "('Brazil', 'O', 2)\n",
      "('along', 'O', 0)\n",
      "('with', 'O', -100)\n",
      "('NCTC13129', 'B', -100)\n",
      "('and', 'O', 2)\n",
      "('NCTC5011', 'B', 0)\n",
      "('from', 'O', -100)\n",
      "('United', 'O', -100)\n",
      "('Kingdom', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('were', 'O', -100)\n",
      "('scattered', 'O', -100)\n",
      "('among', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('non-toxin', 'O', -100)\n",
      "('bearing', 'O', -100)\n",
      "('isolates', 'O', 2)\n",
      "('.', 'O', 0)\n",
      "('The', 'O', -100)\n",
      "('type', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('B1', 'B', 2)\n",
      "('^T', 'O', 0)\n",
      "('(', 'O', 2)\n",
      "('=NBRC', 'B', -100)\n",
      "('111850', 'I', 2)\n",
      "('^T', 'O', 0)\n",
      "('=DSM', 'B', -100)\n",
      "('103919', 'I', -100)\n",
      "('^T', 'O', -100)\n",
      "(')', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('was', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', -100)\n",
      "('a', 'O', 0)\n",
      "('highly', 'O', -100)\n",
      "('alkaline', 'O', 1)\n",
      "('serpentinizing', 'O', -100)\n",
      "('spring', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Barnes', 'O', -100)\n",
      "('Spring', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('The', 'O', 2)\n",
      "('Cedars', 'O', 2)\n",
      "('located', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('north', 'O', 2)\n",
      "('California', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('USA', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('The', 'O', -100)\n",
      "('strain', 'O', 2)\n",
      "('LAD', 'B', 2)\n",
      "('was', 'O', 0)\n",
      "('isolated', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('maize', 'O', 2)\n",
      "('rhizosphere', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('our', 'O', 2)\n",
      "('lab', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('initial', 'O', 2)\n",
      "('characterization', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('newly', 'O', 2)\n",
      "('isolated', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('strain', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('performed', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('effects', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('LAD', 'B', 2)\n",
      "('on', 'O', 0)\n",
      "('corn', 'O', 2)\n",
      "('seedlings', 'O', 2)\n",
      "('were', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('evaluated', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('A', 'O', -100)\n",
      "('similar', 'O', 2)\n",
      "('phenotype', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('observed', 'O', 2)\n",
      "('using', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Y.', 'B', 2)\n",
      "('enterocolitica', 'I', 0)\n",
      "('WA-lux', 'I', -100)\n",
      "('strain', 'O', 1)\n",
      "(',', 'O', -100)\n",
      "('indicating', 'O', 1)\n",
      "('that', 'O', -100)\n",
      "('this', 'O', -100)\n",
      "('phenomenon', 'O', 2)\n",
      "('could', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('extended', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('other', 'O', 2)\n",
      "('pathogens', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Fig', 'O', 2)\n",
      "('S4', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Other', 'O', -100)\n",
      "('marine', 'O', 2)\n",
      "('bacterial', 'O', 2)\n",
      "('Tmms', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('to', 'O', -100)\n",
      "('be', 'O', 2)\n",
      "('active', 'O', 2)\n",
      "('have', 'O', 2)\n",
      "('been', 'O', 2)\n",
      "('found', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Alphaproteobacteria', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('include', 'O', 2)\n",
      "('RnTmm', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Roseovarius', 'B', -100)\n",
      "('nubinhibens', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('ISM', 'I', 2)\n",
      "('(', 'O', -100)\n",
      "('15', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 0)\n",
      "('Tmms', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('Methylocella', 'B', -100)\n",
      "('silvestris', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('BL2', 'I', -100)\n",
      "(',', 'O', -100)\n",
      "('Roseovarius', 'B', 1)\n",
      "('sp.', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('217', 'I', 2)\n",
      "(',', 'O', 2)\n",
      "('Ruegeria', 'B', 2)\n",
      "('pomeroyi', 'I', 2)\n",
      "('strain', 'I', 2)\n",
      "('DSS-3', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('Pelagibacter', 'B', 0)\n",
      "('ubique', 'I', -100)\n",
      "('strain', 'I', -100)\n",
      "('HTCC1002', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('and', 'O', -100)\n",
      "('Pelagibacter', 'B', 1)\n",
      "('ubique', 'I', 1)\n",
      "('strain', 'I', -100)\n",
      "('HTCC721', 'I', 2)\n",
      "('(', 'O', 0)\n",
      "('16', 'O', -100)\n",
      "(')', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('These', 'O', -100)\n",
      "('findings', 'O', 2)\n",
      "('indicated', 'O', 2)\n",
      "('again', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('role', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('OsiSh-2', 'B', 2)\n",
      "('in', 'O', 0)\n",
      "('maintaining', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('relative', 'O', -100)\n",
      "('homeostasis', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('response', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('In', 'O', -100)\n",
      "('A.', 'O', 2)\n",
      "('thaliana', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('ROP11', 'O', 2)\n",
      "('expression', 'O', 2)\n",
      "('affected', 'O', 2)\n",
      "('seed', 'O', -100)\n",
      "('germination', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('seedling', 'O', 2)\n",
      "('growth', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('stomatal', 'O', 2)\n",
      "('closure', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('abscisic', 'O', 2)\n",
      "('acid', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('ABA', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('-mediated', 'O', 2)\n",
      "('responses', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('drought', 'O', -100)\n",
      "('stress', 'O', 2)\n",
      "('responses', 'O', 2)\n",
      "('[', 'O', 2)\n",
      "('22', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('However', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('mapping', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('‘meta-Sphingobium', 'O', 2)\n",
      "(\"'\", 'O', 2)\n",
      "('assembly', 'O', -100)\n",
      "('to', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('non-HCH-degrading', 'O', -100)\n",
      "('Sphingobium', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Sphingobium', 'B', 2)\n",
      "('chlorophenolicum', 'I', 2)\n",
      "('L1', 'I', -100)\n",
      "('and', 'O', -100)\n",
      "('Sphingobium', 'O', -100)\n",
      "('sp', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('There', 'O', -100)\n",
      "('are', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('substantial', 'O', 2)\n",
      "('number', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('transcriptional', 'O', 2)\n",
      "('regulators', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('such', 'O', 2)\n",
      "('as', 'O', 2)\n",
      "('Cyanobacterium', 'O', 2)\n",
      "('anabaena', 'O', 2)\n",
      "('AztR', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Liu', 'O', -100)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2005', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Oscillatoria', 'O', 2)\n",
      "('brevis', 'O', 2)\n",
      "('BxmR', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Liu', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2004', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Staphylococcus', 'O', -100)\n",
      "('aureus', 'O', 2)\n",
      "('CadC', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Endo', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('Silver', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('1995', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('Mycobacterium', 'O', 2)\n",
      "('tuberculosis', 'O', 2)\n",
      "('CmtR', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Cavet', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('2002', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Synechococcus', 'O', 2)\n",
      "('elongatus', 'O', 2)\n",
      "('SmtB', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Vanzile', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2002', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Pseudomonas', 'O', -100)\n",
      "('putida', 'O', 2)\n",
      "('CadR', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Lee', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2001', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('E.', 'O', 2)\n",
      "('coli', 'O', -100)\n",
      "('ZntR', 'O', -100)\n",
      "('(', 'O', 2)\n",
      "('Brocklehurst', 'O', 2)\n",
      "('et', 'O', -100)\n",
      "('al.', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('1999', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('identified', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('recognize', 'O', 2)\n",
      "('cadmium', 'O', 2)\n",
      "('ions', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('PGG', 'O', -100)\n",
      "('increased', 'O', 2)\n",
      "('Caspases', 'O', -100)\n",
      "('3', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('4', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('14', 'O', 2)\n",
      "('expressions', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('MM-231', 'O', 2)\n",
      "('cells', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('but', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('MM-468', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('only', 'O', 2)\n",
      "('Caspase', 'O', 2)\n",
      "('3', 'O', 2)\n",
      "('was', 'O', -100)\n",
      "('significantly', 'O', -100)\n",
      "('upregulated', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('Meeks', 'O', -100)\n",
      "('isolated', 'O', 2)\n",
      "('strains', 'O', -100)\n",
      "('N1', 'B', -100)\n",
      "('(', 'O', 2)\n",
      "('UCD120', 'B', 2)\n",
      "(')', 'O', 0)\n",
      "(',', 'O', 2)\n",
      "('A1', 'B', 0)\n",
      "('(', 'O', -100)\n",
      "('UCD121', 'B', -100)\n",
      "(')', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('A2', 'B', 0)\n",
      "('(', 'O', 2)\n",
      "('UCD122', 'B', 0)\n",
      "(')', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('an', 'O', 2)\n",
      "('extract', 'O', 2)\n",
      "('of', 'O', 0)\n",
      "('cyanobacterial', 'O', 2)\n",
      "('cells', 'O', 0)\n",
      "('from', 'O', -100)\n",
      "('Azolla', 'O', -100)\n",
      "('caroliniana', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('used', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('infect', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('hornwort', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('Anthoceros', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('Notably', 'O', -100)\n",
      "(',', 'O', 2)\n",
      "('peptidoglycan-associated', 'O', 2)\n",
      "('lipoproteins', 'O', 2)\n",
      "('occurred', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('all', 'O', 2)\n",
      "('P.', 'O', 2)\n",
      "('distincta', 'O', 2)\n",
      "('host-associated', 'O', 2)\n",
      "('isolates', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('Supplementary', 'O', 2)\n",
      "('Table', 'O', -100)\n",
      "('S6', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "(',', 'O', -100)\n",
      "('but', 'O', 2)\n",
      "('UDP-2,4-diacetamido-2,4', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('6-trideoxy-beta-L-altropyranose', 'O', 2)\n",
      "('hydrolase', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('which', 'O', 2)\n",
      "('is', 'O', 2)\n",
      "('involved', 'O', 2)\n",
      "('in', 'O', -100)\n",
      "('biosynthesis', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('pseudaminic', 'O', -100)\n",
      "('acid', 'O', -100)\n",
      "('as', 'O', -100)\n",
      "('a', 'O', -100)\n",
      "('component', 'O', -100)\n",
      "('of', 'O', -100)\n",
      "('polysaccharides', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('certain', 'O', -100)\n",
      "('gut', 'O', 2)\n",
      "('pathogens', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('occurred', 'O', -100)\n",
      "('in', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('strains', 'O', -100)\n",
      "('KMM', 'B', -100)\n",
      "('3548', 'I', -100)\n",
      "('and', 'O', -100)\n",
      "('2-2A-13', 'B', -100)\n",
      "('with', 'O', -100)\n",
      "('carotenoid-like', 'O', -100)\n",
      "('phenotypes', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('56', 'O', -100)\n",
      "(']', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('There', 'O', -100)\n",
      "('were', 'O', 2)\n",
      "('also', 'O', 2)\n",
      "('relevant', 'O', 2)\n",
      "('studies', 'O', 2)\n",
      "('investigating', 'O', 2)\n",
      "('weight', 'O', 2)\n",
      "('changes', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('3724', 'O', 2)\n",
      "('obese', 'O', 2)\n",
      "('men', 'O', -100)\n",
      "('and', 'O', 2)\n",
      "('women', 'O', 2)\n",
      "('taking', 'O', 2)\n",
      "('Lactobacillus', 'B', 2)\n",
      "('rhamnosus', 'I', 2)\n",
      "('CGMCC1', 'I', 0)\n",
      "(',', 'O', 1)\n",
      "('fructooligosaccharide', 'O', -100)\n",
      "(',', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('inulin', 'O', -100)\n",
      "('over', 'O', 2)\n",
      "('24', 'O', 2)\n",
      "('weeks', 'O', -100)\n",
      "('[', 'O', -100)\n",
      "('80', 'O', -100)\n",
      "(']', 'O', -100)\n",
      "('.', 'O', -100)\n",
      "('E', 'B', -100)\n",
      "('.', 'I', 0)\n",
      "('coli', 'I', 1)\n",
      "('HT115', 'I', 1)\n",
      "('pellet', 'O', 1)\n",
      "('had', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('highest', 'O', 2)\n",
      "('GABA', 'O', 2)\n",
      "('levels', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('whereas', 'O', 2)\n",
      "('OP50', 'B', 2)\n",
      "('and', 'O', 2)\n",
      "('HT115Δ', 'B', 2)\n",
      "('gad', 'I', 0)\n",
      "('were', 'O', -100)\n",
      "('indistinguishable', 'O', 2)\n",
      "('from', 'O', 0)\n",
      "('each', 'O', -100)\n",
      "('other', 'O', -100)\n",
      "('(', 'O', -100)\n",
      "('Fig', 'O', 1)\n",
      "('8B', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('PS1', 'B', -100)\n",
      "(',', 'O', 0)\n",
      "('PS21', 'B', -100)\n",
      "('and', 'O', 2)\n",
      "('PS75', 'B', 0)\n",
      "('(', 'O', -100)\n",
      "('P.', 'B', 2)\n",
      "('glaciei', 'I', 0)\n",
      "('423', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('97.38', 'O', 0)\n",
      "('%', 'O', -100)\n",
      "('ID', 'O', 1)\n",
      "(';', 'O', -100)\n",
      "('Rhodobacter', 'B', -100)\n",
      "('maris', 'I', 1)\n",
      "('JA276', 'I', -100)\n",
      "(',', 'O', 2)\n",
      "('98.89', 'O', 2)\n",
      "('%', 'O', -100)\n",
      "('ID', 'O', -100)\n",
      "(';', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('Bacillus', 'B', 2)\n",
      "('megaterium', 'I', 0)\n",
      "('NBRC', 'I', -100)\n",
      "('15308', 'I', 1)\n",
      "(',', 'O', -100)\n",
      "('100', 'O', 1)\n",
      "('%', 'O', -100)\n",
      "('ID', 'O', -100)\n",
      "('respectively', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('were', 'O', -100)\n",
      "('the', 'O', -100)\n",
      "('three', 'O', 2)\n",
      "('control', 'O', 2)\n",
      "('strains', 'O', 2)\n",
      "('selected', 'O', 2)\n",
      "(',', 'O', 0)\n",
      "('all', 'O', 1)\n",
      "('of', 'O', -100)\n",
      "('them', 'O', -100)\n",
      "('recovered', 'O', 1)\n",
      "('from', 'O', -100)\n",
      "('solar', 'O', -100)\n",
      "('panels', 'O', 1)\n",
      "('and', 'O', -100)\n",
      "('previously', 'O', -100)\n",
      "('tested', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('C.', 'O', 2)\n",
      "('elegans', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('in', 'O', 2)\n",
      "('vivo', 'O', 2)\n",
      "('protection', 'O', 2)\n",
      "('against', 'O', 2)\n",
      "('oxidative', 'O', 2)\n",
      "('stress', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('Tanner', 'O', 2)\n",
      "('et', 'O', 2)\n",
      "('al.', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('2019', 'O', 2)\n",
      "(')', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('To', 'O', -100)\n",
      "('provide', 'O', 2)\n",
      "('molecular', 'O', 2)\n",
      "('details', 'O', 2)\n",
      "('into', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('essentiality', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('C.', 'O', -100)\n",
      "('albicans', 'O', 2)\n",
      "('Aro1', 'O', 2)\n",
      "(',', 'O', -100)\n",
      "('we', 'O', 2)\n",
      "('first', 'O', 2)\n",
      "('pursued', 'O', -100)\n",
      "('its', 'O', 2)\n",
      "('structural', 'O', 2)\n",
      "('characterization', 'O', 2)\n",
      "('.', 'O', 2)\n",
      "('Taking', 'O', -100)\n",
      "('into', 'O', 2)\n",
      "('consideration', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('above', 'O', 2)\n",
      "('evidence', 'O', 2)\n",
      "('and', 'O', 2)\n",
      "('uncertainties', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('Panel', 'O', 2)\n",
      "('considers', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('possibility', 'O', 2)\n",
      "('of', 'O', 2)\n",
      "('entry', 'O', 2)\n",
      "('into', 'O', 2)\n",
      "('the', 'O', 2)\n",
      "('nursery', 'O', 2)\n",
      "('infecting', 'O', 2)\n",
      "('apple', 'O', -100)\n",
      "('plants', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('surrounding', 'O', 2)\n",
      "('orchards', 'O', 2)\n",
      "('may', 'O', 2)\n",
      "('be', 'O', 2)\n",
      "('unlikely', 'O', -100)\n",
      "('.', 'O', 2)\n",
      "('[', 'O', -100)\n",
      "('24', 'O', 2)\n",
      "(']', 'O', 2)\n",
      "('For', 'O', 2)\n",
      "('example', 'O', 2)\n",
      "(',', 'O', 2)\n",
      "('it', 'O', 2)\n",
      "('was', 'O', 2)\n",
      "('recently', 'O', 2)\n",
      "('shown', 'O', 2)\n",
      "('that', 'O', 2)\n",
      "('cubic', 'O', 2)\n",
      "('Zr_8O_6', 'O', 2)\n",
      "('clusters', 'O', 2)\n",
      "('appearing', 'O', -100)\n",
      "('via', 'O', -100)\n",
      "('structure', 'O', -100)\n",
      "('refinements', 'O', -100)\n",
      "('from', 'O', -100)\n",
      "('single-crystal', 'O', 2)\n",
      "('X-ray', 'O', 2)\n",
      "('diffraction', 'O', 2)\n",
      "('(', 'O', 2)\n",
      "('SCXRD', 'O', 2)\n",
      "(')', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('PCN-221', 'O', 2)\n",
      "('actually', 'O', -100)\n",
      "('resulted', 'O', -100)\n",
      "('from', 'O', 2)\n",
      "('the', 'O', -100)\n",
      "('superposition', 'O', -100)\n",
      "('of', 'O', 2)\n",
      "('disordered', 'O', 2)\n",
      "('Zr_6O_4', 'O', 2)\n",
      "('(', 'O', -100)\n",
      "('OH', 'O', -100)\n",
      "(')', 'O', 2)\n",
      "('_4', 'O', 2)\n",
      "('SBUs', 'O', 2)\n",
      "('E', 'B', -100)\n",
      "('.', 'I', 0)\n",
      "('coli', 'I', 1)\n",
      "('HT115', 'I', 1)\n",
      "('bacteria', 'O', 1)\n",
      "('were', 'O', -100)\n",
      "('inoculated', 'O', -100)\n",
      "('in', 'O', 2)\n",
      "('LB', 'O', 2)\n",
      "('starting', 'O', 2)\n",
      "('from', 'O', 2)\n",
      "('a', 'O', 2)\n",
      "('-80', 'O', 2)\n",
      "('°C', 'O', 2)\n",
      "('glycerol', 'O', 2)\n",
      "('stock', 'O', 2)\n",
      "('and', 'O', -100)\n",
      "('allowed', 'O', 2)\n",
      "('to', 'O', 2)\n",
      "('grow', 'O', 2)\n",
      "('for', 'O', 2)\n",
      "('1', 'O', 2)\n",
      "('hour', 'O', 2)\n",
      "('.', 'O', 2)\n"
     ]
    }
   ],
   "source": [
    "for j in range(len(df)):\n",
    "\tfor i in zip(df.loc[j,\"tokens\"],df.loc[j,\"ner_tags\"],df.loc[j,\"label_ids\"]):\n",
    "\t\tprint(i)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Predict sentences again"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "ner = \"ORGANISM\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "path= f\"../NER_output/{ner}\"\n",
    "\n",
    "tokenizer = AutoTokenizer.from_pretrained(path, model_max_length=512)\n",
    "model = AutoModelForTokenClassification.from_pretrained(path)\n",
    "\n",
    "nlp = pipeline(task='ner', model=model, tokenizer=tokenizer,\n",
    "               aggregation_strategy=\"average\", device=2)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The type strain, Sr54 ^T (=CGMCC 1.9249 ^T = NBRC 114034 ^T ), was isolated from the ice sample obtained from the supraglacial zone of No. 1 glacier in Xinjiang Uygur Autonomous Region, China.\n",
      "[]\n",
      "Organophosphorus hydrolase (OPH; EC 3.1.8.1), encoded by the identical opd genes from soil bacterium Pseudomonas diminuta MG and Flavobacterium sp.\n",
      "[]\n",
      "A human stool-derived Bilophila wadsworthia strain caused systemic inflammation in specific-pathogen-free mice (49).\n",
      "[{'entity_group': 'I', 'score': 0.6806049, 'word': 'pathogen - free mice', 'start': 92, 'end': 110}]\n",
      "The same applied to the potential donor E. coli 10716 which did not transfer plasmids to the three potential recipients Serratia marcescens subsp. marcescens DSM 30122, Enterobacter cloacae DSM 30060 and Proteus mirabilis DSM 4479.\n",
      "[]\n",
      "To further verify the function of c-di-GMP in controlling the switch between the two stages, a c-di-GMP phosphodiesterase (PDE), PA4781, ^28 was overexpressed in KT1115 and PAO1Δ wspF to test whether the alteration of the c-di-GMP level would affect the switch phenomenon and the final RLs production.\n",
      "[]\n",
      "A recent study showed that Deinococcus radiodurans RecN stimulates RecA-mediated HR in vitro [36].\n",
      "[]\n",
      "In addition, the P248L substitution abolished the RDAR colony morphology of FORC_078 and led to the SAW colony morphology similar to that of FORC_075 ( Fig 3b ).\n",
      "[]\n",
      "Results showed that groEL gene sequences (GP1-1 and GP2-1) identical to that of the HKU75 ^T isolate could be amplified from the oral cavities of both guinea pigs ( Fig 2 ), indicating that guinea pigs are likely a reservoir of S. oriscaviae .\n",
      "[{'entity_group': 'B', 'score': 0.5875229, 'word': 'guinea pigs', 'start': 151, 'end': 162}, {'entity_group': 'B', 'score': 0.5788751, 'word': 'guinea pigs', 'start': 190, 'end': 201}]\n",
      "We subjected natural accessions of A. thaliana to infection by the Xanthomonas campestris pv. campestris (Xcc) strain 568.\n",
      "[{'entity_group': 'B', 'score': 0.7148529, 'word': 'a', 'start': 35, 'end': 36}, {'entity_group': 'I', 'score': 0.5736308, 'word': '. thaliana', 'start': 36, 'end': 46}]\n",
      "In order to obtain a genome-wide picture of genes controlled by kstR, we carried out competitive hybridizations between cDNA from wild-type M. smegmatis and the mutant strain ΔkstR1 using M. smegmatis microarrays.\n",
      "[]\n",
      "Subsequently, the infected DF-1 cells were washed with sterile 1X PBS to remove non-adherent mycoplasmas, and incubated for 3 h either with gentamicin media (+Gm) to kill adherent mycoplasmas, or antibiotic-free media (−Gm), retaining all cell-associated bacteria, both invaded and adhered (Fig 3C).\n",
      "[{'entity_group': 'B', 'score': 0.5854736, 'word': 'df -', 'start': 27, 'end': 30}, {'entity_group': 'I', 'score': 0.5258041, 'word': '1', 'start': 30, 'end': 31}]\n",
      "To identify bacterial traits important for growth in potato ( Solanum tuberosum ) tubers, we examined three strains across two species: D. dadantii 3937 ( Dda 3937), D. dianthicola ME23 ( Ddia ME23), and D. dianthicola 67-19 ( Ddia 6719).\n",
      "[{'entity_group': 'B', 'score': 0.6315271, 'word': 'potato', 'start': 53, 'end': 59}, {'entity_group': 'B', 'score': 0.60047144, 'word': 'solanum', 'start': 62, 'end': 69}, {'entity_group': 'I', 'score': 0.55361146, 'word': 'tuberosum', 'start': 70, 'end': 79}]\n",
      "However, Spiraeoideae-infecting strains with identical plasmid content (e.g. only pEA29) do not always exhibit identical phenotypes [22], [51].\n",
      "[]\n",
      "Considering that the B. distachyon BdCYP711A29 is involved in orobanchol biosynthesis, 4 weeks-old (end of the vegetative stage in our conditions) plants of overexpressing lines and TILLING mutants and their respective control lines were examined for the number of tillers and compared to the WT Bd21-3 ecotype.\n",
      "[{'entity_group': 'B', 'score': 0.5711704, 'word': 'b', 'start': 21, 'end': 22}, {'entity_group': 'I', 'score': 0.540351, 'word': 'distachyon', 'start': 24, 'end': 34}, {'entity_group': 'B', 'score': 0.49041498, 'word': 'bd21', 'start': 296, 'end': 300}, {'entity_group': 'I', 'score': 0.6023549, 'word': '- 3', 'start': 300, 'end': 302}]\n",
      "MOV97 was started from P. gossypiella collected in the Mohave Valley of western Arizona in 1997 and was not exposed to toxin in the laboratory (Tabashnik et al.\n",
      "[{'entity_group': 'B', 'score': 0.76892966, 'word': 'p', 'start': 23, 'end': 24}, {'entity_group': 'I', 'score': 0.72877127, 'word': '. gossypiella', 'start': 24, 'end': 37}]\n",
      "The RNA-Seq data revealed transcription of all 10 histidine biosynthesis genes, but it did not give any evidence for transcripts spanning the non-coding region between cg2301 and hisH in C. glutamicum ATCC 13032 under all tested conditions (complex and minimal media, various stresses; R.K.\n",
      "[]\n",
      "Innate immunity is crucial for the elimination of SARS-CoV-2, the removal of dead cells, and restoring tissue function, as well as coordinating/sustaining the protective adaptive immune responses of virus-specific T and B lymphocytes [3,4].\n",
      "[]\n",
      "This result was similar to the relationship reported between strains KA5-A and KA5-B ^T ( Sakai et al. , 2014 ), indicating that rhizosphere soils are one of the preferential habitats for agar-degrading bacteria, such as KA5-B ^T .\n",
      "[]\n",
      "However, RP437 exhibits a growth rate which is approximately 10% lower than MG1655-motile and a migration rate in rich medium a factor of two lower than MG1655-motile.\n",
      "[]\n",
      "For whole-cell fatty acid and polar lipid analysis, the strains 16-SW-7, P. distincta KMM 638 ^T and P. paragorgicola KMM 3548 ^T were grown under optimal physiological conditions for all strains (at 28°C for 24 h on MA).\n",
      "[]\n",
      "Compounds H-151 (InvivoGen) and amlexanox (Abcam) were dissolved in DMSO to create stock solutions.\n",
      "[]\n",
      "Coincidentally, both V. fujianensis FJ201301 ^T and V. metschnikovii JCM 21189 ^T had 3284 core genes.\n",
      "[]\n",
      "Our study showed that IAA production is also a feature of SRS89 strain isolated from stevia seeds.\n",
      "[]\n",
      "Note that only the latter scheme has attached cgST definitions, i.e., unique cgST identifiers are attached to each distinct allelic profile.\n",
      "[]\n",
      "S. citri strain C189 was established in 1972 from a Navel orange tree ( Citrus sinensis (L.)\n",
      "[{'entity_group': 'B', 'score': 0.4725687, 'word': 'navel', 'start': 52, 'end': 57}, {'entity_group': 'I', 'score': 0.7178272, 'word': 'orange tree', 'start': 58, 'end': 69}, {'entity_group': 'B', 'score': 0.48589888, 'word': 'citrus', 'start': 72, 'end': 78}, {'entity_group': 'I', 'score': 0.7028686, 'word': 'sinensis ( l. )', 'start': 79, 'end': 92}]\n",
      "Trichomonas vaginalis isolate T016 and Tritrichomonas foetus (02-97) were grown in Trypticase-yeast extract-maltose (TYM) medium supplemented with 10% heat-inactivated horse serum [32].\n",
      "[]\n",
      "In the FO-36b genome, this phage element contains 94 genes (RS87_14155 to RS87_14625).\n",
      "[]\n",
      "We assembled a dataset of complete genomes of Agrobacterium G3, including sequences from strains isolated from various regions and environments: an eosin flask from a hospital environment in France (CFBP 6623, sequenced twice), a cave in Lechuguilla, United States (LC34), algae in the United Kingdom (SUL3), Arabidopsis roots in Germany (Root651), and bioprospecting for lignocellulolytic microbes and enzymes from natural, highly evolved plant biomass-degrading systems in the United States (UGM030330-04).\n",
      "[]\n",
      "What triggers H. fraxineus to switch from a commensal endophyte on Asian ash to a virulent pathogen on European ash is, as yet, unknown.\n",
      "[{'entity_group': 'B', 'score': 0.45004126, 'word': 'h', 'start': 14, 'end': 15}, {'entity_group': 'I', 'score': 0.50234044, 'word': 'fraxineus', 'start': 17, 'end': 26}, {'entity_group': 'B', 'score': 0.62248945, 'word': 'asian', 'start': 67, 'end': 72}, {'entity_group': 'I', 'score': 0.62699497, 'word': 'ash', 'start': 73, 'end': 76}, {'entity_group': 'B', 'score': 0.62808925, 'word': 'european', 'start': 103, 'end': 111}, {'entity_group': 'I', 'score': 0.59427035, 'word': 'ash', 'start': 112, 'end': 115}]\n",
      "On the species status of the root‐knot nematode Meloidogyne ulmi Palmisano & Ambrogioni, 2000 (Nematoda, Meloidogynidae).\n",
      "[{'entity_group': 'B', 'score': 0.40622178, 'word': 'root', 'start': 29, 'end': 33}, {'entity_group': 'I', 'score': 0.59696895, 'word': '‐ knot nematode meloidogyne ulmi palmisano & ambrogioni', 'start': 33, 'end': 87}]\n",
      "The CDW generated by P. ganghwense C2.2 ( Figure 5 ) reached 13.55 g/L (±2.15) after only 48 h of using CG as the carbon source and was maintained until 96 h.\n",
      "[]\n",
      "P. pentosaceus LI05 (CGMCC 7049), isolated from the fecal samples of a healthy volunteer, demonstrated acid-tolerant and bile-tolerant traits and our previous results demonstrated a potentially protective effects against enteropathogens (Lv et al., 2014a,b; Shi et al., 2017).\n",
      "[]\n",
      "The HLA-A*0201-positive melanoma line mel526 was obtained from the Surgery Branch of the NCI.\n",
      "[]\n",
      "Metastatic PCa cell lines PC3, DU145, LNCaP were purchased from the ATCC Centre; non-metastatic PC346 Flu-1 [69] and PC346 Flu-2 [69] were kindly provided by Prof Jenster group (University Medical Center Rotterdam, the Netherlands).\n",
      "[]\n",
      "This article reports the complete genome sequence of R. planticola strain HH15, containing multiple metal and AMR genes and isolated in 2010 from a pretreated sample in a Canadian wastewater treatment facility.\n",
      "[]\n",
      "As a result, strains CPCC 101082 ^T and CPCC 101083 ^T , isolated from sandy soil samples collected from Badain Jaran Desert, showed indole-3-acetic acid (IAA)-producing activity.\n",
      "[]\n",
      "Known gold-copper detoxification systems (CupR/CueR, CupA, CupC and CopA) found in C. metallidurans CH34 were also present in C. metallidurans BS1.\n",
      "[]\n",
      "We hypothesized that Orientalis strains MG05-1020, IP275 and FV-1 came into being by losing certain DFRs from genomovar09.\n",
      "[]\n",
      "Loganin is an iridoid glycoside extracted from the plant Strychnos nux-vomica Linn L; it is also distributed in the plants Oleander Branch and Columelliaceae.\n",
      "[{'entity_group': 'I', 'score': 0.76159674, 'word': 'strychnos nux - vomica linn l', 'start': 57, 'end': 84}, {'entity_group': 'B', 'score': 0.41247973, 'word': 'columelliaceae', 'start': 143, 'end': 157}]\n",
      "This study aimed to create an in-depth view of the utilization of Photobacterium ganghwense C2.2 for PHA production by linking a wide array of characterization methods: metabolic pathway annotation from the strain’s complete genome, high-throughput phenotypic tests, and biomass analyses through plate-based assays and flask and bioreactor cultivations.\n",
      "[]\n",
      "The biofilm exopolysaccharides production was significantly decreased in KT1115 and PAO1Δ wspF after PA4781 overexpressed ( Figures 4 B-4D).\n",
      "[]\n",
      "H . pylori bacteria deficient in the bifunctional LPS biosynthesis enzyme hldE ( rfaE , gene HP0858 in strain 26695 [ 86 ]), which abolishes the biosynthesis of HBP, would, by inference from the LPS biosynthesis pathways of other Gram-negative bacteria [ 87 ], accumulate the intermediate metabolite sedoheptulose-7-phosphate.\n",
      "[]\n",
      "Aliivibrio (formerly Vibrio) fischeri NRRL B-11177 (ATCC 49387) was grown in marine broth (MB) at 30 °C and 100-rpm rotary shaking.\n",
      "[]\n",
      "V. mediterranei strain CyArs1 was isolated from Cinachyrella specimens collected in Aimen (Penghu archipelago, Taiwan), after plating host homogenates on half-strength marine agar medium and incubating at 17°C ( 5 ).\n",
      "[{'entity_group': 'I', 'score': 0.50291026, 'word': 'cinachyrella', 'start': 48, 'end': 60}]\n",
      "The studied products included mainly grain-based gins (comprising three rye-based Finnish gins) and one wine distillate-based gin.\n",
      "[]\n",
      "Total RNA was isolated using RNeasy Lipid Tissue Mini Kit (Qiagen, Hilden, Germany).\n",
      "[]\n",
      "The MT-4 cells transduced with CPSF6ΔNCs and the GFP dual-expressing lentiviral vector were assessed for HIV susceptibility using a VSV pseudotyped CFP-expressing lentiviral vector.\n",
      "[{'entity_group': 'B', 'score': 0.71797216, 'word': 'mt -', 'start': 4, 'end': 7}]\n",
      "The cells of strain YIM 96748 ^T were Gram-reaction-positive, aerobic, non-motile, catalase-positive actinomycetes, which formed a septal substrate mycelium that fragmented into rod-like elements.\n",
      "[]\n",
      "For the housekeeping gene GAPDH, 20 semitryptic peptides were identified in the melanoma cell line M14 and over 10 semitryptic peptides in the cell lines CCRF-CEM, Colo-205, MCF-7, and RXF-393.\n",
      "[{'entity_group': 'B', 'score': 0.54582864, 'word': 'ccrf', 'start': 154, 'end': 158}, {'entity_group': 'B', 'score': 0.53565, 'word': 'colo', 'start': 164, 'end': 168}, {'entity_group': 'B', 'score': 0.60325354, 'word': 'mcf -', 'start': 174, 'end': 178}, {'entity_group': 'B', 'score': 0.5859534, 'word': 'rxf', 'start': 185, 'end': 188}]\n",
      "Our findings were similar to those of a recent 30-year study at two institutions which reported the presence of digestive symptoms including abdominal pain, reflux, nausea/vomiting, bloating, constipation, and diarrhea following the Ladd procedure as part of a newly defined clinicopathologic gut malrotation syndrome [29].\n",
      "[]\n",
      "SAM-B isolated from a mineral cold spring located in Tyva, southern Siberia\n",
      "[]\n",
      "Drosophila S2 cells were grown in SFX medium (HyClone, Logan, UT, USA) at 25 °C.\n",
      "[{'entity_group': 'I', 'score': 0.72565633, 'word': 'drosophila s2 cells', 'start': 0, 'end': 19}]\n",
      "We have shown that Acss2 augments stress signaling by HIF-2 in fibrosarcoma (HT1080) and hepatocarcinoma (Hep3B) derived cell lines subjected to oxygen or glucose deprivation [15, 16].\n",
      "[]\n",
      "[ 32 ], N. oceani strain NS58 was isolated into pure culture from Tokyo Bay sediment at the West East end of the North Pacific Gyre, which is clockwise connected to the location of AFC27 by the Kuroshio and North Pacific currents as well as the California and North Equatorial Currents ( Figure 1 ).\n",
      "[]\n",
      "Svante Pääbo, a renowned founder of paleogenetics, with his collaborators has provided a sequence of the genome of the Homo Neanderthalis (34).\n",
      "[]\n",
      "Among rare taxa that showed sex-dependent associations with NAFLD, order RF39 has been positively associated with healthy diets [16,52] and negatively associated with BMI, blood triglycerides, and frailty among older adults [52,53,54,55,56], suggesting its potential health benefits.\n",
      "[]\n",
      "Here, we present the genome of Synechococcus sp. strain LA31, which was recently isolated from a temperate estuary.\n",
      "[]\n",
      "With the aim to evaluate the influence on the attachment and the effect over the solubilization of a polymetallic mineral concentrate, the behavior of a microbial consortium formed by Acidithiobacillus thiooxidans DSM 14887T and Leptospirillum ferrooxidans DSM 2705T previously induced with D-galactose for the early formation of EPS was studied.\n",
      "[]\n",
      "The genomic G+C content of R. centenum (70.5%) is higher than the genome of the purple non-sulfur bacterium Rhodospirillum rubrum ATCC 11170 (65.4%; GenBank accession CP000230), its closest sequenced relative.\n",
      "[]\n",
      "Taken together, these results demonstrate that the multicircuited regulatory networks governing fat usage appear well conserved between C. elegans nhr-49 and the mammalian PPARα.\n",
      "[{'entity_group': 'B', 'score': 0.5939047, 'word': 'c', 'start': 136, 'end': 137}]\n",
      "Currently, we cannot discriminate between these two possibilities, and more research is needed to further clarify the molecular pathways involved in GABA- mediated effects in neuronal integrity preservation by HT115.\n",
      "[]\n",
      "Escherichia coli strains DH5alpha, DH10 EMBacY [38] (kindly provided by Imre Berger, University of Bristol, UK) and BL21-CodonPlus (DE3)-RIL (Agilent) were grown at 37°C on LB agar plates or in LB liquid medium with shaking at 200 rpm.\n",
      "[]\n",
      "Indeed, a PAM250 matrix showed that B. elkanii strains SEMIA 587, CCBAU 05737, CCBAU 43297 and USDA 94 shared 100% global sequence similarity among themselves for both NodD _1 and NodD _2 ; consequently, each protein set for these strains grouped together.\n",
      "[]\n",
      "They show that populations of T. eungellensis now persist with stable, endemic infections of B. dendrobatidis.\n",
      "[]\n",
      "However, each pol3-01,L612M,eex strain produced proportionally similar levels of C → T/G → A and A → G/T → C transitions, in contrast to the POL3/pol3-01,L612M, pol3-01, and pol3-L612M strains (Fig 2B, Table S1).\n",
      "[]\n",
      "As with the initial study, one MBO Ravenel-infected steer had a single small granuloma in the left caudal lung lobe.\n",
      "[{'entity_group': 'B', 'score': 0.64046156, 'word': 'steer', 'start': 52, 'end': 57}]\n",
      "Two new peptides laxaphycins B5 and B6 isolated from a freshwater cyanobacterium, Phormidium sp., UIC 10484 completed this series [26].\n",
      "[]\n",
      "Two Gram-staining negative strains (CPCC 101082 T and CPCC 101083 T ) were isolated from biological sandy soil crusts samples collected from Badain Jaran desert, China.\n",
      "[]\n",
      "E. ulmoides is a tree cultivated in China for the production of gutta-percha [116].\n",
      "[{'entity_group': 'B', 'score': 0.5627506, 'word': 'e', 'start': 0, 'end': 1}, {'entity_group': 'I', 'score': 0.67860126, 'word': '. ulmoides', 'start': 1, 'end': 11}, {'entity_group': 'I', 'score': 0.37555605, 'word': 'tree', 'start': 17, 'end': 21}]\n",
      "This suggests that P. luminescens HIM3 has a larger arsenal of insecticidal toxins or greater specificity for D. magnifactella.\n",
      "[{'entity_group': 'B', 'score': 0.5670065, 'word': 'd', 'start': 110, 'end': 111}, {'entity_group': 'I', 'score': 0.6116712, 'word': '. magnifactella', 'start': 111, 'end': 126}]\n",
      "E. coli SJ7, 2EC1, NCCP15955, and RM-103-MS were isolated from the feces of homo sapiens.\n",
      "[{'entity_group': 'B', 'score': 0.576217, 'word': 'homo', 'start': 76, 'end': 80}, {'entity_group': 'I', 'score': 0.4850844, 'word': 'sapiens', 'start': 81, 'end': 88}]\n",
      "A significant difference is the higher presence of C _15:1 ω 6c detected in the P4023 ^T triad that enables its clear separation from the compared strains.\n",
      "[]\n",
      "The major menaquinones of AN20 ^T were MK-5 (60.8%) and MK-11 (29.32%), while the major menaquinones of AN421 ^T and AN502 were MK-5 (58.5% and 55.1%, respectively) and MK-10 (35.5% and 40.1%, respectively).\n",
      "[]\n",
      "In addition, we used as a control the expression pattern of V. anguillarum RV22 grown at 25°C in CM9 supplemented with iron (high-iron conditions).\n",
      "[]\n",
      "Three bacterial acidophile species were used in this study, L. ferriphilum DSM 14647^T, S. thermosulfidooxidans DSM 9293^T, and A. caldus DSM 8584^T.\n",
      "[]\n",
      "The dDDH values ranged from 97.7 to 98.0% compared to R. badensis DSM 100043 ^T , and were less than 24.0% when compared to R. silvae 213 ^T , R. chamberiensis 130333 ^T , and R. aceris SAP-1 ^T ( Table 2 ).\n",
      "[]\n",
      "In this report, we systematically characterize 32 response regulators (RRs) from a metal tolerant groundwater isolate, Pseudomonas stutzeri RCH2 to assess the impact of host-derived post-translational phosphorylation.\n",
      "[]\n",
      "Samples of no added DNA, human Donor DNA, N. lactamica HPA1, N. meningitidis strain Z5005 (Serogroup A), N. meningitidis strain Z4673 (Serogroup B) and S. agalactiae amplified targets were used as negative and positive controls.\n",
      "[]\n",
      "However, in BS1 another homologous czc determinant was identified on the chromid, most similar to the czc determinant from pMOL30 in CH34.\n",
      "[]\n",
      "Bacterial strains were grown in solid LB media with 25 μg/mL of streptomycin for E . coli OP50, 25 μg/mL of tetracycline for E . coli HT115, and 20 μg/mL of kanamycin and chloramphenicol for HT115Δ gad at 37 °C for 10 hours.\n",
      "[]\n",
      "There are a substantial number of transcriptional regulators, such as Cyanobacterium anabaena AztR (Liu et al., 2005), Oscillatoria brevis BxmR (Liu et al., 2004), Staphylococcus aureus CadC (Endo and Silver, 1995), Mycobacterium tuberculosis CmtR (Cavet et al., 2002), Synechococcus elongatus SmtB (Vanzile et al., 2002), Pseudomonas putida CadR (Lee et al., 2001), and E. coli ZntR (Brocklehurst et al., 1999), identified to recognize cadmium ions.\n",
      "[]\n",
      "Furthermore, two additional subunits of H _2 -sensing hydrogenase gene (hupUV) along with a histidine kinase (hupT) gene were exclusively found in the EPR9N genome among the Bathymodiolus mussel symbionts (Fig 4 a).\n",
      "[{'entity_group': 'I', 'score': 0.4651025, 'word': 'bathymodiolus mussel', 'start': 174, 'end': 194}]\n",
      "Rodríguez et al. [65] investigated GABA-producing LAB strains during spontaneous fermented quinoa sourdough, and found that Lb. plantarum strain CRL1905, Leuconostoc mesenteroides strain CRL1907 and Lb. brevis were dominant species in the consortium.\n",
      "[]\n",
      "We had previously reported 15 peroxide resistance genes in SAFR-032, of which 2 were not shared by either the earlier draft version of FO-36b, or the type strain ATCC7061 [ 38 ].\n",
      "[]\n",
      "Though LB and PDB are both complex rich media, pH and/or specific available nutrients differed enough to clearly separate the gene fitness profiles for Dda 3937 and Ddia 6719, though not for Ddia ME23 ( Figure 1 ).\n",
      "[]\n",
      "Significantly increased piperacillin-tazobactam resistance rates were observed among several Gram-negative bacteria; Pseudomonas aeruginosa 0 to 5 (0-19.2%), Klebsiella oxytoca 0-3 (0-13%), and Enterobacter cloacae 0-6 (0-17.1%) (<0.01).\n",
      "[]\n",
      "B. anthracis Sterne 34F2 was obtained from Colorado Serum Company (USA) and grown with BHI broth or agar (Becton Dickinson: Franklin Lakes, NJ, USA) or heart infusion broth (HIB); (Research Products International: Mount Prospect, IL, USA) at 37 °C.\n",
      "[]\n",
      "RTK17.1 was previously isolated from geothermally heated soils sampled from Rotokawa, New Zealand ( Carere et al., 2017 ), and shares 99% 16S rRNA gene sequence identity to Methylacidiphilum infernorum V4.\n",
      "[]\n",
      "HGT5 ( fig 4 E ) is unique to B. apis strains A29, B8, and C6, all strains that had been isolated from honey bee larvae.\n",
      "[{'entity_group': 'B', 'score': 0.71946156, 'word': 'honey', 'start': 103, 'end': 108}, {'entity_group': 'I', 'score': 0.51708204, 'word': 'bee larvae', 'start': 109, 'end': 119}]\n",
      "1,4-bis[(7-Hydroxy-4-methylcoumarin-8-yl)methyl] piperazine (2): FT-IR: (KBr), cm^−1: 3200-3050 (OH); 3048 (aromatic C-H); 2958 (aliphatic C-H); 1722 (C=O, lactone); 1627 (C=C, alkene); 1602 (C=C, aromatic).\n",
      "[]\n",
      "• That E. coli HT115 produces the neurotransmitter GABA, which is identified as the main molecule protecting from neurodegeneration.\n",
      "[]\n",
      "Cultures of L. reuteri AMBV336 contained 1.07 ± 0.12 μg riboflavin/ml, cultures of L. reuteri RC-14 contained 0.68 ± 0.22 μg riboflavin/ml and production of riboflavin by L. rhamnosus GG in MRS medium was undetectable with the HPLC-UV method used.\n",
      "[]\n",
      "To assess the efficiency of surface sterilization, 20 seeds were sterilized as described above, imprints were made by placing sterilized seeds on Lysogeny Broth (LB) (Sigma Aldrich L9234) agar for ∼30 s, followed by incubation at either 18 or 27°C for 24 h.\n",
      "[]\n",
      "pTP819, in which the cat gene is flanked on both sides by lac sequences, PaeR7 sites, and λ sequences (for crossing into the phage), has been described [2]. pTP828 and pTP829 were described previously (without names) as intermediate plasmids in the construction of pTP819 [2]; each bears the cat gene with a single lac flank and PaeR7 site. pTP922 was constructed by deleting the lac and cat sequences between the two PaeR7 sites in pTP819.\n",
      "[]\n",
      "This production level of BLA by B. licheniformis CBBD302(pHY-amyL) is amongst the highest levels in Gram-positive bacteria reported so far.\n",
      "[]\n",
      "The strains P285 and O277 were Gram negative short rod arranged in pairs and single, and non-motile.\n",
      "[]\n",
      "Figure 1A showed that Sun‐R cells exhibited increased cell viability compared with the control group in both ACHN and 786‐O cells.\n",
      "[{'entity_group': 'B', 'score': 0.7249248, 'word': 'sun', 'start': 22, 'end': 25}, {'entity_group': 'B', 'score': 0.62480026, 'word': 'achn', 'start': 109, 'end': 113}, {'entity_group': 'B', 'score': 0.7280545, 'word': '786', 'start': 118, 'end': 121}, {'entity_group': 'I', 'score': 0.51070297, 'word': '‐', 'start': 121, 'end': 122}]\n",
      "The differential occurrence of compensatory mtDNA mutations among strains might explain the relative stability of ΔmtDNA-C levels in EG4181 and PB800 compared with the other four strains.\n",
      "[]\n",
      "G. soehngenii” and the organisms studied previously, a strict dehalogenide-respiring Dehalococcoides mccartyi strain from the Chloroflexi phylum and the phosphite-oxidizing Deltaproteobacterium “Candidatus Phosphitivorax anaerolimi” Phox-21, respectively.\n",
      "[]\n",
      "For growth on bacterial lawns a scraping of Dictyostelium cells was taken from a colony growing on a E. aerogenes lawn using a sterile toothpick and placed into the centre of a Normal Agar plate (20 g·L^−1 agar (Oxoid, Basingstoke, Hampshire, UK), 1 g·L^−1 Bacto peptone (Difco, Detroit, MI, USA), 1.1 g·L^−1 anhydrous glucose, 1.9972 g·L^−1 KH_2PO_4 and 0.356 g·L^−1 Na_2HPO_4.2H_2O, pH 6.0) containing a lawn of E. coli B2.\n",
      "[]\n",
      "Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6 decreased the MDA content by 12 and 6%, respectively, while a decline of 29% (1.89 nmol g^-1 FW) was recorded in plants inoculated with the bacterial consortium.\n",
      "[]\n",
      "Profftella_Dco and Profftella_DC shared all genes for the biosynthesis of diaphorin, hemolysin, riboflavin, biotin, and carotenoids, underlining multiple roles of Profftella, which may contribute to stabilizing symbiotic relationships with the host.\n",
      "[]\n",
      "As reported experimentally under stable conditions of room temperature, viable SARS-CoV-2 was detectable up to 72 h on a plastic surface [75], while plastics remain the most common and most wide-ranging materials used for food packaging [76].\n",
      "[]\n",
      "We performed genome sequence comparisons between X. fastidiosa subsp. multiplex strain CFBP 8418, Xylella fastidiosa subsp. pauca strain 9a5c (isolated from citrus in Brazil) ( 29 ), and Xylella fastidiosa subsp. fastidiosa strain Temecula1 (isolated from grapevine in California) ( 30 ) to estimate the metabolic network conservation between the three subspecies ( Table S1 and Table S2 ).\n",
      "[{'entity_group': 'B', 'score': 0.694834, 'word': 'citrus', 'start': 157, 'end': 163}, {'entity_group': 'B', 'score': 0.5643208, 'word': 'grapevine', 'start': 256, 'end': 265}]\n",
      "5 x 10^5 transfected or treated HCC cells were plated into 60 mm dishes for 48 h or 72 h.\n",
      "[{'entity_group': 'B', 'score': 0.7884612, 'word': 'hcc', 'start': 32, 'end': 35}]\n",
      "One copy of a conserved cspB gene was found in all analyzed genomes and one copy of cspC gene was present in P7388 ^T , F. saccharophilum DSM 1811 ^T and F. pectinovorum DSM 6368 ^T .\n",
      "[]\n",
      "Through the detections of DOX-derived fluorescence in the cells, it can be observed that the DH82 cells exhibited relatively mild phagocytosis to the HIDPPDNE (Fig 4I, A-F) as compared with IDPPDNE (Fig 4I, a-f).\n",
      "[]\n",
      "To measure the MIC of the corresponding antibiotic in E. coli BL21 cells, liquid broth dilution tests were conducted as recommended by CLSI guidelines with Mueller-Hinton (MH) broth (Oxoid, United Kingdom).\n",
      "[]\n",
      "Complete Genome Sequence of Clostridium cadaveris IFB3C5, Isolated from a Human Colonic Adenocarcinoma\n",
      "[]\n",
      "As shown previously [9] , antibiotic resistant and tolerant variant colonies were cultured by exposing a P. aeruginosa PAO1 biofilm lawn to high concentrations of tobramycin through exposure using a CaSO _4 bead containing 1 mg of tobramycin (Fig 1 ).\n",
      "[]\n",
      "GH4 maltose-6′-phosphate glucosidase was conserved in all whisky strains and four milk strains, JCM 1120, CAUH18, H10 and R0052.\n",
      "[]\n",
      "The marine bacterium Moritella marina (MP-1) has been reported to produce unusually high levels of the PUFA docosahexaenoic acid (DHA) ( 3 , 4 ).\n",
      "[]\n",
      "FGT57.6, a male of Pakistani descent, presented with a retroperitoneal paraganglioma, and had three male relatives (from a large extended family) with similar clinical profiles.\n",
      "[]\n",
      "B. salanitronis BL78^T, DSM 18170, was grown anaerobically in DSMZ medium 104 (Peptone-Yeast extract-Glucose broth) [32] at 37°C.\n",
      "[]\n",
      "WT and mutant fusions were integrated into the chromosomal λ att site of E. coli strains CF7789 and TRCF7789 as described previously (35).\n",
      "[]\n",
      "Strains P7388 ^T , P9670, P7381, P7475, and CCM 9065 formed a well-supported cluster close to Flavobacterium resistens BD-b365 ^T and F. branchiicola 59B-3-09 ^T .\n",
      "[]\n",
      "CRL-1942; RRID CVCL_1714; ATCC) were cultured in RPMI 1640 supplemented with 10% (vol/vol) heat-inactivated FBS (Sigma), 0.1 mg ml^−1 streptomycin, and 100 U ml^−1 penicillin (all from Gibco) at 37°C under 5% CO_2.\n",
      "[]\n",
      "Lastly, for n -caproate as the electron acceptor, the initial total concentration of 102.4 ± 0.5 mmol C L ^-1 resulted in an undissociated n -caproic acid concentration of ∼19.8 mmol C L ^-1 (∼3.3 mM) at a pH value of 5.5, which completely inhibited the metabolism of strain 7D4C2 ( Figures 3A,G , Supplementary Figure 3 , and Supplementary Table 2 ).\n",
      "[]\n",
      "Strain PR63039 (Hu et al., 2017 ) and CCUG12901 were isolated from the urine of patients with postinjury urinary tract infection.\n",
      "[]\n",
      "Briefly, an overnight culture of P. aeruginosa PAO1 was diluted to an OD _600 of 0.1 using sterile LB broth.\n",
      "[]\n",
      "This may raise a serious public health concern, because HAdV-55 has the potential to spread and cause severe epidemics in China, and it may become a major etiological agent causing pneumonia among the Chinese population.\n",
      "[]\n",
      "Cholera toxin ctxA , thermosolve hemolysin tlh / tdh , and other toxins (e.g., hlyA and rtxA / B ) were not detected in V. fujianensis FJ201301 ^T .\n",
      "[]\n",
      "This domain exhibits high similarity to other NADP^+ binding domains such as the 3-hydroxyisobutyrate dehydrogenase (HIBADH) from Thermus thermophilus HB8[35], with a root-mean-square deviation (rmsd) of 2.3 Å.\n",
      "[]\n",
      "K2606 strain also showed a significantly higher virulence levels compared to HS11286 and J53 strains in both mice and wax moth larva infection models ( Figure 2 ).\n",
      "[{'entity_group': 'B', 'score': 0.6058931, 'word': 'wax moth', 'start': 118, 'end': 126}, {'entity_group': 'I', 'score': 0.45579103, 'word': 'larva', 'start': 127, 'end': 132}]\n",
      "In agreement with this, a new classification of fungi based exclusively on genome-derived analysis of CAZymes has grouped H. fraxineus with polymertroph pathogens with a narrow host range, corresponding to necrotrophs (Hane et al.\n",
      "[{'entity_group': 'B', 'score': 0.5438062, 'word': 'fungi', 'start': 48, 'end': 53}, {'entity_group': 'B', 'score': 0.7336919, 'word': 'h', 'start': 122, 'end': 123}, {'entity_group': 'I', 'score': 0.49387604, 'word': '. fraxineus', 'start': 123, 'end': 134}]\n",
      "The complete genome sequences, as determined by a combination of short- and long-read sequencing, of three Lactiplantibacillus plantarum strains (M8, M17, and M19) that were isolated from Iranian motal cheese are reported.\n",
      "[]\n",
      "Interestingly, C_2-ceramide induced an extra effect of senescence phenotype in MCF-7 rather than MDA-MB-231 (Figure 2).\n",
      "[{'entity_group': 'B', 'score': 0.7235984, 'word': 'mcf -', 'start': 79, 'end': 83}, {'entity_group': 'I', 'score': 0.49983412, 'word': '7', 'start': 83, 'end': 84}, {'entity_group': 'B', 'score': 0.5853888, 'word': 'mda - mb', 'start': 97, 'end': 103}, {'entity_group': 'I', 'score': 0.5768789, 'word': '- 231', 'start': 103, 'end': 107}]\n",
      "Klebsiella oxytoca fermented with a high concentration of molasses exhibited increased production of 2, 3-butanediol (Afschar et al.\n",
      "[]\n",
      "Our results show that in human macrophages, the protein expression for cathepsins B and S was lowered during infection with MTB, regardless of the activation state of the host cells (Fig 2A).\n",
      "[{'entity_group': 'B', 'score': 0.6334911, 'word': 'human', 'start': 25, 'end': 30}]\n",
      "Indeed, cardiomyocyte-detubulation has been shown to provoke alterations in EC-coupling and calcium dynamics.\n",
      "[]\n",
      "[147] AGDC1 from Arxula adenivorans and PPP2 from Madurella mycetomatis both form trimers and do not require any organic cofactor.\n",
      "[]\n",
      "Factors affecting the prognosis such as ICD implantations, medication intake, and compliance could not be investigated.\n",
      "[]\n",
      "To investigate, whether α_2β_2 TS complexes are also formed in vivo, protein extracts of B73 wildtype maize and the mutant lines bx1, Zmtsb1, and Zmtsb2 were separated by size exclusion chromatography.\n",
      "[{'entity_group': 'B', 'score': 0.5761077, 'word': 'b73 wildtype maize', 'start': 89, 'end': 107}]\n",
      "However, in our analysis of relative transcript levels by Illumina arrays, ITGB1 appeared to be expressed in all cell types and conditions, with relatively high levels in the fBAT-derived preadipocytes, but relatively low levels in NP88 and NP110 progenitors (Additional file 7: Table S4).\n",
      "[{'entity_group': 'B', 'score': 0.5280152, 'word': 'fbat', 'start': 175, 'end': 179}, {'entity_group': 'B', 'score': 0.4839759, 'word': 'np88', 'start': 232, 'end': 236}, {'entity_group': 'B', 'score': 0.36228684, 'word': 'np110', 'start': 241, 'end': 246}]\n",
      "The voltage-dependence of currents induced by dicarboxylates in the EAAT3 D444S/T448A mutant showed that the current reversal potential was shifted approximately 15-20 mV negative relative to EAAT3, suggesting that the ratio of the uncoupled chloride current [30] to the coupled transport current may be increased in the mutant.\n",
      "[]\n",
      "A similar mechanism of activation, i.e. addition to fumarate, was proposed for the anaerobic degradation of alkanes based on the finding of alkylsuccinates in cultures of sulfate-reducing bacteria growing with n-dodecane [57], and of a nitrate-reducing bacterium, ‘Aromatoleum’ sp. strain HxN1 growing on n-hexane [58].\n",
      "[]\n",
      "Recently, the latent carboxylation activity of the propionyl-CoA synthase from Erythrobacter sp.\n",
      "[]\n",
      "Because a previous study suggested that the ZirT-dependent secretion system may play a potential role in biofilm formation [ 60 ], the zirT gene was deleted in the FORC_078 genetic background, and the biofilm- and RDAR colony-forming abilities were determined.\n",
      "[]\n",
      "Analyses of differentially expressed genes showed that compared with the stationary phase (24 h), strain P. agarivorans Hao 2018 had upregulated expression of genes related to the synthesis of extracellular polysaccharides in the logarithmic growth phase (2 h), and that the expression of these genes affected extracellular polysaccharide transport, nucleotide sugar synthesis, and glycosyltransferase synthesis.\n",
      "[]\n",
      "For anaerobic photoautotrophic growth, AB26 strains were grown with H _2 (80% H _2 : 20% CO _2 at 50 kPa) or 10 mM sodium thiosulfate or 5 mM Fe(II) chloride in SW medium.\n",
      "[]\n",
      "The same arrangement of van genes was observed in Bl. pseudococcoides SCSK and Blautia sp.\n",
      "[]\n",
      "SLAB51 contains Streptococcus thermophilus DSM 32245, Bifidobacterium lactis DSM32246, Bifidobacterium lactis DSM 32247, Lactobacillus acidophilus DSM 32241, Lactobacillus helveticus DSM 32242, Lactobacillus paracasei DSM 32243, Lactobacillus plantarum DSM 32244 and Lactobacillus brevis DSM 27961 (Sivomixx800^®, Ormendes SA, Lausanne, Switzerland).\n",
      "[]\n",
      "The bacterial agents selected for this study were as follows: Bacillus anthracis Sterne, a Gram-positive spore-former, Staphylococcus aureus, a Gram-positive non-spore forming organism, and Gram-negative organisms Francisella novicida U112 and Yersinia pestis KIM D27.\n",
      "[]\n",
      "In silico analysis revealed the presence of a PlcR recognition site (TATGNAN_4TNCATA) about 100 nucleotides upstream from the three pep genes in B. cereus ATCC 14579.\n",
      "[]\n",
      "ZH07 was grown at 30°C on Ashby medium for 7 days, and then the growth of the colony was observed.\n",
      "[]\n",
      "SK-CO-15, a transformed human colonic epithelial cell line, was a gift from Dr.\n",
      "[{'entity_group': 'B', 'score': 0.6084122, 'word': 'sk -', 'start': 0, 'end': 3}, {'entity_group': 'I', 'score': 0.5560773, 'word': 'co - 15', 'start': 3, 'end': 8}, {'entity_group': 'B', 'score': 0.5871952, 'word': 'human', 'start': 24, 'end': 29}, {'entity_group': 'I', 'score': 0.45484045, 'word': 'epithelial', 'start': 38, 'end': 48}]\n",
      "In this study, E. coli DH5α, E. coli O157: H7, P. putida KT2440, P. aeruginosa PAO1, R. solanacearum, Rhodococcus RHA1, S. thermophilus ND03, S. aureus and other common strains were used to study the antibacterial activity of GA (C15:1), and GA was found to have significant antibacterial activity against Gram-positive bacteria but little effect on the growth of Gram-negative bacteria.\n",
      "[]\n",
      "The type strain is LMG 20602^T (=ATCC BAA-109^T = CDC G9805^T) and was isolated from sputum of a non-cystic fibrosis patient in the United States in 1996.\n",
      "[]\n",
      "The genome of H. fraxineus is about 62 Mbp and encodes a large and diversified set of host-interacting genes (McMullan et al.\n",
      "[{'entity_group': 'B', 'score': 0.49172902, 'word': 'h', 'start': 14, 'end': 15}, {'entity_group': 'I', 'score': 0.49182105, 'word': '. fraxineus', 'start': 15, 'end': 26}]\n",
      "Nevertheless, we set out to investigate the pathogenicity of all three Hymenoscyphus species, studied here.\n",
      "[{'entity_group': 'I', 'score': 0.42711926, 'word': 'hymenoscyphus', 'start': 71, 'end': 84}]\n",
      "Strains IME678, IME681 and IME682 were obtained by transforming strain IMX581 with pUDE672, pUDE1119 and pUDE1120, respectively.\n",
      "[]\n",
      "KT1115 showed rugose colonies with darker colors on Congo red agar and formed robust biofilms with approximately 1.6-fold higher levels of biofilm exopolysaccharides than PAO1, a model laboratory strain ( Figures 1 A and 1D).\n",
      "[]\n",
      "While the exon 1-targeted VU-1729-1:2 line was somewhat less invasive compared to the parental line (P = 0.0114), the exon 2-targeted line was profoundly impaired (Fig 4A), paralleling expression data.\n",
      "[]\n",
      "To elucidate this direct protection of OsiSh-2 on rice, further study of the isolation and purification of the antagonistic products of OsiSh-2 is under way.\n",
      "[{'entity_group': 'B', 'score': 0.7814906, 'word': 'rice', 'start': 50, 'end': 54}]\n",
      "Regarding the experimental procedure, the same method as that to evaluate the polysaccharide-degrading ability of strain KA5-B ^T was used ( Sakai et al. , 2014 ).\n",
      "[]\n",
      "Enterobacter cloacae EM was isolated from a contaminated batch of a commercial mineral supplement that is normally well stabilized with a mixture of methyl and propyl parabens.\n",
      "[]\n",
      "Therefore, this study aimed to reveal the potential of Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6 isolated from agricultural soils irrigated with wastewater to mitigate multi-metal toxicity in wheat.\n",
      "[{'entity_group': 'B', 'score': 0.7677976, 'word': 'wheat', 'start': 221, 'end': 226}]\n",
      "We next determined whether the label-free assay was able to generate kinetic parameters of aminoacylation, which would shed light on the mechanism of aminoacylation and would be particularly important to elucidate how the mechanism is perturbed by mutations.\n",
      "[]\n",
      "These images indicated that Methylomarinum strain SSMP-1 initially existed in close association with smaller rod-shaped bacteria (Fig 2A).\n",
      "[]\n",
      "Rhodopseudomonas sp. strain AZUL was isolated from a water sample obtained from the high altitude lagoon called Laguna Azul (27°34’ 17.3” S 68°32’ 19.6” W) kindly provided by María Eugenia Farías (PROIMI-CONICET).\n",
      "[{'entity_group': 'I', 'score': 0.58912843, 'word': 'farias', 'start': 189, 'end': 195}]\n",
      "Time-kill assays indicated the in vitro pharmacodynamic potential of AF4.\n",
      "[]\n",
      "Next, 100 μg of the resulting MVs were incubated with DQ12-45-1b in defined minimal medium, and the OD _600 was recorded.\n",
      "[]\n",
      "To test our hypothesis that overexpression of IP subunits derives alternative, more immunogenic peptides, we overexpressed both PSMB8 and PSMB9 (OE) or a vector control (EV) in three different melanoma cell lines (108T, 12T and A375) (Supplementary Fig 6a-c).\n",
      "[]\n",
      "The resulting PCR products were purified and digested with NheI-HF and XhoI restriction enzymes, ligated into the same sites of a predigested pET-28a vector (Merck Bioscience, Madison, WI, USA) and transformed into electrocompetent E. coli DH5α cells.\n",
      "[]\n",
      "“L. licerasiae” strain VAR 010^T and 2 other isolates from Iquitos, Peru (L. interrogans serovar Icterohaemorrhagiae, strains HAI188 and HAI156) that were isolated from leptospirosis patients were used to infect hamsters (N = 2 in each group).\n",
      "[{'entity_group': 'B', 'score': 0.7141226, 'word': 'hamsters', 'start': 212, 'end': 220}]\n",
      "We found that all the studied acr3 sequences are under negative selection (with similar dN/dS ratios ranging from 0.1202 to 0.1403) diverging from S17 sequence which seems to be from an independent branch.\n",
      "[]\n",
      "The selected PfNPNA-1 V_H/κ gene was excised from the rf JC-M13-88 DNA by digestion with XbaI and HindIII and ligated into similarly digested pAbHIS.\n",
      "[]\n",
      "These nine genes are totally unique to FO-36b with no homologs in the entire NR/NT databases (Table 2 ).\n",
      "[]\n",
      "It could be considered that treatment for a short time with cholesterol in the presence of CA may inflict a revisable inflammatory/proliferation stimulus that is followed by HSC apoptosis and mitigation of fibrosis.\n",
      "[]\n",
      "The antimicrobial activity of EPSs in vivo can also be explained by the prebiotic effect that helps probiotics to colonise the surface of the gastrointestinal tract, thus enhancing their competitive inhibition of pathogenic bacteria in the host [90].\n",
      "[]\n",
      "For T. cruzi, T. brucei, and C. fasciculata TrySs, amidase activities of <1-5% compared with the respective synthetase activities have been reported (3, 7, 9).\n",
      "[{'entity_group': 'B', 'score': 0.5413264, 'word': 'c', 'start': 29, 'end': 30}]\n",
      "Concentration of 2.0% NaCl inhibited growth of AN20 ^T and 1.5% NaCl inhibited growth of AN421 ^T and AN502.\n",
      "[]\n",
      "Mycobacterium marinum M and Mycobacterium smegmatis mc^2155 strains were grown at 30°C and 37°C, respectively, in Sauton's medium containing 0.025% of tyloxapol (Sigma) or on Middlebrook 7H10 agar plates supplemented with 10% Oleic acid-Albumin-Dextrose-Catalase (OADC) enrichment, in the presence of kanamycin (25 µg/ml), hygromycin (80 µg/ml) and zeocin (25 µg/ml), when required.\n",
      "[]\n",
      "Strains of P. agglomerans associated with the rhizosphere and plant tissues efficiently promote the growth of many cultivated plants such as the staple foods rice [ 32 , 33 ] and wheat [ 34 , 35 ], mostly by a combination of different plant growth-promoting mechanisms.\n",
      "[]\n",
      "Paired-end reads were run through Breseq ( Deatherage and Barrick, 2014 ) once using the ancestral phage EfV12-phi1 as the reference genome and once using the ancestral E. faecium TX1330 with default parameters.\n",
      "[]\n",
      "In addition, lcpA contributed to oxacillin resistance in two MRSA strains COLn and USA300 (Hubscher et al., 2009; Schaefer et al., 2017) and a methicillin-susceptible S. aureus (MSSA) MSSA1112 (Hubscher et al., 2009).\n",
      "[]\n",
      "To test the downstream effects of these differences in wasp infection strategies, as well as to reveal the molecular basis of Drosophila's response to parasitoid wasps, we undertook a genome-wide gene expression study of D. melanogaster larvae attacked by Lb17 and Lh14 at three time points post-infection (2-5 h, 9-12 h, and 21-24 h).\n",
      "[{'entity_group': 'B', 'score': 0.65220016, 'word': 'drosophila', 'start': 126, 'end': 136}, {'entity_group': 'B', 'score': 0.48778135, 'word': 'parasitoid', 'start': 151, 'end': 161}, {'entity_group': 'I', 'score': 0.42123508, 'word': 'wasps', 'start': 162, 'end': 167}, {'entity_group': 'B', 'score': 0.7996333, 'word': 'd', 'start': 221, 'end': 222}, {'entity_group': 'I', 'score': 0.56213754, 'word': '. melanogaster larvae', 'start': 222, 'end': 243}]\n",
      "More interestingly, the complex C K-edge composition of the organic carbon shell of the C/S microstructures is very similar to the to the composition of microbial cells, represented here by the spectrum of the bacterium Escherichia coli25, as well as extracellular twisted stalks produced by the Fe-oxidizing bacterium Mariprofundus ferrooxydans24.\n",
      "[]\n",
      "Arcobacter butzleri strain RM4018 is a motile bacterium that synthesizes a polar flagellum.\n",
      "[]\n",
      "In this study, A. melanogenum, P. laurentii, Ap. scarabaeorum and Wickerhamomyces sp. 2 were found to be positive for siderophore production.\n",
      "[]\n",
      "The type strain KOR34^T (DSM 104303^T = LMG 31362^T) was isolated from the biofilm community of Posidonia oceanica.\n",
      "[]\n",
      "The fluorescence intensity obtained for L. reuteri AMBV339 was close to the riboflavin control at 25 μg/ml, while L. reuteri AMBV336, L. reuteri RC-14, and L. rhamnosus GG culture fluorescence in MRS was much less pronounced.\n",
      "[]\n",
      "The 23S rRNA allele table was incremented with a sequence defined as allele 13, which carries the only mutation described so far as being associated with macrolide resistance [41] .\n",
      "[]\n",
      "However, a recent reinvestigation of the database showed that the initial annotation was erroneous, and that the sequence thought to belong to the genome of S. aureus, was in fact from another species of this genus, Staphylococcus sciuri^22.\n",
      "[]\n",
      "Histophilus somni OMP40 was identified as a porin with homology between its N-terminal amino acid sequence and the sequences of porins of other gram-negative bacteria The aim of this study was to produce recombinant H. somni OMP40 (rOMP40), optimize its production and evaluate its immunogenic properties in calves.\n",
      "[{'entity_group': 'B', 'score': 0.7841524, 'word': 'calves', 'start': 308, 'end': 314}]\n",
      "Pairwise distances between luminescent expression reporters show the similar effects of anthrose and decoyinine on expression profiles from atxA , lef , and pagA promoters in WT and Δ antC Sterne (Fig S6 ).\n",
      "[]\n",
      "The polymer produced by the soluble fraction of EPSB strain was identified as a class 1 dextran with a linear backbone containing consecutive α-(1→6)-linked D-glucopyranosyl units with few α-(1→3)-linked branches, whereas the cell-associated EPS is a polymer mixture consisting of a levan composed of linear chains of (2→6)-linked β-D-fructofuranosyl residues with β-(2→6) connections, and a class 1 dextran.\n",
      "[]\n",
      "We show that type II alveolar epithelial cells A549 express a range of inflammatory mediators in response to various virulence factors.\n",
      "[]\n",
      "On the contrary, the engineering strain BL21(DE3) Star-pMM had a lower biomass and produced a low titer of FPP in LB or YM9 medium.\n",
      "[]\n",
      "The results demonstrated that PRC-1 was a typical cephalosporinase with a high k_cat and strong hydrolytic activity (k_cat/K_m ratios were 220 × 10^-2 μM^-1⋅s^-1) for the first-generation cephalosporin cefazolin.\n",
      "[]\n",
      "However, 50% of Ln. pseudomesenteroides strains carried severely truncated dextransucrase gene which were primarily isolated from dairy except for Dm-9 (i.e., apple dumpster).\n",
      "[]\n",
      "Transcriptomics data of V. cholerae V52 single cultures in minimal medium were also generated and used to further refine i AM-Vc960 reconstruction and biomass objective function (see Table S1 at https://github.com/alyamahmoud/coinfection_modeling/blob/master/supplementary_material/supplementary_tables.xlsx ). i AM-Vc960 accounts for 2,172 reactions, 1,741 metabolites across three compartments (cytosol, periplasm, and extracellular compartments), and 960 metabolic genes.\n",
      "[]\n",
      "MAFF 211479 was isolated from ginger in Kahoku, Kochi Prefecture, Japan, in 1997.\n",
      "[]\n",
      "Analysis of the A. pleuropneumoniae L20 genome sequence revealed the presence of an intact sap operon (Figure 2A).\n",
      "[]\n",
      "As the green fluorescence of 2’,7’-dichlorofluorescein indicated, both mesoporous and non-mesoporous polydopamine nanoparticles could eliminate the ROS in an SH-SY5Y cell model, in which the former featured higher efficiency at the same particle concentration (Supplementary Fig 9).\n",
      "[]\n",
      "We hypothesized that the sortase A mutants, unable to attach to the polystyrene surface, would be defective for biofilm formation as recently reported for S. gordonii srtA mutant [42].\n",
      "[]\n",
      "At the first level of detoxification, the cytoplasmic efflux, the two strains BS1 and CH34 showed a high degree of similarity between the metal-transporting ATPases ( Figure 8 ).\n",
      "[]\n",
      "In addition, the mRNA levels of A3H in CEM-SS, CEM.NKR, and these cell clones were 100 to 1000-fold lower than those in H9 except that clone #7 expressed slightly higher A3H mRNA (Fig 3C).\n",
      "[]\n",
      "Acidithiobacillus ferrooxidans^T DSM 14882 was grown in Mackintosh (MAC) basal salt medium (Mackintosh, 1978), either with iron(II) ions, elemental sulfur or pyrite as energy source.\n",
      "[]\n",
      "The putative new subdivision consists entirely of uncultured bacteria from diverse ecological habitats, with exception of strain LW23 and Verrucomicrobiaceae bacterium GAS474 from forest soil: for example intestine of the earthworm Eisenia fetida [27] (97% identity), a low-level radioactive waste site [28] (91% identity), marine (coral) and aquatic moss, freshwater lake, as well as terrestrial habitats.\n",
      "[{'entity_group': 'I', 'score': 0.6907737, 'word': 'earthworm eisenia fetida', 'start': 222, 'end': 246}]\n",
      "AB2 is a non-spore forming, pleomorphic, Gram-positive, extracellular protease-producing bacterium that can grow from 12°C to 45°C and can tolerate alkaline (7<pH<11) and haline (0 to 15% wt/vol NaCl and 0 to 10% wt/vol MgCl_2) conditions.\n",
      "[]\n",
      "Under 200 ppm, the chlorophyll b content ranged from 1.81 to 2.25 mg/g FW and an increase of 22 and 23% was observed in plants inoculated with Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6, respectively.\n",
      "[]\n",
      "The present study contributes to the exploration of different strains of L. paracasei derived from non-conventional isolation sources-the South African traditional fermented drink mahewu (strains MA2 and MA3) and kefir grains (strains KF1 and ABK).\n",
      "[]\n",
      "The degradation of naphthalene by a hypersaline Phormidium tenue strain resulted in the formation of [1,2]naphthoquinone and naphthalene-1,2-diol presumably through an inducible enzyme system.\n",
      "[]\n",
      "Previously, we isolated two strains, M1 and HL1, from kefir grains of different sources, and identified tentatively L. kefiranofaciens by polymerase chain reaction-denaturing gradient gel electrophoresis (PCR-DGGE) targeting the V3 variable region of the 16S rRNA gene [ 6 ].\n",
      "[]\n",
      "Besides, strain AB211 showed positive motility through diffused cloudy growth away from the line of inoculation in semisolid M9 agar medium (Supplementary Figure S7D ).\n",
      "[]\n",
      "For the most part, the conductance value ranges are similar among S.1, S.2, and SD, with the exception of the potassium channels, which generally have higher conductance values in S.2 and SD, likely due to the removal of the slow delayed rectifier channel.\n",
      "[]\n",
      "Our results suggest that BME_RS128801 is also involved in biofilm responses to osmotic stress, The average biomass of ΔBME_RS12880 biofilm under 1.5 M NaCl was characted as 1.76 μm^3/μm^2 and average and maximum aggregates thickness, which were 2.16 and 4.38 μm, respectively.\n",
      "[]\n",
      "Reversible dissociation of V-ATPase is well characterized in S. cerevisiae14, but more recent studies suggest that the mammalian enzyme is regulated by a similar process in some cell types1516171819.\n",
      "[{'entity_group': 'B', 'score': 0.6932992, 'word': 's', 'start': 61, 'end': 62}]\n",
      "In our results, ZH07 showed the capacity of nitrogen fixation, phosphorus solubilization, potassium dissolution, siderophore production, and IAA synthesis, suggesting that ZH07 has the potential to promote plant growth.\n",
      "[]\n",
      "Therefore, KA5-B ^T like agar-degrading bacteria are still present and may be isolated from rhizosphere soils.\n",
      "[]\n",
      "Among non-synonymous SNPs unique to FORC_075, an SNP in envZ leading to an amino acid change from Pro248 to Leu248 was identified to result in the impaired biofilm formation and SAW colony morphology of S.\n",
      "[]\n",
      "The toxin genotyping of JXJA17 and Cp FORC_003 were type A, notwithstanding the fact that Cp FORC_003 was isolated from aquarium water in South Korea.\n",
      "[]\n",
      "We report the complete genome sequence of Mucilaginibacter strain 21P, which was isolated from estuarine soil contaminated with mine tailings from the Samarco disaster, which occurred in 2015 in Brazil.\n",
      "[]\n",
      "The variations in sugar content of the O-antigens contribute to the wide variety of antigenic types between the species and strains: UDP- N -acetyl-alpha;-D-mannosaminouronate biosynthesis in P. arctica , UDP-alpha-D-galacturonate biosynthesis I (from UDP-D-glucuronate) in P. distincta , with the exception of 16-SW-7 and U2A.\n",
      "[]\n",
      "The spore production at 72 h ranged from 5.64 to 6.97 log10 optical density (OD), and RT883 and 027 showed similar sporulation rates ( Fig 1 ).\n",
      "[]\n",
      "The genus Euryleptodes presently comprises of two species: Euryleptodes cavicola Heath & McGregor, 1912 and Euryleptodes insularis Hyman, 1953, both from California.\n",
      "[]\n",
      "Indeed, these protein families were the most divergent between G. intestinalis WB and GS [7].\n",
      "[{'entity_group': 'B', 'score': 0.6794353, 'word': 'g', 'start': 63, 'end': 64}, {'entity_group': 'I', 'score': 0.53825724, 'word': '. intestinalis wb', 'start': 64, 'end': 81}, {'entity_group': 'B', 'score': 0.45955116, 'word': 'gs', 'start': 86, 'end': 88}, {'entity_group': 'I', 'score': 0.60978353, 'word': '7', 'start': 90, 'end': 91}]\n",
      "The four currently published Acidihalobacter assembly sequences (GCF_000754095.2, GCF_001753245.1, GCF_001753165.1 and GCF_001975725.1) as well as the outgroup Halothiobacillus neapolitanus c2 (GCF_000024765.1) were downloaded from the National Center for Biotechnology Information (NCBI) GenBank genomic database in October 2019 ( Pruitt et al., 2012 ).\n",
      "[]\n",
      "Consistently, our own ChIP assays with primers targeting RD3 intron 1 showed that an anti-N-Myc antibody efficiently immunoprecipitated the RD3 gene intron 1 region enriched in Sp1-binding sites in BE(2)-C neuroblastoma cells (Figure 3E).\n",
      "[{'entity_group': 'B', 'score': 0.5805619, 'word': 'be', 'start': 198, 'end': 200}]\n",
      "Constructs NHBA_133-427 and NHBA_312-427, which include residues A133-D427 and M312-D427, respectively, were derived from NHBA peptide 20 (N. meningitidis strain 2996 [20] Uniprot Q9JPP1).\n",
      "[]\n",
      "The positive impact of Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6 under HM stress was clearly evident from morphological and biochemical improvement of plants, i.e., root and shoot length, plant biomass, and chlorophyll content, due to successful root colonization of the studied bacterial strains in wheat.\n",
      "[{'entity_group': 'B', 'score': 0.5622442, 'word': 'wheat', 'start': 329, 'end': 334}]\n",
      "This makes MP-1 interesting with regard to the biotechnological production of DHA ( 5 ).\n",
      "[]\n",
      "Hydrogen peroxide achieved similar concentration at 8 h of growth in most strains, except in strain SK678, which showed reduction in the production of H _2 O _2 ( Figure 3B ).\n",
      "[]\n",
      "We, therefore, assembled a dataset of complete genomes of Agrobacterium G3, including previously released sequences from five strains isolated from various geographical regions and environments: an eosin flask from a hospital environment in France (CFBP 6623, sequenced twice), a cave in Lechuguilla, United States (LC34), algae in the United Kingdom (SUL3), Arabidopsis roots in Germany (Root651), and bioprospecting for lignocellulolytic microbes and enzymes from natural, highly evolved plant biomass-degrading systems in the United States (UGM030330-04).\n",
      "[]\n",
      "For WF-MT5-A ^T , the major fatty acids are C _16 : 0 , followed by C _19 : 0 cyclo ω 8 c .\n",
      "[]\n",
      "Photobacterium ganghwense strain C2.2 (DSM 109767) is a marine bacterium isolated from Black Sea shore sediment, which has been shown to produce PHA using pure glycerol as a sole carbon source [ 7 ].\n",
      "[]\n",
      "Enterobacter cloacae strain JWM6 inoculated plants showed the highest increase in fresh shoot weight (143%) and dry weight (117%) as compared to uninoculated plants (Figure 3A).\n",
      "[]\n",
      "To demonstrate that RV22 synthesizes piscibactin in addition to vanchrobactin ( Soengas et al., 2006 , 2007 ), we used the previously described methodology, based on the use of HLB cartridges and liquid chromatography-mass spectrometry (LC-MS), that we developed for isolation of piscibactin ( Souto et al., 2012 ).\n",
      "[]\n",
      "AN421 ^T and AN502 contained minor amounts of MK-11, whereas AN20 ^T contained minor amounts of MK-12.\n",
      "[]\n",
      "P. luteoviolacea 2ta16 is a known producer of several natural products including the bright purple‐coloured antibiotic, violacein (Yang et al., 2007 ), and the thiomarinol family of dithiolopyrrolone antibiotics (Maansson et al., 2016 ).\n",
      "[]\n",
      "It is possible that hybrid bioelectronic processors will scale better than DNA-only systems.\n",
      "[]\n",
      "Thus, yellow pigmentation of the strain P7388 ^T is likely a result of a pigment mixture of flexirubin-type and carotenoid pigments resulting in the dark yellow to orange pigmentation of colonies.\n",
      "[]\n",
      "Position 627 of the PB2 subunit in A/HongKong/156/97 and A/duck/Fujian/01/02 is glutamic acid (E); the PB2 subunits of the other strains have a lysine (K) at 627.\n",
      "[]\n",
      "[6] Diacetyl, a butter-flavoring compound used in plant bakeries, has been linked to increased rates of bronchiolitis obliterans, while monosodium glutamate has been linked to obesity, metabolic disorders, neurotoxic effects, and reproductive organ damage.\n",
      "[]\n",
      "The apparent similarity of the main functional groups determined by FTIR spectrometry of the commercial rhamnolipid (R90) from Pseudomonas aeruginosa (AGAE Technologies, Corvallis, OR, USA) and the isolated biomolecule from strain BUN14 (Fig 2 ), suggested that the BUN14 biosurfactant product was composed on rhamnose rings with long hydrocarbon chains.\n",
      "[]\n",
      "Whether this holds true also for M. catarrhalis Msp22 remains to be elucidated.\n",
      "[]\n",
      "For instance, the type strain (168) makes polymers of glycerol, whereas another frequently studied strain (W23) makes polymers of ribitol.\n",
      "[]\n",
      "Two strains of MBO were used for the challenge inoculum: (1) MBO strain 95-1315 [USDA, Animal Plant and Health Inspection Service (APHIS) designation] originally isolated from a white-tailed deer in Michigan [ 20 ], USA and (2) MBO strain Ravenel (ATCC 35720) obtained from John Chan at Albert Einstein College of Medicine, Bronx, NY and freezer stocks were kept at NADC.\n",
      "[{'entity_group': 'B', 'score': 0.53412765, 'word': 'white', 'start': 178, 'end': 183}, {'entity_group': 'I', 'score': 0.7018769, 'word': '- tailed deer', 'start': 183, 'end': 195}]\n",
      "The predominant fatty acids of KPC-SM-21^T were C_18:1 w9c (44.17%), C_16:0 (21.67%) and summed feature 3* (15.34%) (containing C_16:1 ω7c and/or iso-C_15:0 2-OH that was not determined by Sherlock version).\n",
      "[]\n",
      "L. crispatus BC5 and L. gasseri BC12, previously isolated from human vaginal swabs [ 2 ], were routinely cultured in de Man, Rogosa and Sharpe (MRS) broth under static conditions or on MRS agar plates supplemented with 0.05 % (w/v) l -cysteine, at 37 °C in anaerobic jars containing Anaerocult A (Merck Millipore).\n",
      "[]\n",
      "Finally, we showed that Asaia-GFP is able to colonize two Wolbachia artificially and stably infected strains of mosquito (Ae. aegypti and An. stephensi).\n",
      "[{'entity_group': 'B', 'score': 0.5264914, 'word': 'mosquito', 'start': 112, 'end': 120}, {'entity_group': 'B', 'score': 0.71181846, 'word': 'ae', 'start': 122, 'end': 124}, {'entity_group': 'I', 'score': 0.6510396, 'word': '. aegypti', 'start': 124, 'end': 133}, {'entity_group': 'B', 'score': 0.65916806, 'word': 'an', 'start': 138, 'end': 140}, {'entity_group': 'I', 'score': 0.7161418, 'word': '. stephensi', 'start': 140, 'end': 151}]\n",
      "Interestingly, BphA1c from the strain F199, AhdA1c from P2 and PhnA1b from CHY-1 of type IV can also be grouped into type IIIαβ with HybB from JB2 (2-hydroxybenzoate 5-salicylate hydroxylase) based on the PD value of 0.61 (Figure 8).\n",
      "[]\n",
      "Moreover, the total RLs production of KT1115/pPA4781 was 53.5% lower than that of KT1115.\n",
      "[]\n",
      "The criteria of successful model were that rats presented unconscious, bilateral dilation of pupils, loss of spontaneous voluntary movements and the righting reflex throughout the ischemia and initial reperfusion periods.\n",
      "[{'entity_group': 'B', 'score': 0.7670499, 'word': 'rats', 'start': 43, 'end': 47}]\n",
      "Whether the same c-LPs and homologues are being produced during interaction of B. subtilis HC8 with plants and which of them are involved in its biocontrol activity remains to be established.\n",
      "[]\n",
      "The type strain was NEAU-SSA 1^T (=CCTCC AA 2017028^T = DSM 105692^T), isolated from soil collected from Mount Song, Dengfeng, Henan Province, China.\n",
      "[]\n",
      "The major difference of cell wall structure is that the aliphatic chain length of mycolic acids (ACLM) in DQ12-45-1b is much shorter than that in mycobacteria [ 61 ].\n",
      "[]\n",
      "This periplasmic cleft can be open or closed and forms an entrance drug-binding site to allow for substrates to enter the pump, as observed in EcAcrB (16, 17, 22), P. aeruginosa MexB (18), A. baumannii AdeB (21, 23), A. baumannii AdeJ (24, 25), Campylobacter jejuni CmeB (20), and Neisseria gonorrhoeae MtrD (19, 26, 27).\n",
      "[]\n",
      "For example, many strains of Chlamydia trachomatis are almost identical (>99.99%) to one another (e.g., Chlamydia trachomatis D/UW-3/CX and Chlamydia trachomatis strain Ia/CS190/96).\n",
      "[]\n",
      "Phospholipids bands (deposition line) were scraped, extracted, and used for FA determination.\n",
      "[]\n",
      "A Trichoderma harzianum strain (B13-1) showed good extracellular lipolytic activity (205 UmL−1).\n",
      "[]\n",
      "Together, these results indicate that (i) B. distachyon increased phytosiderophore production in response to bacterial colonization and (ii) SBW25 consumed or otherwise degraded these phytosiderophores, resulting in overall lower abundances within the root exudate solution.\n",
      "[{'entity_group': 'I', 'score': 0.46553087, 'word': 'distachyon', 'start': 45, 'end': 55}]\n",
      "Mechanically, we found that AB4 enhanced colorectal cancer cell apoptosis through Src-dependent caspase-9 apoptotic pathway.\n",
      "[]\n",
      "We found that all strains carrying the kguT deletion produced more pyocyanin, pyoverdine and rhamnolipids than PAO1, whereas the glt deletion had no effect on their production.\n",
      "[]\n",
      "These included: (1) a new Klebsormidium isolate with long, tangled filaments, collected from the acidic (pH 4.3) environment of a former mining site termed Schwarzwand (47°9′ 36.84″N, 13° 13′13.28″E) (Großarl Valley in Salzburg, Austria, Adlassnig et al. 2013); and (2) the previously described K. flaccidum KUE1 (alpine biological soil crust, Tyrolean Alps, Austria) and grouping into the B-Clade according to ITS-phylogeny (Karsten et al.\n",
      "[]\n",
      "The unknown protein related to O-antigen has been described only in V. fujianensis FJ201301 ^T and in V. cholerae MO45 ( Table S3 ).\n",
      "[]\n",
      "Also missing for the above mentioned genomes, but present for strain O3.65 and located next to the subunit pcaIJ, is a regulatory protein (coded by pcaR; 2609025149, TRIHO_43610) needed for functionality of the enzyme 3-oxiadipate CoA transferase.\n",
      "[]\n",
      "The constitutive PME expression displayed by all Bacteroidetes OSR isolates irrespective of Pi concentrations (Fig 1) is a unique and potentially important trait in the context of improving plant phosphorus uptake at the root-soil interface [33, 39, 59].\n",
      "[]\n",
      "Moreover, ZH07 produced 8.33 ± 0.15 mg·L ^−1 of IAA after 4 days of incubation with 200 μg·L ^−1 of l -tryptophan at 30°C ( Table 1 ).\n",
      "[]\n",
      "Using primer pairs Fha1F/Fha1R and Fha2F/Fha2R, two fragments of the expected length (340 bp and 440 bp) could be amplified from chromosomal DNA of B. holmesii G7702.\n",
      "[]\n",
      "Predation, for example by predatory mites reduces Collembola abundances, and is a trophic driver of community dynamics.\n",
      "[{'entity_group': 'I', 'score': 0.5330201, 'word': 'mites', 'start': 36, 'end': 41}, {'entity_group': 'B', 'score': 0.43000266, 'word': 'collembola', 'start': 50, 'end': 60}]\n",
      "The type strain SJCon^T =MTCC 10104^T =DSM 23165^T was isolated from a pesticide-contaminated soil sample collected in Punjab State, India.\n",
      "[]\n",
      "For this reason two isolates (S9 and 85) each representing a subcluster containing the M. ruminalis type strain (HPA92^T) and a subcluster including no reference strain, respectively, were characterized by 16S rRNA gene sequencing.\n",
      "[]\n",
      "The sporulation phenotype of Bt ΔleuB in LB medium, however, was quite different from that in two other glucose free media (1/2 LB and NSM media).\n",
      "[]\n",
      "CP255 was isolated in the Democratic Republic of Congo (then Zaire) in 1991, from the blood of a child at the Institut Médical Evangélique, Kimpese, and is phenotypically MDR (amoxicillin, tetracycline, chloramphenicol, and streptomycin resistant) ( 5 , 6 ).\n",
      "[]\n",
      "E36 and E616 were shown to have diminished ability to grow in coculture with V. cholerae V52.\n",
      "[]\n",
      "Anabaena variabilis ATCC 29413 (=IUCC 1444 = PCC 7937) is a semi-thermophilic, filamentous, heterocyst-forming cyanobacterium.\n",
      "[]\n",
      "More detailed, the initial ranks of the alternatives (i.e., A_5 ≻ A_3 ≻ A_4 ≻ A_1 ≻ A_2 ≻ A_6) are retained in scenarios 1-21, when the weight of the economic criterion is 0.25 < w_1 ≤ 0.3774.\n",
      "[]\n",
      "Initial research studies were performed on V_2O_5 photocatalytic raw material mainly extracted from vanadium ore directly, but its photocatalytic performance was low.\n",
      "[]\n",
      "Age as a factor related to death has been associated with greater comorbidity and shorter survival in patients on dialysis [20].\n",
      "[]\n",
      "B. thermophilus EPR9N symbiont genome possesses two incomplete gene sets for CO _2 -fixation, two complete gene sets for sulfur metabolism, and a gene set for nitrogen metabolism, as previously observed [11] , [12] .\n",
      "[]\n",
      "EAE was induced in 8- to 10-week-old female mice (three Rai^−/− and three wild-type C57BL/6J mice) by subcutaneous injection of 200 μg MOG_35−55 peptide emulsified in an equal volume of complete Freund's adjuvant (CFA) containing 6 mg/ml M. tuberculosis H37Ra (Difco Laboratories, Detroit, MI).\n",
      "[{'entity_group': 'B', 'score': 0.51783264, 'word': 'rai', 'start': 56, 'end': 59}, {'entity_group': 'I', 'score': 0.49475503, 'word': '^ − / −', 'start': 59, 'end': 63}, {'entity_group': 'B', 'score': 0.6520274, 'word': 'c57bl', 'start': 84, 'end': 89}, {'entity_group': 'I', 'score': 0.6111832, 'word': '/ 6j mice', 'start': 89, 'end': 97}]\n",
      "CP4-EPSPS (Agrobacterium sp. strain CP4 5-enolpyruvylshikimate-3-phosphate synthase) protein showed remarkable thermostability and was highly resistant to proteases, such as trypsin.\n",
      "[]\n",
      "In wheat, an increased copy number of Ppd-B1 and Vrn-A1 is associated with altered flowering time [18]\n",
      "[{'entity_group': 'B', 'score': 0.6826566, 'word': 'in wheat', 'start': 0, 'end': 8}]\n",
      "Three of these food isolates and the two fecal isolates were linked to foodborne outbreaks of diarrheal disease ( Johler et al., 2018 ); three food isolates were recovered during controls by a cantonal laboratory (CH_69, CH_72, CH_81) from lasagna, vegetable juice, and sauce; two isolates were obtained from honey during the production process; and one isolate was obtained from tarragon at the retail level.\n",
      "[]\n",
      "In the remaining eleven reactions, clone sequencing determined that the products were due to non-specific amplification, and included phosphatases, methylases, oxidoreductases, helicases, a metallobetalactamase, and hypothetical proteins, from Dehalococcoides, Geobacter, and other, non-dechlorinating organisms in the KB-1 consortium.\n",
      "[]\n",
      "Bdellovibrio bacteriovorus HD100 are terrestrial bacteria which show natural control of Pseudomonas tolaasii, a spoilage pathogen of mushroom crops, on the non-sterile, biotic surface of the mushroom pileus.\n",
      "[{'entity_group': 'B', 'score': 0.6936795, 'word': 'mushroom', 'start': 133, 'end': 141}, {'entity_group': 'I', 'score': 0.34896082, 'word': 'crops', 'start': 142, 'end': 147}, {'entity_group': 'B', 'score': 0.65634793, 'word': 'mushroom', 'start': 191, 'end': 199}, {'entity_group': 'I', 'score': 0.5461817, 'word': 'pileus', 'start': 200, 'end': 206}]\n",
      "In the present study, it is identified only in HCoV-OC43 sequences from the USA and France and data suggest that it has been circulating since the 1940s after being the first known group to differentiate from BCoV.\n",
      "[]\n",
      "Nostoc UCD120, UCD121 and UCD122, isolated from Azolla , are very closely related to the model strain N. punctiforme and Nostoc Moss2, their closest known relatives ( Fig 1 ).\n",
      "[{'entity_group': 'B', 'score': 0.44437513, 'word': 'azolla', 'start': 48, 'end': 54}]\n",
      "We discovered that PGE2 is able to decrease intracellular temperature of hepatocytes, via increasing some lipogenic genes’ expressions, hampering lipolysis and mitochondrial β-oxidation, reducing intracellular ATP level and elevating cAMP level through EP4 receptor.\n",
      "[]\n",
      "To date, only three bacterial effectors have been shown to be involved in deSUMOyaltion: Listeria listeriolysin O [22], Xanthomonas XopD [29] and Yersinia YopJ [21].\n",
      "[]\n",
      "Intriguingly, only the Scalindua genomes (except for the Scalindua sp001828595) contained an operon encoding Fdp and rubredoxin (Rd) which is a potential electron carrier of Fdp.\n",
      "[]\n",
      "The N2-fixing cyanobacterium Nostoc sp. strain PCC 7120 may contain both an uptake and a bidirectional hydrogenase.\n",
      "[]\n",
      "Compounds 1-5 were tested for potential in vitro cytotoxicity against human lung cancer cell line A549, human nasopharyngeal carcinoma cell line CNE2, human breast adenocarcinoma cell line MCF-7, human hepatocarcinoma cancer cell line HepG2, and the normal hepatic cell line HL7702.\n",
      "[{'entity_group': 'B', 'score': 0.562135, 'word': 'human', 'start': 70, 'end': 75}, {'entity_group': 'I', 'score': 0.54317474, 'word': 'cancer cell line a549', 'start': 81, 'end': 102}, {'entity_group': 'B', 'score': 0.49738565, 'word': 'human', 'start': 104, 'end': 109}, {'entity_group': 'I', 'score': 0.4840222, 'word': 'carcinoma', 'start': 125, 'end': 134}, {'entity_group': 'B', 'score': 0.45293862, 'word': 'human', 'start': 151, 'end': 156}, {'entity_group': 'B', 'score': 0.4857657, 'word': 'mcf', 'start': 189, 'end': 192}, {'entity_group': 'I', 'score': 0.6091785, 'word': '7', 'start': 193, 'end': 194}, {'entity_group': 'B', 'score': 0.4825957, 'word': 'human', 'start': 196, 'end': 201}, {'entity_group': 'I', 'score': 0.53619605, 'word': 'cancer', 'start': 218, 'end': 224}, {'entity_group': 'B', 'score': 0.3673464, 'word': 'hepg2', 'start': 235, 'end': 240}, {'entity_group': 'I', 'score': 0.5325694, 'word': 'cell', 'start': 265, 'end': 269}, {'entity_group': 'I', 'score': 0.4304956, 'word': 'hl7702', 'start': 275, 'end': 281}]\n",
      "We also characterized a novel 16S rRNA methylase with only 42% amino acid identity to an rRNA methylase from Enterobacter cloacae strain 115-824A (GenBank Accession JN968578).\n",
      "[]\n",
      "To analyze the expression pattern of V. anguillarum during iron starvation at cold- and warm-water temperature, an RNA-Seq assay was performed in cells of V. anguillarum RV22 grown in CM9 minimal medium, supplemented with 50 μM of the iron chelator 2,2′-dipyridyl (low-iron conditions), at 15 and 25°C ( Figure 2A ).\n",
      "[]\n",
      "The serial passage of B. vietnamiensis strain G4 on TSA with a gradual increase in HEPB concentrations (to above the MIC) was repeated until growth ceased.\n",
      "[]\n",
      "One example is the non-virulent, non-hemadsorbing Portuguese isolate OURT 88/3 which belongs to the genotype I [19].\n",
      "[]\n",
      "Interestingly, 2-SC 58 was five to six times less potent at inhibiting the growth of MRC-5 cells (GI_50 = 26.9 ± 3.2 μM).\n",
      "[{'entity_group': 'B', 'score': 0.5183097, 'word': 'mrc -', 'start': 85, 'end': 89}]\n",
      "Embryos and neonates were collected from pregnant CD-1 females at different stages, ranging from E11.5-P0 (birth) and whole heads, eyes, or brains were fixed in a modified Carnoy’s fixative overnight at 4 °C.\n",
      "[{'entity_group': 'B', 'score': 0.63086945, 'word': 'cd', 'start': 50, 'end': 52}, {'entity_group': 'I', 'score': 0.5623907, 'word': '- 1 females', 'start': 52, 'end': 62}]\n",
      "Moreover, three heterocyst-forming N_2-fixing species Anabaena variabilis ATCC 29413, Nostoc sp.\n",
      "[]\n",
      "Among the members of the Cry protein family, Cry1 and Cry2 are of particular interest in agriculture because of their high toxicity to insect larvae of the orders Lepidoptera and Coleoptera as well as nematodes and mites [ 64 , 65 , 66 , 67 ].\n",
      "[{'entity_group': 'B', 'score': 0.4800716, 'word': 'lepidoptera', 'start': 163, 'end': 174}, {'entity_group': 'B', 'score': 0.49991637, 'word': 'coleoptera', 'start': 179, 'end': 189}, {'entity_group': 'B', 'score': 0.5572656, 'word': 'nematodes and mites', 'start': 201, 'end': 220}]\n",
      "ATCC controls used and propagated as described by the ATCC bacterial culture guide [19], and Garitty et al. [20] were: Enterobacter cloacae, Pseudomonas aeruginosa ATCC 27853 (positive); Klebsiella pneumoniae ATCC 13882 (negative); and Kingella kingae ATCC 23330 (twitching motility as occurs with the extension and retraction of the type IV pilus).\n",
      "[]\n",
      "Recent research found that basal autophagy negatively modulated primary ciliogenesis by removing essential ciliogenesis proteins, while activated autophagy promoted ciliary growth by degradation of oral-facial-digital syndrome 1 (OFD1, a negative ciliogenesis regulatory protein) in MCF7 breast cancer cells [33-35].\n",
      "[{'entity_group': 'B', 'score': 0.52514255, 'word': 'mcf7', 'start': 283, 'end': 287}, {'entity_group': 'I', 'score': 0.49823618, 'word': 'breast cancer cells', 'start': 288, 'end': 307}]\n",
      "Among the 18 BGCs of S. coelicoflavus NBRC 15399^T, 13 are present also in S. coelicoflavus strain ZG0656 whereas three lantipeptide and two terpene BGCs are not.\n",
      "[]\n",
      "Furthermore, the content of anthocyanin and proanthocyanidins in the selected transgenic lines revealed that the anthocyanin content of OE9 and OE10 lines was higher reaching to it maximum (0.8 mg/g) than that of wild-type and the remaining transgenic lines.\n",
      "[]\n",
      "In addition, a novel triscatecholate siderophore, turnerbactin (46) was isolated from the shipworm endosymbiont Teredinibacter turnerae T7901 [46].\n",
      "[]\n",
      "At 24 h, the amount of A/B toxin produced by RT883 and 885 was similar to that of RT027; however, at 72 h, the A/B production in RT027 was substantially higher than that in all other strains ( Fig 1 ).\n",
      "[]\n",
      "The pAT6100 plasmid DNA extracted from Escherichia coli DH5αMCR was transformed into electrocompetent cells of C. glutamicum RES167, a restriction-deficient strain derived from the wild-type, by electroporation.\n",
      "[]\n",
      "We thus observed that strain KE21 encodes a cluster of all seven urease genes (i.e., ureA/B, ureI, and ureE-H) whose activity is required for adjusting the periplasmic pH as an acid acclimation mechanism to resist and survive in the harsh acidic environment of the stomach [34].\n",
      "[]\n",
      "The type strain is CT3 ^T (=DSM 12730 ^T = VKM B-3582 ^T ) isolated from treatment plants in Bari, Italy.\n",
      "[]\n",
      "This work focused on the use of the laboratory strain P. aeruginosa PAO1, originally isolated in 1954 (referred to as strain 1) from a human wound in Melbourne, Australia ( 29 ).\n",
      "[]\n",
      "Overall, the cgMLST analysis revealed that the E. coli ECU32 strain was closely related to the human and gut origin multidrug isolates.\n",
      "[]\n",
      "From both fittings, the closest distance between the ACE spacecraft path and the rope axis is about 0.3R _c, where R _c ~ 0.2 AU is the radius of the MFR’s cross section.\n",
      "[]\n",
      "The intratumoral injection of miR-34a-5p’s agomiR into G-292 decreased DLL1 expression.\n",
      "[{'entity_group': 'B', 'score': 0.70465726, 'word': 'g', 'start': 55, 'end': 56}, {'entity_group': 'I', 'score': 0.60455346, 'word': '- 292', 'start': 56, 'end': 60}]\n",
      "We found both mushroom pathogens and the non-pathogenic strain NS1 had between one to three T3SS, with elements of broken up T3SS clusters also being evident in the genomes.\n",
      "[{'entity_group': 'B', 'score': 0.6476959, 'word': 'mushroom', 'start': 14, 'end': 22}]\n",
      "These data support the notion that growing the lab strain PAO1 planktonically in LB is a viable model system for studying many aspects of CF lung infection and that many P. aeruginosa genes do not vary significantly in expression regardless of the genotype and growth environment.\n",
      "[]\n",
      "The pattern of large amounts of branched CFAs was also similar to that observed in L. fallonii LLAP-10 ^T and its close relatives ( 9 ); however, the CFA profile drastically changed when cultured at lower temperature.\n",
      "[]\n",
      "BC12 was also found to be slightly more resistant than BC5 to some β-lactams ( Fig 4 ).\n",
      "[]\n",
      "As reported in previous study [14], 19-day-old mice will be useful for further evaluation of RVax-1 and parental rMP-12 strain in this regard due to their age-dependent susceptibility.\n",
      "[{'entity_group': 'B', 'score': 0.58091044, 'word': 'mice', 'start': 47, 'end': 51}]\n",
      "Among the few S. mutans strains from which mutacins were first purified and characterized are strains T8 and UA140 ( 7 - 9 ).\n",
      "[]\n",
      "3:IL-6 (A) and TNF-α (B) production by murine peritoneal macrophages (BALB/c and C57BL/6) exposed to lipophosphoglycan (LPG) (10 µg/mL) from dermotropic (AM65, AM73, AMA161 and AMA614) and viscerotropic strains (BA262 and BH46) of Leishmania infantum.\n",
      "[{'entity_group': 'B', 'score': 0.63742846, 'word': 'murine', 'start': 39, 'end': 45}, {'entity_group': 'B', 'score': 0.666963, 'word': 'balb /', 'start': 70, 'end': 75}, {'entity_group': 'I', 'score': 0.5446663, 'word': 'c', 'start': 75, 'end': 76}, {'entity_group': 'B', 'score': 0.6396159, 'word': 'c57bl /', 'start': 81, 'end': 87}, {'entity_group': 'I', 'score': 0.55656517, 'word': '6', 'start': 87, 'end': 88}, {'entity_group': 'B', 'score': 0.6365374, 'word': 'leishmania', 'start': 231, 'end': 241}, {'entity_group': 'I', 'score': 0.69243, 'word': 'infantum', 'start': 242, 'end': 250}]\n",
      "[32, 35, 36]Streptococcus pneumoniae might mediate impaired TLR2 signaling of alveolar macrophages in patients with frequent exacerbations of COPD.\n",
      "[]\n",
      "L. reuteri E81 and LR5A were both found to contain a putative enterolysin-A, L. rhamnosus LR2 had a carnocin_CP52, and L. plantarum PFC-311 had plantaricin-A, -E, -F, -J, -K, and -N.\n",
      "[]\n",
      "Intriguingly, mutations in KaiB that are distal from these binding surfaces, e.g., R22A and D90G in Synechococcus elongatus KaiB, can impair its interaction with the KaiC hexamer and the consequent circadian rhythm [18,19].\n",
      "[]\n",
      "P. glacincola BNF20 was isolated from a sediment sample collected at King George Island, Antarctica (S62°11′37.6″; W58°56′14.9″) during the ECA-48 Chilean Antarctic Expedition (January 2012).\n",
      "[]\n",
      "Colwellia psychrerythraea 34H was grown in 2216 medium at 10 °C.\n",
      "[]\n",
      "Strain Marseille-P9525 ^T exhibited catalase-negative and oxidase-negative activities.\n",
      "[]\n",
      "We found high-fat diet markedly decreased the relative abundance of Bifidobacterium and Ruminococcaceae_UCG-014 at the genus level in feces, but CHI leaves extract effectively prevented the reduction of these probiotics (Figures 6E, F).\n",
      "[]\n",
      "In the present study, we report that endophytic Streptomyces hygroscopicus OsiSh-2 can form a sophisticated interaction with host rice, maintaining cellular homeostasis under pathogen-infection stress, and optimize plant growth and disease resistance in rice.\n",
      "[{'entity_group': 'B', 'score': 0.59244597, 'word': 'rice', 'start': 130, 'end': 134}, {'entity_group': 'B', 'score': 0.7613039, 'word': 'rice', 'start': 254, 'end': 258}]\n",
      "Remarkably, the P. distincta genes are strongly strain-specific, with the exception of seven common genes for P. distincta strains and P. arctica A 37-1-2 ^T ( Supplementary Table S5 ).\n",
      "[]\n",
      "Bacillus intestinalis strain 1731 is an aerobic, spore-forming, motile, Gram-positive, rod-shaped bacterium that was isolated from the small intestine of a patient with intestinal malignancy.\n",
      "[]\n",
      "MP-1 was obtained from ATCC and grown in marine broth medium 2216 (catalog number BD279110) at 18°C for 72 hours in a conical flask agitated at 150 rpm.\n",
      "[]\n",
      "There was no significant difference between cell-free extracts of NHY1 and OKB105 in the rate of biosurfactant formation as determined by the disappearance of 3-hydroxymyristoyl-CoA, the release of CoA, and the formation of surfactin.\n",
      "[]\n",
      "Our work aimed to study a monocolonization of the intestine of newborn germ-free piglets with mucinolytic and non-mucinolytic strains of Bifidobacterium boum, its impact on the modification of the intestinal barrier, interference of bifidobacteria with S.\n",
      "[{'entity_group': 'B', 'score': 0.44764063, 'word': 'germ', 'start': 71, 'end': 75}, {'entity_group': 'I', 'score': 0.41657308, 'word': 'free', 'start': 76, 'end': 80}, {'entity_group': 'B', 'score': 0.42232504, 'word': 'piglets', 'start': 81, 'end': 88}]\n",
      "We disclosed that the symbiosis relationship of OsiSh-2 with rice might contribute to a simultaneous increase in rice blast resistance and yielding.\n",
      "[{'entity_group': 'B', 'score': 0.58773607, 'word': 'rice', 'start': 61, 'end': 65}]\n",
      "The other three non-ICU isolates were recovered from a sputum sample of a 65-year-old male (Ab19), a urine sample of a 16-year male (Ab40), and a sputum sample of a 63-year female (Ab119).\n",
      "[]\n",
      "To test whether the Sli gene is an efficient pollen SI-breaker, RH and E172 were used as the Sli gene donor to pollinate 125 self-incompatible diploid lines that belong to six groups (50 S. stenotomum, 37 S. goniocalyx, 25 S. phureja11 S. tuberosom, 1 S. ajanhuiri, and 1 S. curtilobum lines, Fig 2b and Supplementary Table 1).\n",
      "[{'entity_group': 'B', 'score': 0.66434246, 'word': 's', 'start': 187, 'end': 188}, {'entity_group': 'I', 'score': 0.64992404, 'word': '. stenotomum', 'start': 188, 'end': 200}, {'entity_group': 'B', 'score': 0.6825454, 'word': 's', 'start': 205, 'end': 206}, {'entity_group': 'I', 'score': 0.5903549, 'word': '. goniocalyx', 'start': 206, 'end': 218}, {'entity_group': 'B', 'score': 0.66413766, 'word': 's', 'start': 223, 'end': 224}, {'entity_group': 'I', 'score': 0.4884227, 'word': '. phureja11', 'start': 224, 'end': 235}, {'entity_group': 'B', 'score': 0.63860303, 'word': 's', 'start': 236, 'end': 237}, {'entity_group': 'I', 'score': 0.4622396, 'word': '. tuberosom', 'start': 237, 'end': 248}, {'entity_group': 'B', 'score': 0.6716357, 'word': 's', 'start': 252, 'end': 253}, {'entity_group': 'I', 'score': 0.5365562, 'word': '. ajanhuiri', 'start': 253, 'end': 264}, {'entity_group': 'B', 'score': 0.669359, 'word': 's', 'start': 272, 'end': 273}, {'entity_group': 'I', 'score': 0.6242689, 'word': '. curtilobum', 'start': 273, 'end': 285}]\n",
      "2)Shop → User: r _s  The shop first checks D to verify if the e-cash is still within the expiration date or not.\n",
      "[]\n",
      "This was followed by transforming the vector into chemically competent E. coli DH5α cells (Invitrogen) and plating on Luria-Bertani (LB) agar fortified with 50 μg ml^−1 kanamycin at 37°C for 16 h.\n",
      "[]\n",
      "Three hundred seven ORFs found in FO-36b are not shared by SAFR-032.\n",
      "[]\n",
      "It also demonstrates the intensity of the flavonoid signal in Deinococcus sp. 43 intracellular and extracellular environments.\n",
      "[]\n",
      "Although the MT5 structure remains similar to that of VcINDY, one important difference centres on the pro-S carboxylate of the bound citrate (Supplementary Fig 8).\n",
      "[]\n",
      "Peptidoglycan type of HKU75 ^T was A3α l -lysine- l -alanine, with rhamnose, ribose, and glucose as major cell-wall constituents.\n",
      "[]\n",
      "Compound 1 (3 g, 0.017 mol) and NaN_3 (3.65 g, 0.056 mol) were dissolved in 25 mL DMSO and stirred for 24 h at 90 °C.\n",
      "[]\n",
      "Here, we report the draft genome sequence of the Gram-negative strain Klebsiella pneumoniae RYC492, which produces the amyloid-forming and antibacterial peptide microcin E492.\n",
      "[]\n",
      "Streptococcus agalactiae (GBS) isolate CJB111 (serotype V) and CJB111Δ adcA Δ adcAII Δ lmb were cultured in Todd-Hewitt broth at 37°C.\n",
      "[]\n",
      "Following the approach previously described (Chambonnier et al., 2016), to measure the level of protein-protein interaction between FHbp from MC58 or L91543 with SecA, LB broth cultures were grown to A_600 0.6 then induced for 3 h with 0.5 mM IPTG.\n",
      "[]\n",
      "The cryo-EM structure of the monomeric LH1-RC complex of Rba . sphaeroides IL106 was determined at 2.9 Å resolution (Fig 1 , Supplementary Table 1 and Supplementary Figs.\n",
      "[]\n",
      "Quantification of F. verticillioides DNA showed an increased level of fungi in caterpillars and pupae, with reduced levels in adults.\n",
      "[{'entity_group': 'I', 'score': 0.4646427, 'word': 'caterpillars', 'start': 79, 'end': 91}, {'entity_group': 'B', 'score': 0.51186734, 'word': 'pupae', 'start': 96, 'end': 101}, {'entity_group': 'B', 'score': 0.44832125, 'word': 'adults', 'start': 126, 'end': 132}]\n",
      "For OsiSh-2 reisolation from inner host plants, the rice tissues were excised and subjected to four-step surface sterilization as described previously ( 52 ).\n",
      "[{'entity_group': 'B', 'score': 0.5789977, 'word': 'rice', 'start': 52, 'end': 56}]\n",
      "Arabidopsis thaliana ecotype Sei-0 was used for transformation [36] and transformed plants were screened for single-copy insertion homozygotes, as previously described [9] to obtain plant lines IPC4S49A-2, IPC4A49T-2, IPC4SII-6 and IPC4A49TSII-1.\n",
      "[{'entity_group': 'I', 'score': 0.68795854, 'word': 'arabidopsis thaliana ecotype sei - 0', 'start': 0, 'end': 34}, {'entity_group': 'I', 'score': 0.4335223, 'word': '6', 'start': 226, 'end': 227}]\n",
      "We therefore tested all the selected primer sets against genomic DNA from three different genetic backgrounds, namely Populus trichocarpa (Nisqually-1), Populus trichocarpa X Populus deltoides (H11-11) and Populus deltoides (ST-70).\n",
      "[{'entity_group': 'B', 'score': 0.58882475, 'word': 'populus', 'start': 118, 'end': 125}, {'entity_group': 'I', 'score': 0.7460718, 'word': 'trichocarpa', 'start': 126, 'end': 137}, {'entity_group': 'I', 'score': 0.6087622, 'word': 'nisqually - 1', 'start': 139, 'end': 150}, {'entity_group': 'B', 'score': 0.6006973, 'word': 'populus', 'start': 153, 'end': 160}, {'entity_group': 'I', 'score': 0.73557043, 'word': 'trichocarpa', 'start': 161, 'end': 172}, {'entity_group': 'B', 'score': 0.5248823, 'word': 'populus', 'start': 175, 'end': 182}, {'entity_group': 'I', 'score': 0.7965678, 'word': 'deltoides', 'start': 183, 'end': 192}, {'entity_group': 'B', 'score': 0.42088962, 'word': 'h11', 'start': 194, 'end': 197}, {'entity_group': 'I', 'score': 0.71826565, 'word': '- 11', 'start': 197, 'end': 200}, {'entity_group': 'B', 'score': 0.55570513, 'word': 'populus', 'start': 206, 'end': 213}, {'entity_group': 'I', 'score': 0.7931792, 'word': 'deltoides', 'start': 214, 'end': 223}, {'entity_group': 'B', 'score': 0.5180978, 'word': 'st', 'start': 225, 'end': 227}, {'entity_group': 'I', 'score': 0.7473805, 'word': '- 70', 'start': 227, 'end': 230}]\n",
      "Under these conditions, not only β-catenin but also NURR1 levels were gradually decreased.\n",
      "[]\n",
      "Provided that Z. mobilis 8b could adjust itself to reduce energy needs for substrate uptake and reserve energy for stress responses via down-regulating the outer membrane transport protein ZMO1522 [39], it is speculated that, although required a further verification, down-regulation of apolipoprotein N-acyltransferase might also play a role in reducing energy expenditure.\n",
      "[]\n",
      "Nitratiruptor was infected multiple times by diverse phages and diversified by the acquisition of prophage genes and various immune mechanisms (phage repressor, restriction-modification system, and CRISPR-Cas systems).\n",
      "[]\n",
      "In this context, we evaluated two HM-resistant PGPB strains, i.e., Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6 isolated from wastewater-irrigated agricultural soils, for their efficacy to mitigate HM (Cd, Ni, and Pb) stress in a pot experiment.\n",
      "[]\n",
      "Briefly, a Rif1 protein fragment from residues 694 - 1094 (Sreesankar et al., 2012) was C-terminally six-histidine tagged and and expressed in E. coli Rossetta DE3 cells and purified using Ni-NTA Agarose beads (Qiagen).\n",
      "[]\n",
      "This group includes enolase 1 (α-enolase) (Spot 2) and cofilin-1 (Spot 9), whose abundance decreased after CMS-2 treatment, both in B16-F10 and Melan-a cell lines.\n",
      "[{'entity_group': 'B', 'score': 0.82456124, 'word': 'b16', 'start': 132, 'end': 135}, {'entity_group': 'I', 'score': 0.5353361, 'word': '- f10', 'start': 135, 'end': 139}, {'entity_group': 'B', 'score': 0.6637733, 'word': 'melan', 'start': 144, 'end': 149}, {'entity_group': 'I', 'score': 0.45376307, 'word': '- a', 'start': 149, 'end': 151}]\n",
      "Type strain P7388 ^T (= CCM 9064 ^T = LMG 31962 ^T ) was isolated from water samples taken in 2016 from a small temporary lake (GPS: −63.795894 S, −57.809928 W).\n",
      "[]\n",
      "We identified 1,838 proteins with high confidence that matched cyanobacterium ESFC-1 in at least 2 of 5 replicate samples.\n",
      "[]\n",
      "This environment appears to be the optimal habitat for agar-degrading bacteria, such as strain KA5-B ^T .\n",
      "[]\n",
      "AZUL cells were grown in PY medium and harvested by centrifugation at 5,000 × g.\n",
      "[]\n",
      "The average colony growth rate (10.72 mm/d) of strain XJ-BZ-5-1_hyg-V was significantly higher than that (10.50 mm/d) of strain XJ-BZ-5-1_hyg (Figure 3E).\n",
      "[]\n",
      "The LC_50 values of GC16 against eggs, larvae, nymphs, and adults of T. pueraricola were in the range of 1.266~2.239 g/L, and the LC_90 values ranged from 5.951 to 26.888 g/L.\n",
      "[{'entity_group': 'B', 'score': 0.3681432, 'word': 'nymphs', 'start': 47, 'end': 53}, {'entity_group': 'B', 'score': 0.63136256, 'word': 't', 'start': 69, 'end': 70}, {'entity_group': 'I', 'score': 0.67232203, 'word': '. pueraricola', 'start': 70, 'end': 83}]\n",
      "Out of 58 bacterial strains isolated in Qassim, Saudi Arabia, four strains were identified by the 16S rRNA as the Streptomyces cinereoruber strain P6-4, Priestia megaterium strain P12, Rossellomorea aquimaris strain P22-2, and Pseudomonas plecoglossicida strain P24.\n",
      "[]\n",
      "Moreover, in situ GABA supplementation or heterologous expression of glutamate decarboxylase in E . coli OP50 conferred neuroprotective activity to this strain.\n",
      "[]\n",
      "Citrobacter werkmanii strain WWN1 and Enterobacter cloacae strain JWM6 were able to increase shoot POD activity by 23 and 24%, respectively, while the bacterial consortium showed an increase of 22% compared to non-treated plants.\n",
      "[]\n",
      "The P. viridiflava strain UASWS0038 was isolated in our lab from a Phytophthora sp.-infected rhododendron leaf.\n",
      "[{'entity_group': 'B', 'score': 0.44203785, 'word': 'rhododendron', 'start': 93, 'end': 105}]\n",
      "In this study, C. sakazakii strain ATCC 29544 (ST1) showed the ability to persist within macrophages cell line RAW 264.7 for up to 48 h of incubation.\n",
      "[{'entity_group': 'B', 'score': 0.47193113, 'word': 'raw', 'start': 111, 'end': 114}]\n",
      "This revealed amino acid uptake deficiencies of the FOT mutants compared to E1 (Figure 8B).\n",
      "[]\n",
      "McClain and colleagues [19] described four antigenic regions on SmD3, of which antigenic region 4 encompasses amino acids 104-126.\n",
      "[]\n",
      "Groups of 30 fish were acclimated at water temperature of 15°C (cold) or 25°C (warm) and infected with the same dose of V. anguillarum RV22 (2-5 × 10 ^5 CFU per fish).\n",
      "[{'entity_group': 'B', 'score': 0.625742, 'word': 'fish', 'start': 13, 'end': 17}]\n",
      "We confirmed that the enhanced protection effect of the OsiSh-2 is based on a priming mechanism instead of consuming energy to express many host defense-related genes and proteins.\n",
      "[]\n",
      "Previously, Mer operon was detected in MDR A. veronii strain MS-18-37 isolated from United States catfish but no study about its function was demonstrated ( Abdelhamed et al., 2019 ).\n",
      "[{'entity_group': 'B', 'score': 0.69773185, 'word': 'catfish', 'start': 98, 'end': 105}]\n",
      "Furthermore, the gene with an “ldcC-like” environment was found to be annotated as cadA in particular strains of Citrobacter freundii, Cronobacter sakazakii, Enterobacter cloacae subsp.\n",
      "[]\n",
      "Overnight cultures of PAO1 and PA80 were grown in PPGAS medium for 24 h.\n",
      "[]\n",
      "Oscillibacter sp. 1-3 and Firmicutes bacterium ASF500 have important characteristics in enhancing intestinal epithelial barrier functions and immunity [35].\n",
      "[]\n",
      "Strains MP1 and DSM 29 both originated from honey or bee-related sources [ 4 , 30 ], whereas the others did not [ 3 , 31 ]; the introduction of a common ancestor to bees could have led to niche specialization that caused speciation.\n",
      "[{'entity_group': 'B', 'score': 0.74652505, 'word': 'bees', 'start': 165, 'end': 169}]\n",
      "Coabalamin-dependent O-demethylase in Blautia sp. strain MRG-PMF1 was found to catalyze the unprecedented allyl aryl ether cleavage reaction.\n",
      "[]\n",
      "Asian isolates: Hymenoscyphus koreanus isolate F52847-04 and H. occultus isolate F52847-22, originally known as KUS-F52847_04 and KUS-F52847_22, respectively, were collected in Korea from petioles of Korean ash and kindly provided by Dr.\n",
      "[{'entity_group': 'B', 'score': 0.8127562, 'word': 'korean', 'start': 200, 'end': 206}, {'entity_group': 'I', 'score': 0.56413984, 'word': 'ash', 'start': 207, 'end': 210}]\n",
      "In comparison to glucose cultivations, K. intermedius ENS15 completely utilized the carbon source, synthesizing 1.3 ± 0.0 g/L BC (yield, 0.7 mg/g _substrate ; BC characterization, Fig S6 and Table S5 ).\n",
      "[]\n",
      "The HKU75 ^T in the DNA extracts, as well as in all Streptococcus species isolated from the swab samples, was detected using PCR targeting its partial 588-bp fragment of the groEL gene.\n",
      "[]\n",
      "The A. baumannii strains DR1, DR2, and AB067 strains were assessed for antibiotic susceptibility using Kirby Bauer disk diffusion assay; antibiotics disks (in μg/ml) from HiMedia Laboratories, India, used were amikacin, ampicillin, azithromycin, ceftazidime-avibactam, ceftazidime, ceftriaxone, chloramphenicol, ciprofloxacin, colistin, and co-trimoxazole (Sulpha/Trimethoprim), and the data were tabulated.\n",
      "[]\n",
      "Toyama and Lidstrom ( Toyama and Lidstrom, 1998 ) have shown that disruption of this gene could reduce PQQ production in AM1 but does not completely abolish its biosynthesis.\n",
      "[]\n",
      "We also assessed whether the conventional and DNA OspC vaccines were able to provide protection against infection with B. burgdorferi ss strain N40 transmitted by I. scapularis nymphs.\n",
      "[{'entity_group': 'B', 'score': 0.52877504, 'word': 'i', 'start': 163, 'end': 164}, {'entity_group': 'I', 'score': 0.8221829, 'word': '. scapularis nymphs', 'start': 164, 'end': 183}]\n",
      "^20 ^, ^21 We then used bacterial aggregation to estimate the biofilm formation of KT1115 in liquid culture.\n",
      "[]\n",
      "The potential of R. nectarea strain 8N4 ^T to produce pili and fimbriae is evident from the presence of the following genes: (i) major type-1 subunit fimbrin (pilin), (ii) prepilin-peptidase-dependent protein D, (iii) minor fimbrial subunit and (iv) fimbrial-chaperone protein.\n",
      "[]\n",
      "The marine thraustochytrid S. limacinum SR21 (ATCC-MYA-1381) was procured from the American Type Culture Collection and was cultured in ATCC 790 By + medium containing yeast extract (1 g), peptone (1 g), and glucose (5 g) in artificial seawater (1000 mL).\n",
      "[]\n",
      "Here we describe a new methanotrophic isolate that shares phenotypic characteristics and phylogenetic relatedness with the marine methanotroph Methylomarinum vadi.\n",
      "[]\n",
      "Samples of the same aqueous extracts of PM_10 analysed by ICP-MS were subsequently instilled into male Wistar rats.\n",
      "[{'entity_group': 'B', 'score': 0.55671406, 'word': 'male', 'start': 98, 'end': 102}, {'entity_group': 'I', 'score': 0.54218525, 'word': 'wistar rats', 'start': 103, 'end': 114}]\n",
      "CTX51T was isolated from a surgically resected tumor from a treatment-naive, male colorectal cancer patient diagnosed with adenocarcinoma in the cecum.\n",
      "[]\n",
      "In this work, we found that the exoelectrogen Cellulomonas fimi strain Clb-11 that we isolated secretes extracellular chromate reductase or extracellular electron mediator to reduce Cr(VI).\n",
      "[]\n",
      "Conversely, down-regulation of miR-148a significantly enhanced p-STAT3 and p-Akt protein levels in SGC-7901 cells (Fig 7B).\n",
      "[{'entity_group': 'B', 'score': 0.6043812, 'word': 'sgc -', 'start': 99, 'end': 103}, {'entity_group': 'I', 'score': 0.4782477, 'word': '7901', 'start': 103, 'end': 107}]\n",
      "The near-saturated transposon insertion library of R. solanacearum GMI1000 preserved at −80°C was adjusted to 1 × 10 ^8 CFU/ml in BG medium and reactivated at 28°C for 1 h.\n",
      "[]\n",
      "Nonetheless, HT115 has a highly increased capacity to produce GABA, even compared with other K-12 strains.\n",
      "[]\n",
      "HeLa cells were maintained in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 4.5 g/L glucose (Thermo Fisher Scientific) and 10% fetal bovine serum (FBS) (Gibco, United States).\n",
      "[{'entity_group': 'B', 'score': 0.40336952, 'word': 'hela', 'start': 0, 'end': 4}]\n",
      "F. faecale WV33 ^T was aerobically cultured in Luria-Bertani (LB; 10 g/L tryptone, 5 g/L yeast extract, and 5 g/L NaCl) medium at 30 °C on a rotary shaker at 250 rpm.\n",
      "[]\n",
      "Living at home for a year after college and carpooling to work with my father was pleasant but I found the job limited and the atmosphere of Baltimore stifling.\n",
      "[]\n",
      "Along these lines, IIV-6 could be used as a non-replicative virus to induce an immune response that can restrict arbovirus infection in mammals.\n",
      "[{'entity_group': 'B', 'score': 0.7588457, 'word': 'mammals', 'start': 136, 'end': 143}]\n",
      "A strain of purple non-sulfur bacteria, Rhodopseudomonas palustris nifA^∗ (McKinlay and Harwood, 2010) was grown in a defined mineral medium containing Na_2HPO_4 (1.77 g/L), KH_2PO_4 (1.77 g/L), 0.1% (v/v) concentrated base solution, 0.5% (v/v) Wolfe’s vitamins solution and 30 mM sodium acetate, as a carbon source.\n",
      "[]\n",
      "The transferase lpxA [UDP-acetylglucosamine acyltransferase], synthase lpxB [lipid A-disaccharide synthase], transferases lpxD [UDP-3- O -[3-hydroxymyristoyl] glucosamine N -acyltransferase], and lpxL [Kdo2-lipid IVA lauroyltransferase/acyltransferase], lipid A phosphoethanolamine transferase (A1S_2752; eptA ) were all found in these MDR strains DR1 (OCL8-type), DR2 (OCL1d-type), and AB067 (OCL1d-type) with 100% identity to proteins (A1S_1965, A1S_1668, and A1S_1967) found in A. baumannii strain ATCC 17978.\n",
      "[]\n",
      "One day preceding experiments, hTCEpi cells were plated on No.\n",
      "[]\n",
      "Without extraction, strain 7D4C2 reached the stationary growth phase by day 5 with substrate left over due to inhibition at an undissociated n -caproic acid concentration of 4.8 mM ( Figures 5A-C and Supplementary Figure 7A ).\n",
      "[]\n",
      "The LH1 subunits of strain 970 are uniformly distributed around the RC forming a closed, slightly elliptical double cylinder composed of 16 pairs of helical α(inner)β(outer)-polypeptides (αβ-dimer), 32 BChls a , 16 carotenoids (3,4,3′,4′-tetrahydrospirilloxanthin) [16] , and 16 Ca ions (Fig 1 and Supplementary Figs. 5 and 6 ).\n",
      "[]\n",
      "The bacteria used in this study was Enterococcus faecium strain TX1330 (BEI HM-204), was isolated from healthy human feces and obtained through BEI Resources, NIAID, NIH as part of the Human Microbiome Project.\n",
      "[]\n",
      "Enterobacter cloacae is a gram-negative Proteobacterium belonging to the Enterobacteriaceae family.\n",
      "[]\n",
      "AN20 ^T hydrolyzed gelatin and weakly hydrolyzed cellulose, whereas AN421 ^T and AN502 were positive for gelatin hydrolysis only.\n",
      "[]\n",
      "In an immune competent model of fungal bronchopneumonia, myeloid-derived HIF-1α was shown to be critical for neutrophil recruitment and ultimately, survival of mice challenged with A. fumigatus CEA10 strain (30).\n",
      "[{'entity_group': 'B', 'score': 0.66468084, 'word': 'mice', 'start': 160, 'end': 164}]\n",
      "In the previously studied Desulfovibrio vulgaris Hildenborough cytochrome c_3, the four iron ions are at the vertices of a deformed tetrahedron [25,26], whereas in the presently studied Shewanella oneidensis MR-1 Small Tetraheme Cytochrome (STC), the irons form approximately the pattern of a hockey stick (a ‘dog leg’ according to others [29]), with some of the Fe-Fe distances significantly longer than in the cytochrome c_3 structure [27].\n",
      "[]\n",
      "Transcripts for each of the genes required for the complete oxidation of CH_4, carbon fixation, and glycogen synthesis were detected (Supplementary Table S1), thereby indicating that Methylacidiphilum sp.\n",
      "[]\n",
      "Moreover previous studies have demonstrated that the absence of the endoglucanases TomCel1 and TomCel2 in tomato and Arabidopsis alters the jasmonic acid signaling network limiting the necrotrophic pathogen Botrytis cinerea invasion and increasing the susceptibility to the hemibiotrophic Pseudomonas syringae DC3000 (Flors et al., 2007; Finiti et al., 2013).\n",
      "[{'entity_group': 'B', 'score': 0.78077763, 'word': 'tomato', 'start': 106, 'end': 112}, {'entity_group': 'B', 'score': 0.7023966, 'word': 'arabidopsis', 'start': 117, 'end': 128}]\n",
      "Future research may help determine whether COVID-19 vaccine acceptance would lead to increased vaccination coverage in the real world.\n",
      "[]\n",
      "B. xenovorans LB400 is a model bacterium for the degradation of PCBs and other aromatic compounds [3], [15], [16].\n",
      "[]\n",
      "Paracoccus pantotrophus strain KS1 is a Gram-negative, motile coccoid bacterium belonging to the family Rhodobacteraceae in the order Rhodobacterales, in the class Alphaproteobacteria.\n",
      "[]\n",
      "PSN38@TPL-nsa exhibited potent antitumor and TME remodeling efficacy in GC intraperitoneal and PDX tumor models.\n",
      "[]\n",
      "In order to determine whether the 11-fold expansion of the N. meningitidis MenW ST-11 clone among UoN students was due to localized clusters and rapid genome evolution, we analyzed WGS data for the 54 MenW isolates obtained from this cross-sectional carriage study (Oldfield et al., 2018).\n",
      "[{'entity_group': 'I', 'score': 0.4698079, 'word': 'students', 'start': 102, 'end': 110}]\n",
      "Interestingly, we observed that even in the case of uninduced CIAT 899 cultures (B^- medium, 55 mM mannitol), the purified extract slightly induced the formation of primordia as previously observed [17].\n",
      "[]\n",
      "Prediction of PULs found seven putative loci involved in degradation of carrageenan, dextran, hemicellulose, chitin, pectin, starch and xylan, which implies the ability of strain P7388 ^T to degrade various polysaccharide substrates.\n",
      "[]\n",
      "The bacterial genus Psychrobacter was first described by Juni & Heym (1986) and includes a group of non-motile, oxidase positive, psychrotolerant, Gram-negative rods or coccobacilli isolated from animals and processed foods ( Bozal et al., 2003 ).\n",
      "[]\n",
      "Following 4 days of Dex treatment B13 cells were also infected overnight with either the liver activating protein (LAP) or the liver inhibitory protein (LIP) both at an MOI of 50.\n",
      "[{'entity_group': 'B', 'score': 0.46460956, 'word': 'b13', 'start': 34, 'end': 37}]\n",
      "The type strain A. acetoxydans, strain INE (=DSM 29876^T = JCM 30553^T), was isolated from the Tinto River, Spain.\n",
      "[]\n",
      "Coxiella burnetii DNA was detected in 12% of cattle meat samples in this study.\n",
      "[{'entity_group': 'B', 'score': 0.47632244, 'word': 'cattle', 'start': 45, 'end': 51}]\n",
      "Parasitological screening was also extended to residents of all houses within a radius of 1-km of the site.\n",
      "[]\n",
      "PI578718 obtained 484,390 transcripts with a maximum length of 15,993 bp, average length of 1,218 bp and N50 of 1,952 bp; PI234881 obtained 483,175 transcripts with a maximum length of 17,029 bp, average length of 1,192 bp and N50 of 1,950 bp.\n",
      "[]\n"
     ]
    }
   ],
   "source": [
    "import jsonlines\n",
    "\n",
    "# Specify the path to the JSONL file\n",
    "jsonl_file = f\"../NER/{ner}/test.jsonls\"\n",
    "\n",
    "# Open the JSONL file\n",
    "with jsonlines.open(jsonl_file) as reader:\n",
    "    # Iterate over each line in the file\n",
    "    for line in reader:\n",
    "        for l in line:\n",
    "\n",
    "            # Get the value from the \"text\" field\n",
    "            text = l[\"data\"][\"text\"]\n",
    "            \n",
    "            # Pass the value to the nlp() function\n",
    "            print(text)\n",
    "            print(nlp(text))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "kk"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
